--- 255117
+++ 255118
@@ -147 +147,23 @@
- Harrison has married twice and has one child with his second wife.
+ Harrison
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ married twice and has one child with his second wife.
+ <ins>
+  He passed away on Thursday,
+ </ins>
+ <ins>
+  <a href="/wiki/November_29" title="November 29">
+   November 29
+  </a>
+  ,
+ </ins>
+ <ins>
+  <a href="/wiki/2001" title="2001">
+   2001
+  </a>
+  .
+ </ins>
@@ -150 +171,0 @@
- <br/>
