--- 753656269
+++ 753750244
@@ -5,9 +5,14 @@
- (born July 18, 1921), (
- <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
-  Col
- </a>
- ,
- <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
-  USMC
- </a>
- , Ret.), is a former
+ <ins>
+  (July 18, 1921 – December 8, 2016) was an American
+ </ins>
+ <del>
+  (born July 18, 1921), (
+  <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
+   Col
+  </a>
+  ,
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   USMC
+  </a>
+  , Ret.), is a former
+ </del>
@@ -27 +32,4 @@
-  United States senator
+  United States Senator
+  Link: /wiki/United_States_Senate from
+ Ohio
+  Link: /wiki/Ohio  . In 1962 he became the first American to orbit the earth, circling three times. Before joining NASA, he was a distinguished fighter pilot in both World War II and Korea.
@@ -29 +37,7 @@
- . He was one of the "
+</p>
+<del>
+ senator
+  Link: /wiki/United_States_Senate .
+</del>
+<p>
+ He was one of the "
@@ -41,7 +55,11 @@
- to become America's first astronauts and fly the
- <a href="/wiki/Project_Mercury" title="Project Mercury">
-  Project Mercury
- </a>
- spacecraft.
-</p>
-<p>
+ to become America's first
+ <ins>
+  astronauts.
+ </ins>
+ <del>
+  astronauts and fly the
+  <a href="/wiki/Project_Mercury" title="Project Mercury">
+   Project Mercury
+  </a>
+  spacecraft.
+ </del>
@@ -89 +107,8 @@
- . Glenn is the earliest-born American to go to orbit, and the second earliest-born man overall after Soviet cosmonaut
+ . Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the earliest-born American to go to orbit, and the second earliest-born man overall after Soviet cosmonaut
@@ -101 +126,8 @@
- in 1990. With the death of
+ in 1990.
+ <ins>
+  Glenn was the last surviving member of the Mercury Seven after
+ </ins>
+ <del>
+  With
+ </del>
+ the death of
@@ -105 +137,6 @@
- on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
+ <ins>
+  in 2013.
+ </ins>
+ <del>
+  on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
+ </del>
@@ -112 +149,7 @@
- ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965. A member of the
+ <ins>
+  , but withdrew after an accident.
+ </ins>
+ <del>
+  ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March.
+ </del>
+ He retired from the Marine Corps on January 1, 1965. A member of the
@@ -120,5 +163,8 @@
- and served through January 3, 1999. With the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- on January 3, 2015, Glenn became the oldest living former United States Senator.
+ and served through January 3, 1999.
+ <del>
+  With the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  on January 3, 2015, Glenn became the oldest living former United States Senator.
+ </del>
@@ -396 +442,2 @@
- . On two occasions, he returned to his base with over 250 holes in his aircraft.
+ .
+ On two occasions, he returned to his base with over 250 holes in his aircraft.
@@ -471 +518,3 @@
- (class 12), graduating in 1954. He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns. He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
+ (class 12), graduating in 1954.
+ He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns.
+ He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
@@ -486 +535,8 @@
- Glenn has nearly 9,000 hours of flying time, with approximately 3,000 hours in
+ Glenn
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ nearly 9,000 hours of flying time, with approximately 3,000 hours in
@@ -504 +560,2 @@
- . The flight from
+ .
+ The flight from
@@ -638 +695,8 @@
- in 1998).
+ in
+ <ins>
+  1998.
+ )
+ </ins>
+ <del>
+  1998).
+ </del>
@@ -730 +794,8 @@
- in 1995, more than 30 years after the hearings. In the late 1970s, Glenn is reported to have supported Shuttle Mission Specialist Astronaut
+ in 1995, more than 30 years after the hearings. In the late 1970s, Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ reported to have supported Shuttle Mission Specialist Astronaut
@@ -804,0 +876,3 @@
+ <ins>
+  He served as one the pallbearers at his funeral.
+ </ins>
@@ -823 +897,6 @@
- . Instead, he challenged Metzenbaum again, whom Gilligan had appointed
+ . Instead, he challenged Metzenbaum again, whom Gilligan had
+ <ins>
+  appointed.
+ </ins>
+ <del>
+  appointed
@@ -825,4 +904,5 @@
- <a href="/wiki/William_B._Saxbe" title="William B. Saxbe">
-  William B. Saxbe
- </a>
- , who had resigned to become Attorney General of the United States.
+  <a href="/wiki/William_B._Saxbe" title="William B. Saxbe">
+   William B. Saxbe
+  </a>
+  , who had resigned to become Attorney General of the United States.
+ </del>
@@ -876,0 +957,3 @@
+ <ins>
+  In 1997, Glenn announced that he would retire from the Senate at the end of his then-current term.
+ </ins>
@@ -911,5 +994,10 @@
- to keep his seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
- <i>
-  on earth
- </i>
- has John Glenn done?"
+ to keep his
+ <ins>
+  seat.
+ </ins>
+ <del>
+  seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
+  <i>
+   on earth
+  </i>
+  has John Glenn done?"
@@ -916,0 +1005 @@
+ </del>
@@ -932 +1021,2 @@
- . Glenn also ran for the
+ .
+ Glenn also ran for the
@@ -936,4 +1026,11 @@
- . A November 1983
- <i>
-  <a href="/wiki/The_New_York_Times" title="The New York Times">
-   New York Times
+ .
+ <del>
+  A November 1983
+  <i>
+   <a href="/wiki/The_New_York_Times" title="The New York Times">
+    New York Times
+   </a>
+  </i>
+  /
+  <a href="/wiki/CBS_News" title="CBS News">
+   CBS News
@@ -941,6 +1038,2 @@
- </i>
- /
- <a href="/wiki/CBS_News" title="CBS News">
-  CBS News
- </a>
- poll found him second, supported by 41% of those polled, to Mondale's 49%.
+  poll found him second, supported by 41% of those polled, to Mondale's 49%.
+ </del>
@@ -977,5 +1070,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (July 2014)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -983,5 +1076,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -999 +1092,2 @@
- , served as chairman of the
+ ,
+ served as chairman of the
@@ -1015 +1109,2 @@
- . Once Republicans regained control of the Senate, Glenn also served as the ranking minority member on a special Senate investigative committee chaired by
+ .
+ Once Republicans regained control of the Senate, Glenn also served as the ranking minority member on a special Senate investigative committee chaired by
@@ -1027 +1122,7 @@
- to U.S. political campaigns for the 1996 election. There was considerable acrimony between the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
+ to U.S. political campaigns for the 1996 election.
+ There was considerable acrimony between the two very high-profile senators during the life of this
+ <ins>
+  committee.
+ </ins>
+ <del>
+  committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
@@ -1029,12 +1130,13 @@
- <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
-  Mary O. Boyle
- </a>
- was the Democratic party nominee. She faced Republican nominee and sitting governor
- <a href="/wiki/George_Voinovich" title="George Voinovich">
-  George Voinovich
- </a>
- in the
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
-  general election
- </a>
- , which Voinovich won.
+  <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
+   Mary O. Boyle
+  </a>
+  was the Democratic party nominee. She faced Republican nominee and sitting governor
+  <a href="/wiki/George_Voinovich" title="George Voinovich">
+   George Voinovich
+  </a>
+  in the
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
+   general election
+  </a>
+  , which Voinovich won.
+ </del>
@@ -1058 +1160,5 @@
- on October 29, 1998, becoming, at age 77, the oldest person to go into space as a
+ on October 29, 1998,
+ <del>
+  becoming, at age 77, the oldest person to go into space
+ </del>
+ as a
@@ -1072 +1178,7 @@
- mission. According to
+ <ins>
+  mission, becoming, at age 77, the oldest person to go into space.
+ </ins>
+ <del>
+  mission.
+ </del>
+ According to
@@ -1142 +1254 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+ <a href="/wiki/Ohio_State_University" title="Ohio State University">
@@ -1149 +1261,7 @@
- . Today Glenn holds an adjunct professorship at the Glenn School.
+ <ins>
+  , and Glenn held
+ </ins>
+ <del>
+  . Today Glenn holds
+ </del>
+ an adjunct professorship at the Glenn School.
@@ -1167 +1285,7 @@
-   Anna and John Glenn in 1965
+   <ins>
+    Glenn and his wife Anna
+   </ins>
+   <del>
+    Anna and John Glenn
+   </del>
+   in 1965
@@ -1202 +1326,8 @@
- Glenn is an honorary member of the
+ Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ an honorary member of the
@@ -1225 +1356,8 @@
- , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
+ , Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a member of Concord Lodge # 688 New Concord, Ohio, and
@@ -1229 +1367,8 @@
- , the Masonic youth organization, and is an ordained elder in the
+ , the Masonic youth organization, and
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ an ordained elder in the
@@ -1233,11 +1378,5 @@
- .
-</p>
-<p>
- Glenn's name was used for the character of
- <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
-  John Tracy
- </a>
- in the 1960s children's TV series
- <i>
-  <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
-   Thunderbirds
+ <del>
+  .
+Glenn's name was used for the character of
+  <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
+   John Tracy
@@ -1245 +1384,7 @@
- </i>
+  in the 1960s children's TV series
+  <i>
+   <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
+    Thunderbirds
+   </a>
+  </i>
+ </del>
@@ -1266 +1411,6 @@
- .
+ <ins>
+  , as himself.
+ </ins>
+ <del>
+  .
+ </del>
@@ -1295 +1445,6 @@
- -
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
@@ -1299,0 +1455,3 @@
+ <ins>
+  Other non-band members to have received this honor include
+ </ins>
@@ -1308,16 +1466,18 @@
- <a href="/wiki/Buster_Douglas" title="Buster Douglas">
-  Buster Douglas
- </a>
- ,
- <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
-  E. Gordon Gee
- </a>
- ,
- <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
-  Novice Fawcett
- </a>
- ,
- <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
-  Robert Ries
- </a>
- , and
+ <del>
+  <a href="/wiki/Buster_Douglas" title="Buster Douglas">
+   Buster Douglas
+  </a>
+  ,
+  <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
+   E. Gordon Gee
+  </a>
+  ,
+  <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
+   Novice Fawcett
+  </a>
+  ,
+  <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
+   Robert Ries
+  </a>
+  , and
+ </del>
@@ -1331 +1491,6 @@
- are the only other non-band members to have received this honor.
+ <ins>
+  .
+ </ins>
+ <del>
+  are the only other non-band members to have received this honor.
+ </del>
@@ -1411,4 +1576,36 @@
- On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
-</p>
-<p>
- Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ On June 28, 2016, the Columbus, Ohio airport
+ <ins>
+  known for many years as Port Columbus
+ </ins>
+ was officially renamed the
+ <a href="/wiki/John_Glenn_Columbus_International_Airport" title="John Glenn Columbus International Airport">
+  John Glenn Columbus International Airport
+ </a>
+ <del>
+  Airport.
+ </del>
+ Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+</p>
+<p>
+ Glenn
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ stated that he
+ <ins>
+  saw
+ </ins>
+ <del>
+  sees
+ </del>
+ no contradiction between believing in God and the knowledge that evolution is "a fact", and that he
+ <ins>
+  believed
+ </ins>
+ <del>
+  believes
+ </del>
+ evolution should be taught in schools.
@@ -1417 +1614 @@
- <span class="mw-headline" id="Illness">
+ <span class="mw-headline" id="Illness_and_death">
@@ -1418,0 +1616,3 @@
+  <ins>
+   and death
+  </ins>
@@ -1423,7 +1623,25 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  The Ohio State University
- </a>
- announced that Glenn was hospitalized at OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
- <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
-  The James Cancer Hospital
- </a>
+ <ins>
+  <a href="/wiki/Ohio_State_University" title="Ohio State University">
+   Ohio State University
+  </a>
+  announced that Glenn was hospitalized at the
+  <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
+   James Cancer Hospital
+  </a>
+  at
+ </ins>
+ The Ohio State
+ <ins>
+  University,
+ </ins>
+ <del>
+  University
+  Link: /wiki/The_Ohio_State_University announced that Glenn was hospitalized at OSU,
+ </del>
+ having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization
+ <del>
+  at
+  <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
+   The James Cancer Hospital
+  </a>
+ </del>
@@ -1431,0 +1650,4 @@
+ <ins>
+  Glenn died December 8, 2016, at the Ohio State University Wexner Medical Center in Columbus, Ohio.
+ A specific cause of death was not immediately announced.
+ </ins>
@@ -1471,3 +1693,5 @@
-     <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
-      Friendship 7
-     </a>
+     <i>
+      <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
+       Friendship 7
+      </a>
+     </i>
@@ -1917,4 +2141,6 @@
-  <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
-   Golden Plate Award
-  </a>
-  for Science and Exploration, 1964
+  <del>
+   <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
+    Golden Plate Award
+   </a>
+   for Science and Exploration, 1964
+  </del>
@@ -2079 +2305,8 @@
- , a mobile landing platform that is scheduled to be delivered to the
+ , a mobile landing platform that
+ <ins>
+  was
+ </ins>
+ <del>
+  is scheduled to be
+ </del>
+ delivered to the
@@ -2083 +2316,7 @@
- in 2014 is named for him. It was christened February 1, 2014, in
+ <ins>
+  on March 12, 2014,
+ </ins>
+ <del>
+  in 2014
+ </del>
+ is named for him. It was christened February 1, 2014, in
@@ -2222 +2461,8 @@
- . Glenn is the 60th person to hold this distinction.
+ . Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the 60th person to hold this distinction.
@@ -2236,3 +2482,5 @@
- <span class="mw-headline" id="Physical_description">
-  Physical description
- </span>
+ <del>
+  <span class="mw-headline" id="Physical_description">
+   Physical description
+  </span>
+ </del>
@@ -2242 +2490,3 @@
-  Weight: 170 lb (77 kg)
+  <del>
+   Weight: 170 lb (77 kg)
+  </del>
@@ -2245 +2495,3 @@
-  Height: 5 ft 10½ in (1.79 m)
+  <del>
+   Height: 5 ft 10½ in (1.79 m)
+  </del>
@@ -2248 +2500,3 @@
-  Hair: Red
+  <del>
+   Hair: Red
+  </del>
@@ -2251 +2505,3 @@
-  Eyes: Green
+  <del>
+   Eyes: Green
+  </del>
@@ -2322,0 +2579,2 @@
+<div style="clear:both;">
+</div>
\ No newline at end of file
