--- 753656269
+++ 753784007
@@ -5,9 +5,14 @@
- (born July 18, 1921), (
- <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
-  Col
- </a>
- ,
- <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
-  USMC
- </a>
- , Ret.), is a former
+ <ins>
+  (July 18, 1921 – December 8, 2016) was an American
+ </ins>
+ <del>
+  (born July 18, 1921), (
+  <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
+   Col
+  </a>
+  ,
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   USMC
+  </a>
+  , Ret.), is a former
+ </del>
@@ -27 +32,8 @@
-  United States senator
+  United States Senator
+  Link: /wiki/United_States_Senate from
+ Ohio
+  Link: /wiki/Ohio  . In 1962 he became the first American to orbit the earth, circling three times. Before joining NASA, he was a distinguished fighter pilot in both World War II and Korea, with five
+ Distinguished Flying Crosses
+  Link: /wiki/Distinguished_Flying_Crosses  and eighteen
+ clusters
+  Link: /wiki/Oak_leaf_cluster  senator
@@ -29 +41,4 @@
- . He was one of the "
+ .
+</p>
+<p>
+ He was one of the "
@@ -41,7 +56,11 @@
- to become America's first astronauts and fly the
- <a href="/wiki/Project_Mercury" title="Project Mercury">
-  Project Mercury
- </a>
- spacecraft.
-</p>
-<p>
+ to become America's first
+ <ins>
+  astronauts.
+ </ins>
+ <del>
+  astronauts and fly the
+  <a href="/wiki/Project_Mercury" title="Project Mercury">
+   Project Mercury
+  </a>
+  spacecraft.
+ </del>
@@ -66,28 +85,35 @@
- and the fifth person in space, after
- <a class="mw-redirect" href="/wiki/Cosmonaut" title="Cosmonaut">
-  cosmonauts
- </a>
- <a href="/wiki/Yuri_Gagarin" title="Yuri Gagarin">
-  Yuri Gagarin
- </a>
- and
- <a href="/wiki/Gherman_Titov" title="Gherman Titov">
-  Gherman Titov
- </a>
- and the
- <a href="/wiki/Sub-orbital_spaceflight" title="Sub-orbital spaceflight">
-  sub-orbital flights
- </a>
- of Mercury astronauts
- <a href="/wiki/Alan_Shepard" title="Alan Shepard">
-  Alan Shepard
- </a>
- and
- <a href="/wiki/Gus_Grissom" title="Gus Grissom">
-  Gus Grissom
- </a>
- . Glenn is the earliest-born American to go to orbit, and the second earliest-born man overall after Soviet cosmonaut
- <a href="/wiki/Georgy_Beregovoy" title="Georgy Beregovoy">
-  Georgy Beregovoy
- </a>
- . Glenn received the
+ and the fifth person in
+ <ins>
+  space.
+ </ins>
+ <del>
+  space, after
+  <a class="mw-redirect" href="/wiki/Cosmonaut" title="Cosmonaut">
+   cosmonauts
+  </a>
+  <a href="/wiki/Yuri_Gagarin" title="Yuri Gagarin">
+   Yuri Gagarin
+  </a>
+  and
+  <a href="/wiki/Gherman_Titov" title="Gherman Titov">
+   Gherman Titov
+  </a>
+  and the
+  <a href="/wiki/Sub-orbital_spaceflight" title="Sub-orbital spaceflight">
+   sub-orbital flights
+  </a>
+  of Mercury astronauts
+  <a href="/wiki/Alan_Shepard" title="Alan Shepard">
+   Alan Shepard
+  </a>
+  and
+  <a href="/wiki/Gus_Grissom" title="Gus Grissom">
+   Gus Grissom
+  </a>
+  . Glenn is the earliest-born American to go to orbit, and the second earliest-born man overall after Soviet cosmonaut
+  <a href="/wiki/Georgy_Beregovoy" title="Georgy Beregovoy">
+   Georgy Beregovoy
+  </a>
+  .
+ </del>
+ Glenn received the
@@ -101 +127,8 @@
- in 1990. With the death of
+ in 1990.
+ <ins>
+  Glenn was the last surviving member of the Mercury Seven after
+ </ins>
+ <del>
+  With
+ </del>
+ the death of
@@ -105,4 +138,16 @@
- on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
-</p>
-<p>
- Glenn resigned from NASA on January 16, 1964, and the next day announced plans to run for a U.S. Senate seat from
+ <ins>
+  .
+ </ins>
+ <del>
+  on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
+ </del>
+</p>
+<p>
+ Glenn resigned from NASA
+ <ins>
+  in 1964 and
+ </ins>
+ <del>
+  on January 16, 1964, and the next day
+ </del>
+ announced plans to run for a U.S. Senate seat from
@@ -112 +157,7 @@
- ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965. A member of the
+ <ins>
+  .
+ </ins>
+ <del>
+  ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965.
+ </del>
+ A member of the
@@ -116 +167,7 @@
- , he finally
+ , he
+ <ins>
+  first
+ </ins>
+ <del>
+  finally
+ </del>
@@ -120,8 +177,29 @@
- and served through January 3, 1999. With the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- on January 3, 2015, Glenn became the oldest living former United States Senator.
-</p>
-<p>
- On October 29, 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
+ <ins>
+  where he
+ </ins>
+ <del>
+  and
+ </del>
+ served through January 3, 1999.
+</p>
+<p>
+ <ins>
+  He retired from the Marine Corps in 1965, after twenty-three years in the military, with over fifteen medals and awards, including the
+  <a href="/wiki/NASA_Distinguished_Service_Medal" title="NASA Distinguished Service Medal">
+   NASA Distinguished Service Medal
+  </a>
+  and the
+  <a href="/wiki/Congressional_Space_Medal_of_Honor" title="Congressional Space Medal of Honor">
+   Congressional Space Medal of Honor
+  </a>
+  . In
+ </ins>
+ <del>
+  With the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  on January 3, 2015, Glenn became the oldest living former United States Senator.
+On October 29,
+ </del>
+ 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
@@ -131,7 +209,3 @@
- , when at age 77, he flew as a
- <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
-  Payload Specialist
- </a>
- on
- <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
-  <i>
+ <ins>
+  as crew member of the
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
@@ -139,7 +213,19 @@
-  </i>
- </a>
- mission
- <a href="/wiki/STS-95" title="STS-95">
-  STS-95
- </a>
- . He was awarded the
+  </a>
+  space shuttle. He was also
+ </ins>
+ <del>
+  , when at age 77, he flew as a
+  <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
+   Payload Specialist
+  </a>
+  on
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
+   Discovery
+  </a>
+  mission
+  <a href="/wiki/STS-95" title="STS-95">
+   STS-95
+  </a>
+  . He was
+ </del>
+ awarded the
@@ -169 +255,5 @@
- , the son of John Herschel Glenn, Sr. (1895–1966) and Teresa (
+ , the son of John Herschel Glenn, Sr. (1895–1966) and
+ <ins>
+  Clara
+ </ins>
+ Teresa (
@@ -396 +486,2 @@
- . On two occasions, he returned to his base with over 250 holes in his aircraft.
+ .
+ On two occasions, he returned to his base with over 250 holes in his aircraft.
@@ -471 +562,3 @@
- (class 12), graduating in 1954. He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns. He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
+ (class 12), graduating in 1954.
+ He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns.
+ He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
@@ -486 +579,8 @@
- Glenn has nearly 9,000 hours of flying time, with approximately 3,000 hours in
+ Glenn
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ nearly 9,000 hours of flying time, with approximately 3,000 hours in
@@ -504 +604,2 @@
- . The flight from
+ .
+ The flight from
@@ -548,0 +650,16 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:172px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    John Glenn in his
+   </ins>
+   <ins>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </ins>
+  </div>
+ </div>
+</div>
@@ -580,0 +698,11 @@
+ <div class="thumbinner" style="width:242px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn (center) with President John F. Kennedy and General Leighton I. Davis celebrating Glenn's orbital flight, 1962
+   </ins>
+  </div>
+ </div>
+</div>
+<div class="thumb tleft">
@@ -582,3 +709,0 @@
-  <a class="image" href="/wiki/File:GPN-2000-001027.jpg">
-   <img alt="" class="thumbimage" data-file-height="2343" data-file-width="1800" height="221" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/170px-GPN-2000-001027.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/255px-GPN-2000-001027.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/340px-GPN-2000-001027.jpg 2x" width="170"/>
-  </a>
@@ -587,2 +711,0 @@
-    <a class="internal" href="/wiki/File:GPN-2000-001027.jpg" title="Enlarge">
-    </a>
@@ -590,4 +713,8 @@
-   John Glenn in his
-   <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
-    Mercury spacesuit
-   </a>
+   <del>
+    John Glenn in his
+   </del>
+   <del>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </del>
@@ -633 +760,2 @@
- , became known worldwide as the "City of Light" when residents turned on their house, car and streetlights as Glenn passed overhead.
+ , became known worldwide as the "City of Light"
+ when residents turned on their house, car and streetlights as Glenn passed overhead.
@@ -638 +766,8 @@
- in 1998).
+ in
+ <ins>
+  1998.
+ )
+ </ins>
+ <del>
+  1998).
+ </del>
@@ -730 +865,8 @@
- in 1995, more than 30 years after the hearings. In the late 1970s, Glenn is reported to have supported Shuttle Mission Specialist Astronaut
+ in 1995, more than 30 years after the hearings. In the late 1970s, Glenn
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ reported to have supported Shuttle Mission Specialist Astronaut
@@ -804,0 +947,3 @@
+ <ins>
+  He served as a pallbearer at Kennedy's funeral.
+ </ins>
@@ -823 +968,6 @@
- . Instead, he challenged Metzenbaum again, whom Gilligan had appointed
+ . Instead, he challenged Metzenbaum again, whom Gilligan had
+ <ins>
+  appointed.
+ </ins>
+ <del>
+  appointed
@@ -825,4 +975,5 @@
- <a href="/wiki/William_B._Saxbe" title="William B. Saxbe">
-  William B. Saxbe
- </a>
- , who had resigned to become Attorney General of the United States.
+  <a href="/wiki/William_B._Saxbe" title="William B. Saxbe">
+   William B. Saxbe
+  </a>
+  , who had resigned to become Attorney General of the United States.
+ </del>
@@ -876,0 +1028,3 @@
+ <ins>
+  In 1997, Glenn announced that he would retire from the Senate at the end of his then-current term.
+ </ins>
@@ -911,5 +1065,10 @@
- to keep his seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
- <i>
-  on earth
- </i>
- has John Glenn done?"
+ to keep his
+ <ins>
+  seat.
+ </ins>
+ <del>
+  seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
+  <i>
+   on earth
+  </i>
+  has John Glenn done?"
@@ -916,0 +1076 @@
+ </del>
@@ -932 +1092,2 @@
- . Glenn also ran for the
+ .
+ Glenn also ran for the
@@ -936,4 +1097,11 @@
- . A November 1983
- <i>
-  <a href="/wiki/The_New_York_Times" title="The New York Times">
-   New York Times
+ .
+ <del>
+  A November 1983
+  <i>
+   <a href="/wiki/The_New_York_Times" title="The New York Times">
+    New York Times
+   </a>
+  </i>
+  /
+  <a href="/wiki/CBS_News" title="CBS News">
+   CBS News
@@ -941,6 +1109,2 @@
- </i>
- /
- <a href="/wiki/CBS_News" title="CBS News">
-  CBS News
- </a>
- poll found him second, supported by 41% of those polled, to Mondale's 49%.
+  poll found him second, supported by 41% of those polled, to Mondale's 49%.
+ </del>
@@ -977,5 +1141,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (July 2014)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -983,5 +1147,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -999 +1163,2 @@
- , served as chairman of the
+ ,
+ served as chairman of the
@@ -1015 +1180,8 @@
- . Once Republicans regained control of the Senate, Glenn also served as the ranking minority member on a special Senate investigative committee chaired by
+ .
+</p>
+<p>
+ Once Republicans regained control of the Senate, Glenn
+ <del>
+  also
+ </del>
+ served as the ranking minority member on a special Senate investigative committee chaired by
@@ -1027 +1199,14 @@
- to U.S. political campaigns for the 1996 election. There was considerable acrimony between the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
+ to U.S. political campaigns for the 1996 election.
+ There was considerable acrimony between the two
+ <ins>
+  men
+ </ins>
+ <del>
+  very high-profile senators
+ </del>
+ during the life of this
+ <ins>
+  committee.
+ </ins>
+ <del>
+  committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
@@ -1029,12 +1214,13 @@
- <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
-  Mary O. Boyle
- </a>
- was the Democratic party nominee. She faced Republican nominee and sitting governor
- <a href="/wiki/George_Voinovich" title="George Voinovich">
-  George Voinovich
- </a>
- in the
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
-  general election
- </a>
- , which Voinovich won.
+  <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
+   Mary O. Boyle
+  </a>
+  was the Democratic party nominee. She faced Republican nominee and sitting governor
+  <a href="/wiki/George_Voinovich" title="George Voinovich">
+   George Voinovich
+  </a>
+  in the
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
+   general election
+  </a>
+  , which Voinovich won.
+ </del>
@@ -1058 +1244,5 @@
- on October 29, 1998, becoming, at age 77, the oldest person to go into space as a
+ on October 29, 1998,
+ <del>
+  becoming, at age 77, the oldest person to go into space
+ </del>
+ as a
@@ -1072 +1262,7 @@
- mission. According to
+ <ins>
+  mission, becoming, at age 77, the oldest person to go into space.
+ </ins>
+ <del>
+  mission.
+ </del>
+ According to
@@ -1078,0 +1275,2 @@
+</p>
+<p>
@@ -1088,3 +1286,7 @@
-</p>
-<p>
- Glenn's participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
+ <ins>
+  His
+ </ins>
+ <del>
+  Glenn's
+ </del>
+ participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
@@ -1142 +1344 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+ <a href="/wiki/Ohio_State_University" title="Ohio State University">
@@ -1149 +1351,7 @@
- . Today Glenn holds an adjunct professorship at the Glenn School.
+ <ins>
+  , and Glenn held
+ </ins>
+ <del>
+  . Today Glenn holds
+ </del>
+ an adjunct professorship at the Glenn School.
@@ -1167 +1375,7 @@
-   Anna and John Glenn in 1965
+   <ins>
+    Glenn and his wife Anna
+   </ins>
+   <del>
+    Anna and John Glenn
+   </del>
+   in 1965
@@ -1172 +1386,8 @@
- On April 6, 1943, Glenn married his childhood sweetheart,
+ On April 6, 1943, Glenn married his
+ <ins>
+  high school
+ </ins>
+ <del>
+  childhood
+ </del>
+ sweetheart,
@@ -1184 +1405,50 @@
- . He also was a member of the Stag Club Fraternity at Muskingum College.
+ <ins>
+  , where he
+ </ins>
+ <del>
+  . He also
+ </del>
+ was a member of the Stag Club
+</p>
+<p>
+ <ins>
+  Fraternity.
+ Together, they had two children, John David and Carolyn Ann, and two grandchildren.
+ They remained married until his death. His boyhood home in New Concord has been
+ </ins>
+ <ins>
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+ </ins>
+ <ins>
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </ins>
+</p>
+<p>
+ <ins>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn was a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and was an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  He set an example of someone whose faith began before he became an astronaut, and whose faith was reinforced after traveling in space. Subsequent astronauts similarly had their faith deepened. "To look out at this kind of creation and not believe in God is to me impossible," said Glenn, after one of his later space voyages.
+ He stated that he saw no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believed evolution should be taught in schools.
+ He explained:
+ </ins>
@@ -1186,2 +1456,15 @@
-<p>
- Glenn was also one of the original owners of a
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   I don't see that I'm any less religious that I can appreciate the fact that science just records that we change with evolution and time, and that's a fact. It doesn't mean it's less wondrous and it doesn't mean that there can't be some power greater than any of us that has been behind and is behind whatever is going on.
+  </ins>
+ </p>
+</blockquote>
+<ins>
+ Glenn was
+</ins>
+<del>
+ Fraternity at Muskingum College.
+Glenn was also
+</del>
+  one of the original owners of a
@@ -1189,2 +1472,2 @@
-  Holiday Inn
- </a>
+ Holiday Inn
+</a>
@@ -1193,2 +1476,2 @@
-  Orlando
- </a>
+ Orlando
+</a>
@@ -1197,2 +1480,2 @@
-  Seralago Hotel &amp; Suites Main Gate East
- </a>
+ Seralago Hotel &amp; Suites Main Gate East
+</a>
@@ -1199,0 +1483,11 @@
+ <p>
+ <ins>
+  His business partner was
+ </ins>
+ <ins>
+  <a href="/wiki/Henri_Landwirth" title="Henri Landwirth">
+   Henri Landwirth
+  </a>
+  , a Holocaust survivor, who became Glenn's "best friend."
+ Glenn recalls learning about Landwirth's background:
+ </ins>
@@ -1201,2 +1495,21 @@
-<p>
- Glenn is an honorary member of the
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   Henri doesn't talk about it much. It was years before he spoke about it with me and then only because of an accident. We were down in Florida during the space program. Everyone was wearing short-sleeved Ban-Lon shirts—everyone but Henri. Then one day I saw Henri at the pool and noticed the number on his arm. I told Henri that if it were me I'd wear that number like a medal with a spotlight on it.
+  </ins>
+ </p>
+</blockquote>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Public_appearances_and_ceremonies">
+   Public appearances and ceremonies
+  </span>
+ </ins>
+</h3>
+<ins>
+ Glenn was
+</ins>
+<del>
+ Glenn is
+</del>
+ an honorary member of the
@@ -1204,2 +1517,2 @@
-  International Academy of Astronautics
- </a>
+ International Academy of Astronautics
+</a>
@@ -1208,2 +1521,2 @@
-  Society of Experimental Test Pilots
- </a>
+ Society of Experimental Test Pilots
+</a>
@@ -1212,2 +1525,2 @@
-  Order of Daedalians
- </a>
+ Order of Daedalians
+</a>
@@ -1216,2 +1529,2 @@
-  Ohio Democratic Party
- </a>
+ Ohio Democratic Party
+</a>
@@ -1218,0 +1532,16 @@
+ <p>
+ <del>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and is an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </del>
@@ -1221,23 +1550,4 @@
- A
- <a href="/wiki/Freemasonry" title="Freemasonry">
-  Freemason
- </a>
- , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
- <a href="/wiki/DeMolay_International" title="DeMolay International">
-  DeMolay International
- </a>
- , the Masonic youth organization, and is an ordained elder in the
- <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
-  Presbyterian Church
- </a>
- .
-</p>
-<p>
- Glenn's name was used for the character of
- <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
-  John Tracy
- </a>
- in the 1960s children's TV series
- <i>
-  <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
-   Thunderbirds
+ <del>
+  Glenn's name was used for the character of
+  <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
+   John Tracy
@@ -1245,2 +1555,8 @@
- </i>
- .
+  in the 1960s children's TV series
+  <i>
+   <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
+    Thunderbirds
+   </a>
+  </i>
+  .
+ </del>
@@ -1249,9 +1565,11 @@
- Glenn's boyhood home in New Concord has been
- <a href="/wiki/Building_restoration" title="Building restoration">
-  restored
- </a>
- and made into an
- <a href="/wiki/Historic_house_museum" title="Historic house museum">
-  historic house museum
- </a>
- and education center.
+ <del>
+  Glenn's boyhood home in New Concord has been
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </del>
@@ -1266,19 +1584,29 @@
- .
-</p>
-<p>
- On August 4, 2006, Glenn and his wife were injured in an automobile accident on
- <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
-  I-270
- </a>
- near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
- <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
-  sternum
- </a>
- . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
-</p>
-<p>
- On September 5, 2009, John and Annie Glenn dotted the "i" during
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  The Ohio State University
- </a>
- 's
+ <ins>
+  , as himself.
+ </ins>
+ <del>
+  .
+ </del>
+</p>
+<p>
+ <del>
+  On August 4, 2006, Glenn and his wife were injured in an automobile accident on
+  <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
+   I-270
+  </a>
+  near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
+  <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
+   sternum
+  </a>
+  . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
+ </del>
+</p>
+<p>
+ On September 5, 2009, John and Annie Glenn dotted the "i" during The Ohio State
+ <ins>
+  University's
+ </ins>
+ <del>
+  University
+  Link: /wiki/The_Ohio_State_University 's
+ </del>
@@ -1295 +1623,6 @@
- -
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
@@ -1299,0 +1633,3 @@
+ <ins>
+  Other non-band members to have received this honor include
+ </ins>
@@ -1308,16 +1644,18 @@
- <a href="/wiki/Buster_Douglas" title="Buster Douglas">
-  Buster Douglas
- </a>
- ,
- <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
-  E. Gordon Gee
- </a>
- ,
- <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
-  Novice Fawcett
- </a>
- ,
- <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
-  Robert Ries
- </a>
- , and
+ <del>
+  <a href="/wiki/Buster_Douglas" title="Buster Douglas">
+   Buster Douglas
+  </a>
+  ,
+  <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
+   E. Gordon Gee
+  </a>
+  ,
+  <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
+   Novice Fawcett
+  </a>
+  ,
+  <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
+   Robert Ries
+  </a>
+  , and
+ </del>
@@ -1331 +1669,6 @@
- are the only other non-band members to have received this honor.
+ <ins>
+  .
+ </ins>
+ <del>
+  are the only other non-band members to have received this honor.
+ </del>
@@ -1354,4 +1697,10 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  Ohio State
- </a>
- , where the public affairs school is named for him.
+ <ins>
+  The Ohio State University,
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+   Ohio State
+  </a>
+  ,
+ </del>
+ where the public affairs school is named for him.
@@ -1400 +1749,24 @@
- In June 2014, Glenn underwent a successful
+ In June
+</p>
+<p>
+ <ins>
+  2016 the Columbus, Ohio airport known for many years as Port Columbus was officially renamed the
+ </ins>
+ <ins>
+  <a href="/wiki/John_Glenn_Columbus_International_Airport" title="John Glenn Columbus International Airport">
+   John Glenn Columbus International Airport
+  </a>
+  . Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke about how visiting that airport as a child inspired his interest in flying.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Illness_and_death">
+   Illness and death
+  </span>
+ </ins>
+</h2>
+<ins>
+ In June
+</ins>
+  2014, Glenn underwent a successful
@@ -1402,2 +1774,2 @@
-  heart valve replacement
- </a>
+ heart valve replacement
+</a>
@@ -1406,2 +1778,2 @@
-  Cleveland Clinic
- </a>
+ Cleveland Clinic
+</a>
@@ -1408,0 +1781,4 @@
+ <p>
+ <ins>
+  At the beginning of December 2016,
+ </ins>
@@ -1411 +1787,3 @@
- On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ <del>
+  On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ </del>
@@ -1414 +1792,3 @@
- Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ <del>
+  Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ </del>
@@ -1417,3 +1797,5 @@
- <span class="mw-headline" id="Illness">
-  Illness
- </span>
+ <del>
+  <span class="mw-headline" id="Illness">
+   Illness
+  </span>
+ </del>
@@ -1421 +1803 @@
-<p>
+<del>
@@ -1422,0 +1805,2 @@
+</del>
+<del>
@@ -1426,4 +1810,20 @@
- announced that Glenn was hospitalized at OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
- <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
-  The James Cancer Hospital
- </a>
+ announced that
+</del>
+ Glenn was hospitalized at <ins>
+ the
+</ins>
+<a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
+ <del>
+  OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
+ The
+ </del>
+ James Cancer Hospital
+</a>
+<ins>
+ of
+ <a href="/wiki/Ohio_State_University_Wexner_Medical_Center" title="Ohio State University Wexner Medical Center">
+  OSU Wexner Medical Center
+ </a>
+ in Columbus.
+</ins>
+<del>
@@ -1431 +1831,34 @@
- A family source said that Glenn was in declining health, that his condition was grave, and that Annie Glenn and his children and grandchildren had joined him at the hospital.
+</del>
+ A family source said that Glenn <ins>
+ had been
+</ins>
+<del>
+ was
+</del>
+ in declining health, <ins>
+ and
+</ins>
+ that his condition was <ins>
+ grave. His wife, Annie, and their
+</ins>
+<del>
+ grave, and that Annie Glenn and his
+</del>
+ children and grandchildren had joined him at the hospital.
+ <p>
+ <ins>
+  Glenn died December 8, 2016, at the OSU Wexner Medical Center.
+ </ins>
+</p>
+<p>
+ <ins>
+  No cause of death has yet been disclosed. Glenn will be interred at
+  <a href="/wiki/Arlington_National_Cemetery" title="Arlington National Cemetery">
+   Arlington National Cemetery
+  </a>
+  after lying in state at the
+  <a href="/wiki/Ohio_Statehouse" title="Ohio Statehouse">
+   Ohio Statehouse
+  </a>
+  and a memorial service at Mershon Auditorium at The Ohio State University.
+ </ins>
@@ -1471,3 +1904,5 @@
-     <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
-      Friendship 7
-     </a>
+     <i>
+      <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
+       Friendship 7
+      </a>
+     </i>
@@ -1766 +2201,7 @@
-    and two
+    and
+    <ins>
+     eighteen
+    </ins>
+    <del>
+     two
+    </del>
@@ -1778 +2219,10 @@
-    with fifteen stars and two clusters
+    with fifteen stars and
+    <ins>
+     eighteen
+     <a href="/wiki/Oak_leaf_cluster" title="Oak leaf cluster">
+      clusters
+     </a>
+    </ins>
+    <del>
+     two clusters
+    </del>
@@ -1917,4 +2367,6 @@
-  <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
-   Golden Plate Award
-  </a>
-  for Science and Exploration, 1964
+  <del>
+   <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
+    Golden Plate Award
+   </a>
+   for Science and Exploration, 1964
+  </del>
@@ -1975 +2427,5 @@
-   ) to Senator John Glenn and
+   ) to Senator John Glenn
+   <ins>
+    (left)
+   </ins>
+   and
@@ -1982,0 +2439,3 @@
+   <ins>
+    (right)
+   </ins>
@@ -2079 +2538,8 @@
- , a mobile landing platform that is scheduled to be delivered to the
+ , a mobile landing platform that
+ <ins>
+  was
+ </ins>
+ <del>
+  is scheduled to be
+ </del>
+ delivered to the
@@ -2083 +2549,7 @@
- in 2014 is named for him. It was christened February 1, 2014, in
+ <ins>
+  on March 12, 2014,
+ </ins>
+ <del>
+  in 2014
+ </del>
+ is named for him. It was christened February 1, 2014, in
@@ -2110,3 +2582,5 @@
-</p>
-<p>
- He also received Honorary Doctorates from
+ He
+ <del>
+  also
+ </del>
+ received Honorary Doctorates from
@@ -2148,2 +2621,0 @@
-</p>
-<p>
@@ -2214,12 +2686,14 @@
- On January 3, 2015, with the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- , Glenn became the
- <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
-  oldest former United States senator still living
- </a>
- . Glenn is the 60th person to hold this distinction.
-</p>
-<p>
- On September 12, 2016,
+ On
+ <del>
+  January 3, 2015, with the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  , Glenn became the
+  <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
+   oldest former United States senator still living
+  </a>
+  . Glenn is the 60th person to hold this distinction.
+On
+ </del>
+ September 12, 2016,
@@ -2236,3 +2710,5 @@
- <span class="mw-headline" id="Physical_description">
-  Physical description
- </span>
+ <del>
+  <span class="mw-headline" id="Physical_description">
+   Physical description
+  </span>
+ </del>
@@ -2242 +2718,3 @@
-  Weight: 170 lb (77 kg)
+  <del>
+   Weight: 170 lb (77 kg)
+  </del>
@@ -2245 +2723,3 @@
-  Height: 5 ft 10½ in (1.79 m)
+  <del>
+   Height: 5 ft 10½ in (1.79 m)
+  </del>
@@ -2248 +2728,3 @@
-  Hair: Red
+  <del>
+   Hair: Red
+  </del>
@@ -2251 +2733,3 @@
-  Eyes: Green
+  <del>
+   Eyes: Green
+  </del>
@@ -2322,0 +2807,2 @@
+<div style="clear:both;">
+</div>
\ No newline at end of file
