--- 753656269
+++ 753722387
@@ -1,5 +1,787 @@
-<p>
- <b>
-  John Herschel Glenn Jr.
- </b>
- (born July 18, 1921), (
+<table class="infobox vcard" style="width:22em">
+ <tr>
+  <p>
+   <ins>
+    December 8, 2016
+   </ins>
+  </p>
+  <div class="hatnote navigation-not-searchable" role="note">
+   <ins>
+    For other people named John Glenn, see
+    <a class="mw-disambig" href="/wiki/John_Glenn_(disambiguation)" title="John Glenn (disambiguation)">
+     John Glenn (disambiguation)
+    </a>
+    .
+   </ins>
+  </div>
+ </tr>
+ <tr>
+  <th colspan="2" style="text-align:center;font-size:125%;font-weight:bold;font-size: 130%;">
+   <ins>
+    <span class="fn">
+     John Glenn
+    </span>
+   </ins>
+  </th>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center">
+  </td>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center">
+  </td>
+ </tr>
+ <tr>
+  <th colspan="2" style="text-align:center;background:lavender">
+   <ins>
+    <a href="/wiki/United_States_Senate" title="United States Senate">
+     United States Senator
+    </a>
+    <br/>
+    from
+    <a href="/wiki/Ohio" title="Ohio">
+     Ohio
+    </a>
+   </ins>
+  </th>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center;border-bottom:none">
+   <ins>
+    <span class="nowrap">
+     <b>
+      In office
+     </b>
+    </span>
+    <br/>
+    December 24, 1974 – January 3, 1999
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="text-align:left">
+   <ins>
+    <span class="nowrap">
+     Preceded by
+    </span>
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Howard_Metzenbaum" title="Howard Metzenbaum">
+     Howard Metzenbaum
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="text-align:left">
+   <ins>
+    <span class="nowrap">
+     Succeeded by
+    </span>
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/George_Voinovich" title="George Voinovich">
+     George Voinovich
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th colspan="2" style="text-align:center;background:lavender">
+   <ins>
+    Chairman of the
+    <a class="mw-redirect" href="/wiki/United_States_Senate_Committee_on_Governmental_Affairs" title="United States Senate Committee on Governmental Affairs">
+     Senate Committee on Governmental Affairs
+    </a>
+   </ins>
+  </th>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center;border-bottom:none">
+   <ins>
+    <span class="nowrap">
+     <b>
+      In office
+     </b>
+    </span>
+    <br/>
+    January 3, 1987 – January 3, 1995
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="text-align:left">
+   <ins>
+    <span class="nowrap">
+     Preceded by
+    </span>
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/William_V._Roth_Jr." title="William V. Roth Jr.">
+     William V. Roth Jr.
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="text-align:left">
+   <ins>
+    <span class="nowrap">
+     Succeeded by
+    </span>
+   </ins>
+  </th>
+  <td>
+   <ins>
+    William V. Roth Jr.
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th colspan="2" style="text-align:center;background:lavender">
+   <ins>
+    Personal details
+   </ins>
+  </th>
+ </tr>
+ <th scope="row">
+  <ins>
+   Born
+  </ins>
+ </th>
+ <td>
+  <span class="nickname">
+   John Herschel Glenn Jr.
+  </span>
+  <p>
+   <ins>
+    <span style="display:none">
+     (
+     <span class="bday">
+      1921-07-18
+     </span>
+     )
+    </span>
+    July 18, 1921
+   </ins>
+   <ins>
+    <a href="/wiki/Cambridge,_Ohio" title="Cambridge, Ohio">
+     Cambridge, Ohio
+    </a>
+    , U.S.
+   </ins>
+  </p>
+ </td>
+ <tr>
+  <th scope="row">
+   <ins>
+    Died
+   </ins>
+  </th>
+  <td>
+   <ins>
+    December 8, 2016
+    <span style="display:none">
+     (
+     <span class="dday deathdate">
+      2016-12-08
+     </span>
+     )
+    </span>
+    (aged 95)
+    <br/>
+    <a href="/wiki/Columbus,_Ohio" title="Columbus, Ohio">
+     Columbus, Ohio
+    </a>
+    , U.S.
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Political party
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Democratic_Party_(United_States)" title="Democratic Party (United States)">
+     Democratic
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Spouse(s)
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <span itemscope="" itemtype="https://en.wikipedia.org/wiki/John Glenn">
+     <a href="/wiki/Annie_Glenn" title="Annie Glenn">
+      Annie Glenn
+     </a>
+     (
+     <abbr title="married">
+      m.
+     </abbr>
+     1943)
+    </span>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Residence
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Columbus,_Ohio" title="Columbus, Ohio">
+     Columbus, Ohio
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    <a href="/wiki/Alma_mater" title="Alma mater">
+     Alma mater
+    </a>
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <span class="nowrap">
+     <a href="/wiki/Muskingum_University" title="Muskingum University">
+      Muskingum University
+     </a>
+     , B.S. 1962
+    </span>
+    <br/>
+    <a class="mw-redirect" href="/wiki/University_of_Maryland" title="University of Maryland">
+     University of Maryland
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Religion
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a class="mw-redirect" href="/wiki/Presbyterian" title="Presbyterian">
+     Presbyterian
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Awards
+   </ins>
+  </th>
+  <td>
+   <div class="plainlist nowrap">
+    <ul>
+     <li>
+      <ins>
+       <a href="/wiki/Distinguished_Flying_Cross_(United_States)" title="Distinguished Flying Cross (United States)">
+        Distinguished Flying Cross
+       </a>
+       × 6
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Air_Medal" title="Air Medal">
+        Air Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Presidential_Unit_Citation_(United_States)" title="Presidential Unit Citation (United States)">
+        Presidential Unit Citation
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Navy_Unit_Commendation" title="Navy Unit Commendation">
+        Navy Unit Commendation
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/American_Campaign_Medal" title="American Campaign Medal">
+        American Campaign Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a class="mw-redirect" href="/wiki/Asiatic-Pacific_Campaign_Medal" title="Asiatic-Pacific Campaign Medal">
+        Asiatic-Pacific Campaign Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/World_War_II_Victory_Medal_(United_States)" title="World War II Victory Medal (United States)">
+        World War II Victory Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/China_Service_Medal" title="China Service Medal">
+        China Service Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Navy_Occupation_Service_Medal" title="Navy Occupation Service Medal">
+        Navy Occupation Service Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/National_Defense_Service_Medal" title="National Defense Service Medal">
+        National Defense Service Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Korean_Service_Medal" title="Korean Service Medal">
+        Korean Service Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/United_Nations_Korea_Medal" title="United Nations Korea Medal">
+        United Nations Korea Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <div style="padding:0.1em 0;line-height:1.2em;">
+       <ins>
+        <a href="/wiki/Republic_of_Korea_Presidential_Unit_Citation" title="Republic of Korea Presidential Unit Citation">
+         South Korea Presidential Unit Citation
+        </a>
+       </ins>
+      </div>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Congressional_Gold_Medal" title="Congressional Gold Medal">
+        Congressional Gold Medal
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Presidential_Medal_of_Freedom" title="Presidential Medal of Freedom">
+        Presidential Medal of Freedom
+       </a>
+      </ins>
+     </li>
+     <li>
+      <div style="padding:0.1em 0;line-height:1.2em;">
+       <ins>
+        <a href="/wiki/Congressional_Space_Medal_of_Honor" title="Congressional Space Medal of Honor">
+         Congressional Space Medal of Honor
+        </a>
+       </ins>
+      </div>
+     </li>
+    </ul>
+   </div>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Signature
+   </ins>
+  </th>
+  <td>
+  </td>
+ </tr>
+ <tr>
+  <th colspan="2" style="text-align:center;background:lavender">
+   <ins>
+    Military service
+   </ins>
+  </th>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Service/branch
+   </ins>
+  </th>
+  <td>
+   <div class="plainlist nowrap">
+    <ul>
+     <li>
+      <ins>
+       <a href="/wiki/United_States_Navy" title="United States Navy">
+        United States Navy
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+        United States Marine Corps
+       </a>
+      </ins>
+     </li>
+    </ul>
+   </div>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Years of service
+   </ins>
+  </th>
+  <td>
+   <ins>
+    1941–1965
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Rank
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
+     Colonel
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Unit
+   </ins>
+  </th>
+  <td>
+   <div class="hlist">
+    <ul>
+     <li>
+      <ins>
+       <a href="/wiki/Marine_Aircraft_Group_15" title="Marine Aircraft Group 15">
+        VMJ-353
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/VMF-155" title="VMF-155">
+        VMF-155
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/VMF-218" title="VMF-218">
+        VMF-218
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/VMA-311" title="VMA-311">
+        VMA-311
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/4th_Fighter_Wing" title="4th Fighter Wing">
+        4th Fighter Wing
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/51st_Fighter_Wing" title="51st Fighter Wing">
+        51st Fighter Wing
+       </a>
+      </ins>
+     </li>
+    </ul>
+   </div>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row">
+   <ins>
+    Battles/wars
+   </ins>
+  </th>
+  <td>
+   <div class="hlist">
+    <ul>
+     <li>
+      <ins>
+       <a href="/wiki/World_War_II" title="World War II">
+        World War II
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a href="/wiki/Korean_War" title="Korean War">
+        Korean War
+       </a>
+      </ins>
+     </li>
+    </ul>
+   </div>
+  </td>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center">
+   <hr/>
+  </td>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center">
+  </td>
+ </tr>
+ <tr>
+  <th class="role" colspan="2" style="text-align:center">
+   <ins>
+    <a href="/wiki/NASA" title="NASA">
+     NASA
+    </a>
+    <a href="/wiki/Astronaut" title="Astronaut">
+     Astronaut
+    </a>
+   </ins>
+  </th>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Nationality
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/United_States" title="United States">
+     United States
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Other names
+    </ins>
+   </div>
+  </th>
+  <td class="nickname">
+   <ins>
+    John Herschel Glenn, Jr.
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Other occupation
+    </ins>
+   </div>
+  </th>
+  <td class="role">
+   <ins>
+    <a href="/wiki/Test_pilot" title="Test pilot">
+     Test pilot
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr class="note">
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Time in space
+    </ins>
+   </div>
+  </th>
+  <td>
+   <ins>
+    4h 55m 23s
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Selection
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Mercury_Seven" title="Mercury Seven">
+     1959 NASA Group 1
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Missions
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Mercury-Atlas_6" title="Mercury-Atlas 6">
+     Mercury-Atlas 6
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Mission insignia
+    </ins>
+   </div>
+  </th>
+  <td>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Retirement
+   </ins>
+  </th>
+  <td>
+   <ins>
+    January 16, 1964
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Awards
+   </ins>
+  </th>
+  <td>
+  </td>
+ </tr>
+ <tr>
+  <td colspan="2" style="text-align:center">
+  </td>
+ </tr>
+ <tr>
+  <th class="role" colspan="2" style="text-align:center">
+   <ins>
+    <a href="/wiki/NASA" title="NASA">
+     NASA
+    </a>
+    <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
+     Payload Specialist
+    </a>
+   </ins>
+  </th>
+ </tr>
+ <tr class="note">
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Time in space
+    </ins>
+   </div>
+  </th>
+  <td>
+   <ins>
+    9d 2h 39m
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Missions
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/STS-95" title="STS-95">
+     STS-95
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <div style="padding:0.1em 0;line-height:1.2em;">
+    <ins>
+     Mission insignia
+    </ins>
+   </div>
+  </th>
+  <td>
+  </td>
+ </tr>
+ <tr>
+  <th scope="row" style="padding-right:0.65em;">
+   <ins>
+    Awards
+   </ins>
+  </th>
+  <td>
+  </td>
+ </tr>
+ <ins>
+  <b>
+   John Herschel Glenn Jr.
+  </b>
+  (July 18, 1921 – December 8, 2016),
+ </ins>
+</table>
+<del>
+ (born July 18, 1921),
+</del>
+ (
@@ -7,2 +789,2 @@
-  Col
- </a>
+ Col
+</a>
@@ -11,6 +793,11 @@
-  USMC
- </a>
- , Ret.), is a former
- <a href="/wiki/United_States_Marine_Corps_Aviation" title="United States Marine Corps Aviation">
-  aviator
- </a>
+ USMC
+</a>
+ , Ret.), <ins>
+ was an
+</ins>
+<del>
+ is a former
+</del>
+<a href="/wiki/United_States_Marine_Corps_Aviation" title="United States Marine Corps Aviation">
+ aviator
+</a>
@@ -19,2 +806,2 @@
-  engineer
- </a>
+ engineer
+</a>
@@ -23,2 +810,2 @@
-  astronaut
- </a>
+ astronaut
+</a>
@@ -27,2 +814,2 @@
-  United States senator
- </a>
+ United States senator
+</a>
@@ -31,2 +818,2 @@
-  Mercury Seven
- </a>
+ Mercury Seven
+</a>
@@ -35,2 +822,2 @@
-  test pilots
- </a>
+ test pilots
+</a>
@@ -39,2 +826,2 @@
-  NASA
- </a>
+ NASA
+</a>
@@ -43,2 +830,2 @@
-  Project Mercury
- </a>
+ Project Mercury
+</a>
@@ -46,2 +833 @@
-</p>
-<p>
+ <p>
@@ -124 +910,7 @@
- on January 3, 2015, Glenn became the oldest living former United States Senator.
+ on January 3, 2015, Glenn became the oldest living former United States
+ <ins>
+  Senator, and has now died at 95
+ </ins>
+ <del>
+  Senator.
+ </del>
@@ -1417 +2209 @@
- <span class="mw-headline" id="Illness">
+ <span class="mw-headline" id="Illness_and_death">
@@ -1418,0 +2211,3 @@
+  <ins>
+   and death
+  </ins>
@@ -1431,0 +2227,3 @@
+ <ins>
+  Glenn died December 8, 2016 at the Ohio State University Wexner Medical Center in Columbus, Ohio.
+ </ins>
@@ -2322 +3120 @@
-</ul>
+</ul>
\ No newline at end of file
