--- 753656269
+++ 753720863
@@ -5 +5,8 @@
- (born July 18, 1921), (
+ (born July 18,
+ <ins>
+  1921, died December 8, 2016),
+ </ins>
+ <del>
+  1921),
+ </del>
+ (
@@ -149,0 +157,3 @@
+ <ins>
+  Glenn died on December 8, 2016.
+ </ins>
@@ -1431,0 +1442,3 @@
+ <ins>
+  Glenn died December 8, 2016 at the Ohio State University Wexner Medical Center in Columbus, Ohio.
+ </ins>
@@ -2322 +2335 @@
-</ul>
+</ul>
\ No newline at end of file
