--- 753656269
+++ 753722913
@@ -5 +5,7 @@
- (born July 18, 1921), (
+ <ins>
+  (July 18, 1921 – December 8, 2016),
+ </ins>
+ <del>
+  (born July 18, 1921),
+ </del>
+ (
@@ -13 +19,7 @@
- , Ret.), is a former
+ , Ret.),
+ <ins>
+  was an American
+ </ins>
+ <del>
+  is a former
+ </del>
@@ -27 +39,4 @@
-  United States senator
+  <del>
+   United States
+  </del>
+  senator
@@ -47,0 +63,11 @@
+ <ins>
+  Sadly John Glenn has just died at the age of 95 on December, 8 2016. AAS
+ </ins>
+</p>
+<p>
+ <br/>
+</p>
+<p>
+ <br/>
+</p>
+<p>
@@ -124 +150,7 @@
- on January 3, 2015, Glenn became the oldest living former United States Senator.
+ on January 3, 2015, Glenn became the oldest living former United States
+ <ins>
+  Senator, and has now died at 95
+ </ins>
+ <del>
+  Senator.
+ </del>
@@ -1149 +1181,7 @@
- . Today Glenn holds an adjunct professorship at the Glenn School.
+ <ins>
+  , and Glenn held
+ </ins>
+ <del>
+  . Today Glenn holds
+ </del>
+ an adjunct professorship at the Glenn School.
@@ -1417 +1455 @@
- <span class="mw-headline" id="Illness">
+ <span class="mw-headline" id="Illness_and_death">
@@ -1418,0 +1457,3 @@
+  <ins>
+   and death
+  </ins>
@@ -1431,0 +1473,3 @@
+ <ins>
+  Glenn died December 8, 2016 at the Ohio State University Wexner Medical Center in Columbus, Ohio.
+ </ins>
@@ -2322 +2366 @@
-</ul>
+</ul>
\ No newline at end of file
