--- 753656269
+++ 753948769
@@ -5,9 +5,14 @@
- (born July 18, 1921), (
- <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
-  Col
- </a>
- ,
- <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
-  USMC
- </a>
- , Ret.), is a former
+ <ins>
+  (July 18, 1921 – December 8, 2016) was an American
+ </ins>
+ <del>
+  (born July 18, 1921), (
+  <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
+   Col
+  </a>
+  ,
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   USMC
+  </a>
+  , Ret.), is a former
+ </del>
@@ -27 +32,15 @@
-  United States senator
+  United States  Senator
+  Link: /wiki/United_States_Senate from
+ Ohio
+  Link: /wiki/Ohio  . In 1962 he became the first American to orbit the
+ Earth
+  Link: /wiki/Earth  , circling three times. Before joining NASA, he was a distinguished fighter pilot in both
+ World War II
+  Link: /wiki/World_War_II  and
+ Korea
+  Link: /wiki/Korean_War  , with five
+ Distinguished Flying Crosses
+  Link: /wiki/Distinguished_Flying_Cross_(United_States)  and eighteen
+ clusters
+  Link: /wiki/Oak_leaf_cluster  .
+Glenn
@@ -29 +48,6 @@
- . He was one of the "
+</p>
+<del>
+ senator
+  Link: /wiki/United_States_Senate . He
+</del>
+ was one of the "
@@ -31,2 +55,2 @@
-  Mercury Seven
- </a>
+ Mercury Seven
+</a>
@@ -35,2 +59,2 @@
-  test pilots
- </a>
+ test pilots
+</a>
@@ -39,3 +63,7 @@
-  NASA
- </a>
- to become America's first astronauts and fly the
+ NASA
+</a>
+ to become America's first <ins>
+ astronauts.
+</ins>
+<del>
+ astronauts and fly the
@@ -46,3 +74,8 @@
-</p>
-<p>
- On February 20, 1962, Glenn flew the
+</del>
+ On February 20, 1962, <ins>
+ he
+</ins>
+<del>
+ Glenn
+</del>
+ flew the
@@ -50,4 +83,4 @@
-  <a href="/wiki/Mercury-Atlas_6" title="Mercury-Atlas 6">
-   Friendship 7
-  </a>
- </i>
+ <a href="/wiki/Mercury-Atlas_6" title="Mercury-Atlas 6">
+  Friendship 7
+ </a>
+</i>
@@ -56,2 +89,2 @@
-  American
- </a>
+ American
+</a>
@@ -60,2 +93,2 @@
-  orbit
- </a>
+ orbit
+</a>
@@ -64,3 +97,7 @@
-  Earth
- </a>
- and the fifth person in space, after
+ Earth
+</a>
+ and the fifth person in <ins>
+ space.
+</ins>
+<del>
+ space, after
@@ -93 +130,3 @@
- . Glenn received the
+ .
+</del>
+ Glenn received the
@@ -95,3 +134,6 @@
-  Congressional Space Medal of Honor
- </a>
- in 1978, and was inducted into the
+ Congressional Space Medal of Honor
+</a>
+ in 1978, <del>
+ and
+</del>
+ was inducted into the
@@ -99,3 +141,7 @@
-  U.S. Astronaut Hall of Fame
- </a>
- in 1990. With the death of
+ U.S. Astronaut Hall of Fame
+</a>
+ in <ins>
+ 1990, and was
+</ins>
+<del>
+ 1990. With the death of
@@ -105,4 +151,18 @@
- on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
-</p>
-<p>
- Glenn resigned from NASA on January 16, 1964, and the next day announced plans to run for a U.S. Senate seat from
+ on October 10, 2013, Glenn became
+</del>
+ the last surviving member of the Mercury Seven.
+ <p>
+ <ins>
+  After he
+ </ins>
+ <del>
+  Glenn
+ </del>
+ resigned from NASA
+ <ins>
+  in 1964, Glenn planned
+ </ins>
+ <del>
+  on January 16, 1964, and the next day announced plans
+ </del>
+ to run for a U.S. Senate seat from
@@ -112 +172,7 @@
- ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965. A member of the
+ <ins>
+  .
+ </ins>
+ <del>
+  ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965.
+ </del>
+ A member of the
@@ -116 +182,7 @@
- , he finally
+ , he
+ <ins>
+  first
+ </ins>
+ <del>
+  finally
+ </del>
@@ -120,8 +192,29 @@
- and served through January 3, 1999. With the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- on January 3, 2015, Glenn became the oldest living former United States Senator.
-</p>
-<p>
- On October 29, 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
+ <ins>
+  where he
+ </ins>
+ <del>
+  and
+ </del>
+ served through January 3, 1999.
+</p>
+<p>
+ <ins>
+  He retired from the Marine Corps in 1965, after twenty-three years in the military, with over fifteen medals and awards, including the
+  <a href="/wiki/NASA_Distinguished_Service_Medal" title="NASA Distinguished Service Medal">
+   NASA Distinguished Service Medal
+  </a>
+  and the
+  <a href="/wiki/Congressional_Space_Medal_of_Honor" title="Congressional Space Medal of Honor">
+   Congressional Space Medal of Honor
+  </a>
+  . In
+ </ins>
+ <del>
+  With the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  on January 3, 2015, Glenn became the oldest living former United States Senator.
+On October 29,
+ </del>
+ 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
@@ -131,7 +224,3 @@
- , when at age 77, he flew as a
- <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
-  Payload Specialist
- </a>
- on
- <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
-  <i>
+ <ins>
+  as crew member of the
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
@@ -139,7 +228,19 @@
-  </i>
- </a>
- mission
- <a href="/wiki/STS-95" title="STS-95">
-  STS-95
- </a>
- . He was awarded the
+  </a>
+  space shuttle. He was also
+ </ins>
+ <del>
+  , when at age 77, he flew as a
+  <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
+   Payload Specialist
+  </a>
+  on
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
+   Discovery
+  </a>
+  mission
+  <a href="/wiki/STS-95" title="STS-95">
+   STS-95
+  </a>
+  . He was
+ </del>
+ awarded the
@@ -159,0 +261,11 @@
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:252px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn's childhood home in New Concord
+   </ins>
+  </div>
+ </div>
+</div>
@@ -169 +281,5 @@
- , the son of John Herschel Glenn, Sr. (1895–1966) and Teresa (
+ , the son of John Herschel Glenn, Sr. (1895–1966) and
+ <ins>
+  Clara
+ </ins>
+ Teresa (
@@ -176 +292,7 @@
-  New Concord, Ohio
+  <ins>
+   nearby
+ New Concord
+  </ins>
+  <del>
+   New Concord, Ohio
+  </del>
@@ -396 +518,2 @@
- . On two occasions, he returned to his base with over 250 holes in his aircraft.
+ .
+ On two occasions, he returned to his base with over 250 holes in his aircraft.
@@ -471 +594,3 @@
- (class 12), graduating in 1954. He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns. He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
+ (class 12), graduating in 1954.
+ He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns.
+ He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
@@ -486 +611,8 @@
- Glenn has nearly 9,000 hours of flying time, with approximately 3,000 hours in
+ Glenn
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ nearly 9,000 hours of flying time, with approximately 3,000 hours in
@@ -504 +636,2 @@
- . The flight from
+ .
+ The flight from
@@ -548,0 +682,16 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:172px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    John Glenn in his
+   </ins>
+   <ins>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </ins>
+  </div>
+ </div>
+</div>
@@ -550,25 +699,50 @@
- In 1958, the newly formed NASA began a recruiting program for astronauts. Requirements were that each had to be a military test pilot between the ages of 25 and 40 with sufficient flight hours, no more than 5'11" in height, and possess a degree in a scientific field. 508 pilots were subjected to rigorous mental and physical tests, and finally the selection was narrowed down to seven astronauts (Glenn,
- <a href="/wiki/Alan_Shepard" title="Alan Shepard">
-  Alan Shepard
- </a>
- ,
- <a href="/wiki/Gus_Grissom" title="Gus Grissom">
-  Gus Grissom
- </a>
- ,
- <a href="/wiki/Scott_Carpenter" title="Scott Carpenter">
-  Scott Carpenter
- </a>
- ,
- <a href="/wiki/Wally_Schirra" title="Wally Schirra">
-  Wally Schirra
- </a>
- ,
- <a href="/wiki/Gordon_Cooper" title="Gordon Cooper">
-  Gordon Cooper
- </a>
- , and
- <a href="/wiki/Deke_Slayton" title="Deke Slayton">
-  Deke Slayton
- </a>
- ), who were introduced to the public at a NASA press conference in April 1959. Glenn just barely met the requirements as he was close to the age cutoff of 40 and also lacked the required science-based degree at the time. During this time, he remained an officer in the
+ <ins>
+  While Glenn was on duty at Patuxent and Washington, Glenn began to read everything he could about space. His office was requested to furnish a test pilot to be sent to the Langley Air Force Base in Virginia to make some runs on a spaceflight simulator, which was a part of NASA research on reentry vehicle shapes. The officer would also be sent to the Naval Air Development Center in Johnsville, Pennsylvania. The test pilot would be subjected to high g-forces in a centrifuge to compare to the data collected in the simulator. Glenn requested this position and was granted it. He spent a few days at Langley and a week in Johnsville for the testing.
+ </ins>
+</p>
+<p>
+ <ins>
+  Prior to Glenn's appointment as an astronaut in the Mercury program, he participated in the capsule design. NASA had requested that military service members participate in planning the mockup of the capsule. Since Glenn had participated in the research at Langley and Johnsville, combined he with his experience sitting on mock-up boards in the Navy and his knowledge of the capsule procedures, he was sent to the McDonnell plant in St. Louis and acted as a service adviser on the mock-up board.
+ </ins>
+</p>
+<p>
+ In 1958, the newly formed NASA began a recruiting program for
+ <ins>
+  astronauts,
+ and
+ </ins>
+ <del>
+  astronauts. Requirements were that each had to be a military test pilot between the ages of 25 and 40 with sufficient flight hours, no more than 5'11" in height, and possess a degree in a scientific field. 508 pilots were subjected to rigorous mental and physical tests, and finally the selection was narrowed down to seven astronauts (Glenn,
+  <a href="/wiki/Alan_Shepard" title="Alan Shepard">
+   Alan Shepard
+  </a>
+  ,
+  <a href="/wiki/Gus_Grissom" title="Gus Grissom">
+   Gus Grissom
+  </a>
+  ,
+  <a href="/wiki/Scott_Carpenter" title="Scott Carpenter">
+   Scott Carpenter
+  </a>
+  ,
+  <a href="/wiki/Wally_Schirra" title="Wally Schirra">
+   Wally Schirra
+  </a>
+  ,
+  <a href="/wiki/Gordon_Cooper" title="Gordon Cooper">
+   Gordon Cooper
+  </a>
+  , and
+  <a href="/wiki/Deke_Slayton" title="Deke Slayton">
+   Deke Slayton
+  </a>
+  ), who were introduced to the public at a NASA press conference in April 1959.
+ </del>
+ Glenn just barely met the requirements as he was close to the age cutoff of 40 and also lacked the required science-based degree at the time.
+ <ins>
+  He
+ </ins>
+ <del>
+  During this time, he
+ </del>
+ remained an officer in the
@@ -578 +752,7 @@
- .
+ <ins>
+  after he was selected in 1959.
+ After his selection, he was assigned to the NASA Space Task Group in 1959, which was located at Langley Research Center in Hampton, Virginia.
+ The task force was moved to Houston in 1962 and became a part of the NASA Manned Spacecraft Center.
+ Glenn was a backup pilot to Alan Shepard and Gus Grissom, on the Freedom 7 and Liberty Bell 7 respectively.
+ Astronauts were given an additional role in the spaceflight program, and Glenn's was the cockpit layout and control functioning, not only for Mercury but also early designs for Apollo.
+ </ins>
@@ -580,0 +761,14 @@
+ <div class="thumbinner" style="width:242px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn (center) with President John F. Kennedy and General Leighton I. Davis celebrating Glenn's orbital flight, 1962
+   </ins>
+  </div>
+ </div>
+</div>
+<del>
+ .
+</del>
+<div class="thumb tleft">
@@ -582,3 +775,0 @@
-  <a class="image" href="/wiki/File:GPN-2000-001027.jpg">
-   <img alt="" class="thumbimage" data-file-height="2343" data-file-width="1800" height="221" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/170px-GPN-2000-001027.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/255px-GPN-2000-001027.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/340px-GPN-2000-001027.jpg 2x" width="170"/>
-  </a>
@@ -587,2 +777,0 @@
-    <a class="internal" href="/wiki/File:GPN-2000-001027.jpg" title="Enlarge">
-    </a>
@@ -590,4 +779,8 @@
-   John Glenn in his
-   <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
-    Mercury spacesuit
-   </a>
+   <del>
+    John Glenn in his
+   </del>
+   <del>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </del>
@@ -608 +801,7 @@
- on February 20, 1962, on the Mercury-Atlas 6 mission, circling the globe three times during a flight lasting 4 hours, 55 minutes, and 23 seconds.
+ on February 20, 1962, on the Mercury-Atlas 6 mission, circling the globe three times during a flight lasting
+ <ins>
+  nearly five hours.
+ </ins>
+ <del>
+  4 hours, 55 minutes, and 23 seconds.
+ </del>
@@ -610,0 +810,15 @@
+<p>
+ <ins>
+  For Glenn the day became the "best day of his life," while it also renewed America's confidence.
+ His voyage took place while America and the Soviet Union were in the midst of the
+  <a href="/wiki/Cold_War" title="Cold War">
+   Cold War
+  </a>
+  and competing in the "
+  <a href="/wiki/Space_Race" title="Space Race">
+   Space Race
+  </a>
+  ."
+During the flight, Glenn's
+ </ins>
+</p>
@@ -613 +827,3 @@
-  Zero G, and I feel fine.
+  <del>
+   Zero G, and I feel fine.
+  </del>
@@ -616,7 +832,9 @@
-  <cite>
-   — Glenn's first words on becoming
-   <a href="/wiki/Weightlessness" title="Weightlessness">
-    weightless
-   </a>
-   upon reaching orbit, February 20, 1962,
-  </cite>
+  <del>
+   <cite>
+    — Glenn's first words on becoming
+    <a href="/wiki/Weightlessness" title="Weightlessness">
+     weightless
+    </a>
+    upon reaching orbit, February 20, 1962,
+   </cite>
+  </del>
@@ -625 +843 @@
-<p>
+<del>
@@ -629,0 +848,2 @@
+</del>
+<del>
@@ -634,0 +855,2 @@
+</del>
+<del>
@@ -640,4 +862,11 @@
- <a class="mw-redirect" href="/wiki/Ablative_heat_shield" title="Ablative heat shield">
-  heat shield
- </a>
- had come loose, which could allow it to fail during
+</del>
+<a class="mw-redirect" href="/wiki/Ablative_heat_shield" title="Ablative heat shield">
+ heat shield
+</a>
+ had <ins>
+ been thought to have come loose and likely
+</ins>
+<del>
+ come loose, which could allow it
+</del>
+ to fail during
@@ -645,3 +874,9 @@
-  re-entry
- </a>
- through the atmosphere, causing his capsule to burn up. Flight controllers had Glenn modify his re-entry procedure by keeping his
+ re-entry
+</a>
+<ins>
+ , which would cause the entire space
+</ins>
+<del>
+ through the atmosphere, causing his
+</del>
+ capsule to burn up. Flight controllers had Glenn modify his re-entry procedure by keeping his
@@ -649,3 +884,9 @@
-  retrorocket
- </a>
- pack on over the shield in an attempt to keep it in place. He made his
+ retrorocket
+</a>
+ pack on over the shield <ins>
+ to help
+</ins>
+<del>
+ in an attempt to
+</del>
+ keep it in place. He made his
@@ -653,2 +894,2 @@
-  splashdown
- </a>
+ splashdown
+</a>
@@ -656 +897,7 @@
-</p>
+ <ins>
+ Glenn's flight and fiery splashdown was portrayed in the 1983 film
+ <a href="/wiki/The_Right_Stuff_(film)" title="The Right Stuff (film)">
+  The Right Stuff
+ </a>
+ .
+</ins>
@@ -721,14 +968,23 @@
- in favor of excluding women from the NASA astronaut program. Although NASA had no official policy prohibiting women, in practice the requirement that astronauts had to be military test pilots excluded them entirely.
- The impact of the testimony of so prestigious a hero is debatable, but no female astronaut flew on a NASA mission until
- <a href="/wiki/Sally_Ride" title="Sally Ride">
-  Sally Ride
- </a>
- in 1983 (in the meantime, the Soviets had flown two women on space missions), and none piloted a mission until
- <a href="/wiki/Eileen_Collins" title="Eileen Collins">
-  Eileen Collins
- </a>
- in 1995, more than 30 years after the hearings. In the late 1970s, Glenn is reported to have supported Shuttle Mission Specialist Astronaut
- <a href="/wiki/Judith_Resnik" title="Judith Resnik">
-  Judith Resnik
- </a>
- in her career.
+ in favor of excluding women from the NASA astronaut program. Although NASA had no official policy prohibiting women, in
+ <ins>
+  practice,
+ </ins>
+ <del>
+  practice
+ </del>
+ the requirement that astronauts had to be military test pilots excluded them entirely.
+ <del>
+  The impact of the testimony of so prestigious a hero is debatable, but no female astronaut flew on a NASA mission until
+  <a href="/wiki/Sally_Ride" title="Sally Ride">
+   Sally Ride
+  </a>
+  in 1983 (in the meantime, the Soviets had flown two women on space missions), and none piloted a mission until
+  <a href="/wiki/Eileen_Collins" title="Eileen Collins">
+   Eileen Collins
+  </a>
+  in 1995, more than 30 years after the hearings. In the late 1970s, Glenn is reported to have supported Shuttle Mission Specialist Astronaut
+  <a href="/wiki/Judith_Resnik" title="Judith Resnik">
+   Judith Resnik
+  </a>
+  in her career.
+ </del>
@@ -744 +1000,6 @@
-  Colonel
+  <ins>
+   colonel
+  </ins>
+  <del>
+   Colonel
+  </del>
@@ -765,0 +1027,31 @@
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     This section
+     <b>
+      is missing information about what Glenn did during his 24 years in the Senate
+     </b>
+     .
+     <span class="hide-when-compact">
+      Please expand the section to include this information. Further details may exist on the
+      <a href="/wiki/Talk:John_Glenn#Senate_career" title="Talk:John Glenn">
+       talk page
+      </a>
+      .
+     </span>
+     <small>
+      <i>
+       (December 2016)
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
@@ -804,0 +1097,3 @@
+ <ins>
+  He served as a pallbearer at Kennedy's funeral.
+ </ins>
@@ -823 +1118,6 @@
- . Instead, he challenged Metzenbaum again, whom Gilligan had appointed
+ . Instead, he challenged Metzenbaum
+ <ins>
+  again for the Senate seat vacated by
+ </ins>
+ <del>
+  again, whom Gilligan had appointed
@@ -824,0 +1125 @@
+ </del>
@@ -826 +1127 @@
-  William B. Saxbe
+  William B. Saxbe
@@ -828 +1129,6 @@
- , who had resigned to become Attorney General of the United States.
+ <ins>
+  (R-OH).
+ </ins>
+ <del>
+  , who had resigned to become Attorney General of the United States.
+ </del>
@@ -851,8 +1157,11 @@
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
-  In 1986, Glenn defeated challenger
- </a>
- U.S. Representative
- <a href="/wiki/Tom_Kindness" title="Tom Kindness">
-  Tom Kindness
- </a>
- . Metzenbaum would go on to seek a rematch against Taft in 1976, winning a close race on
+ <del>
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
+   In 1986, Glenn defeated challenger
+  </a>
+  U.S. Representative
+  <a href="/wiki/Tom_Kindness" title="Tom Kindness">
+   Tom Kindness
+  </a>
+  .
+ </del>
+ Metzenbaum would go on to seek a rematch against Taft in 1976, winning a close race on
@@ -876,0 +1186,10 @@
+ <ins>
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
+   In 1986, Glenn defeated challenger
+  </a>
+  U.S. Representative
+  <a href="/wiki/Tom_Kindness" title="Tom Kindness">
+   Tom Kindness
+  </a>
+  . In 1997, Glenn announced that he would retire from the Senate at the end of his then-current term.
+ </ins>
@@ -911,5 +1230,10 @@
- to keep his seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
- <i>
-  on earth
- </i>
- has John Glenn done?"
+ to keep his
+ <ins>
+  seat.
+ </ins>
+ <del>
+  seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
+  <i>
+   on earth
+  </i>
+  has John Glenn done?"
@@ -916,0 +1241 @@
+ </del>
@@ -932 +1257,2 @@
- . Glenn also ran for the
+ .
+ Glenn also ran for the
@@ -936,4 +1262,11 @@
- . A November 1983
- <i>
-  <a href="/wiki/The_New_York_Times" title="The New York Times">
-   New York Times
+ .
+ <del>
+  A November 1983
+  <i>
+   <a href="/wiki/The_New_York_Times" title="The New York Times">
+    New York Times
+   </a>
+  </i>
+  /
+  <a href="/wiki/CBS_News" title="CBS News">
+   CBS News
@@ -941,6 +1274,2 @@
- </i>
- /
- <a href="/wiki/CBS_News" title="CBS News">
-  CBS News
- </a>
- poll found him second, supported by 41% of those polled, to Mondale's 49%.
+  poll found him second, supported by 41% of those polled, to Mondale's 49%.
+ </del>
@@ -977,5 +1306,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (July 2014)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -983,5 +1312,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -999 +1328,2 @@
- , served as chairman of the
+ ,
+ served as chairman of the
@@ -1015,8 +1345,33 @@
- . Once Republicans regained control of the Senate, Glenn also served as the ranking minority member on a special Senate investigative committee chaired by
- <a href="/wiki/Tennessee" title="Tennessee">
-  Tennessee
- </a>
- senator
- <a class="mw-redirect" href="/wiki/Fred_Dalton_Thompson" title="Fred Dalton Thompson">
-  Fred Dalton Thompson
- </a>
+ .
+</p>
+<p>
+ Once Republicans regained control of the Senate, Glenn
+ <del>
+  also
+ </del>
+ served as the ranking minority member on
+ <ins>
+  the
+  <a href="/wiki/United_States_Senate_Homeland_Security_Permanent_Subcommittee_on_Investigations" title="United States Senate Homeland Security Permanent Subcommittee on Investigations">
+   Permanent Subcommittee on Investigations
+  </a>
+  , chaired by
+  <a href="/wiki/Maine" title="Maine">
+   Maine
+  </a>
+  senator
+  <a href="/wiki/Susan_Collins" title="Susan Collins">
+   Susan Collins
+  </a>
+  ,
+ </ins>
+ <del>
+  a special Senate investigative committee chaired by
+  <a href="/wiki/Tennessee" title="Tennessee">
+   Tennessee
+  </a>
+  senator
+  <a class="mw-redirect" href="/wiki/Fred_Dalton_Thompson" title="Fred Dalton Thompson">
+   Fred Dalton Thompson
+  </a>
+ </del>
@@ -1027 +1382,15 @@
- to U.S. political campaigns for the 1996 election. There was considerable acrimony between the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
+ to U.S. political campaigns for the 1996 election.
+ There was considerable acrimony between
+ <ins>
+  Glenn and the overseeing committee chair,
+  <a href="/wiki/Fred_Thompson" title="Fred Thompson">
+   Fred Thompson
+  </a>
+  of
+  <a href="/wiki/Tennessee" title="Tennessee">
+   Tennessee
+  </a>
+  .
+ </ins>
+ <del>
+  the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
@@ -1029,12 +1398,13 @@
- <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
-  Mary O. Boyle
- </a>
- was the Democratic party nominee. She faced Republican nominee and sitting governor
- <a href="/wiki/George_Voinovich" title="George Voinovich">
-  George Voinovich
- </a>
- in the
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
-  general election
- </a>
- , which Voinovich won.
+  <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
+   Mary O. Boyle
+  </a>
+  was the Democratic party nominee. She faced Republican nominee and sitting governor
+  <a href="/wiki/George_Voinovich" title="George Voinovich">
+   George Voinovich
+  </a>
+  in the
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
+   general election
+  </a>
+  , which Voinovich won.
+ </del>
@@ -1052,0 +1423,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Senator-astronaut John Glenn on the shuttle Discovery, 1998
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1058 +1439,5 @@
- on October 29, 1998, becoming, at age 77, the oldest person to go into space as a
+ on October 29, 1998,
+ <del>
+  becoming, at age 77, the oldest person to go into space
+ </del>
+ as a
@@ -1072 +1457,7 @@
- mission. According to
+ <ins>
+  mission, becoming, at age 77, the oldest person to go into space.
+ </ins>
+ <del>
+  mission.
+ </del>
+ According to
@@ -1078,0 +1470,9 @@
+ <ins>
+  Shortly before the flight, researchers learned that Glenn had to be disqualified from one of the flight's two main priority human experiments (about the effects of
+  <a href="/wiki/Melatonin" title="Melatonin">
+   melatonin
+  </a>
+  ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+ </ins>
+</p>
+<p>
@@ -1088,3 +1488,7 @@
-</p>
-<p>
- Glenn's participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
+ <ins>
+  His
+ </ins>
+ <del>
+  Glenn's
+ </del>
+ participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
@@ -1101,5 +1505,12 @@
- It was noted that Glenn's flight offered valuable research on weightlessness and other aspects of space flight on the same person at two points in life 36 years apart—by far the longest interval between space flights by the same person—providing information on the effects of spaceflight and weightlessness on the elderly, with an ideal
- <a href="/wiki/Control_variable" title="Control variable">
-  control subject
- </a>
- .
+ <ins>
+  In a 2012 interview, Glenn said that the purpose of his flight was "to make measurements and do research on me at the age of 77 [...] comparing the results on me in space with the younger [astronauts] and maybe get [insights] on the immune system or protein turnover or vestibular functions and other things — heart changes.
+ He regretted that NASA did not follow up on this research about aging by sending more people from this age range into space.
+ </ins>
+</p>
+<p>
+ <del>
+  It was noted that Glenn's flight offered valuable research on weightlessness and other aspects of space flight on the same person at two points in life 36 years apart—by far the longest interval between space flights by the same person—providing information on the effects of spaceflight and weightlessness on the elderly, with an ideal
+  <a href="/wiki/Control_variable" title="Control variable">
+   control subject
+  </a>
+  .
@@ -1107,4 +1518,5 @@
- <a href="/wiki/Melatonin" title="Melatonin">
-  melatonin
- </a>
- ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+  <a href="/wiki/Melatonin" title="Melatonin">
+   melatonin
+  </a>
+  ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+ </del>
@@ -1142 +1554 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+ <a href="/wiki/Ohio_State_University" title="Ohio State University">
@@ -1149 +1561,7 @@
- . Today Glenn holds an adjunct professorship at the Glenn School.
+ <ins>
+  , and Glenn held
+ </ins>
+ <del>
+  . Today Glenn holds
+ </del>
+ an adjunct professorship at the Glenn School.
@@ -1159,3 +1576,0 @@
-  <a class="image" href="/wiki/File:Annie_and_John_Glenn_1965.jpg">
-   <img alt="" class="thumbimage" data-file-height="1695" data-file-width="1420" height="263" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Annie_and_John_Glenn_1965.jpg/220px-Annie_and_John_Glenn_1965.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Annie_and_John_Glenn_1965.jpg/330px-Annie_and_John_Glenn_1965.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Annie_and_John_Glenn_1965.jpg/440px-Annie_and_John_Glenn_1965.jpg 2x" width="220"/>
-  </a>
@@ -1164,2 +1578,0 @@
-    <a class="internal" href="/wiki/File:Annie_and_John_Glenn_1965.jpg" title="Enlarge">
-    </a>
@@ -1167 +1580,9 @@
-   Anna and John Glenn in 1965
+   <ins>
+    Glenn and his wife Anna
+   </ins>
+   <div class="magnify">
+   </div>
+   <del>
+    Anna and John Glenn
+   </del>
+   in 1965
@@ -1172 +1593,8 @@
- On April 6, 1943, Glenn married his childhood sweetheart,
+ On April 6, 1943, Glenn married his
+ <ins>
+  high school
+ </ins>
+ <del>
+  childhood
+ </del>
+ sweetheart,
@@ -1184 +1612,55 @@
- . He also was a member of the Stag Club Fraternity at Muskingum College.
+ <ins>
+  , where he
+ </ins>
+ <del>
+  . He also
+ </del>
+ was a member of the Stag Club
+</p>
+<p>
+ <ins>
+  Fraternity.
+ Together, they had two children, John David and Carolyn Ann, and two grandchildren.
+ They remained married until his death. His boyhood home in New Concord has been
+ </ins>
+ <ins>
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+ </ins>
+ <ins>
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </ins>
+</p>
+<p>
+ <ins>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn was a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and was an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  He set an example of someone whose faith began before he became an astronaut, and whose faith was reinforced after traveling in space.
+ </ins>
+</p>
+<p>
+ <ins>
+  "To look out at this kind of creation and not believe in God is to me impossible," said Glenn, after his second and final space voyage.
+ He stated that he saw no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believed evolution should be taught in schools.
+ He explained:
+ </ins>
@@ -1186,2 +1668,15 @@
-<p>
- Glenn was also one of the original owners of a
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   I don't see that I'm any less religious that I can appreciate the fact that science just records that we change with evolution and time, and that's a fact. It doesn't mean it's less wondrous and it doesn't mean that there can't be some power greater than any of us that has been behind and is behind whatever is going on.
+  </ins>
+ </p>
+</blockquote>
+<ins>
+ Glenn was
+</ins>
+<del>
+ Fraternity at Muskingum College.
+Glenn was also
+</del>
+  one of the original owners of a
@@ -1189,2 +1684,2 @@
-  Holiday Inn
- </a>
+ Holiday Inn
+</a>
@@ -1193,2 +1688,2 @@
-  Orlando
- </a>
+ Orlando
+</a>
@@ -1197,2 +1692,2 @@
-  Seralago Hotel &amp; Suites Main Gate East
- </a>
+ Seralago Hotel &amp; Suites Main Gate East
+</a>
@@ -1199,0 +1695,11 @@
+ <p>
+ <ins>
+  His business partner was
+ </ins>
+ <ins>
+  <a href="/wiki/Henri_Landwirth" title="Henri Landwirth">
+   Henri Landwirth
+  </a>
+  , a Holocaust survivor, who became Glenn's "best friend."
+ Glenn recalls learning about Landwirth's background:
+ </ins>
@@ -1201,2 +1707,38 @@
-<p>
- Glenn is an honorary member of the
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   Henri doesn't talk about it much. It was years before he spoke about it with me and then only because of an accident. We were down in Florida during the space program. Everyone was wearing short-sleeved Ban-Lon shirts—everyone but Henri. Then one day I saw Henri at the pool and noticed the number on his arm. I told Henri that if it were me I'd wear that number like a medal with a spotlight on it.
+  </ins>
+ </p>
+</blockquote>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Public_appearances_and_ceremonies">
+   Public appearances and ceremonies
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn appears with President Kennedy and Soviet cosmonaut
+   </ins>
+   <ins>
+    <a href="/wiki/Gherman_Titov" title="Gherman Titov">
+     Gherman Titov
+    </a>
+    , 1962
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ Glenn was
+</ins>
+<del>
+ Glenn is
+</del>
+ an honorary member of the
@@ -1204,2 +1746,2 @@
-  International Academy of Astronautics
- </a>
+ International Academy of Astronautics
+</a>
@@ -1208,2 +1750,2 @@
-  Society of Experimental Test Pilots
- </a>
+ Society of Experimental Test Pilots
+</a>
@@ -1212,2 +1754,2 @@
-  Order of Daedalians
- </a>
+ Order of Daedalians
+</a>
@@ -1216,2 +1758,2 @@
-  Ohio Democratic Party
- </a>
+ Ohio Democratic Party
+</a>
@@ -1218,0 +1761,16 @@
+ <p>
+ <del>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and is an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </del>
@@ -1221,23 +1779,4 @@
- A
- <a href="/wiki/Freemasonry" title="Freemasonry">
-  Freemason
- </a>
- , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
- <a href="/wiki/DeMolay_International" title="DeMolay International">
-  DeMolay International
- </a>
- , the Masonic youth organization, and is an ordained elder in the
- <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
-  Presbyterian Church
- </a>
- .
-</p>
-<p>
- Glenn's name was used for the character of
- <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
-  John Tracy
- </a>
- in the 1960s children's TV series
- <i>
-  <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
-   Thunderbirds
+ <del>
+  Glenn's name was used for the character of
+  <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
+   John Tracy
@@ -1245,2 +1784,8 @@
- </i>
- .
+  in the 1960s children's TV series
+  <i>
+   <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
+    Thunderbirds
+   </a>
+  </i>
+  .
+ </del>
@@ -1249,9 +1794,11 @@
- Glenn's boyhood home in New Concord has been
- <a href="/wiki/Building_restoration" title="Building restoration">
-  restored
- </a>
- and made into an
- <a href="/wiki/Historic_house_museum" title="Historic house museum">
-  historic house museum
- </a>
- and education center.
+ <del>
+  Glenn's boyhood home in New Concord has been
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </del>
@@ -1266,19 +1813,29 @@
- .
-</p>
-<p>
- On August 4, 2006, Glenn and his wife were injured in an automobile accident on
- <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
-  I-270
- </a>
- near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
- <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
-  sternum
- </a>
- . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
-</p>
-<p>
- On September 5, 2009, John and Annie Glenn dotted the "i" during
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  The Ohio State University
- </a>
- 's
+ <ins>
+  , as himself.
+ </ins>
+ <del>
+  .
+ </del>
+</p>
+<p>
+ <del>
+  On August 4, 2006, Glenn and his wife were injured in an automobile accident on
+  <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
+   I-270
+  </a>
+  near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
+  <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
+   sternum
+  </a>
+  . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
+ </del>
+</p>
+<p>
+ On September 5, 2009, John and Annie Glenn dotted the "i" during The Ohio State
+ <ins>
+  University's
+ </ins>
+ <del>
+  University
+  Link: /wiki/The_Ohio_State_University 's
+ </del>
@@ -1295 +1852,6 @@
- -
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
@@ -1299,0 +1862,3 @@
+ <ins>
+  Other non-band members to have received this honor include
+ </ins>
@@ -1308,16 +1873,18 @@
- <a href="/wiki/Buster_Douglas" title="Buster Douglas">
-  Buster Douglas
- </a>
- ,
- <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
-  E. Gordon Gee
- </a>
- ,
- <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
-  Novice Fawcett
- </a>
- ,
- <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
-  Robert Ries
- </a>
- , and
+ <del>
+  <a href="/wiki/Buster_Douglas" title="Buster Douglas">
+   Buster Douglas
+  </a>
+  ,
+  <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
+   E. Gordon Gee
+  </a>
+  ,
+  <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
+   Novice Fawcett
+  </a>
+  ,
+  <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
+   Robert Ries
+  </a>
+  , and
+ </del>
@@ -1331 +1898,6 @@
- are the only other non-band members to have received this honor.
+ <ins>
+  .
+ </ins>
+ <del>
+  are the only other non-band members to have received this honor.
+ </del>
@@ -1354,4 +1926,10 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  Ohio State
- </a>
- , where the public affairs school is named for him.
+ <ins>
+  The Ohio State University,
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+   Ohio State
+  </a>
+  ,
+ </del>
+ where the public affairs school is named for him.
@@ -1400 +1978,24 @@
- In June 2014, Glenn underwent a successful
+ In June
+</p>
+<p>
+ <ins>
+  2016 the Columbus, Ohio airport known for many years as Port Columbus was officially renamed the
+ </ins>
+ <ins>
+  <a href="/wiki/John_Glenn_Columbus_International_Airport" title="John Glenn Columbus International Airport">
+   John Glenn Columbus International Airport
+  </a>
+  . Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke about how visiting that airport as a child inspired his interest in flying.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Illness_and_death">
+   Illness and death
+  </span>
+ </ins>
+</h2>
+<ins>
+ In June
+</ins>
+  2014, Glenn underwent a successful
@@ -1402,2 +2003,2 @@
-  heart valve replacement
- </a>
+ heart valve replacement
+</a>
@@ -1406,2 +2007,2 @@
-  Cleveland Clinic
- </a>
+ Cleveland Clinic
+</a>
@@ -1408,0 +2010,4 @@
+ <p>
+ <ins>
+  At the beginning of December 2016,
+ </ins>
@@ -1411 +2016,3 @@
- On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ <del>
+  On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ </del>
@@ -1414 +2021,3 @@
- Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ <del>
+  Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ </del>
@@ -1417,3 +2026,5 @@
- <span class="mw-headline" id="Illness">
-  Illness
- </span>
+ <del>
+  <span class="mw-headline" id="Illness">
+   Illness
+  </span>
+ </del>
@@ -1421 +2032 @@
-<p>
+<del>
@@ -1422,0 +2034,2 @@
+</del>
+<del>
@@ -1426,4 +2039,20 @@
- announced that Glenn was hospitalized at OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
- <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
-  The James Cancer Hospital
- </a>
+ announced that
+</del>
+ Glenn was hospitalized at <ins>
+ the
+</ins>
+<a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
+ <del>
+  OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
+ The
+ </del>
+ James Cancer Hospital
+</a>
+<ins>
+ of
+ <a href="/wiki/Ohio_State_University_Wexner_Medical_Center" title="Ohio State University Wexner Medical Center">
+  OSU Wexner Medical Center
+ </a>
+ in Columbus.
+</ins>
+<del>
@@ -1431 +2060,30 @@
- A family source said that Glenn was in declining health, that his condition was grave, and that Annie Glenn and his children and grandchildren had joined him at the hospital.
+</del>
+ A family source said that Glenn <ins>
+ had been
+</ins>
+<del>
+ was
+</del>
+ in declining health, <ins>
+ and
+</ins>
+ that his condition was <ins>
+ grave. His wife, Annie, and their
+</ins>
+<del>
+ grave, and that Annie Glenn and his
+</del>
+ children and grandchildren had joined him at the hospital.
+ <p>
+ <ins>
+  Glenn died December 8, 2016, at the OSU Wexner Medical Center.
+ No cause of death has yet been disclosed. Glenn will be interred at
+  <a href="/wiki/Arlington_National_Cemetery" title="Arlington National Cemetery">
+   Arlington National Cemetery
+  </a>
+  after lying in state at the
+  <a href="/wiki/Ohio_Statehouse" title="Ohio Statehouse">
+   Ohio Statehouse
+  </a>
+  and a memorial service at Mershon Auditorium at The Ohio State University.
+ </ins>
@@ -1432,0 +2091,52 @@
+<h3>
+ <ins>
+  <span class="mw-headline" id="Tributes">
+   Tributes
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:242px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn looks into a celestial training device before his 1962 launch.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Among those honoring Glenn were President
+  <a href="/wiki/Barack_Obama" title="Barack Obama">
+   Barack Obama
+  </a>
+  , who said that Glenn, "the first American to orbit the Earth, reminded us that with courage and a spirit of discovery there's no limit to the heights we can reach together."
+ Tributes were also given by former Secretary of State
+  <a href="/wiki/Hillary_Clinton" title="Hillary Clinton">
+   Hillary Clinton
+  </a>
+  ,
+ and President-elect
+  <a href="/wiki/Donald_Trump" title="Donald Trump">
+   Donald Trump
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  The phrase "Godspeed," that hailed Glenn's historic launch into space, became a social media
+  <a href="/wiki/Hashtag" title="Hashtag">
+   hashtag
+  </a>
+  . Past and current astronauts added their own tributes, along with NASA Administrator and former shuttle astronaut,
+  <a href="/wiki/Charles_Bolden" title="Charles Bolden">
+   Charles Bolden
+  </a>
+  , who added that "John Glenn's legacy is one of risk and accomplishment, of history created and duty to country carried out under great pressure with the whole world watching."
+ </ins>
+</p>
+<div style="clear:both;">
+</div>
@@ -1471,3 +2181,5 @@
-     <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
-      Friendship 7
-     </a>
+     <i>
+      <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
+       Friendship 7
+      </a>
+     </i>
@@ -1766 +2478,7 @@
-    and two
+    and
+    <ins>
+     eighteen
+    </ins>
+    <del>
+     two
+    </del>
@@ -1778 +2496,10 @@
-    with fifteen stars and two clusters
+    with fifteen stars and
+    <ins>
+     eighteen
+     <a href="/wiki/Oak_leaf_cluster" title="Oak leaf cluster">
+      clusters
+     </a>
+    </ins>
+    <del>
+     two clusters
+    </del>
@@ -1917,4 +2644,6 @@
-  <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
-   Golden Plate Award
-  </a>
-  for Science and Exploration, 1964
+  <del>
+   <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
+    Golden Plate Award
+   </a>
+   for Science and Exploration, 1964
+  </del>
@@ -1975 +2704,5 @@
-   ) to Senator John Glenn and
+   ) to Senator John Glenn
+   <ins>
+    (left)
+   </ins>
+   and
@@ -1982,0 +2716,3 @@
+   <ins>
+    (right)
+   </ins>
@@ -2079 +2815,8 @@
- , a mobile landing platform that is scheduled to be delivered to the
+ , a mobile landing platform that
+ <ins>
+  was
+ </ins>
+ <del>
+  is scheduled to be
+ </del>
+ delivered to the
@@ -2083 +2826,7 @@
- in 2014 is named for him. It was christened February 1, 2014, in
+ <ins>
+  on March 12, 2014,
+ </ins>
+ <del>
+  in 2014
+ </del>
+ is named for him. It was christened February 1, 2014, in
@@ -2091 +2840,6 @@
- ’
+ <ins>
+  '
+ </ins>
+ <del>
+  ’
+ </del>
@@ -2110,3 +2864,5 @@
-</p>
-<p>
- He also received Honorary Doctorates from
+ He
+ <del>
+  also
+ </del>
+ received Honorary Doctorates from
@@ -2148,2 +2903,0 @@
-</p>
-<p>
@@ -2214,12 +2968,14 @@
- On January 3, 2015, with the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- , Glenn became the
- <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
-  oldest former United States senator still living
- </a>
- . Glenn is the 60th person to hold this distinction.
-</p>
-<p>
- On September 12, 2016,
+ On
+ <del>
+  January 3, 2015, with the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  , Glenn became the
+  <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
+   oldest former United States senator still living
+  </a>
+  . Glenn is the 60th person to hold this distinction.
+On
+ </del>
+ September 12, 2016,
@@ -2236,3 +2992,5 @@
- <span class="mw-headline" id="Physical_description">
-  Physical description
- </span>
+ <del>
+  <span class="mw-headline" id="Physical_description">
+   Physical description
+  </span>
+ </del>
@@ -2242 +3000,3 @@
-  Weight: 170 lb (77 kg)
+  <del>
+   Weight: 170 lb (77 kg)
+  </del>
@@ -2245 +3005,3 @@
-  Height: 5 ft 10½ in (1.79 m)
+  <del>
+   Height: 5 ft 10½ in (1.79 m)
+  </del>
@@ -2248 +3010,3 @@
-  Hair: Red
+  <del>
+   Hair: Red
+  </del>
@@ -2251 +3015,3 @@
-  Eyes: Green
+  <del>
+   Eyes: Green
+  </del>
@@ -2322,0 +3089,2 @@
+<div style="clear:both;">
+</div>
\ No newline at end of file
