--- 753656269
+++ 754105229
@@ -5,9 +5,14 @@
- (born July 18, 1921), (
- <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
-  Col
- </a>
- ,
- <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
-  USMC
- </a>
- , Ret.), is a former
+ <ins>
+  (July 18, 1921 – December 8, 2016) was an American
+ </ins>
+ <del>
+  (born July 18, 1921), (
+  <a href="/wiki/Colonel_(United_States)" title="Colonel (United States)">
+   Col
+  </a>
+  ,
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   USMC
+  </a>
+  , Ret.), is a former
+ </del>
@@ -27 +32,15 @@
-  United States senator
+  United States  Senator
+  Link: /wiki/United_States_Senate from
+ Ohio
+  Link: /wiki/Ohio  . In 1962 he became the first American to orbit the Earth, circling three times. Before joining NASA, he was a distinguished fighter pilot in both
+ World War II
+  Link: /wiki/World_War_II  and
+ Korea
+  Link: /wiki/Korean_War  , with six
+ Distinguished Flying Crosses
+  Link: /wiki/Distinguished_Flying_Cross_(United_States)  and eighteen
+ clusters
+  Link: /wiki/Oak_leaf_cluster  to the
+ Air Medal
+  Link: /wiki/Air_Medal  .
+Glenn
@@ -29 +48,6 @@
- . He was one of the "
+</p>
+<del>
+ senator
+  Link: /wiki/United_States_Senate . He
+</del>
+ was one of the "
@@ -31,2 +55,2 @@
-  Mercury Seven
- </a>
+ Mercury Seven
+</a>
@@ -35,2 +59,2 @@
-  test pilots
- </a>
+ test pilots
+</a>
@@ -39,3 +63,7 @@
-  NASA
- </a>
- to become America's first astronauts and fly the
+ NASA
+</a>
+ to become America's first <ins>
+ astronauts.
+</ins>
+<del>
+ astronauts and fly the
@@ -46,3 +74,8 @@
-</p>
-<p>
- On February 20, 1962, Glenn flew the
+</del>
+ On February 20, 1962, <ins>
+ he
+</ins>
+<del>
+ Glenn
+</del>
+ flew the
@@ -50,7 +83,2 @@
-  <a href="/wiki/Mercury-Atlas_6" title="Mercury-Atlas 6">
-   Friendship 7
-  </a>
- </i>
- mission and became the first
- <a href="/wiki/United_States" title="United States">
-  American
+ <a href="/wiki/Mercury-Atlas_6" title="Mercury-Atlas 6">
+  Friendship 7
@@ -58 +86,6 @@
- to
+</i>
+ mission and became the first American <ins>
+ to orbit the Earth
+</ins>
+<del>
+ Link: /wiki/United_States to
@@ -66 +99,6 @@
- and the fifth person in space, after
+</del>
+ and the fifth person in <ins>
+ space.
+</ins>
+<del>
+ space, after
@@ -93 +131,3 @@
- . Glenn received the
+ .
+</del>
+ Glenn received the
@@ -95,3 +135,6 @@
-  Congressional Space Medal of Honor
- </a>
- in 1978, and was inducted into the
+ Congressional Space Medal of Honor
+</a>
+ in 1978, <del>
+ and
+</del>
+ was inducted into the
@@ -99,3 +142,7 @@
-  U.S. Astronaut Hall of Fame
- </a>
- in 1990. With the death of
+ U.S. Astronaut Hall of Fame
+</a>
+ in <ins>
+ 1990, and was
+</ins>
+<del>
+ 1990. With the death of
@@ -105,8 +152,28 @@
- on October 10, 2013, Glenn became the last surviving member of the Mercury Seven.
-</p>
-<p>
- Glenn resigned from NASA on January 16, 1964, and the next day announced plans to run for a U.S. Senate seat from
- <a href="/wiki/Ohio" title="Ohio">
-  Ohio
- </a>
- ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965. A member of the
+ on October 10, 2013, Glenn became
+</del>
+ the last surviving member of the Mercury Seven.
+ <p>
+ <ins>
+  After he
+ </ins>
+ <del>
+  Glenn
+ </del>
+ resigned from NASA
+ <ins>
+  in 1964, Glenn planned
+ </ins>
+ <del>
+  on January 16, 1964, and the next day announced plans
+ </del>
+ to run for a U.S. Senate seat from
+ <ins>
+  Ohio.
+ </ins>
+ <del>
+  <a href="/wiki/Ohio" title="Ohio">
+   Ohio
+  </a>
+  ; however, a bathroom fall which resulted in a concussion caused him to withdraw from the race in March. He retired from the Marine Corps on January 1, 1965.
+ </del>
+ A member of the
@@ -116 +183,7 @@
- , he finally
+ , he
+ <ins>
+  first
+ </ins>
+ <del>
+  finally
+ </del>
@@ -120,8 +193,25 @@
- and served through January 3, 1999. With the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- on January 3, 2015, Glenn became the oldest living former United States Senator.
-</p>
-<p>
- On October 29, 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
+ <ins>
+  where he
+ </ins>
+ <del>
+  and
+ </del>
+ served through January 3, 1999.
+</p>
+<p>
+ <ins>
+  He retired from the Marine Corps in 1965, after twenty-three years in the military, with over fifteen medals and awards, including the
+  <a href="/wiki/NASA_Distinguished_Service_Medal" title="NASA Distinguished Service Medal">
+   NASA Distinguished Service Medal
+  </a>
+  and the Congressional Space Medal of Honor. In
+ </ins>
+ <del>
+  With the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  on January 3, 2015, Glenn became the oldest living former United States Senator.
+On October 29,
+ </del>
+ 1998, while still a sitting senator, he became the oldest person to fly in space, and the only one to fly in both the Mercury and
@@ -131,7 +221,3 @@
- , when at age 77, he flew as a
- <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
-  Payload Specialist
- </a>
- on
- <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
-  <i>
+ <ins>
+  as crew member of the
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
@@ -139,7 +225,19 @@
-  </i>
- </a>
- mission
- <a href="/wiki/STS-95" title="STS-95">
-  STS-95
- </a>
- . He was awarded the
+  </a>
+  space shuttle. He was also
+ </ins>
+ <del>
+  , when at age 77, he flew as a
+  <a class="mw-redirect" href="/wiki/Payload_Specialist" title="Payload Specialist">
+   Payload Specialist
+  </a>
+  on
+  <a href="/wiki/Space_Shuttle_Discovery" title="Space Shuttle Discovery">
+   Discovery
+  </a>
+  mission
+  <a href="/wiki/STS-95" title="STS-95">
+   STS-95
+  </a>
+  . He was
+ </del>
+ awarded the
@@ -156,2 +254,25 @@
- <span class="mw-headline" id="Early_life.2C_education_and_military_service">
-  Early life, education and military service
+ <span class="mw-headline" id="Early_life_and_education">
+  Early
+  <p>
+   <ins>
+    life and education
+   </ins>
+  </p>
+  <div class="thumb tright">
+   <div class="thumbinner" style="width:252px;">
+    <div class="thumbcaption">
+     <div class="magnify">
+     </div>
+     <ins>
+      Glenn's childhood home in New Concord
+     </ins>
+    </div>
+   </div>
+  </div>
+  <ins>
+   John Herschel Glenn, Jr.,
+  </ins>
+  <del>
+   life, education and military service
+ John Glenn
+  </del>
@@ -160,2 +281 @@
-<p>
- John Glenn was born on July 18, 1921, in
+ was born on July 18, 1921, in
@@ -163,2 +283,2 @@
-  Cambridge
- </a>
+ Cambridge
+</a>
@@ -167,3 +287,9 @@
-  Ohio
- </a>
- , the son of John Herschel Glenn, Sr. (1895–1966) and Teresa (
+ Ohio
+</a>
+ , the son of John Herschel Glenn, Sr. <ins>
+ (1895–1966), who worked for a plumbing firm, and Clara
+</ins>
+<del>
+ (1895–1966) and
+</del>
+ Teresa (
@@ -171,2 +297,6 @@
-  née
- </a>
+ née
+</a>
+<ins>
+ Sproat; 1897–1971), a school teacher.
+</ins>
+<del>
@@ -174 +304,11 @@
- He was raised in
+</del>
+ He was raised in <ins>
+ nearby
+ <a href="/wiki/New_Concord,_Ohio" title="New Concord, Ohio">
+  New Concord
+ </a>
+ ,
+ where his father had a plumbing business, the Glenn Plumbing Company. He had an adopted sister, Jean.
+ He attended New Concord Elementary School.
+</ins>
+<del>
@@ -179 +319 @@
-</p>
+</del>
@@ -181 +321,5 @@
- After graduating from New Concord High School in 1939, he studied
+ After graduating from
+ <a href="/wiki/John_Glenn_High_School_(New_Concord,_Ohio)" title="John Glenn High School (New Concord, Ohio)">
+  New Concord High School
+ </a>
+ in 1939, he studied
@@ -202 +346,8 @@
- degree. However, the school granted Glenn his degree in 1962, after his
+ degree.
+ <ins>
+  The
+ </ins>
+ <del>
+  However, the
+ </del>
+ school granted Glenn his degree in 1962, after his
@@ -207,0 +359,7 @@
+<h2>
+ <ins>
+  <span class="mw-headline" id="Military_career">
+   Military career
+  </span>
+ </ins>
+</h2>
@@ -234 +392 @@
-  World War II
+  World War II
@@ -238 +396 @@
-  U.S. Army Air Corps
+  U.S. Army Air Corps
@@ -240 +398,2 @@
- . However, he was never called to duty, and in March 1942 enlisted as a
+ .
+ However, he was never called to duty, and in March 1942 enlisted as a
@@ -262 +421 @@
-  U.S. Marine Corps
+  U.S. Marine Corps
@@ -267 +426,16 @@
- Upon completing his training in 1943, Glenn was assigned to Marine Squadron
+ Upon completing his training
+ <ins>
+  on March 31, 1943, Glenn was commissioned as a
+  <a href="/wiki/Second_lieutenant" title="Second lieutenant">
+   second lieutenant
+  </a>
+  and, after advanced training at
+  <a href="/wiki/Camp_Kearny" title="Camp Kearny">
+   Camp Kearny
+  </a>
+  , California,
+ </ins>
+ <del>
+  in 1943, Glenn
+ </del>
+ was assigned to Marine Squadron
@@ -275,5 +449,27 @@
- transport planes. He transferred to
- <a href="/wiki/VMF-155" title="VMF-155">
-  VMF-155
- </a>
- as an
+ transport planes. He
+ <ins>
+  was posted to the
+  <a class="mw-redirect" href="/wiki/Marine_Corps_Air_Station_El_Centro" title="Marine Corps Air Station El Centro">
+   Marine Corps Air Station El Centro
+  </a>
+  in California on July 4, 1943, where he joined
+  <a class="mw-redirect" href="/wiki/VMO-155" title="VMO-155">
+   VMO-155
+  </a>
+  , which flew the
+  <a class="mw-redirect" href="/wiki/F4F_Wildcat" title="F4F Wildcat">
+   F4F Wildcat
+  </a>
+  <a href="/wiki/Fighter_aircraft" title="Fighter aircraft">
+   fighter
+  </a>
+  .
+ VMO-155 re-equipped with the
+ </ins>
+ <del>
+  transferred to
+  <a href="/wiki/VMF-155" title="VMF-155">
+   VMF-155
+  </a>
+  as an
+ </del>
@@ -281 +477 @@
-  F4U Corsair
+  F4U Corsair
@@ -282,0 +479,26 @@
+</p>
+<p>
+ <ins>
+  in September 1943.
+ He was promoted to
+  <a href="/wiki/First_lieutenant" title="First lieutenant">
+   first lieutenant
+  </a>
+  in October 1943 and on January 6, 1944, he was shipped out to Hawaii.
+ It was intended that VMO-155 should move to the
+  <a href="/wiki/Marshall_Islands" title="Marshall Islands">
+   Marshall Islands
+  </a>
+  , but this was delayed, and on February 21, it moved to
+  <a href="/wiki/Midway_Atoll" title="Midway Atoll">
+   Midway Atoll
+  </a>
+  , where it became part of the garrison.
+On June 28, 1944, VMO-155 sailed for the
+  <a href="/wiki/Marshall_Islands" title="Marshall Islands">
+   Marshall Islands
+  </a>
+  where, over the next two years he
+ </ins>
+</p>
+<del>
@@ -286,3 +508,11 @@
- , and flew 59
- <a href="/wiki/Aerial_warfare" title="Aerial warfare">
-  combat missions
+ , and
+</del>
+ flew 59 combat <ins>
+ missions, bombing and strafing Japanese positions in the area.
+ He was hit by antiaircraft fire five times, and was awarded two
+ <a href="/wiki/Distinguished_Flying_Cross_(United_States)" title="Distinguished Flying Cross (United States)">
+  Distinguished Flying Crosses
+ </a>
+ and ten
+ <a href="/wiki/Air_Medal" title="Air Medal">
+  Air Medals
@@ -290 +520,5 @@
- in the
+ . Returning to the United States at the end of his one-year tour of duty in
+</ins>
+<del>
+ missions
+  Link: /wiki/Aerial_warfare in the
@@ -303,3 +537,6 @@
- . In 1945, he was assigned to
- <a class="mw-redirect" href="/wiki/NAS_Patuxent_River" title="NAS Patuxent River">
-  NAS Patuxent River
+ . In
+</del>
+ 1945, he was assigned <ins>
+ to
+ <a href="/wiki/Marine_Corps_Air_Station_Cherry_Point" title="Marine Corps Air Station Cherry Point">
+  Marine Corps Air Station Cherry Point
@@ -306,0 +544,6 @@
+ in North Carolina, and then
+</ins>
+ to
+ <a class="mw-redirect" href="/wiki/NAS_Patuxent_River" title="NAS Patuxent River">
+ NAS Patuxent River
+</a>
@@ -309,3 +552,9 @@
-  Maryland
- </a>
- , and was promoted to
+ Maryland
+</a>
+<ins>
+ . He
+</ins>
+<del>
+ , and
+</del>
+ was promoted to
@@ -313,3 +562,39 @@
-  captain
- </a>
- shortly before the war's end.
+ captain
+</a>
+<ins>
+ in July 1945,
+</ins>
+ shortly before the war's <p>
+ <ins>
+  end, but was still unsure as to whether he would receive a regular commission in the Marine Corps. He received orders to return to Cherry Point, where he joined
+  <a class="new" href="/w/index.php?title=VMF-913&amp;action=edit&amp;redlink=1" title="VMF-913 (page does not exist)">
+   VMF-913
+  </a>
+  , another Corsair squadron. There, he was finally informed that he had qualified for a regular commission.
+Glenn next was assigned to
+  <a href="/wiki/Marine_Corps_Air_Station_El_Toro" title="Marine Corps Air Station El Toro">
+   Marine Corps Air Station El Toro
+  </a>
+  in California, in March 1946. He volunteered for service in
+  <a href="/wiki/North_China" title="North China">
+   North China
+  </a>
+  , believing that it would be a short tour. He joined
+  <a href="/wiki/VMF-218" title="VMF-218">
+   VMF-218
+  </a>
+  , yet another Corsair squadron, which was based at
+  <a href="/wiki/Beijing_Nanyuan_Airport" title="Beijing Nanyuan Airport">
+   Nanyuan Field
+  </a>
+  , near
+  <a href="/wiki/Beijing" title="Beijing">
+   Beijing
+  </a>
+  , in December 1946.
+ He
+ </ins>
+ <del>
+  end.
+Glenn
+ </del>
@@ -317,2 +602,4 @@
-<p>
- Glenn flew patrol missions in
+ flew patrol missions in <ins>
+ with the
+</ins>
+<del>
@@ -325,0 +613 @@
+</del>
@@ -328,2 +616,6 @@
-  Guam
- </a>
+ Guam
+</a>
+<ins>
+ in March 1947. He finally returned home in December 1948. He was then posted to
+</ins>
+<del>
@@ -335,3 +627,4 @@
- <a class="mw-redirect" href="/wiki/NAS_Corpus_Christi" title="NAS Corpus Christi">
-  NAS Corpus Christi
- </a>
+</del>
+<a class="mw-redirect" href="/wiki/NAS_Corpus_Christi" title="NAS Corpus Christi">
+ NAS Corpus Christi
+</a>
@@ -340,5 +633,6 @@
-  Texas
- </a>
- , followed by attending the
- <a href="/wiki/Marine_Corps_University#Schools.2C_programs_and_courses" title="Marine Corps University">
-  Amphibious Warfare School
+ Texas
+</a>
+ , <ins>
+ initially as a student, and then as a
+ <a href="/wiki/Flight_instructor" title="Flight instructor">
+  flight instructor
@@ -347 +641,16 @@
-</p>
+ In July 1951, he was sent to
+</ins>
+<del>
+ followed by attending
+</del>
+ the
+ <a class="mw-redirect" href="/wiki/Amphibious_Warfare_School" title="Amphibious Warfare School">
+ Amphibious Warfare School
+  Link: /wiki/Amphibious_Warfare_School at the
+ Marine Corps Base Quantico
+  Link: /wiki/Marine_Corps_Base_Quantico  in
+ Virginia
+  Link: /wiki/Virginia  for a six-month course.
+ He was then posted to the staff of the Commandant, Marine Corps Schools. Given only four hours a month of flying time, Glenn maintained his proficiency (and flight pay) by flying on the weekends.
+  Link: /wiki/Marine_Corps_University#Schools.2C_programs_and_courses .
+</a>
@@ -376 +685,11 @@
- During the
+ <ins>
+  In October 1952, Glenn was ordered to
+  <a href="/wiki/South_Korea" title="South Korea">
+   South Korea
+  </a>
+  during final stages of
+ </ins>
+ <del>
+  During
+ </del>
+ the
@@ -380 +699,12 @@
- , Glenn was assigned to
+ <ins>
+  .
+ On February 3, 1953, he reported to
+  <a href="/wiki/Pohang_Airport" title="Pohang Airport">
+   K-3
+  </a>
+  , where he
+ </ins>
+ <del>
+  , Glenn
+ </del>
+ was assigned to
@@ -384 +714,8 @@
- , flying the new
+ ,
+ <ins>
+  one of the two Marine fighter squadrons there, as its operations officer.
+ VMF-311 was equipped with the
+ </ins>
+ <del>
+  flying the new
+ </del>
@@ -386 +723 @@
-  F9F Panther
+  F9F Panther
@@ -389,4 +726,15 @@
- <a href="/wiki/Interceptor_aircraft" title="Interceptor aircraft">
-  interceptor
- </a>
- . He flew his Panther in 63 combat missions, gaining the nickname "magnet ass" from his alleged ability to attract enemy
+ <ins>
+  <a href="/wiki/Fighter-bomber" title="Fighter-bomber">
+   fighter-bomber
+  </a>
+  , and was assigned a variety of missions. Glenn flew his first, a reconnaissance flight, on February 26, 1953.
+ He flew
+ </ins>
+ <del>
+  <a href="/wiki/Interceptor_aircraft" title="Interceptor aircraft">
+   interceptor
+  </a>
+  . He flew his Panther in
+ </del>
+ 63 combat missions,
+ gaining the nickname "magnet ass" from his alleged ability to attract enemy
@@ -396 +744,11 @@
- . On two occasions, he returned to his base with over 250 holes in his aircraft.
+ <ins>
+  , an occupational hazard of low-level
+  <a href="/wiki/Close_air_support" title="Close air support">
+   close air support
+  </a>
+  missions.
+ </ins>
+ <del>
+  .
+ </del>
+ On two occasions, he returned to his base with over 250 holes in his aircraft.
@@ -413 +771,2 @@
- . He also flew with future Major General
+ .
+ He also flew with future Major General
@@ -420 +779,7 @@
- Glenn flew a second Korean combat tour in an interservice exchange program with the
+ <ins>
+  Before leaving for Korea, Glenn had applied for a interservice exchange position
+ </ins>
+ <del>
+  Glenn flew a second Korean combat tour in an interservice exchange program
+ </del>
+ with the
@@ -425,31 +790,83 @@
- <a href="/wiki/51st_Fighter_Wing" title="51st Fighter Wing">
-  51st Fighter Wing
- </a>
- . He logged 27 missions in the faster
- <a class="mw-redirect" href="/wiki/F-86_Sabre#Variants" title="F-86 Sabre">
-  F-86F Sabre
- </a>
- and shot down three
- <a href="/wiki/Mikoyan-Gurevich_MiG-15" title="Mikoyan-Gurevich MiG-15">
-  MiG-15s
- </a>
- near the
- <a href="/wiki/Yalu_River" title="Yalu River">
-  Yalu River
- </a>
- in the final days before the ceasefire.
-</p>
-<p>
- For his service in 149 combat missions in two wars, he received numerous honors, including the
- <a href="/wiki/Distinguished_Flying_Cross_(United_States)" title="Distinguished Flying Cross (United States)">
-  Distinguished Flying Cross
- </a>
- (six occasions) and the
- <a href="/wiki/Air_Medal" title="Air Medal">
-  Air Medal
- </a>
- with eighteen
- <a class="mw-redirect" href="/wiki/Award_star" title="Award star">
-  award stars
- </a>
- .
+ <ins>
+  flying the
+  <a class="mw-redirect" href="/wiki/F-86_Sabre" title="F-86 Sabre">
+   F-86 Sabre
+  </a>
+  jet
+  <a class="mw-redirect" href="/wiki/Fighter-interceptor" title="Fighter-interceptor">
+   fighter-interceptor
+  </a>
+  . He arranged with Colonel Leon W. Gray to check out in the F-86 at
+  <a class="mw-redirect" href="/wiki/Otis_Air_Force_Base" title="Otis Air Force Base">
+   Otis Air Force Base
+  </a>
+  in preparation.
+ Glenn later wrote that "Since the days of the
+  <a href="/wiki/Lafayette_Escadrille" title="Lafayette Escadrille">
+   Lafayette Escadrille
+  </a>
+  during World War I, pilots have viewed air-to-air combat as the ultimate test not only of their machines but of their own personal determination and flying skills. I was no exception."
+ In June 1953, he reported for duty with the
+  <a class="mw-redirect" href="/wiki/25th_Fighter-Interceptor_Squadron" title="25th Fighter-Interceptor Squadron">
+   25th Fighter-Interceptor Squadron
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/51st_Fighter_Wing" title="51st Fighter Wing">
+   51st Fighter Wing
+  </a>
+ </del>
+ . He logged 27 missions in the faster
+ <ins>
+  F-86, and shot down his first
+  <a class="mw-redirect" href="/wiki/MiG-15" title="MiG-15">
+   MiG-15s
+  </a>
+  in a dogfight on July 12, 1953. A second followed on July 19, and a third on July 22, in a fight in which a flight of four sabres
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/F-86_Sabre#Variants" title="F-86 Sabre">
+   F-86F Sabre
+  </a>
+  and
+ </del>
+ shot down three
+ <ins>
+  MiGs. As it turned out, these were the last aerial victories of the war, which ended with an armistice five days later.
+ </ins>
+ <del>
+  <a href="/wiki/Mikoyan-Gurevich_MiG-15" title="Mikoyan-Gurevich MiG-15">
+   MiG-15s
+  </a>
+  near the
+  <a href="/wiki/Yalu_River" title="Yalu River">
+   Yalu River
+  </a>
+  in the final days before the ceasefire.
+ </del>
+ For his service in
+ <ins>
+  Korea, he was awarded
+ </ins>
+ <del>
+  149 combat missions in two wars, he received numerous honors, including
+ </del>
+ the Distinguished Flying Cross
+ <ins>
+  twice more,
+ </ins>
+ <del>
+  Link: /wiki/Distinguished_Flying_Cross_(United_States) (six occasions)
+ </del>
+ and the Air
+ <ins>
+  Medala further eight times.
+ </ins>
+ <del>
+  Medal
+  Link: /wiki/Air_Medal with eighteen
+  <a class="mw-redirect" href="/wiki/Award_star" title="Award star">
+   award stars
+  </a>
+  .
+ </del>
@@ -471 +888,3 @@
- (class 12), graduating in 1954. He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns. He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
+ (class 12), graduating in 1954.
+ He served as an armament officer, flying planes to high altitude and testing their cannons and machine guns.
+ He was assigned to the Fighter Design Branch of the Navy Bureau of Aeronautics (now
@@ -486 +905,8 @@
- Glenn has nearly 9,000 hours of flying time, with approximately 3,000 hours in
+ Glenn
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ nearly 9,000 hours of flying time, with approximately 3,000 hours in
@@ -504 +930,2 @@
- . The flight from
+ .
+ The flight from
@@ -516 +943 @@
- , took 3 hours, 23 minutes and 8.3 seconds. As he passed over his hometown, a child in the neighborhood reportedly ran to the Glenn house shouting "Johnny dropped a bomb! Johnny dropped a bomb! Johnny dropped a bomb!" as the
+ , took 3 hours, 23 minutes and 8.3 seconds. As he passed over his hometown, a child in the neighborhood reportedly ran to the Glenn house shouting "Johnny dropped a bomb! Johnny dropped a bomb! Johnny dropped a bomb!" as the
@@ -528 +955 @@
- during which speeds dropped below 300 mph), and the first continuous transcontinental
+ during which speeds dropped below 300 mph), and the first continuous transcontinental
@@ -542,0 +970,61 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:172px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    John Glenn in his
+   </ins>
+   <ins>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </ins>
+  </div>
+ </div>
+</div>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Pre-selection">
+   Pre-selection
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  While Glenn was on duty at Patuxent and Washington, Glenn began to read everything he could about space. His office was requested to furnish a test pilot to be sent to the Langley Air Force Base in Virginia to make some runs on a spaceflight simulator, which was a part of NASA research on reentry vehicle shapes. The officer would also be sent to the Naval Air Development Center in Johnsville, Pennsylvania. The test pilot would be subjected to high g-forces in a centrifuge to compare to the data collected in the simulator. Glenn requested this position and was granted it. He spent a few days at Langley and a week in Johnsville for the testing.
+ </ins>
+</p>
+<p>
+ <ins>
+  Prior to Glenn's appointment as an astronaut in the Mercury program, he participated in the capsule design. NASA had requested that military service members participate in planning the mockup of the capsule. Since Glenn had participated in the research at Langley and Johnsville, combined he with his experience sitting on mock-up boards in the Navy and his knowledge of the capsule procedures, he was sent to the McDonnell plant in St. Louis and acted as a service adviser on the mock-up board.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Selection">
+   Selection
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  In 1958, the newly formed NASA began a recruiting program for astronauts,
+ and Glenn just barely met the requirements as he was close to the age cutoff of 40 and also lacked the required science-based degree at the time. He remained an officer in the
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   United States Marine Corps
+  </a>
+  after he was selected in 1959.
+ After his selection, he was assigned to the NASA Space Task Group in 1959, which was located at Langley Research Center in Hampton, Virginia.
+ The task force was moved to Houston in 1962 and became a part of the NASA Manned Spacecraft Center.
+ Glenn was a backup pilot to Alan Shepard and Gus Grissom, on the Freedom 7 and Liberty Bell 7 respectively.
+ Astronauts were given an additional role in the spaceflight program, and Glenn's was the cockpit layout and control functioning, not only for Mercury but also early designs for Apollo.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Project_Mercury">
+   Project Mercury
+  </span>
+ </ins>
+</h3>
@@ -548,0 +1037,11 @@
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:242px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn (center) with President John F. Kennedy and General Leighton I. Davis celebrating Glenn's orbital flight, 1962
+   </ins>
+  </div>
+ </div>
+</div>
@@ -550,29 +1049,31 @@
- In 1958, the newly formed NASA began a recruiting program for astronauts. Requirements were that each had to be a military test pilot between the ages of 25 and 40 with sufficient flight hours, no more than 5'11" in height, and possess a degree in a scientific field. 508 pilots were subjected to rigorous mental and physical tests, and finally the selection was narrowed down to seven astronauts (Glenn,
- <a href="/wiki/Alan_Shepard" title="Alan Shepard">
-  Alan Shepard
- </a>
- ,
- <a href="/wiki/Gus_Grissom" title="Gus Grissom">
-  Gus Grissom
- </a>
- ,
- <a href="/wiki/Scott_Carpenter" title="Scott Carpenter">
-  Scott Carpenter
- </a>
- ,
- <a href="/wiki/Wally_Schirra" title="Wally Schirra">
-  Wally Schirra
- </a>
- ,
- <a href="/wiki/Gordon_Cooper" title="Gordon Cooper">
-  Gordon Cooper
- </a>
- , and
- <a href="/wiki/Deke_Slayton" title="Deke Slayton">
-  Deke Slayton
- </a>
- ), who were introduced to the public at a NASA press conference in April 1959. Glenn just barely met the requirements as he was close to the age cutoff of 40 and also lacked the required science-based degree at the time. During this time, he remained an officer in the
- <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
-  United States Marine Corps
- </a>
- .
+ <del>
+  In 1958, the newly formed NASA began a recruiting program for astronauts. Requirements were that each had to be a military test pilot between the ages of 25 and 40 with sufficient flight hours, no more than 5'11" in height, and possess a degree in a scientific field. 508 pilots were subjected to rigorous mental and physical tests, and finally the selection was narrowed down to seven astronauts (Glenn,
+  <a href="/wiki/Alan_Shepard" title="Alan Shepard">
+   Alan Shepard
+  </a>
+  ,
+  <a href="/wiki/Gus_Grissom" title="Gus Grissom">
+   Gus Grissom
+  </a>
+  ,
+  <a href="/wiki/Scott_Carpenter" title="Scott Carpenter">
+   Scott Carpenter
+  </a>
+  ,
+  <a href="/wiki/Wally_Schirra" title="Wally Schirra">
+   Wally Schirra
+  </a>
+  ,
+  <a href="/wiki/Gordon_Cooper" title="Gordon Cooper">
+   Gordon Cooper
+  </a>
+  , and
+  <a href="/wiki/Deke_Slayton" title="Deke Slayton">
+   Deke Slayton
+  </a>
+  ), who were introduced to the public at a NASA press conference in April 1959. Glenn just barely met the requirements as he was close to the age cutoff of 40 and also lacked the required science-based degree at the time. During this time, he remained an officer in the
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   United States Marine Corps
+  </a>
+  .
+ </del>
@@ -582,3 +1082,0 @@
-  <a class="image" href="/wiki/File:GPN-2000-001027.jpg">
-   <img alt="" class="thumbimage" data-file-height="2343" data-file-width="1800" height="221" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/170px-GPN-2000-001027.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/255px-GPN-2000-001027.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/93/GPN-2000-001027.jpg/340px-GPN-2000-001027.jpg 2x" width="170"/>
-  </a>
@@ -587,2 +1084,0 @@
-    <a class="internal" href="/wiki/File:GPN-2000-001027.jpg" title="Enlarge">
-    </a>
@@ -590,4 +1086,8 @@
-   John Glenn in his
-   <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
-    Mercury spacesuit
-   </a>
+   <del>
+    John Glenn in his
+   </del>
+   <del>
+    <a href="/wiki/Navy_Mark_IV" title="Navy Mark IV">
+     Mercury spacesuit
+    </a>
+   </del>
@@ -608 +1108,7 @@
- on February 20, 1962, on the Mercury-Atlas 6 mission, circling the globe three times during a flight lasting 4 hours, 55 minutes, and 23 seconds.
+ on February 20, 1962, on the Mercury-Atlas 6 mission, circling the globe three times during a flight lasting
+ <ins>
+  nearly five hours.
+ </ins>
+ <del>
+  4 hours, 55 minutes, and 23 seconds.
+ </del>
@@ -610,0 +1117,27 @@
+<p>
+ <ins>
+  For Glenn the day became the "best day of his life," while it also renewed America's confidence.
+ His voyage took place while America and the Soviet Union were in the midst of the
+ </ins>
+ <ins>
+  <a href="/wiki/Cold_War" title="Cold War">
+   Cold War
+  </a>
+  and competing in the "
+ </ins>
+ <ins>
+  <a href="/wiki/Space_Race" title="Space Race">
+   Space Race
+  </a>
+  ."
+ The flight took him to an altutide of approximately 162 statute miles at a velocity of about 17,500 mph.
+ </ins>
+</p>
+<p>
+ <ins>
+  During Glenn's first orbit, there was a scheduled 30 minute test to see if Glenn could fly the spacecraft manually. This test became significant once a failure of the automatic control system was detected at the end of the first orbit. This forced Glenn to operate in manual mode for the second and third orbits, as well as re-entry.
+ </ins>
+</p>
+<ins>
+ Later in the flight, telemetry indicated that the
+</ins>
@@ -613 +1146,3 @@
-  Zero G, and I feel fine.
+  <del>
+   Zero G, and I feel fine.
+  </del>
@@ -616,7 +1151,9 @@
-  <cite>
-   — Glenn's first words on becoming
-   <a href="/wiki/Weightlessness" title="Weightlessness">
-    weightless
-   </a>
-   upon reaching orbit, February 20, 1962,
-  </cite>
+  <del>
+   <cite>
+    — Glenn's first words on becoming
+    <a href="/wiki/Weightlessness" title="Weightlessness">
+     weightless
+    </a>
+    upon reaching orbit, February 20, 1962,
+   </cite>
+  </del>
@@ -625 +1162 @@
-<p>
+<del>
@@ -629,0 +1167,2 @@
+</del>
+<del>
@@ -634,0 +1174,2 @@
+</del>
+<del>
@@ -640,4 +1181,11 @@
- <a class="mw-redirect" href="/wiki/Ablative_heat_shield" title="Ablative heat shield">
-  heat shield
- </a>
- had come loose, which could allow it to fail during
+</del>
+<a class="mw-redirect" href="/wiki/Ablative_heat_shield" title="Ablative heat shield">
+ heat shield
+</a>
+ had <ins>
+ become loose. If the telemetry was correct, Glenn's spacecraft would likely have been destroyed
+</ins>
+<del>
+ come loose, which could allow it to fail
+</del>
+ during
@@ -645,3 +1193,9 @@
-  re-entry
- </a>
- through the atmosphere, causing his capsule to burn up. Flight controllers had Glenn modify his re-entry procedure by keeping his
+ re-entry
+</a>
+<ins>
+ due to the intense heat.
+</ins>
+<del>
+ through the atmosphere, causing his capsule to burn up.
+</del>
+ Flight controllers had Glenn modify his re-entry procedure by keeping his
@@ -649,3 +1203,12 @@
-  retrorocket
- </a>
- pack on over the shield in an attempt to keep it in place. He made his
+ retrorocket
+</a>
+ pack on over the shield <ins>
+ to help
+</ins>
+<del>
+ in an attempt to
+</del>
+ keep it in place. <ins>
+ Leaving the retrorocket pack on caused large chunks of flaming debris to fly past the window of the capsule during re-entry, although Glenn thought it could have also been the heat shield. He told an interviewer, "Fortunately it was the rocket pack - or I wouldn't be answering these questions."
+</ins>
+ He made his
@@ -653 +1216,13 @@
-  splashdown
+ splashdown
+</a>
+<ins>
+ 800 miles southeast of Cape Canaveral safely after his 4 hour and 55 minute flight,
+</ins>
+<del>
+ safely,
+</del>
+ and afterwards it was determined that the indicator was faulty.
+ <ins>
+ Glenn's flight and fiery splashdown were portrayed in the 1983 film
+ <a href="/wiki/The_Right_Stuff_(film)" title="The Right Stuff (film)">
+  The Right Stuff
@@ -655,2 +1230,2 @@
- safely, and afterwards it was determined that the indicator was faulty.
-</p>
+ .
+</ins>
@@ -699,0 +1275,7 @@
+ <ins>
+  However, he became "so valuable to the nation as an iconic figure," said NASA administrator
+  <a href="/wiki/Charles_Bolden" title="Charles Bolden">
+   Charles Bolden
+  </a>
+  , that Kennedy would not "risk putting him back in space again."
+ </ins>
@@ -721 +1303,17 @@
- in favor of excluding women from the NASA astronaut program. Although NASA had no official policy prohibiting women, in practice the requirement that astronauts had to be military test pilots excluded them entirely.
+ in favor of excluding women from the NASA astronaut program. Although NASA had no official policy prohibiting women, in
+ <ins>
+  practice,
+ </ins>
+ <del>
+  practice
+ </del>
+ the requirement that astronauts had to be military test pilots excluded them entirely.
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Resignation">
+   Resignation
+  </span>
+ </ins>
+</h3>
+<del>
@@ -735 +1333 @@
-</p>
+</del>
@@ -744 +1342,6 @@
-  Colonel
+  <ins>
+   colonel
+  </ins>
+  <del>
+   Colonel
+  </del>
@@ -765,0 +1369,31 @@
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     This section
+     <b>
+      is missing information about what Glenn did during his 24 years in the Senate
+     </b>
+     .
+     <span class="hide-when-compact">
+      Please expand the section to include this information. Further details may exist on the
+      <a href="/wiki/Talk:John_Glenn#Senate_career" title="Talk:John Glenn">
+       talk page
+      </a>
+      .
+     </span>
+     <small>
+      <i>
+       (December 2016)
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
@@ -804,0 +1439,3 @@
+ <ins>
+  He served as a pallbearer at Kennedy's funeral.
+ </ins>
@@ -823 +1460,6 @@
- . Instead, he challenged Metzenbaum again, whom Gilligan had appointed
+ . Instead, he challenged Metzenbaum
+ <ins>
+  again for the Senate seat vacated by
+ </ins>
+ <del>
+  again, whom Gilligan had appointed
@@ -824,0 +1467 @@
+ </del>
@@ -826 +1469 @@
-  William B. Saxbe
+  William B. Saxbe
@@ -828 +1471,6 @@
- , who had resigned to become Attorney General of the United States.
+ <ins>
+  (R-OH).
+ </ins>
+ <del>
+  , who had resigned to become Attorney General of the United States.
+ </del>
@@ -851,8 +1499,11 @@
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
-  In 1986, Glenn defeated challenger
- </a>
- U.S. Representative
- <a href="/wiki/Tom_Kindness" title="Tom Kindness">
-  Tom Kindness
- </a>
- . Metzenbaum would go on to seek a rematch against Taft in 1976, winning a close race on
+ <del>
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
+   In 1986, Glenn defeated challenger
+  </a>
+  U.S. Representative
+  <a href="/wiki/Tom_Kindness" title="Tom Kindness">
+   Tom Kindness
+  </a>
+  .
+ </del>
+ Metzenbaum would go on to seek a rematch against Taft in 1976, winning a close race on
@@ -876,0 +1528,10 @@
+ <ins>
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1986" title="United States Senate election in Ohio, 1986">
+   In 1986, Glenn defeated challenger
+  </a>
+  U.S. Representative
+  <a href="/wiki/Tom_Kindness" title="Tom Kindness">
+   Tom Kindness
+  </a>
+  . In 1997, Glenn announced that he would retire from the Senate at the end of his then-current term.
+ </ins>
@@ -911,5 +1572,10 @@
- to keep his seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
- <i>
-  on earth
- </i>
- has John Glenn done?"
+ to keep his
+ <ins>
+  seat.
+ </ins>
+ <del>
+  seat, though his percentage was reduced to a career low of 51%. DeWine used the memorable campaign slogan, "What
+  <i>
+   on earth
+  </i>
+  has John Glenn done?"
@@ -916,0 +1583 @@
+ </del>
@@ -932 +1599,2 @@
- . Glenn also ran for the
+ .
+ Glenn also ran for the
@@ -936,4 +1604,11 @@
- . A November 1983
- <i>
-  <a href="/wiki/The_New_York_Times" title="The New York Times">
-   New York Times
+ .
+ <del>
+  A November 1983
+  <i>
+   <a href="/wiki/The_New_York_Times" title="The New York Times">
+    New York Times
+   </a>
+  </i>
+  /
+  <a href="/wiki/CBS_News" title="CBS News">
+   CBS News
@@ -941,6 +1616,2 @@
- </i>
- /
- <a href="/wiki/CBS_News" title="CBS News">
-  CBS News
- </a>
- poll found him second, supported by 41% of those polled, to Mondale's 49%.
+  poll found him second, supported by 41% of those polled, to Mondale's 49%.
+ </del>
@@ -977,5 +1648,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (July 2014)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -983,5 +1654,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -999 +1670,2 @@
- , served as chairman of the
+ ,
+ served as chairman of the
@@ -1015,8 +1687,33 @@
- . Once Republicans regained control of the Senate, Glenn also served as the ranking minority member on a special Senate investigative committee chaired by
- <a href="/wiki/Tennessee" title="Tennessee">
-  Tennessee
- </a>
- senator
- <a class="mw-redirect" href="/wiki/Fred_Dalton_Thompson" title="Fred Dalton Thompson">
-  Fred Dalton Thompson
- </a>
+ .
+</p>
+<p>
+ Once Republicans regained control of the Senate, Glenn
+ <del>
+  also
+ </del>
+ served as the ranking minority member on
+ <ins>
+  the
+  <a href="/wiki/United_States_Senate_Homeland_Security_Permanent_Subcommittee_on_Investigations" title="United States Senate Homeland Security Permanent Subcommittee on Investigations">
+   Permanent Subcommittee on Investigations
+  </a>
+  , chaired by
+  <a href="/wiki/Maine" title="Maine">
+   Maine
+  </a>
+  senator
+  <a href="/wiki/Susan_Collins" title="Susan Collins">
+   Susan Collins
+  </a>
+  ,
+ </ins>
+ <del>
+  a special Senate investigative committee chaired by
+  <a href="/wiki/Tennessee" title="Tennessee">
+   Tennessee
+  </a>
+  senator
+  <a class="mw-redirect" href="/wiki/Fred_Dalton_Thompson" title="Fred Dalton Thompson">
+   Fred Dalton Thompson
+  </a>
+ </del>
@@ -1027 +1724,15 @@
- to U.S. political campaigns for the 1996 election. There was considerable acrimony between the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
+ to U.S. political campaigns for the 1996 election.
+ There was considerable acrimony between
+ <ins>
+  Glenn and the overseeing committee chair,
+  <a href="/wiki/Fred_Thompson" title="Fred Thompson">
+   Fred Thompson
+  </a>
+  of
+  <a href="/wiki/Tennessee" title="Tennessee">
+   Tennessee
+  </a>
+  .
+ </ins>
+ <del>
+  the two very high-profile senators during the life of this committee, which reached a level of public disagreement between the five leaders of a congressional committee seldom seen in recent years, amid allegations that Glenn suppressed these issues prior to his subsequent Space Shuttle flight which had to be approved by President Clinton.
@@ -1029,12 +1740,13 @@
- <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
-  Mary O. Boyle
- </a>
- was the Democratic party nominee. She faced Republican nominee and sitting governor
- <a href="/wiki/George_Voinovich" title="George Voinovich">
-  George Voinovich
- </a>
- in the
- <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
-  general election
- </a>
- , which Voinovich won.
+  <a href="/wiki/Mary_O._Boyle" title="Mary O. Boyle">
+   Mary O. Boyle
+  </a>
+  was the Democratic party nominee. She faced Republican nominee and sitting governor
+  <a href="/wiki/George_Voinovich" title="George Voinovich">
+   George Voinovich
+  </a>
+  in the
+  <a href="/wiki/United_States_Senate_election_in_Ohio,_1998" title="United States Senate election in Ohio, 1998">
+   general election
+  </a>
+  , which Voinovich won.
+ </del>
@@ -1052,0 +1765,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Senator-astronaut John Glenn on the shuttle Discovery, 1998
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1058 +1781,5 @@
- on October 29, 1998, becoming, at age 77, the oldest person to go into space as a
+ on October 29, 1998,
+ <del>
+  becoming, at age 77, the oldest person to go into space
+ </del>
+ as a
@@ -1072 +1799,7 @@
- mission. According to
+ <ins>
+  mission, becoming, at age 77, the oldest person to go into space.
+ </ins>
+ <del>
+  mission.
+ </del>
+ According to
@@ -1078,0 +1812,9 @@
+ <ins>
+  Shortly before the flight, researchers learned that Glenn had to be disqualified from one of the flight's two main priority human experiments (about the effects of
+  <a href="/wiki/Melatonin" title="Melatonin">
+   melatonin
+  </a>
+  ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+ </ins>
+</p>
+<p>
@@ -1088,3 +1830,7 @@
-</p>
-<p>
- Glenn's participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
+ <ins>
+  His
+ </ins>
+ <del>
+  Glenn's
+ </del>
+ participation in the nine-day mission was criticized by some in the space community as a political favor granted to Glenn by
@@ -1101,5 +1847,12 @@
- It was noted that Glenn's flight offered valuable research on weightlessness and other aspects of space flight on the same person at two points in life 36 years apart—by far the longest interval between space flights by the same person—providing information on the effects of spaceflight and weightlessness on the elderly, with an ideal
- <a href="/wiki/Control_variable" title="Control variable">
-  control subject
- </a>
- .
+ <ins>
+  In a 2012 interview, Glenn said that the purpose of his flight was "to make measurements and do research on me at the age of 77 [...] comparing the results on me in space with the younger [astronauts] and maybe get [insights] on the immune system or protein turnover or vestibular functions and other things — heart changes.
+ He regretted that NASA did not follow up on this research about aging by sending more people from this age range into space.
+ </ins>
+</p>
+<p>
+ <del>
+  It was noted that Glenn's flight offered valuable research on weightlessness and other aspects of space flight on the same person at two points in life 36 years apart—by far the longest interval between space flights by the same person—providing information on the effects of spaceflight and weightlessness on the elderly, with an ideal
+  <a href="/wiki/Control_variable" title="Control variable">
+   control subject
+  </a>
+  .
@@ -1107,4 +1860,5 @@
- <a href="/wiki/Melatonin" title="Melatonin">
-  melatonin
- </a>
- ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+  <a href="/wiki/Melatonin" title="Melatonin">
+   melatonin
+  </a>
+  ) because he did not meet one of the study's medical conditions; he still participated in two other experiments about sleep monitoring and protein use.
+ </del>
@@ -1142 +1896 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+ <a href="/wiki/Ohio_State_University" title="Ohio State University">
@@ -1149 +1903,7 @@
- . Today Glenn holds an adjunct professorship at the Glenn School.
+ <ins>
+  , and Glenn held
+ </ins>
+ <del>
+  . Today Glenn holds
+ </del>
+ an adjunct professorship at the Glenn School.
@@ -1167 +1927,7 @@
-   Anna and John Glenn in 1965
+   <ins>
+    Glenn and his wife Anna
+   </ins>
+   <del>
+    Anna and John Glenn
+   </del>
+   in 1965
@@ -1172 +1938,8 @@
- On April 6, 1943, Glenn married his childhood sweetheart,
+ On April 6, 1943, Glenn married his
+ <ins>
+  high school
+ </ins>
+ <del>
+  childhood
+ </del>
+ sweetheart,
@@ -1184 +1956,0 @@
- . He also was a member of the Stag Club Fraternity at Muskingum College.
@@ -1187 +1959,76 @@
- Glenn was also one of the original owners of a
+ <ins>
+  , where he was a member of the Stag Club Fraternity.
+ Together, they had two children, John David and Carolyn Ann, and two grandchildren.
+ They remained married for 73 years, until his death. His NASA friend
+ </ins>
+ <ins>
+  <a href="/wiki/Charles_Bolden" title="Charles Bolden">
+   Charles Bolden
+  </a>
+  saw Glenn's marriage as an inspiration:
+ </ins>
+</p>
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   For anyone who's contemplating marriage, you ought to go to school on the Glenns, because they can teach us a lot about what unending love, with undying respect and admiration for each other means...There was never a question whether Annie was the love of his life.
+  </ins>
+ </p>
+</blockquote>
+<p>
+ <ins>
+  His boyhood home in New Concord has been
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </ins>
+</p>
+<p>
+ <ins>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn was a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and was an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  He set an example of someone whose faith began before he became an astronaut, and whose faith was reinforced after traveling in space.
+ </ins>
+</p>
+<p>
+ <ins>
+  "To look out at this kind of creation and not believe in God is to me impossible," said Glenn, after his second and final space voyage.
+ He stated that he saw no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believed evolution should be taught in schools.
+ He explained:
+ </ins>
+</p>
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   I don't see that I'm any less religious that I can appreciate the fact that science just records that we change with evolution and time, and that's a fact. It doesn't mean it's less wondrous and it doesn't mean that there can't be some power greater than any of us that has been behind and is behind whatever is going on.
+  </ins>
+ </p>
+</blockquote>
+<ins>
+ Glenn was
+</ins>
+<del>
+ . He also was a member of the Stag Club Fraternity at Muskingum College.
+Glenn was also
+</del>
+  one of the original owners of a
@@ -1189,2 +2036,2 @@
-  Holiday Inn
- </a>
+ Holiday Inn
+</a>
@@ -1193,2 +2040,2 @@
-  Orlando
- </a>
+ Orlando
+</a>
@@ -1197,2 +2044,2 @@
-  Seralago Hotel &amp; Suites Main Gate East
- </a>
+ Seralago Hotel &amp; Suites Main Gate East
+</a>
@@ -1199,0 +2047,11 @@
+ <p>
+ <ins>
+  His business partner was
+ </ins>
+ <ins>
+  <a href="/wiki/Henri_Landwirth" title="Henri Landwirth">
+   Henri Landwirth
+  </a>
+  , a Holocaust survivor, who became Glenn's "best friend."
+ Glenn recalls learning about Landwirth's background:
+ </ins>
@@ -1201,2 +2059,38 @@
-<p>
- Glenn is an honorary member of the
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   Henri doesn't talk about it much. It was years before he spoke about it with me and then only because of an accident. We were down in Florida during the space program. Everyone was wearing short-sleeved Ban-Lon shirts—everyone but Henri. Then one day I saw Henri at the pool and noticed the number on his arm. I told Henri that if it were me I'd wear that number like a medal with a spotlight on it.
+  </ins>
+ </p>
+</blockquote>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Public_appearances_and_ceremonies">
+   Public appearances and ceremonies
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn appears with President Kennedy and Soviet cosmonaut
+   </ins>
+   <ins>
+    <a href="/wiki/Gherman_Titov" title="Gherman Titov">
+     Gherman Titov
+    </a>
+    , 1962
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ Glenn was
+</ins>
+<del>
+ Glenn is
+</del>
+ an honorary member of the
@@ -1204,2 +2098,2 @@
-  International Academy of Astronautics
- </a>
+ International Academy of Astronautics
+</a>
@@ -1208,2 +2102,2 @@
-  Society of Experimental Test Pilots
- </a>
+ Society of Experimental Test Pilots
+</a>
@@ -1212,2 +2106,2 @@
-  Order of Daedalians
- </a>
+ Order of Daedalians
+</a>
@@ -1216,2 +2110,2 @@
-  Ohio Democratic Party
- </a>
+ Ohio Democratic Party
+</a>
@@ -1218,0 +2113,16 @@
+ <p>
+ <del>
+  A
+  <a href="/wiki/Freemasonry" title="Freemasonry">
+   Freemason
+  </a>
+  , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
+  <a href="/wiki/DeMolay_International" title="DeMolay International">
+   DeMolay International
+  </a>
+  , the Masonic youth organization, and is an ordained elder in the
+  <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
+   Presbyterian Church
+  </a>
+  .
+ </del>
@@ -1221,23 +2131,4 @@
- A
- <a href="/wiki/Freemasonry" title="Freemasonry">
-  Freemason
- </a>
- , Glenn is a member of Concord Lodge # 688 New Concord, Ohio, and
- <a href="/wiki/DeMolay_International" title="DeMolay International">
-  DeMolay International
- </a>
- , the Masonic youth organization, and is an ordained elder in the
- <a class="mw-redirect" href="/wiki/Presbyterian_Church_(U.S.A.)" title="Presbyterian Church (U.S.A.)">
-  Presbyterian Church
- </a>
- .
-</p>
-<p>
- Glenn's name was used for the character of
- <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
-  John Tracy
- </a>
- in the 1960s children's TV series
- <i>
-  <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
-   Thunderbirds
+ <del>
+  Glenn's name was used for the character of
+  <a href="/wiki/John_Tracy_(Thunderbirds)" title="John Tracy (Thunderbirds)">
+   John Tracy
@@ -1245,2 +2136,8 @@
- </i>
- .
+  in the 1960s children's TV series
+  <i>
+   <a href="/wiki/Thunderbirds_(TV_series)" title="Thunderbirds (TV series)">
+    Thunderbirds
+   </a>
+  </i>
+  .
+ </del>
@@ -1249,9 +2146,11 @@
- Glenn's boyhood home in New Concord has been
- <a href="/wiki/Building_restoration" title="Building restoration">
-  restored
- </a>
- and made into an
- <a href="/wiki/Historic_house_museum" title="Historic house museum">
-  historic house museum
- </a>
- and education center.
+ <del>
+  Glenn's boyhood home in New Concord has been
+  <a href="/wiki/Building_restoration" title="Building restoration">
+   restored
+  </a>
+  and made into an
+  <a href="/wiki/Historic_house_museum" title="Historic house museum">
+   historic house museum
+  </a>
+  and education center.
+ </del>
@@ -1266,19 +2165,29 @@
- .
-</p>
-<p>
- On August 4, 2006, Glenn and his wife were injured in an automobile accident on
- <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
-  I-270
- </a>
- near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
- <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
-  sternum
- </a>
- . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
-</p>
-<p>
- On September 5, 2009, John and Annie Glenn dotted the "i" during
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  The Ohio State University
- </a>
- 's
+ <ins>
+  , as himself.
+ </ins>
+ <del>
+  .
+ </del>
+</p>
+<p>
+ <del>
+  On August 4, 2006, Glenn and his wife were injured in an automobile accident on
+  <a href="/wiki/Interstate_270_(Ohio)" title="Interstate 270 (Ohio)">
+   I-270
+  </a>
+  near Columbus, Ohio, and were hospitalized for two days. Glenn suffered a "very sore chest" and a fractured
+  <a class="mw-redirect" href="/wiki/Human_sternum" title="Human sternum">
+   sternum
+  </a>
+  . Annie Glenn was treated for minor injuries. Glenn was cited for failure to yield the right-of-way.
+ </del>
+</p>
+<p>
+ On September 5, 2009, John and Annie Glenn dotted the "i" during The Ohio State
+ <ins>
+  University's
+ </ins>
+ <del>
+  University
+  Link: /wiki/The_Ohio_State_University 's
+ </del>
@@ -1295 +2204,6 @@
- -
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
@@ -1299,0 +2214,3 @@
+ <ins>
+  Other non-band members to have received this honor include
+ </ins>
@@ -1308,16 +2225,18 @@
- <a href="/wiki/Buster_Douglas" title="Buster Douglas">
-  Buster Douglas
- </a>
- ,
- <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
-  E. Gordon Gee
- </a>
- ,
- <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
-  Novice Fawcett
- </a>
- ,
- <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
-  Robert Ries
- </a>
- , and
+ <del>
+  <a href="/wiki/Buster_Douglas" title="Buster Douglas">
+   Buster Douglas
+  </a>
+  ,
+  <a href="/wiki/E._Gordon_Gee" title="E. Gordon Gee">
+   E. Gordon Gee
+  </a>
+  ,
+  <a class="mw-redirect" href="/wiki/Novice_Fawcett" title="Novice Fawcett">
+   Novice Fawcett
+  </a>
+  ,
+  <a class="new" href="/w/index.php?title=Robert_Ries&amp;action=edit&amp;redlink=1" title="Robert Ries (page does not exist)">
+   Robert Ries
+  </a>
+  , and
+ </del>
@@ -1331 +2250,6 @@
- are the only other non-band members to have received this honor.
+ <ins>
+  .
+ </ins>
+ <del>
+  are the only other non-band members to have received this honor.
+ </del>
@@ -1354,4 +2278,10 @@
- <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
-  Ohio State
- </a>
- , where the public affairs school is named for him.
+ <ins>
+  The Ohio State University,
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/The_Ohio_State_University" title="The Ohio State University">
+   Ohio State
+  </a>
+  ,
+ </del>
+ where the public affairs school is named for him.
@@ -1400 +2330,24 @@
- In June 2014, Glenn underwent a successful
+ In June
+</p>
+<p>
+ <ins>
+  2016 the Columbus, Ohio airport known for many years as Port Columbus was officially renamed the
+ </ins>
+ <ins>
+  <a href="/wiki/John_Glenn_Columbus_International_Airport" title="John Glenn Columbus International Airport">
+   John Glenn Columbus International Airport
+  </a>
+  . Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke about how visiting that airport as a child inspired his interest in flying.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Illness_and_death">
+   Illness and death
+  </span>
+ </ins>
+</h2>
+<ins>
+ In June
+</ins>
+  2014, Glenn underwent a successful
@@ -1402,2 +2355,2 @@
-  heart valve replacement
- </a>
+ heart valve replacement
+</a>
@@ -1406,2 +2359,2 @@
-  Cleveland Clinic
- </a>
+ Cleveland Clinic
+</a>
@@ -1408,0 +2362,4 @@
+ <p>
+ <ins>
+  At the beginning of December 2016,
+ </ins>
@@ -1411 +2368,3 @@
- On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ <del>
+  On June 28, 2016, the Columbus, Ohio airport was officially renamed the John Glenn Columbus International Airport. Just before his 95th birthday, Glenn and his wife Annie attended the ceremony, and he spoke eloquently about how visiting that airport as a child inspired his interest in flying.
+ </del>
@@ -1414 +2373,3 @@
- Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ <del>
+  Glenn has stated that he sees no contradiction between believing in God and the knowledge that evolution is "a fact", and that he believes evolution should be taught in schools.
+ </del>
@@ -1417,3 +2378,5 @@
- <span class="mw-headline" id="Illness">
-  Illness
- </span>
+ <del>
+  <span class="mw-headline" id="Illness">
+   Illness
+  </span>
+ </del>
@@ -1421 +2384 @@
-<p>
+<del>
@@ -1422,0 +2386,2 @@
+</del>
+<del>
@@ -1426,4 +2391,20 @@
- announced that Glenn was hospitalized at OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
- <a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
-  The James Cancer Hospital
- </a>
+ announced that
+</del>
+ Glenn was hospitalized at <ins>
+ the
+</ins>
+<a href="/wiki/The_James_Cancer_Hospital" title="The James Cancer Hospital">
+ <del>
+  OSU, having been admitted "more than a week" before. The spokesman indicated he did not know Glenn's condition or diagnosis, and cautioned that his hospitalization at
+ The
+ </del>
+ James Cancer Hospital
+</a>
+<ins>
+ of
+ <a href="/wiki/Ohio_State_University_Wexner_Medical_Center" title="Ohio State University Wexner Medical Center">
+  OSU Wexner Medical Center
+ </a>
+ in Columbus.
+</ins>
+<del>
@@ -1431 +2412,81 @@
- A family source said that Glenn was in declining health, that his condition was grave, and that Annie Glenn and his children and grandchildren had joined him at the hospital.
+</del>
+ A family source said that Glenn <ins>
+ had been
+</ins>
+<del>
+ was
+</del>
+ in declining health, <ins>
+ and
+</ins>
+ that his condition was <ins>
+ grave. His wife, Annie, and their
+</ins>
+<del>
+ grave, and that Annie Glenn and his
+</del>
+ children and grandchildren had joined him at the hospital.
+ <p>
+ <ins>
+  Glenn died December 8, 2016, at the OSU Wexner Medical Center.
+ No cause of death has yet been disclosed. Glenn will be interred at
+  <a href="/wiki/Arlington_National_Cemetery" title="Arlington National Cemetery">
+   Arlington National Cemetery
+  </a>
+  after lying in state at the
+  <a href="/wiki/Ohio_Statehouse" title="Ohio Statehouse">
+   Ohio Statehouse
+  </a>
+  and a memorial service at Mershon Auditorium at The Ohio State University.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Tributes">
+   Tributes
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:242px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Glenn looks into a celestial training device before his 1962 launch.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Among those honoring Glenn were President
+  <a href="/wiki/Barack_Obama" title="Barack Obama">
+   Barack Obama
+  </a>
+  , who said that Glenn, "the first American to orbit the Earth, reminded us that with courage and a spirit of discovery there's no limit to the heights we can reach together."
+ Tributes were also given by former Secretary of State
+  <a href="/wiki/Hillary_Clinton" title="Hillary Clinton">
+   Hillary Clinton
+  </a>
+  ,
+ and President-elect
+  <a href="/wiki/Donald_Trump" title="Donald Trump">
+   Donald Trump
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  The phrase "Godspeed," that hailed Glenn's historic launch into space, became a social media
+  <a href="/wiki/Hashtag" title="Hashtag">
+   hashtag
+  </a>
+  #Godspeedjohnglenn. Past and current astronauts added their own tributes, along with NASA Administrator and former shuttle astronaut,
+  <a href="/wiki/Charles_Bolden" title="Charles Bolden">
+   Charles Bolden
+  </a>
+  , who added that "John Glenn's legacy is one of risk and accomplishment, of history created and duty to country carried out under great pressure with the whole world watching."
+ President Obama has ordered all flags half staff until his interment.
+ </ins>
@@ -1432,0 +2494,2 @@
+<div style="clear:both;">
+</div>
@@ -1471,3 +2534,5 @@
-     <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
-      Friendship 7
-     </a>
+     <i>
+      <a class="mw-redirect" href="/wiki/Friendship_7" title="Friendship 7">
+       Friendship 7
+      </a>
+     </i>
@@ -1766 +2831,7 @@
-    and two
+    and
+    <ins>
+     eighteen
+    </ins>
+    <del>
+     two
+    </del>
@@ -1778 +2849,10 @@
-    with fifteen stars and two clusters
+    with fifteen stars and
+    <ins>
+     eighteen
+     <a href="/wiki/Oak_leaf_cluster" title="Oak leaf cluster">
+      clusters
+     </a>
+    </ins>
+    <del>
+     two clusters
+    </del>
@@ -1917,4 +2997,6 @@
-  <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
-   Golden Plate Award
-  </a>
-  for Science and Exploration, 1964
+  <del>
+   <a class="mw-redirect" href="/wiki/Golden_Plate_Award" title="Golden Plate Award">
+    Golden Plate Award
+   </a>
+   for Science and Exploration, 1964
+  </del>
@@ -1928,0 +3011,5 @@
+ <li>
+  <ins>
+   Ambassador of Space Exploration Award
+  </ins>
+ </li>
@@ -1975 +3062,5 @@
-   ) to Senator John Glenn and
+   ) to Senator John Glenn
+   <ins>
+    (left)
+   </ins>
+   and
@@ -1982,0 +3074,3 @@
+   <ins>
+    (right)
+   </ins>
@@ -2079 +3173,8 @@
- , a mobile landing platform that is scheduled to be delivered to the
+ , a mobile landing platform that
+ <ins>
+  was
+ </ins>
+ <del>
+  is scheduled to be
+ </del>
+ delivered to the
@@ -2083 +3184,7 @@
- in 2014 is named for him. It was christened February 1, 2014, in
+ <ins>
+  on March 12, 2014,
+ </ins>
+ <del>
+  in 2014
+ </del>
+ is named for him. It was christened February 1, 2014, in
@@ -2091 +3198,6 @@
- ’
+ <ins>
+  '
+ </ins>
+ <del>
+  ’
+ </del>
@@ -2110,3 +3222,5 @@
-</p>
-<p>
- He also received Honorary Doctorates from
+ He
+ <del>
+  also
+ </del>
+ received Honorary Doctorates from
@@ -2148,2 +3261,0 @@
-</p>
-<p>
@@ -2214,12 +3326,14 @@
- On January 3, 2015, with the death of
- <a href="/wiki/Edward_Brooke" title="Edward Brooke">
-  Edward Brooke
- </a>
- , Glenn became the
- <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
-  oldest former United States senator still living
- </a>
- . Glenn is the 60th person to hold this distinction.
-</p>
-<p>
- On September 12, 2016,
+ On
+ <del>
+  January 3, 2015, with the death of
+  <a href="/wiki/Edward_Brooke" title="Edward Brooke">
+   Edward Brooke
+  </a>
+  , Glenn became the
+  <a href="/wiki/List_of_longest-living_United_States_Senators" title="List of longest-living United States Senators">
+   oldest former United States senator still living
+  </a>
+  . Glenn is the 60th person to hold this distinction.
+On
+ </del>
+ September 12, 2016,
@@ -2236,3 +3350,5 @@
- <span class="mw-headline" id="Physical_description">
-  Physical description
- </span>
+ <del>
+  <span class="mw-headline" id="Physical_description">
+   Physical description
+  </span>
+ </del>
@@ -2242 +3358,3 @@
-  Weight: 170 lb (77 kg)
+  <del>
+   Weight: 170 lb (77 kg)
+  </del>
@@ -2245 +3363,3 @@
-  Height: 5 ft 10½ in (1.79 m)
+  <del>
+   Height: 5 ft 10½ in (1.79 m)
+  </del>
@@ -2248 +3368,3 @@
-  Hair: Red
+  <del>
+   Hair: Red
+  </del>
@@ -2251 +3373,3 @@
-  Eyes: Green
+  <del>
+   Eyes: Green
+  </del>
@@ -2322,0 +3447,2 @@
+<div style="clear:both;">
+</div>
\ No newline at end of file
