--- 1094270
+++ 1094303
@@ -2,4 +2,12 @@
- <b>
-  Katharine Hepburn
- </b>
- (born
+ <ins>
+  <b>
+   Katharine Hepburn
+  </b>
+  (
+ </ins>
+ <del>
+  <b>
+   Katharine Hepburn
+  </b>
+  (born
+ </del>
@@ -13 +21,15 @@
- ) is a notable
+ <ins>
+  -
+  <a href="/wiki/June_29" title="June 29">
+   June 29
+  </a>
+  ,
+  <a href="/wiki/2003" title="2003">
+   2003
+  </a>
+  ) was
+ </ins>
+ <del>
+  ) is
+ </del>
+ a notable
@@ -36 +58,8 @@
- and acted on
+ and
+ <ins>
+  debuted
+ </ins>
+ <del>
+  acted
+ </del>
+ on
@@ -40 +69,11 @@
- before her screen-test for
+ <ins>
+  that same year in
+  <i>
+   Night Hostess
+  </i>
+  . In 1932
+ </ins>
+ <del>
+  before
+ </del>
+ her screen-test for
@@ -486 +525 @@
-</ul>
+</ul>
\ No newline at end of file
