--- 1094270
+++ 1096680
@@ -1,0 +2,23 @@
+</p>
+<p>
+</p>
+<table align="right">
+ <tr>
+  <td>
+   <p>
+    <ins>
+     <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Lionwint.JPG" title="File:Lionwint.JPG">
+      File:Lionwint.JPG
+     </a>
+    </ins>
+   </p>
+  </td>
+ </tr>
+</table>
+<ins>
+ <b>
+  Katharine Hepburn
+ </b>
+ (
+</ins>
+<del>
@@ -6,3 +29,4 @@
- <a href="/wiki/May_12" title="May 12">
-  May 12
- </a>
+</del>
+<a href="/wiki/May_12" title="May 12">
+ May 12
+</a>
@@ -11 +35,6 @@
-  1907
+ 1907
+</a>
+<ins>
+ -
+ <a href="/wiki/June_29" title="June 29">
+  June 29
@@ -13,6 +42,3 @@
- ) is a notable
- <a href="/wiki/United_States" title="United States">
-  American
- </a>
- <a class="mw-redirect" href="/wiki/Actress" title="Actress">
-  actress
+ ,
+ <a href="/wiki/2003" title="2003">
+  2003
@@ -19,0 +46,12 @@
+ ) was
+</ins>
+<del>
+ ) is
+</del>
+ a notable
+ <a href="/wiki/United_States" title="United States">
+ American
+</a>
+<a class="mw-redirect" href="/wiki/Actress" title="Actress">
+ actress
+</a>
@@ -22,2 +60,2 @@
-  Hartford, Connecticut
- </a>
+ Hartford, Connecticut
+</a>
@@ -26,2 +64,2 @@
-  Bryn Mawr College
- </a>
+ Bryn Mawr College
+</a>
@@ -30,2 +68,2 @@
-  Pennsylvania
- </a>
+ Pennsylvania
+</a>
@@ -34,3 +72,9 @@
-  1928
- </a>
- and acted on
+ 1928
+</a>
+ and <ins>
+ debuted
+</ins>
+<del>
+ acted
+</del>
+ on
@@ -38,3 +82,13 @@
-  Broadway
- </a>
- before her screen-test for
+ Broadway
+</a>
+<ins>
+ that same year in
+ <i>
+  Night Hostess
+ </i>
+ . In 1932
+</ins>
+<del>
+ before
+</del>
+ her screen-test for
@@ -42,2 +96,2 @@
-  RKO
- </a>
+ RKO
+</a>
@@ -46,2 +100,2 @@
-  George Cukor
- </a>
+ George Cukor
+</a>
@@ -50,2 +104,2 @@
-  A Bill of Divorcement
- </i>
+ A Bill of Divorcement
+</i>
@@ -54,2 +108,2 @@
-  John Barrymore
- </a>
+ John Barrymore
+</a>
@@ -58,2 +112,2 @@
-  Academy Award
- </a>
+ Academy Award
+</a>
@@ -62,2 +116,2 @@
-  1933
- </a>
+ 1933
+</a>
@@ -66,2 +120,2 @@
-  Meryl Streep
- </a>
+ Meryl Streep
+</a>
@@ -69,2 +123 @@
-</p>
-<p>
+ <p>
@@ -92,0 +146,164 @@
+<p>
+ <ins>
+  Katharine Hepburn died at her home in
+  <a href="/wiki/Old_Saybrook,_Connecticut" title="Old Saybrook, Connecticut">
+   Old Saybrook, Connecticut
+  </a>
+  .
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Broadway">
+   Broadway
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   <i>
+    <a class="mw-disambig" href="/wiki/These_Days" title="These Days">
+     These Days
+    </a>
+   </i>
+   -
+   <a href="/wiki/1928" title="1928">
+    1928
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a class="new" href="/w/index.php?title=Art_and_Mrs._Bottle&amp;action=edit&amp;redlink=1" title="Art and Mrs. Bottle (page does not exist)">
+     Art and Mrs. Bottle
+    </a>
+   </i>
+   -
+   <a href="/wiki/1930" title="1930">
+    1930
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a href="/wiki/The_Warrior%27s_Husband" title="The Warrior's Husband">
+     The Warrior's Husband
+    </a>
+   </i>
+   -
+   <a href="/wiki/1932" title="1932">
+    1932
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a class="mw-disambig" href="/wiki/The_Lake" title="The Lake">
+     The Lake
+    </a>
+   </i>
+   -
+   <a href="/wiki/1933" title="1933">
+    1933
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a class="mw-disambig" href="/wiki/The_Philadelphia_Story" title="The Philadelphia Story">
+     The Philadelphia Story
+    </a>
+   </i>
+   -
+   <a href="/wiki/1939" title="1939">
+    1939
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a class="mw-disambig" href="/wiki/Without_Love" title="Without Love">
+     Without Love
+    </a>
+   </i>
+   -
+   <a href="/wiki/1942" title="1942">
+    1942
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a href="/wiki/As_You_Like_It" title="As You Like It">
+     As You Like It
+    </a>
+   </i>
+   -
+   <a href="/wiki/1950" title="1950">
+    1950
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a href="/wiki/The_Millionairess" title="The Millionairess">
+     The Millionairess
+    </a>
+   </i>
+   -
+   <a href="/wiki/1952" title="1952">
+    1952
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a class="mw-disambig" href="/wiki/Coco" title="Coco">
+     Coco
+    </a>
+   </i>
+   -
+   <a href="/wiki/1969" title="1969">
+    1969
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a href="/wiki/A_Matter_of_Gravity" title="A Matter of Gravity">
+     A Matter of Gravity
+    </a>
+   </i>
+   -
+   <a href="/wiki/1976" title="1976">
+    1976
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    <a href="/wiki/The_West_Side_Waltz" title="The West Side Waltz">
+     The West Side Waltz
+    </a>
+   </i>
+   -
+   <a href="/wiki/1981" title="1981">
+    1981
+   </a>
+  </ins>
+ </li>
+</ul>
+<p>
+ <br/>
+</p>
@@ -486 +703 @@
-</ul>
+</ul>
\ No newline at end of file
