--- 426997211
+++ 426998029
@@ -1572,0 +1573,8 @@
+<p>
+ <ins>
+  <b>
+   May 2011
+  </b>
+  Obama announced Osama Bin Laden has been killed.
+ </ins>
+</p>
@@ -1703 +1711,7 @@
- On May 2, 2011, the White House declared that bin Laden was dead.
+ On May
+ <ins>
+  1st, 2011, rumors have abounded in the wake of an impromptu presidential press conference that Bin Laden was caught and killed.
+ </ins>
+ <del>
+  2, 2011, the White House declared that bin Laden was dead.
+ </del>
@@ -1813,0 +1828,5 @@
+<p>
+ <ins>
+  i hate chohee
+ </ins>
+</p>
\ No newline at end of file
