--- 426997211
+++ 426997322
@@ -1572,0 +1573,8 @@
+<p>
+ <ins>
+  <b>
+   May 2011
+  </b>
+  : Reports of Bin Ladin's death have hit the news.
+ </ins>
+</p>
@@ -1749,0 +1758,14 @@
+<p>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  The White House reported bin Laden dead on May 1st, 2011.
+ </ins>
+</p>
@@ -1813 +1835 @@
-</div>
+</div>
\ No newline at end of file
