--- 426997211
+++ 427007992
@@ -19 +19,8 @@
- ; born March 10, 1957) is a member of the wealthy
+ ;
+ <ins>
+  (March 10, 1957 – May 1, 2011) was
+ </ins>
+ <del>
+  born March 10, 1957) is
+ </del>
+ a member of the wealthy
@@ -29,0 +37,5 @@
+ <ins>
+  <a class="mw-redirect" href="/wiki/Terrorist" title="Terrorist">
+   terrorist
+  </a>
+ </ins>
@@ -34 +46,8 @@
- , most widely recognized for the
+ ,
+ <ins>
+  responsible
+ </ins>
+ <del>
+  most widely recognized
+ </del>
+ for the
@@ -52 +71,8 @@
- Bin Laden is on the American
+ Bin Laden
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ on the American
@@ -71 +97,8 @@
- Since 2001, Osama bin Laden and his organization have been major targets of the U.S.
+ Since 2001, Osama bin Laden and his organization
+ <ins>
+  had
+ </ins>
+ <del>
+  have
+ </del>
+ been major targets of the U.S.
@@ -75 +108,8 @@
- . Bin Laden and fellow al-Qaeda leaders are believed to be hiding near the border of
+ . Bin Laden and fellow al-Qaeda leaders
+ <ins>
+  were
+ </ins>
+ <del>
+  are
+ </del>
+ believed to be hiding near the border of
@@ -89,0 +130,13 @@
+ <ins>
+  On May 1, 2011, U.S. President Barack Obama announced on national television that bin Laden had been killed in
+  <a href="/wiki/Abbottabad" title="Abbottabad">
+   Abbottabad
+  </a>
+  ,
+  <a href="/wiki/Pakistan" title="Pakistan">
+   Pakistan
+  </a>
+  and that his body was in U.S. custody.
+ </ins>
+</p>
+<p>
@@ -232 +285,6 @@
- .
+ <ins>
+  along with lifelong friends Anthony Kardous and Enos Fingy.
+ </ins>
+ <del>
+  .
+ </del>
@@ -262 +320,8 @@
- He also writes poetry.
+ He also
+ <ins>
+  wrote
+ </ins>
+ <del>
+  writes
+ </del>
+ poetry.
@@ -297 +362,8 @@
- Bin Laden believes that the restoration of
+ Bin Laden
+ <ins>
+  believed
+ </ins>
+ <del>
+  believes
+ </del>
+ that the restoration of
@@ -315 +387,8 @@
- He believes Afghanistan under the rule of
+ He
+ <ins>
+  believed
+ </ins>
+ <del>
+  believes
+ </del>
+ Afghanistan under the rule of
@@ -324 +403,5 @@
- Bin Laden has consistently dwelt on the need for violent jihad to right what he believes are injustices against Muslims perpetrated by the United States and sometimes by other non-Muslim states,
+ Bin Laden
+ <del>
+  has
+ </del>
+ consistently dwelt on the need for violent jihad to right what he believes are injustices against Muslims perpetrated by the United States and sometimes by other non-Muslim states,
@@ -329 +412,5 @@
- , and the necessity of forcing the US to withdraw from the Middle East. He has also called on Americans to "reject the immoral acts of
+ , and the necessity of forcing the US to withdraw from the Middle East. He
+ <del>
+  has
+ </del>
+ also called on Americans to "reject the immoral acts of
@@ -352,2 +439,15 @@
- Probably the most infamous part of Bin Laden's ideology is that civilians, including women and children, are legitimate targets of jihad.
- Bin Laden is
+ Probably the most infamous part of Bin Laden's ideology
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ that civilians, including women and children, are legitimate targets of jihad.
+ Bin Laden
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
@@ -357 +457,5 @@
- , and has delivered warnings against alleged Jewish conspiracies: "These Jews are masters of usury and leaders in treachery. They will leave you nothing, either in this world or the next."
+ , and
+ <del>
+  has
+ </del>
+ delivered warnings against alleged Jewish conspiracies: "These Jews are masters of usury and leaders in treachery. They will leave you nothing, either in this world or the next."
@@ -365,2 +469,16 @@
- bin Laden opposes music on religious grounds,
- and his attitude towards technology is mixed. He is interested in "earth-moving machinery and
+ bin Laden
+ <ins>
+  opposed
+ </ins>
+ <del>
+  opposes
+ </del>
+ music on religious grounds,
+ and his attitude towards technology
+ <ins>
+  was mixed. He was
+ </ins>
+ <del>
+  is mixed. He is
+ </del>
+ interested in "earth-moving machinery and
@@ -370,4 +488,25 @@
- of plants" on the one hand, but rejects "chilled water" on the other.
-</p>
-<p>
- His viewpoints and methods of achieving them have led to him been designated as a "
+ of plants" on the one hand, but
+ <ins>
+  rejected
+ </ins>
+ <del>
+  rejects
+ </del>
+ "chilled water" on the other.
+</p>
+<p>
+ His viewpoints and methods of achieving them
+ <ins>
+  had
+ </ins>
+ <del>
+  have
+ </del>
+ led to him
+ <ins>
+  being
+ </ins>
+ <del>
+  been
+ </del>
+ designated as a "
@@ -1095,0 +1235,5 @@
+<p>
+ <ins>
+  His death announcement by US President Barack Obama presented a codicil to both the attacks of 9/11/2001, and the ensuing "War on Terror" waged by the US government in response.
+ </ins>
+</p>
@@ -1211 +1355,8 @@
- Capturing Osama bin Laden has been an objective of the United States government since the presidency of
+ Capturing Osama bin Laden
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ been an objective of the United States government since the presidency of
@@ -1293 +1444,7 @@
- Bin laden escaped into the Tribal areas across the border into Pakistan.
+ <ins>
+  that Bin Laden
+ </ins>
+ <del>
+  Bin laden
+ </del>
+ escaped into the Tribal areas across the border into Pakistan.
@@ -1387,13 +1544,15 @@
- <span class="mw-headline" id="Reports_of_his_death">
-  Reports of his death
- </span>
-</h3>
-<p>
- <b>
-  December 2001
- </b>
- Quoting an unnamed Taliban official, the
- <i>
-  Pakistan Observer
- </i>
- reported that Bin Laden died of untreated lung complications and was buried in an unmarked grave in Tora Bora on December 15.
+ <span class="mw-headline" id="Reports_of_whereabouts">
+  Reports of
+  <del>
+   his death
+  </del>
+  <p>
+   <del>
+    <b>
+     December 2001
+    </b>
+    Quoting an unnamed Taliban official, the
+    <i>
+     Pakistan Observer
+    </i>
+    reported that Bin Laden died of untreated lung complications and was buried in an unmarked grave in Tora Bora on December 15.
@@ -1402,20 +1561,24 @@
- <a class="mw-redirect" href="/wiki/AlWafd_-_Daily" title="AlWafd - Daily">
-  AlWafd - Daily
- </a>
- carried a short
- <a href="/wiki/Obituary" title="Obituary">
-  obituary
- </a>
- by a
- <i>
-  prominent official
- </i>
- of the Afghan Taliban, who was allegedly present at the funeral, stating Bin Laden had been buried on or about December 13:
-</p>
-<blockquote>
- <p>
-  "(Osama bin Laden) suffered serious complications and died a natural, quiet death. He was buried in Tora Bora, a funeral attended by 30 Al Qaeda fighters, close members of his family and friends from the Taliban. By the Wahhabi tradition, no mark was left on the grave"
- </p>
-</blockquote>
-<p>
- A videotape was released on December 27 showing a gaunt, unwell Bin Laden, prompting an unnamed White House aide to comment that it could have been made shortly before his death.
+    <a class="mw-redirect" href="/wiki/AlWafd_-_Daily" title="AlWafd - Daily">
+     AlWafd - Daily
+    </a>
+    carried a short
+    <a href="/wiki/Obituary" title="Obituary">
+     obituary
+    </a>
+    by a
+    <i>
+     prominent official
+    </i>
+    of the Afghan Taliban, who was allegedly present at the funeral, stating Bin Laden had been buried on or about December 13:
+   </del>
+  </p>
+  <blockquote>
+   <p>
+    <del>
+     "(Osama bin Laden) suffered serious complications and died a natural, quiet death. He was buried in Tora Bora, a funeral attended by 30 Al Qaeda fighters, close members of his family and friends from the Taliban. By the Wahhabi tradition, no mark was left on the grave"
+    </del>
+   </p>
+  </blockquote>
+  <p>
+   <del>
+    A videotape was released on December 27 showing a gaunt, unwell Bin Laden, prompting an unnamed White House aide to comment that it could have been made shortly before his death.
@@ -1428,52 +1591,60 @@
-</p>
-<p>
- <b>
-  October 2002
- </b>
- : In a CNN interview, Afghan President Hamid Karzai stated that "I would come to believe that [bin Laden] probably is dead."
-</p>
-<p>
- <b>
-  April 2005
- </b>
- : The
- <i>
-  <a class="mw-redirect" href="/wiki/Sydney_Morning_Herald" title="Sydney Morning Herald">
-   Sydney Morning Herald
-  </a>
- </i>
- stated "Dr Clive Williams, director of terrorism studies at the
- <a href="/wiki/Australian_National_University" title="Australian National University">
-  Australian National University
- </a>
- , says documents provided by an Indian colleague suggested bin Laden died of massive organ failure in April last year ... 'It's hard to prove or disprove these things because there hasn't really been anything that allows you to make a judgment one way or the other,' Dr. Williams said."
-</p>
-<p>
- <b>
-  Late 2005
- </b>
- CIA disbands "
- <a href="/wiki/Bin_Laden_Issue_Station" title="Bin Laden Issue Station">
-  Bin Laden Issue Station
- </a>
- " codenamed "Alec Station", the CIA's bin Laden tracking unit, 1996–2005
-</p>
-<p>
- <b>
-  September 2006
- </b>
- : On September 23, 2006, the French newspaper
- <i>
-  <a href="/wiki/L%27Est_R%C3%A9publicain" title="L'Est Républicain">
-   L'Est Républicain
-  </a>
- </i>
- quoted a report from the French secret service (
- <a class="mw-redirect" href="/wiki/Direction_g%C3%A9n%C3%A9rale_de_la_s%C3%A9curit%C3%A9_ext%C3%A9rieure" title="Direction générale de la sécurité extérieure">
-  Direction générale de la sécurité extérieure
- </a>
- , DGSE) stating that Osama bin Laden had died in Pakistan on August 23, 2006, after contracting a case of
- <a href="/wiki/Typhoid_fever" title="Typhoid fever">
-  typhoid fever
- </a>
- that paralyzed his lower limbs.
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     October 2002
+    </b>
+    : In a CNN interview, Afghan President Hamid Karzai stated that "I would come to believe that [bin Laden] probably is dead."
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     April 2005
+    </b>
+    : The
+    <i>
+     <a class="mw-redirect" href="/wiki/Sydney_Morning_Herald" title="Sydney Morning Herald">
+      Sydney Morning Herald
+     </a>
+    </i>
+    stated "Dr Clive Williams, director of terrorism studies at the
+    <a href="/wiki/Australian_National_University" title="Australian National University">
+     Australian National University
+    </a>
+    , says documents provided by an Indian colleague suggested bin Laden died of massive organ failure in April last year ... 'It's hard to prove or disprove these things because there hasn't really been anything that allows you to make a judgment one way or the other,' Dr. Williams said."
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     Late 2005
+    </b>
+    CIA disbands "
+    <a href="/wiki/Bin_Laden_Issue_Station" title="Bin Laden Issue Station">
+     Bin Laden Issue Station
+    </a>
+    " codenamed "Alec Station", the CIA's bin Laden tracking unit, 1996–2005
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     September 2006
+    </b>
+    : On September 23, 2006, the French newspaper
+    <i>
+     <a href="/wiki/L%27Est_R%C3%A9publicain" title="L'Est Républicain">
+      L'Est Républicain
+     </a>
+    </i>
+    quoted a report from the French secret service (
+    <a class="mw-redirect" href="/wiki/Direction_g%C3%A9n%C3%A9rale_de_la_s%C3%A9curit%C3%A9_ext%C3%A9rieure" title="Direction générale de la sécurité extérieure">
+     Direction générale de la sécurité extérieure
+    </a>
+    , DGSE) stating that Osama bin Laden had died in Pakistan on August 23, 2006, after contracting a case of
+    <a href="/wiki/Typhoid_fever" title="Typhoid fever">
+     typhoid fever
+    </a>
+    that paralyzed his lower limbs.
@@ -1482,12 +1653,12 @@
- <a href="/wiki/Saudi_Arabia" title="Saudi Arabia">
-  Saudi Arabian
- </a>
- secret service to its government, which reported it to the French secret service. The French defense minister
- <a href="/wiki/Mich%C3%A8le_Alliot-Marie" title="Michèle Alliot-Marie">
-  Michèle Alliot-Marie
- </a>
- expressed her regret that the report had been published while French President
- <a href="/wiki/Jacques_Chirac" title="Jacques Chirac">
-  Jacques Chirac
- </a>
- declared that bin Laden's death had not been confirmed.
+    <a href="/wiki/Saudi_Arabia" title="Saudi Arabia">
+     Saudi Arabian
+    </a>
+    secret service to its government, which reported it to the French secret service. The French defense minister
+    <a href="/wiki/Mich%C3%A8le_Alliot-Marie" title="Michèle Alliot-Marie">
+     Michèle Alliot-Marie
+    </a>
+    expressed her regret that the report had been published while French President
+    <a href="/wiki/Jacques_Chirac" title="Jacques Chirac">
+     Jacques Chirac
+    </a>
+    declared that bin Laden's death had not been confirmed.
@@ -1496,4 +1667,4 @@
- <a href="/wiki/Condoleezza_Rice" title="Condoleezza Rice">
-  Condoleezza Rice
- </a>
- saying only, "No comment, and no knowledge."
+    <a href="/wiki/Condoleezza_Rice" title="Condoleezza Rice">
+     Condoleezza Rice
+    </a>
+    saying only, "No comment, and no knowledge."
@@ -1501,22 +1672,24 @@
- <a class="mw-redirect" href="/wiki/Water-borne_illness" title="Water-borne illness">
-  water-borne illness
- </a>
- , but that he had heard no reports that it was specifically typhoid or that he had died.
-</p>
-<p>
- <b>
-  November 2007
- </b>
- : In an interview with political interviewer
- <a href="/wiki/David_Frost" title="David Frost">
-  David Frost
- </a>
- taken on November 2, 2007, the Pakistani politician and Pakistan Peoples Party leader
- <a href="/wiki/Benazir_Bhutto" title="Benazir Bhutto">
-  Benazir Bhutto
- </a>
- claimed that bin Laden had been murdered by
- <a href="/wiki/Ahmed_Omar_Saeed_Sheikh" title="Ahmed Omar Saeed Sheikh">
-  Omar Sheikh
- </a>
- . During her answer to a question pertaining to the identities of those who had previously attempted her own assassination, Bhutto named Sheikh as a possible suspect while referring to him as "the man who murdered Osama bin Laden." Despite the weight of such a statement, neither Bhutto nor Frost attempted to clarify it during the remainder of the interview.
+    <a class="mw-redirect" href="/wiki/Water-borne_illness" title="Water-borne illness">
+     water-borne illness
+    </a>
+    , but that he had heard no reports that it was specifically typhoid or that he had died.
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     November 2007
+    </b>
+    : In an interview with political interviewer
+    <a href="/wiki/David_Frost" title="David Frost">
+     David Frost
+    </a>
+    taken on November 2, 2007, the Pakistani politician and Pakistan Peoples Party leader
+    <a href="/wiki/Benazir_Bhutto" title="Benazir Bhutto">
+     Benazir Bhutto
+    </a>
+    claimed that bin Laden had been murdered by
+    <a href="/wiki/Ahmed_Omar_Saeed_Sheikh" title="Ahmed Omar Saeed Sheikh">
+     Omar Sheikh
+    </a>
+    . During her answer to a question pertaining to the identities of those who had previously attempted her own assassination, Bhutto named Sheikh as a possible suspect while referring to him as "the man who murdered Osama bin Laden." Despite the weight of such a statement, neither Bhutto nor Frost attempted to clarify it during the remainder of the interview.
@@ -1524,4 +1697,4 @@
- <a href="/wiki/Daniel_Pearl" title="Daniel Pearl">
-  Daniel Pearl
- </a>
- and not Osama Bin Laden.
+    <a href="/wiki/Daniel_Pearl" title="Daniel Pearl">
+     Daniel Pearl
+    </a>
+    and not Osama Bin Laden.
@@ -1529,8 +1702,8 @@
- <a class="external text" href="http://www.bbc.co.uk/go/homepage/int/news/-/mediaselector/check/nolavconsole/ukfs_news/hi?redirect=fs.stm&amp;nbram=1&amp;bbram=1&amp;nbwm=1&amp;bbwm=1&amp;news=1&amp;nol_storyid=7075843" rel="nofollow">
-  edited version
- </a>
- with the
- <a class="external text" href="http://www.youtube.com/watch?v=oIO8B6fpFSQ" rel="nofollow">
-  complete interview
- </a>
- .
+    <a class="external text" href="http://www.bbc.co.uk/go/homepage/int/news/-/mediaselector/check/nolavconsole/ukfs_news/hi?redirect=fs.stm&amp;nbram=1&amp;bbram=1&amp;nbwm=1&amp;bbwm=1&amp;news=1&amp;nol_storyid=7075843" rel="nofollow">
+     edited version
+    </a>
+    with the
+    <a class="external text" href="http://www.youtube.com/watch?v=oIO8B6fpFSQ" rel="nofollow">
+     complete interview
+    </a>
+    .
@@ -1538,38 +1711,46 @@
-</p>
-<p>
- <b>
-  March 2009
- </b>
- : In an essay published in
- <i>
-  <a href="/wiki/The_American_Spectator" title="The American Spectator">
-   The American Spectator
-  </a>
- </i>
- in March 2009,
- <a href="/wiki/International_relations" title="International relations">
-  international relations
- </a>
- professor Angelo Codevilla of Boston University argued that Osama bin Laden had been dead for many years.
-</p>
-<p>
- <b>
-  April 2009
- </b>
- : During an interview with the Telegraph, Pakistan's President Asif Ali Zardari raised the prospect that Osama bin Laden could be dead after he said that intelligence officials could find "no trace" of the al-Qaeda chief. Mr Zardari's predecessor, Pervez Musharraf, similarly suggested that the Saudi terror chief could be dead. Additionally, Pakistan's intelligence agencies also believe Osama bin Laden may be dead.
-</p>
-<p>
- <b>
-  October 2009
- </b>
- : An article in the British tabloid
- <i>
-  <a href="/wiki/Daily_Mail" title="Daily Mail">
-   Daily Mail
-  </a>
- </i>
- points out that the theory that Bin Laden died in 2001 "is gaining credence among political commentators, respected academics and even terror experts" and notes that the mounting evidence that supports the claim makes the theory "worthy of examination".
-</p>
-<h3>
- <span class="mw-headline" id="Reports_of_his_current_whereabouts">
-  Reports of his current whereabouts
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     March 2009
+    </b>
+    : In an essay published in
+    <i>
+     <a href="/wiki/The_American_Spectator" title="The American Spectator">
+      The American Spectator
+     </a>
+    </i>
+    in March 2009,
+    <a href="/wiki/International_relations" title="International relations">
+     international relations
+    </a>
+    professor Angelo Codevilla of Boston University argued that Osama bin Laden had been dead for many years.
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     April 2009
+    </b>
+    : During an interview with the Telegraph, Pakistan's President Asif Ali Zardari raised the prospect that Osama bin Laden could be dead after he said that intelligence officials could find "no trace" of the al-Qaeda chief. Mr Zardari's predecessor, Pervez Musharraf, similarly suggested that the Saudi terror chief could be dead. Additionally, Pakistan's intelligence agencies also believe Osama bin Laden may be dead.
+   </del>
+  </p>
+  <p>
+   <del>
+    <b>
+     October 2009
+    </b>
+    : An article in the British tabloid
+    <i>
+     <a href="/wiki/Daily_Mail" title="Daily Mail">
+      Daily Mail
+     </a>
+    </i>
+    points out that the theory that Bin Laden died in 2001 "is gaining credence among political commentators, respected academics and even terror experts" and notes that the mounting evidence that supports the claim makes the theory "worthy of examination".
+   </del>
+  </p>
+  <del>
+   Reports of his current
+  </del>
+  whereabouts
@@ -1585 +1766,39 @@
- Claims as to the location of Osama bin Laden have been made since December 2001, although none have been definitively proven and some have placed Osama in different locations during overlapping time periods. Since a major military offensive in Afghanistan in the wake of the 2001 al Qaeda attacks in the United States failed to uncover his whereabouts, Pakistan had regularly been identified as his suspected hiding place.
+ <ins>
+  Many claims
+ </ins>
+ <del>
+  Claims
+ </del>
+ as to the location of Osama bin Laden
+ <ins>
+  were made in the wake of 9/11, although none were ever
+ </ins>
+ <del>
+  have been made since December 2001, although none have been
+ </del>
+ definitively proven and some
+ <del>
+  have
+ </del>
+ placed Osama in different locations during overlapping time periods.
+ <ins>
+  After military offensives
+ </ins>
+ <del>
+  Since a major military offensive
+ </del>
+ in Afghanistan in the wake of
+ <ins>
+  9/11
+ </ins>
+ <del>
+  the 2001 al Qaeda attacks in the United States
+ </del>
+ failed to uncover his whereabouts, Pakistan
+ <ins>
+  was regularly
+ </ins>
+ <del>
+  had regularly been
+ </del>
+ identified as his suspected hiding place.
@@ -1608 +1827 @@
- , "Atiyah" instructs Zarqawi to "send messengers from your end to Waziristan so that they meet with the brothers of the leadership ... I am now on a visit to them and I am writing you this letter as I am with them..." Al-Rahman also indicates that bin Laden and al-Qaeda are "weak" and "have many of their own problems." The letter has been deemed authentic by military and counterterrorism officials, according to
+ , "Atiyah" instructs Zarqawi to "send messengers from your end to Waziristan so that they meet with the brothers of the leadership ... I am now on a visit to them and I am writing you this letter as I am with them..." Al-Rahman also indicates that bin Laden and al-Qaeda are "weak" and "have many of their own problems." The letter has been deemed authentic by military and counterterrorism officials, according to the
@@ -1610 +1829,3 @@
-  the Washington Post
+  <a class="mw-redirect" href="/wiki/Washington_Post" title="Washington Post">
+   Washington Post
+  </a>
@@ -1627 +1848,7 @@
- as likely hideouts of Al-Qaeda leader Osama bin Laden.
+ as
+ <ins>
+  bin Laden's likely hideouts.
+ </ins>
+ <del>
+  likely hideouts of Al-Qaeda leader Osama bin Laden.
+ </del>
@@ -1651 +1878,5 @@
- In the first week of December 2009, a Taliban detainee in Pakistan said he had information that Bin Laden was in Afghanistan in 2009. The detainee said that in January or February (of 2009) he met a trusted contact who had seen Bin Laden about 15 to 20 days earlier in Afghanistan. But, the US has had no reliable information on the whereabouts of Bin Laden in years, US Defense Secretary Robert Gates admitted on December 6, 2009.
+ In the first week of December 2009, a Taliban detainee in Pakistan said he had information that Bin Laden was in Afghanistan in 2009. The detainee said that in January or February (of 2009) he met a trusted contact who had seen Bin Laden about 15 to 20 days earlier in Afghanistan. But, the US has had no reliable information on the whereabouts of Bin Laden in years, US Defense Secretary
+ <a href="/wiki/Robert_Gates" title="Robert Gates">
+  Robert Gates
+ </a>
+ admitted on December 6, 2009.
@@ -1668,0 +1900,3 @@
+ <ins>
+  ,
+ </ins>
@@ -1674,3 +1908,13 @@
- <i>
-  stupefied by the FBI's decision to use his photograph to compose its latest image of al Qaeda leader Osama bin Laden
- </i>
+ <ins>
+  "stupefied
+ </ins>
+ <del>
+  stupefied
+ </del>
+ by the FBI's decision to use his photograph to compose its latest image of al Qaeda leader Osama bin
+ <ins>
+  Laden"
+ </ins>
+ <del>
+  Laden
+ </del>
@@ -1700 +1944,117 @@
- On October 18, 2010, an unnamed NATO official suggested that bin Laden is "alive and well and living comfortably" in Pakistan, protected by elements of the country's intelligence services. A senior Pakistani official denied the allegations and said they were designed to put pressure on the Pakistani government ahead of talks aimed at strengthening ties between Pakistan and the United States.
+ On October 18, 2010, an unnamed NATO official suggested that bin Laden
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ "alive and well and living comfortably" in Pakistan, protected by elements of the country's intelligence services. A senior Pakistani official denied the allegations and said
+ <ins>
+  the accusations
+ </ins>
+ <del>
+  they
+ </del>
+ were designed to put pressure on the Pakistani government ahead of talks aimed at strengthening ties between Pakistan and the United States.
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<div class="hatnote navigation-not-searchable" role="note">
+ <ins>
+  Main article:
+  <a href="/wiki/Death_of_Osama_bin_Laden" title="Death of Osama bin Laden">
+   Death of Osama bin Laden
+  </a>
+ </ins>
+</div>
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     <b>
+      This section documents a
+      <a href="/wiki/Portal:Current_events" title="Portal:Current events">
+       current event
+      </a>
+      .
+     </b>
+     Information may change rapidly as the event progresses, and initial news reports may be
+     <a href="/wiki/Wikipedia:Identifying_reliable_sources#Breaking_news" title="Wikipedia:Identifying reliable sources">
+      unreliable
+     </a>
+     . The
+     <a class="external text" href="//en.wikipedia.org/w/index.php?title=Osama_bin_Laden&amp;action=history">
+      last updates
+     </a>
+     to this section
+     <a href="/wiki/Wikipedia:Risk_disclaimer" title="Wikipedia:Risk disclaimer">
+      may not reflect
+     </a>
+     the most current information.
+     <small class="hide-when-compact">
+      <i>
+       (
+       <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
+        Learn how and when to remove this template message
+       </a>
+       )
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    Wikinews has related news:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikinews.org/wiki/Osama_bin_Laden_dead,_report_US_officials" title="wikinews:Osama bin Laden dead, report US officials">
+       Osama bin Laden dead, report US officials
+      </a>
+     </b>
+    </i>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  On May 1, 2011, President Obama announced that bin Laden was killed by "a small team of Americans" acting under his direct orders, in a covert operation in
+  <a href="/wiki/Abbottabad" title="Abbottabad">
+   Abbottabad
+  </a>
+  , Pakistan, 150km north of Islamabad.
+ , affirming earlier confirmation by US officials to the media.
+ DNA from bin Laden's body, compared with DNA samples on record from his dead sister, confirmed bin Laden's identity.
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+  The body was recovered by the US military and is currently in its custody.
+ </ins>
+</p>
+<p>
+ <ins>
+  President Obama said that the operation was conducted with assistance from the Pakistani government.
+ </ins>
@@ -1703 +2063,3 @@
- On May 2, 2011, the White House declared that bin Laden was dead.
+ <del>
+  On May 2, 2011, the White House declared that bin Laden was dead.
+ </del>
@@ -1748 +2110,8 @@
- claimed that bin Laden is a spy employed by the United States.
+ claimed that bin Laden
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a spy employed by the United States.
@@ -1755 +2124 @@
-<div class="div-col columns column-width" style="-moz-column-width: 20em; -webkit-column-width: 20em; column-width: 20em;">
+<div class="div-col columns column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em;">
@@ -1813 +2182 @@
-</div>
+</div>
\ No newline at end of file
