--- 426997211
+++ 427000424
@@ -19 +19,8 @@
- ; born March 10, 1957) is a member of the wealthy
+ ;
+ <ins>
+  March 10, 1957 – 2011) was
+ </ins>
+ <del>
+  born March 10, 1957) is
+ </del>
+ a member of the wealthy
@@ -49,0 +57,4 @@
+ <ins>
+  On May 1st 2011, U.S. officials announced that bin Laden was dead
+ and that the United States was in possession of his corpse.
+ </ins>
@@ -71 +82,8 @@
- Since 2001, Osama bin Laden and his organization have been major targets of the U.S.
+ Since 2001, Osama bin Laden and his organization
+ <ins>
+  had
+ </ins>
+ <del>
+  have
+ </del>
+ been major targets of the U.S.
@@ -75 +93,8 @@
- . Bin Laden and fellow al-Qaeda leaders are believed to be hiding near the border of
+ . Bin Laden and fellow al-Qaeda leaders
+ <ins>
+  were
+ </ins>
+ <del>
+  are
+ </del>
+ believed to be hiding near the border of
@@ -1390,0 +1416,8 @@
+<div class="hatnote navigation-not-searchable" role="note">
+ <ins>
+  Main article:
+  <a href="/wiki/Death_of_Osama_bin_Laden" title="Death of Osama bin Laden">
+   Death of Osama bin Laden
+  </a>
+ </ins>
+</div>
@@ -1572,0 +1606,13 @@
+<p>
+ <ins>
+  <b>
+   May 2011
+  </b>
+  <i>
+   <a href="/wiki/The_New_York_Times" title="The New York Times">
+    The New York Times
+   </a>
+  </i>
+  announced Osama Bin Laden has been killed.
+ </ins>
+</p>
@@ -1703 +1749,4 @@
- On May 2, 2011, the White House declared that bin Laden was dead.
+ On May
+ <ins>
+  1st, 2011, rumors abounded in the wake of an impromptu presidential press conference that Bin Laden was caught and killed.
+ </ins>
@@ -1705,0 +1755,25 @@
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  In April 2011, it was reported that bin Laden was killed during a US military action.
+ DNA from bin Laden's body, compared with DNA samples on record, confirmed bin Laden's identity. The body was recovered by the US military and is currently in its custody.
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
+</p>
+<del>
+ 2, 2011, the White House declared that bin Laden was dead.
+</del>
+<h2>
@@ -1813 +1887 @@
-</div>
+</div>
\ No newline at end of file
