--- 10484392
+++ 10485453
@@ -5,4 +5,9 @@
- (born
- <a href="/wiki/Louisville,_Kentucky" title="Louisville, Kentucky">
-  Louisville, Kentucky
- </a>
+ <ins>
+  (
+ </ins>
+ <del>
+  (born
+  <a href="/wiki/Louisville,_Kentucky" title="Louisville, Kentucky">
+   Louisville, Kentucky
+  </a>
+ </del>
@@ -16 +21,15 @@
- ) is an
+ <ins>
+  –
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  ) was
+ </ins>
+ <del>
+  ) is
+ </del>
+ an
@@ -45 +64,8 @@
- ” to comically spin outlandish tales that are completely unbelievable, yet providing a unique viewpoint to accurately describe the underling reality at hand. Thompson almost always writes in the
+ ” to comically spin outlandish tales that are completely unbelievable, yet providing a unique viewpoint to accurately describe the
+ <ins>
+  underlying
+ </ins>
+ <del>
+  underling
+ </del>
+ reality at hand. Thompson almost always writes in the
@@ -49 +75,8 @@
- narrative, and his stories become so colorfully contrived that they easily slip into the realm of fiction, however the basic framework of the story he is telling is often very true. Thompson’s writing style exploits this void between the realms of fiction and non-fiction, and it has become known as
+ narrative, and his stories become so colorfully contrived that they easily slip into the realm of fiction, however the basic framework of the story he is telling is
+ <ins>
+  very often
+ </ins>
+ <del>
+  often very
+ </del>
+ true. Thompson’s writing style exploits this void between the realms of fiction and non-fiction, and it has become known as
@@ -74,3 +107,5 @@
- <span class="mw-headline" id="Early_Years">
-  Early Years
- </span>
+ <ins>
+  <span class="mw-headline" id="Middle_Years">
+   Middle Years
+  </span>
+ </ins>
@@ -79,37 +113,0 @@
- Thompson began his career in
- <a href="/wiki/1956" title="1956">
-  1956
- </a>
- as a
- <a class="mw-redirect" href="/wiki/Sports" title="Sports">
-  sports
- </a>
- journalist, writing for the base paper at
- <a href="/wiki/Eglin_Air_Force_Base" title="Eglin Air Force Base">
-  Eglin Air Force Base
- </a>
- in
- <a class="mw-redirect" href="/wiki/Fort_Walton_Beach" title="Fort Walton Beach">
-  Fort Walton Beach
- </a>
- ,
- <a href="/wiki/Florida" title="Florida">
-  Florida
- </a>
- , where he was enlisted. After the Air Force, Thompson worked briefly as a
- <a href="/wiki/Copy_editing" title="Copy editing">
-  copyboy
- </a>
- for
- <a class="mw-redirect" href="/wiki/Time_Magazine" title="Time Magazine">
-  Time Magazine
- </a>
- while living a
- <a class="mw-disambig" href="/wiki/Beat" title="Beat">
-  beat
- </a>
- -inspired lifestyle in
- <a href="/wiki/New_York_City" title="New York City">
-  New York City
- </a>
- .
@@ -118,28 +116,13 @@
- Thompson began vacationing extensively in the
- <a class="mw-redirect" href="/wiki/Caribbean_Islands" title="Caribbean Islands">
-  Caribbean Islands
- </a>
- and
- <a href="/wiki/South_America" title="South America">
-  South America
- </a>
- , using his unique locale to
- <a href="/wiki/Freelancer" title="Freelancer">
-  freelance
- </a>
- articles to a number of daily newspapers stateside. He befriended the journalist
- <a href="/wiki/William_Kennedy_(author)" title="William Kennedy (author)">
-  William Kennedy
- </a>
- while in
- <a href="/wiki/Puerto_Rico" title="Puerto Rico">
-  Puerto Rico
- </a>
- , who would later win the
- <a href="/wiki/Pulitzer_Prize" title="Pulitzer Prize">
-  Pulitzer Prize
- </a>
- for his novel
- <i>
-  <a class="mw-disambig" href="/wiki/Ironweed" title="Ironweed">
-   Ironweed
+</p>
+<h2>
+ <del>
+  <span class="mw-headline" id="Early_Years">
+   Early Years
+  </span>
+ </del>
+</h2>
+<p>
+ <del>
+  Thompson began his career in
+  <a href="/wiki/1956" title="1956">
+   1956
@@ -147,9 +130,3 @@
- </i>
- . Thompson eventually became a somewhat celebrated South American correspondent for a
- <a class="mw-redirect" href="/wiki/Dow_Jones" title="Dow Jones">
-  Dow Jones
- </a>
- owned weekly magazine called
- <i>
-  <a href="/wiki/The_Observer" title="The Observer">
-   The Observer
+  as a
+  <a class="mw-redirect" href="/wiki/Sports" title="Sports">
+   sports
@@ -157,2 +134,75 @@
- </i>
- .
+  journalist, writing for the base paper at
+  <a href="/wiki/Eglin_Air_Force_Base" title="Eglin Air Force Base">
+   Eglin Air Force Base
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/Fort_Walton_Beach" title="Fort Walton Beach">
+   Fort Walton Beach
+  </a>
+  ,
+  <a href="/wiki/Florida" title="Florida">
+   Florida
+  </a>
+  , where he was enlisted. After the Air Force, Thompson worked briefly as a
+  <a href="/wiki/Copy_editing" title="Copy editing">
+   copyboy
+  </a>
+  for
+  <a class="mw-redirect" href="/wiki/Time_Magazine" title="Time Magazine">
+   Time Magazine
+  </a>
+  while living a
+  <a class="mw-disambig" href="/wiki/Beat" title="Beat">
+   beat
+  </a>
+  -inspired lifestyle in
+  <a href="/wiki/New_York_City" title="New York City">
+   New York City
+  </a>
+  .
+ </del>
+</p>
+<p>
+ <del>
+  Thompson began vacationing extensively in the
+  <a class="mw-redirect" href="/wiki/Caribbean_Islands" title="Caribbean Islands">
+   Caribbean Islands
+  </a>
+  and
+  <a href="/wiki/South_America" title="South America">
+   South America
+  </a>
+  , using his unique locale to
+  <a href="/wiki/Freelancer" title="Freelancer">
+   freelance
+  </a>
+  articles to a number of daily newspapers stateside. He befriended the journalist
+  <a href="/wiki/William_Kennedy_(author)" title="William Kennedy (author)">
+   William Kennedy
+  </a>
+  while in
+  <a href="/wiki/Puerto_Rico" title="Puerto Rico">
+   Puerto Rico
+  </a>
+  , who would later win the
+  <a href="/wiki/Pulitzer_Prize" title="Pulitzer Prize">
+   Pulitzer Prize
+  </a>
+  for his novel
+  <i>
+   <a class="mw-disambig" href="/wiki/Ironweed" title="Ironweed">
+    Ironweed
+   </a>
+  </i>
+  . Thompson eventually became a somewhat celebrated South American correspondent for a
+  <a class="mw-redirect" href="/wiki/Dow_Jones" title="Dow Jones">
+   Dow Jones
+  </a>
+  owned weekly magazine called
+  <i>
+   <a href="/wiki/The_Observer" title="The Observer">
+    The Observer
+   </a>
+  </i>
+  .
+ </del>
@@ -162,15 +212,3 @@
-  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Gonzo_quote.PNG" title="File:Gonzo quote.PNG">
-   File:Gonzo quote.PNG
-  </a>
-  <div class="thumbcaption">
-   A modification of one of Thompson's original
-   <a class="mw-disambig" href="/wiki/Gonzo" title="Gonzo">
-    Gonzo
-   </a>
-   flyers during his bid for
-   <a class="mw-redirect mw-disambig" href="/wiki/Sherriff" title="Sherriff">
-    sherriff
-   </a>
-   of
-   <a href="/wiki/Aspen,_Colorado" title="Aspen, Colorado">
-    Aspen, Colorado
+  <del>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Gonzo_quote.PNG" title="File:Gonzo quote.PNG">
+    File:Gonzo quote.PNG
@@ -178 +216,17 @@
-   .
+  </del>
+  <div class="thumbcaption">
+   <del>
+    A modification of one of Thompson's original
+    <a class="mw-disambig" href="/wiki/Gonzo" title="Gonzo">
+     Gonzo
+    </a>
+    flyers during his bid for
+    <a class="mw-redirect mw-disambig" href="/wiki/Sherriff" title="Sherriff">
+     sherriff
+    </a>
+    of
+    <a href="/wiki/Aspen,_Colorado" title="Aspen, Colorado">
+     Aspen, Colorado
+    </a>
+    .
+   </del>
@@ -183,10 +237,16 @@
- In the meantime, Thompson wrote two serious novels (
- <i>
-  <a href="/wiki/Prince_Jellyfish" title="Prince Jellyfish">
-   Prince Jellyfish
-  </a>
- </i>
- and
- <i>
-  <a class="mw-redirect" href="/wiki/The_Rum_Diary" title="The Rum Diary">
-   The Rum Diary
+ <del>
+  In the meantime, Thompson wrote two serious novels (
+  <i>
+   <a href="/wiki/Prince_Jellyfish" title="Prince Jellyfish">
+    Prince Jellyfish
+   </a>
+  </i>
+  and
+  <i>
+   <a class="mw-redirect" href="/wiki/The_Rum_Diary" title="The Rum Diary">
+    The Rum Diary
+   </a>
+  </i>
+  ) and copious amounts of
+  <a class="mw-redirect" href="/wiki/Short_stories" title="Short stories">
+   short stories
@@ -194,6 +254,2 @@
- </i>
- ) and copious amounts of
- <a class="mw-redirect" href="/wiki/Short_stories" title="Short stories">
-  short stories
- </a>
- , none of which were published despite relentless submissions to publishers. Kennedy later remarked that he and Thompson were cohorts in the fact that they were, at the time, failed novelists who had turned to journalism in order to make a living.
+  , none of which were published despite relentless submissions to publishers. Kennedy later remarked that he and Thompson were cohorts in the fact that they were, at the time, failed novelists who had turned to journalism in order to make a living.
+ </del>
@@ -202,4 +258,10 @@
- Thompson got his big break when he pitched a story to
- <i>
-  <a href="/wiki/Harper%27s_Magazine" title="Harper's Magazine">
-   Harper's Magazine
+ <del>
+  Thompson got his big break when he pitched a story to
+  <i>
+   <a href="/wiki/Harper%27s_Magazine" title="Harper's Magazine">
+    Harper's Magazine
+   </a>
+  </i>
+  about his relationship with the
+  <a class="mw-redirect" href="/wiki/Hell%27s_Angels" title="Hell's Angels">
+   Hell's Angels
@@ -207,6 +269,2 @@
- </i>
- about his relationship with the
- <a class="mw-redirect" href="/wiki/Hell%27s_Angels" title="Hell's Angels">
-  Hell's Angels
- </a>
- motorcycle gang, known erroneously at the time for exploits in murder and gang-rape. After the article was published, numerous book offers on the subject were awarded his way.
+  motorcycle gang, known erroneously at the time for exploits in murder and gang-rape. After the article was published, numerous book offers on the subject were awarded his way.
+ </del>
@@ -215,3 +273,5 @@
- <span class="mw-headline" id="Middle_Years_to_Present">
-  Middle Years to Present
- </span>
+ <del>
+  <span class="mw-headline" id="Middle_Years_to_Present">
+   Middle Years to Present
+  </span>
+ </del>
@@ -325 +385,8 @@
- , prompting Thompson to shave his head bald and refer to his opposition as "my long haired opponent."
+ , prompting Thompson to shave his head bald and refer to his opposition as "my
+ <ins>
+  long-haired
+ </ins>
+ <del>
+  long haired
+ </del>
+ opponent."
@@ -343 +410,8 @@
- He has also been quoted as saying "I hate to advocate drugs, alcohol, violence or insanity to anyone, but they've always worked for me."
+ He has also been quoted as
+ <ins>
+  saying,
+ </ins>
+ <del>
+  saying
+ </del>
+ "I hate to advocate drugs, alcohol, violence or insanity to anyone, but they've always worked for me."
@@ -346,27 +420,5 @@
- His latest book,
- <i>
-  Kingdom of Fear
- </i>
- , is an angry commentary on the passing of the
- <a href="/wiki/American_Century" title="American Century">
-  American Century
- </a>
- . Thompson currently writes a Web column, "Hey Rube," for
- <a href="/wiki/ESPN" title="ESPN">
-  ESPN
- </a>
- . He has at times also toured on the lecture circuit, once with
- <a href="/wiki/John_Belushi" title="John Belushi">
-  John Belushi
- </a>
- . Another modern appearance is through
- <a href="/wiki/Cartoonist" title="Cartoonist">
-  cartoonist
- </a>
- <a href="/wiki/Garry_Trudeau" title="Garry Trudeau">
-  Garry Trudeau
- </a>
- , who based his
- <i>
-  <a href="/wiki/Doonesbury" title="Doonesbury">
-   Doonesbury
+</p>
+<div class="floatright">
+ <ins>
+  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Hunter_s_thompson.jpg" title="File:Hunter s thompson.jpg">
+   Hunter S. Thompson
@@ -374,13 +426,53 @@
- </i>
- character
- <a href="/wiki/Uncle_Duke" title="Uncle Duke">
-  Uncle Duke
- </a>
- on Thompson, to loud protests from Thompson himself. Similarly, Spider Jerusalem, the gonzo journalist protagonist of
- <a href="/wiki/Warren_Ellis" title="Warren Ellis">
-  Warren Ellis
- </a>
- 's
- <i>
-  <a href="/wiki/Transmetropolitan" title="Transmetropolitan">
-   Transmetropolitan
+ </ins>
+</div>
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     <b>
+      This article documents a
+      <a href="/wiki/Portal:Current_events" title="Portal:Current events">
+       current event
+      </a>
+      .
+     </b>
+     Information may change rapidly as the event progresses, and initial news reports may be
+     <a href="/wiki/Wikipedia:Identifying_reliable_sources#Breaking_news" title="Wikipedia:Identifying reliable sources">
+      unreliable
+     </a>
+     . The
+     <a class="external text" href="//en.wikipedia.org/w/index.php?title=Hunter_S._Thompson&amp;action=history">
+      last updates
+     </a>
+     to this article
+     <a href="/wiki/Wikipedia:Risk_disclaimer" title="Wikipedia:Risk disclaimer">
+      may not reflect
+     </a>
+     the most current information.
+     <small class="hide-when-compact">
+      <i>
+       (
+       <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
+        Learn how and when to remove this template message
+       </a>
+       )
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  <b>
+   Hunter Stockton Thompson
+  </b>
+  (
+  <a href="/wiki/July_18" title="July 18">
+   July 18
@@ -389,2 +481,24 @@
- </i>
- is (more lovingly) based on Thompson.
+  <a href="/wiki/1937" title="1937">
+   1937
+  </a>
+  –
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  ) was an
+  <a href="/wiki/United_States" title="United States">
+   American
+  </a>
+  <a href="/wiki/Journalist" title="Journalist">
+   journalist
+  </a>
+  and
+  <a href="/wiki/Author" title="Author">
+   author
+  </a>
+  .
+ </ins>
@@ -393,9 +507,11 @@
- Thompson is an admitted fan of
- <a class="mw-redirect" href="/wiki/Firearms" title="Firearms">
-  firearms
- </a>
- and is known to keep a keg of
- <a href="/wiki/Gunpowder" title="Gunpowder">
-  gunpowder
- </a>
- in his basement.
+ <ins>
+  Hunter S. Thompson is often portrayed in popular culture as a dangerously absurd,
+  <a class="mw-redirect" href="/wiki/Illegal_drugs" title="Illegal drugs">
+   drug
+  </a>
+  -crazed journalist bent on comic self-destruction. While this portrayal is certainly not completely inaccurate, he is also considered by many to be one of the most important American writers of the
+  <a href="/wiki/20th_century" title="20th century">
+   20th century
+  </a>
+  .
+ </ins>
@@ -403,5 +518,0 @@
-<h2>
- <span class="mw-headline" id="Movies">
-  Movies
- </span>
-</h2>
@@ -409,17 +520,29 @@
- The film
- <i>
-  Where The Buffalo Roam
- </i>
- (
- <a href="/wiki/1980_in_film" title="1980 in film">
-  1980
- </a>
- ) depicts Thompson's attempts at writing stories for both the Super Bowl and the 1972 Presidential Election. It stars
- <a href="/wiki/Bill_Murray" title="Bill Murray">
-  Bill Murray
- </a>
- as Thompson and
- <a href="/wiki/Peter_Boyle" title="Peter Boyle">
-  Peter Boyle
- </a>
- as Thompson's attorney Oscar Acosta, refered to in the movie as Carl Laslow, Esq.
+ <ins>
+  Thompson is known for an explosive writing style that employs what he calls “
+  <a class="mw-redirect" href="/wiki/Action_verbs" title="Action verbs">
+   action verbs
+  </a>
+  ” to comically spin outlandish tales that are completely unbelievable, yet providing a unique viewpoint to accurately describe the underlying reality at hand. Thompson almost always writes in the
+  <a class="mw-disambig" href="/wiki/First_person" title="First person">
+   first person
+  </a>
+  narrative, and his stories become so colorfully contrived that they easily slip into the realm of fiction, however the basic framework of the story he is telling is very often true. Thompson’s writing style exploits this void between the realms of fiction and non-fiction, and it has become known as
+  <a class="mw-redirect" href="/wiki/Gonzo_Journalism" title="Gonzo Journalism">
+   Gonzo Journalism
+  </a>
+  . Other notable
+  <a href="/wiki/Gonzo_journalism" title="Gonzo journalism">
+   gonzo journalists
+  </a>
+  include
+  <a class="new" href="/w/index.php?title=Timothy_Edward_Jones&amp;action=edit&amp;redlink=1" title="Timothy Edward Jones (page does not exist)">
+   Timothy Edward Jones
+  </a>
+  , author of the new cult classic,
+  <i>
+   <a class="new" href="/w/index.php?title=European_Confession&amp;action=edit&amp;redlink=1" title="European Confession (page does not exist)">
+    European Confession
+   </a>
+   .
+  </i>
+ </ins>
@@ -426,0 +550,50 @@
+<div class="floatright">
+ <ins>
+  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Hunter_s_thompson.jpg" title="File:Hunter s thompson.jpg">
+   Hunter S. Thompson
+  </a>
+ </ins>
+</div>
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     <b>
+      This article documents a
+      <a href="/wiki/Portal:Current_events" title="Portal:Current events">
+       current event
+      </a>
+      .
+     </b>
+     Information may change rapidly as the event progresses, and initial news reports may be
+     <a href="/wiki/Wikipedia:Identifying_reliable_sources#Breaking_news" title="Wikipedia:Identifying reliable sources">
+      unreliable
+     </a>
+     . The
+     <a class="external text" href="//en.wikipedia.org/w/index.php?title=Hunter_S._Thompson&amp;action=history">
+      last updates
+     </a>
+     to this article
+     <a href="/wiki/Wikipedia:Risk_disclaimer" title="Wikipedia:Risk disclaimer">
+      may not reflect
+     </a>
+     the most current information.
+     <small class="hide-when-compact">
+      <i>
+       (
+       <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
+        Learn how and when to remove this template message
+       </a>
+       )
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
@@ -428,4 +601,7 @@
- The film
- <i>
-  <a href="/wiki/Fear_and_Loathing_in_Las_Vegas" title="Fear and Loathing in Las Vegas">
-   Fear and Loathing in Las Vegas
+ <ins>
+  <b>
+   Hunter Stockton Thompson
+  </b>
+  (
+  <a href="/wiki/July_18" title="July 18">
+   July 18
@@ -433,27 +609,25 @@
-  : A Savage Journey to the Heart of the American Dream
- </i>
- , based on the book was released in
- <a href="/wiki/1998_in_film" title="1998 in film">
-  1998
- </a>
- , starring
- <a href="/wiki/Johnny_Depp" title="Johnny Depp">
-  Johnny Depp
- </a>
- and
- <a class="mw-redirect" href="/wiki/Benicio_Del_Toro" title="Benicio Del Toro">
-  Benicio Del Toro
- </a>
- as the journalist and the attorney, respectively. The film, directed by
- <a href="/wiki/Monty_Python" title="Monty Python">
-  Monty Python
- </a>
- veteran
- <a href="/wiki/Terry_Gilliam" title="Terry Gilliam">
-  Terry Gilliam
- </a>
- , has achieved something of a
- <a href="/wiki/Cult_film" title="Cult film">
-  cult following
- </a>
- .
+  ,
+  <a href="/wiki/1937" title="1937">
+   1937
+  </a>
+  –
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  ) was an
+  <a href="/wiki/United_States" title="United States">
+   American
+  </a>
+  <a href="/wiki/Journalist" title="Journalist">
+   journalist
+  </a>
+  and
+  <a href="/wiki/Author" title="Author">
+   author
+  </a>
+  .
+ </ins>
@@ -462,9 +636,42 @@
- Also, a new film is currently (
- <a href="/wiki/2005_in_film" title="2005 in film">
-  2005
- </a>
- ) in production, based upon Thompson's novel
- <i>
-  The Rum Diary.
- </i>
- Both Depp and Del Toro will be starring in this new Thompson film. Del Toro was supposed to have directed, but he withdrew in January of 2004. The director is yet to be announced.
+ <ins>
+  Hunter S. Thompson is often portrayed in popular culture as a dangerously absurd,
+  <a class="mw-redirect" href="/wiki/Illegal_drugs" title="Illegal drugs">
+   drug
+  </a>
+  -crazed journalist bent on comic self-destruction. While this portrayal is certainly not completely inaccurate, he is also considered by many to be one of the most important American writers of the
+  <a href="/wiki/20th_century" title="20th century">
+   20th century
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson is known for an explosive writing style that employs what he calls “
+  <a class="mw-redirect" href="/wiki/Action_verbs" title="Action verbs">
+   action verbs
+  </a>
+  ” to comically spin outlandish tales that are completely unbelievable, yet providing a unique viewpoint to accurately describe the underlying reality at hand. Thompson almost always writes in the
+  <a class="mw-disambig" href="/wiki/First_person" title="First person">
+   first person
+  </a>
+  narrative, and his stories become so colorfully contrived that they easily slip into the realm of fiction, however the basic framework of the story he is telling is very often true. Thompson’s writing style exploits this void between the realms of fiction and non-fiction, and it has become known as
+  <a class="mw-redirect" href="/wiki/Gonzo_Journalism" title="Gonzo Journalism">
+   Gonzo Journalism
+  </a>
+  . Other notable
+  <a href="/wiki/Gonzo_journalism" title="Gonzo journalism">
+   gonzo journalists
+  </a>
+  include
+  <a class="new" href="/w/index.php?title=Timothy_Edward_Jones&amp;action=edit&amp;redlink=1" title="Timothy Edward Jones (page does not exist)">
+   Timothy Edward Jones
+  </a>
+  , author of the new cult classic,
+  <i>
+   <a class="new" href="/w/index.php?title=European_Confession&amp;action=edit&amp;redlink=1" title="European Confession (page does not exist)">
+    European Confession
+   </a>
+   .
+  </i>
+ </ins>
@@ -473,3 +680,5 @@
- <span class="mw-headline" id="Bibliography">
-  Bibliography
- </span>
+ <ins>
+  <span class="mw-headline" id="Middle_Years_2">
+   Middle Years
+  </span>
+ </ins>
@@ -477,2 +686,3 @@
-<ul>
- <li>
+<p>
+ <ins>
+  He later worked for
@@ -480 +690,3 @@
-   The Rum Diary: The Long Lost Novel
+   <a class="mw-redirect" href="/wiki/Rolling_Stone_magazine" title="Rolling Stone magazine">
+    Rolling Stone magazine
+   </a>
@@ -482,3 +694 @@
-  (1959)
- </li>
- <li>
+  , where his next two books
@@ -486 +696 @@
-   Hell's Angels: A Strange and Terrible Saga
+   Fear and Loathing in Las Vegas
@@ -488,3 +698 @@
-  (New York, Random House, 1966)
- </li>
- <li>
+  and
@@ -492 +700 @@
-   Fear and Loathing in Las Vegas, and Other American Stories
+   Fear and Loathing on the campaign trail 1972
@@ -494,3 +702,603 @@
-  (New York, Random House, 1971)
- </li>
- <li>
+  were first serialized.
+ </ins>
+</p>
+<p>
+ <ins>
+  Published in
+  <a href="/wiki/1971" title="1971">
+   1971
+  </a>
+  ,
+  <i>
+   Fear and Loathing in Las Vegas
+  </i>
+  is a first-person account by a journalist of a trip to Las Vegas with his "300-pound
+  <a href="/wiki/Samoa" title="Samoa">
+   Samoan
+  </a>
+  " attorney to cover a narcotics officers' convention and the "fabulous Mint 400" motorcycle race. During the trip, the journalist and his lawyer become sidetracked by a search for the
+  <a class="mw-redirect" href="/wiki/American_dream" title="American dream">
+   American dream
+  </a>
+  , with the aid of heroic amounts of
+  <a class="mw-redirect" href="/wiki/LSD" title="LSD">
+   LSD
+  </a>
+  ,
+  <a href="/wiki/Ether" title="Ether">
+   ether
+  </a>
+  ,
+  <a href="/wiki/Adrenochrome" title="Adrenochrome">
+   adrenochrome
+  </a>
+  ,
+  <a href="/wiki/Ibogaine" title="Ibogaine">
+   ibogaine
+  </a>
+  and numerous other drugs. The journalist and the attorney are never named, but they are obviously thinly-disguised versions of Thompson himself, and his friend and lawyer
+  <a href="/wiki/Oscar_Zeta_Acosta" title="Oscar Zeta Acosta">
+   Oscar Zeta Acosta
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson frequently refers to himself as "Raoul Duke" or "Dr. Gonzo." He received his '
+  <a href="/wiki/Doctorate" title="Doctorate">
+   doctorate
+  </a>
+  ' from a mail-order church in the sixties.
+ </ins>
+</p>
+<p>
+ <ins>
+  Some of Thompson's other books include
+  <i>
+   Fear and Loathing on the
+   <a class="mw-redirect" href="/wiki/U.S._presidential_election,_1972" title="U.S. presidential election, 1972">
+    Campaign Trail '72
+   </a>
+  </i>
+  , a collection of
+  <i>
+   Rolling Stone
+  </i>
+  articles he wrote while covering the campaigns of President
+  <a class="mw-redirect" href="/wiki/Richard_M._Nixon" title="Richard M. Nixon">
+   Richard M. Nixon
+  </a>
+  and his unsuccessful opponent, Senator
+  <a href="/wiki/George_McGovern" title="George McGovern">
+   George McGovern
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1970" title="1970">
+   1970
+  </a>
+  , Thompson made an unsuccessful bid for
+  <a href="/wiki/Sheriff" title="Sheriff">
+   Sheriff
+  </a>
+  of
+  <a href="/wiki/Pitkin_County,_Colorado" title="Pitkin County, Colorado">
+   Pitkin County, Colorado
+  </a>
+  . He ran on a platform promoting decriminalization of drugs and the sale thereof, and renaming
+  <a href="/wiki/Aspen,_Colorado" title="Aspen, Colorado">
+   Aspen, Colorado
+  </a>
+  as "Fat City." The Republican sheriff against whom he ran had a
+  <a href="/wiki/Crew_cut" title="Crew cut">
+   crew cut
+  </a>
+  , prompting Thompson to shave his head bald and refer to his opposition as "my long-haired opponent."
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson has identified the death of the
+  <a href="/wiki/American_Dream" title="American Dream">
+   American Dream
+  </a>
+  as his “reporter’s beat”, and has covered the subject in one fashion or another throughout his writing. During an interview with
+  <a class="mw-redirect" href="/wiki/Salon.com" title="Salon.com">
+   salon.com
+  </a>
+  , however, Thompson was asked whether he was not, in fact, the living embodiment of the classic American Dream. He answered the question by first screaming a string of frustrated obscenities and then admitting that, in actuality, he probably was.
+ </ins>
+</p>
+<p>
+ <ins>
+  A slogan of Thompson's, "When the going gets weird, the weird turn pro," appears as a chapter heading in
+  <i>
+   Fear and Loathing in Las Vegas.
+  </i>
+  He has also been quoted as saying, "I hate to advocate drugs, alcohol, violence or insanity to anyone, but they've always worked for me."
+ </ins>
+</p>
+<div class="floatright">
+ <ins>
+  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Hunter_s_thompson.jpg" title="File:Hunter s thompson.jpg">
+   Hunter S. Thompson
+  </a>
+ </ins>
+</div>
+<table class="plainlinks metadata ambox ambox-notice" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     <b>
+      This article documents a
+      <a href="/wiki/Portal:Current_events" title="Portal:Current events">
+       current event
+      </a>
+      .
+     </b>
+     Information may change rapidly as the event progresses, and initial news reports may be
+     <a href="/wiki/Wikipedia:Identifying_reliable_sources#Breaking_news" title="Wikipedia:Identifying reliable sources">
+      unreliable
+     </a>
+     . The
+     <a class="external text" href="//en.wikipedia.org/w/index.php?title=Hunter_S._Thompson&amp;action=history">
+      last updates
+     </a>
+     to this article
+     <a href="/wiki/Wikipedia:Risk_disclaimer" title="Wikipedia:Risk disclaimer">
+      may not reflect
+     </a>
+     the most current information.
+     <small class="hide-when-compact">
+      <i>
+       (
+       <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
+        Learn how and when to remove this template message
+       </a>
+       )
+      </i>
+     </small>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  <b>
+   Hunter Stockton Thompson
+  </b>
+  (
+  <a href="/wiki/July_18" title="July 18">
+   July 18
+  </a>
+  ,
+  <a href="/wiki/1937" title="1937">
+   1937
+  </a>
+  –
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  ) was an
+  <a href="/wiki/United_States" title="United States">
+   American
+  </a>
+  <a href="/wiki/Journalist" title="Journalist">
+   journalist
+  </a>
+  and
+  <a href="/wiki/Author" title="Author">
+   author
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Hunter S. Thompson is often portrayed in popular culture as a dangerously absurd,
+  <a class="mw-redirect" href="/wiki/Illegal_drugs" title="Illegal drugs">
+   drug
+  </a>
+  -crazed journalist bent on comic self-destruction. While this portrayal is certainly not completely inaccurate, he is also considered by many to be one of the most important American writers of the
+  <a href="/wiki/20th_century" title="20th century">
+   20th century
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson is known for an explosive writing style that employs what he calls “
+  <a class="mw-redirect" href="/wiki/Action_verbs" title="Action verbs">
+   action verbs
+  </a>
+  ” to comically spin outlandish tales that are completely unbelievable, yet providing a unique viewpoint to accurately describe the underlying reality at hand. Thompson almost always writes in the
+  <a class="mw-disambig" href="/wiki/First_person" title="First person">
+   first person
+  </a>
+  narrative, and his stories become so colorfully contrived that they easily slip into the realm of fiction, however the basic framework of the story he is telling is very often true. Thompson’s writing style exploits this void between the realms of fiction and non-fiction, and it has become known as
+  <a class="mw-redirect" href="/wiki/Gonzo_Journalism" title="Gonzo Journalism">
+   Gonzo Journalism
+  </a>
+  . Other notable
+  <a href="/wiki/Gonzo_journalism" title="Gonzo journalism">
+   gonzo journalists
+  </a>
+  include
+  <a class="new" href="/w/index.php?title=Timothy_Edward_Jones&amp;action=edit&amp;redlink=1" title="Timothy Edward Jones (page does not exist)">
+   Timothy Edward Jones
+  </a>
+  , author of the new cult classic,
+  <i>
+   <a class="new" href="/w/index.php?title=European_Confession&amp;action=edit&amp;redlink=1" title="European Confession (page does not exist)">
+    European Confession
+   </a>
+   .
+  </i>
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Middle_Years_3">
+   Middle Years
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  He later worked for
+  <i>
+   <a class="mw-redirect" href="/wiki/Rolling_Stone_magazine" title="Rolling Stone magazine">
+    Rolling Stone magazine
+   </a>
+  </i>
+  , where his next two books
+  <i>
+   Fear and Loathing in Las Vegas
+  </i>
+  and
+  <i>
+   Fear and Loathing on the campaign trail 1972
+  </i>
+  were first serialized.
+ </ins>
+</p>
+<p>
+ <ins>
+  Published in
+  <a href="/wiki/1971" title="1971">
+   1971
+  </a>
+  ,
+  <i>
+   Fear and Loathing in Las Vegas
+  </i>
+  is a first-person account by a journalist of a trip to Las Vegas with his "300-pound
+  <a href="/wiki/Samoa" title="Samoa">
+   Samoan
+  </a>
+  " attorney to cover a narcotics officers' convention and the "fabulous Mint 400" motorcycle race. During the trip, the journalist and his lawyer become sidetracked by a search for the
+  <a class="mw-redirect" href="/wiki/American_dream" title="American dream">
+   American dream
+  </a>
+  , with the aid of heroic amounts of
+  <a class="mw-redirect" href="/wiki/LSD" title="LSD">
+   LSD
+  </a>
+  ,
+  <a href="/wiki/Ether" title="Ether">
+   ether
+  </a>
+  ,
+  <a href="/wiki/Adrenochrome" title="Adrenochrome">
+   adrenochrome
+  </a>
+  ,
+  <a href="/wiki/Ibogaine" title="Ibogaine">
+   ibogaine
+  </a>
+  and numerous other drugs. The journalist and the attorney are never named, but they are obviously thinly-disguised versions of Thompson himself, and his friend and lawyer
+  <a href="/wiki/Oscar_Zeta_Acosta" title="Oscar Zeta Acosta">
+   Oscar Zeta Acosta
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson frequently refers to himself as "Raoul Duke" or "Dr. Gonzo." He received his '
+  <a href="/wiki/Doctorate" title="Doctorate">
+   doctorate
+  </a>
+  ' from a mail-order church in the sixties.
+ </ins>
+</p>
+<p>
+ <ins>
+  Some of Thompson's other books include
+  <i>
+   Fear and Loathing on the
+   <a class="mw-redirect" href="/wiki/U.S._presidential_election,_1972" title="U.S. presidential election, 1972">
+    Campaign Trail '72
+   </a>
+  </i>
+  , a collection of
+  <i>
+   Rolling Stone
+  </i>
+  articles he wrote while covering the campaigns of President
+  <a class="mw-redirect" href="/wiki/Richard_M._Nixon" title="Richard M. Nixon">
+   Richard M. Nixon
+  </a>
+  and his unsuccessful opponent, Senator
+  <a href="/wiki/George_McGovern" title="George McGovern">
+   George McGovern
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1970" title="1970">
+   1970
+  </a>
+  , Thompson made an unsuccessful bid for
+  <a href="/wiki/Sheriff" title="Sheriff">
+   Sheriff
+  </a>
+  of
+  <a href="/wiki/Pitkin_County,_Colorado" title="Pitkin County, Colorado">
+   Pitkin County, Colorado
+  </a>
+  . He ran on a platform promoting decriminalization of drugs and the sale thereof, and renaming
+  <a href="/wiki/Aspen,_Colorado" title="Aspen, Colorado">
+   Aspen, Colorado
+  </a>
+  as "Fat City." The Republican sheriff against whom he ran had a
+  <a href="/wiki/Crew_cut" title="Crew cut">
+   crew cut
+  </a>
+  , prompting Thompson to shave his head bald and refer to his opposition as "my long-haired opponent."
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson has identified the death of the
+  <a href="/wiki/American_Dream" title="American Dream">
+   American Dream
+  </a>
+  as his “reporter’s beat”, and has covered the subject in one fashion or another throughout his writing. During an interview with
+  <a class="mw-redirect" href="/wiki/Salon.com" title="Salon.com">
+   salon.com
+  </a>
+  , however, Thompson was asked whether he was not, in fact, the living embodiment of the classic American Dream. He answered the question by first screaming a string of frustrated obscenities and then admitting that, in actuality, he probably was.
+ </ins>
+</p>
+<p>
+ <ins>
+  A slogan of Thompson's, "When the going gets weird, the weird turn pro," appears as a chapter heading in
+  <i>
+   Fear and Loathing in Las Vegas.
+  </i>
+  He has also been quoted as saying, "I hate to advocate drugs, alcohol, violence or insanity to anyone, but they've always worked for me."
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Later_Years">
+   Later Years
+  </span>
+ </ins>
+</h2>
+<ins>
+ Thompson's last
+</ins>
+<del>
+ His latest
+</del>
+ book,
+ <i>
+ Kingdom of Fear
+</i>
+ , is an angry commentary on the passing of the
+ <a href="/wiki/American_Century" title="American Century">
+ American Century
+</a>
+ . Thompson currently writes a Web column, "Hey Rube," for
+ <a href="/wiki/ESPN" title="ESPN">
+ ESPN
+</a>
+ . He <ins>
+ had
+</ins>
+<del>
+ has
+</del>
+ at times also toured on the lecture circuit, once with
+ <a href="/wiki/John_Belushi" title="John Belushi">
+ John Belushi
+</a>
+ . Another modern appearance is through
+ <a href="/wiki/Cartoonist" title="Cartoonist">
+ cartoonist
+</a>
+<a href="/wiki/Garry_Trudeau" title="Garry Trudeau">
+ Garry Trudeau
+</a>
+ , who based his
+ <i>
+ <a href="/wiki/Doonesbury" title="Doonesbury">
+  Doonesbury
+ </a>
+</i>
+ character
+ <a href="/wiki/Uncle_Duke" title="Uncle Duke">
+ Uncle Duke
+</a>
+ on Thompson, to loud protests from Thompson himself. Similarly, Spider Jerusalem, the gonzo journalist protagonist <ins>
+ off
+</ins>
+<del>
+ of
+</del>
+<a href="/wiki/Warren_Ellis" title="Warren Ellis">
+ Warren Ellis
+</a>
+ 's
+ <i>
+ <a href="/wiki/Transmetropolitan" title="Transmetropolitan">
+  Transmetropolitan
+ </a>
+ ,
+</i>
+ is (more lovingly) based on Thompson.
+ <p>
+ Thompson
+ <ins>
+  died of a self-inflicted gunshot wound at his home in
+  <a href="/wiki/Woody_Creek,_Colorado" title="Woody Creek, Colorado">
+   Woody Creek
+  </a>
+  ,
+  <a href="/wiki/Colorado" title="Colorado">
+   Colorado
+  </a>
+  , on Sunday,
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  . He was 67.
+ </ins>
+ <del>
+  is an admitted fan of
+  <a class="mw-redirect" href="/wiki/Firearms" title="Firearms">
+   firearms
+  </a>
+  and is known to keep a keg of
+  <a href="/wiki/Gunpowder" title="Gunpowder">
+   gunpowder
+  </a>
+  in his basement.
+ </del>
+</p>
+<h2>
+ <span class="mw-headline" id="Movies">
+  Movies
+ </span>
+</h2>
+<p>
+ The film
+ <i>
+  Where The Buffalo Roam
+ </i>
+ (
+ <a href="/wiki/1980_in_film" title="1980 in film">
+  1980
+ </a>
+ ) depicts Thompson's attempts at writing stories for both the Super Bowl and the 1972 Presidential Election. It stars
+ <a href="/wiki/Bill_Murray" title="Bill Murray">
+  Bill Murray
+ </a>
+ as Thompson and
+ <a href="/wiki/Peter_Boyle" title="Peter Boyle">
+  Peter Boyle
+ </a>
+ as Thompson's attorney Oscar Acosta,
+ <ins>
+  referred
+ </ins>
+ <del>
+  refered
+ </del>
+ to in the movie as Carl Laslow, Esq.
+</p>
+<p>
+ The film
+ <i>
+  <a href="/wiki/Fear_and_Loathing_in_Las_Vegas" title="Fear and Loathing in Las Vegas">
+   Fear and Loathing in Las Vegas
+  </a>
+  : A Savage Journey to the Heart of the American Dream
+ </i>
+ , based on the book was released in
+ <a href="/wiki/1998_in_film" title="1998 in film">
+  1998
+ </a>
+ , starring
+ <a href="/wiki/Johnny_Depp" title="Johnny Depp">
+  Johnny Depp
+ </a>
+ and
+ <a class="mw-redirect" href="/wiki/Benicio_Del_Toro" title="Benicio Del Toro">
+  Benicio Del Toro
+ </a>
+ as the journalist and the attorney, respectively. The film, directed by
+ <a href="/wiki/Monty_Python" title="Monty Python">
+  Monty Python
+ </a>
+ veteran
+ <a href="/wiki/Terry_Gilliam" title="Terry Gilliam">
+  Terry Gilliam
+ </a>
+ , has achieved something of a
+ <a href="/wiki/Cult_film" title="Cult film">
+  cult following
+ </a>
+ .
+</p>
+<p>
+ Also, a new film is currently (
+ <a href="/wiki/2005_in_film" title="2005 in film">
+  2005
+ </a>
+ ) in production, based upon Thompson's novel
+ <i>
+  The Rum Diary.
+ </i>
+ Both Depp and Del Toro will be starring in this new Thompson film. Del Toro was supposed to have directed, but he withdrew in January of 2004. The director is yet to be announced.
+</p>
+<h2>
+ <span class="mw-headline" id="Bibliography">
+  Bibliography
+ </span>
+</h2>
+<ul>
+ <li>
+  <i>
+   The Rum Diary: The Long Lost Novel
+  </i>
+  (1959)
+ </li>
+ <li>
+  <i>
+   Hell's Angels: A Strange and Terrible Saga
+  </i>
+  (New York, Random House, 1966)
+ </li>
+ <li>
+  <i>
+   Fear and Loathing in Las Vegas, and Other American Stories
+  </i>
+  (New York, Random House, 1971)
+ </li>
+ <li>
@@ -500 +1308,383 @@
-  (San Francisco, Straight Arrow Books, 1973)
+  (San Francisco, Straight Arrow Books, 1973)
+ </li>
+ <li>
+  <i>
+   Gonzo Papers, Vol. 1: The Great Shark Hunt: Strange Tales from a Strange Time
+  </i>
+  (New York, Summit Books, 1979)
+ </li>
+ <li>
+  <i>
+   The Curse of Lono,
+  </i>
+  illustrated by
+  <a href="/wiki/Ralph_Steadman" title="Ralph Steadman">
+   Ralph Steadman
+  </a>
+  (Bantam Books, 1983)
+ </li>
+ <li>
+  <i>
+   Gonzo Papers, Vol. 2: Generation of Swine: Tales of Shame and Degradation in the '80s
+  </i>
+  (New York, Summit Books, 1988)
+ </li>
+ <li>
+  <i>
+   Gonzo Papers, Vol. 3: Songs of the Doomed: More Notes on the Death of the American Dream
+  </i>
+  (New York, Summit Books, 1990)
+ </li>
+ <li>
+  <i>
+   Screwjack and Other Stories.
+  </i>
+  (Santa Barbara, Neville Press, 1991)
+ </li>
+ <li>
+  <i>
+   Gonzo Papers, Vol. 4: Better Than Sex: Confessions of a Political Junkie
+  </i>
+  Trapped Like a Rat in Mr. Bill's Neighborhood (New York, Random House, 1994)
+ </li>
+ <li>
+  <i>
+   The Fear and Loathing Letters, Volume I -- The Proud Highway -- Saga of a Desperate Southern Gentleman 1955 - 1967
+  </i>
+  (New York, Random House, 1997)
+ </li>
+ <li>
+  <i>
+   Fear and Loathing in America: The Brutal Odyssey of an Outlaw Journalist 1968-1976
+  </i>
+  (Collection of Papers first appeared in Time magazine, 1997)
+ </li>
+ <li>
+  <i>
+   Kingdom of Fear: Loathsome Secrets of a Star-Crossed Child in the Final Days of the American Century
+  </i>
+  (Simon &amp; Schuster; 1st Simon edition, November 1, 2003)
+ </li>
+ <li>
+  --Hey Rube : Blood Sport, the Bush Doctrine, and the Downward Spiral of Dumbness Modern History from the Sports Desk (Simon &amp; Schuster, August 11, 2004)
+ </li>
+</ul>
+<h2>
+ <span class="mw-headline" id="External_link">
+  External link
+ </span>
+</h2>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+   <img alt="" class="noviewer" data-file-height="355" data-file-width="300" height="40" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/34px-Wikiquote-logo.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/51px-Wikiquote-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/68px-Wikiquote-logo.svg.png 2x" width="34"/>
+  </td>
+  <td class="mbox-text plainlist">
+   Wikiquote has quotations related to:
+   <i>
+    <b>
+     <a class="extiw" href="https://en.wikiquote.org/wiki/Special:Search/Hunter_S._Thompson" title="q:Special:Search/Hunter S. Thompson">
+      Hunter S. Thompson
+     </a>
+    </b>
+   </i>
+  </td>
+ </tr>
+</table>
+<ul>
+ <li>
+  <a class="external text" href="http://www.gonzo.org" rel="nofollow">
+   The Great Thompson Hunt
+  </a>
+ </li>
+ <li>
+  <a class="external text" href="http://sports.espn.go.com/espn/archive?columnist=hunter_s._thompson&amp;root=page2" rel="nofollow">
+   Thompson's ESPN column
+  </a>
+ </li>
+ <li>
+  <a class="external text" href="http://www.imdb.com/title/tt0081748/" rel="nofollow">
+   IMDB site for
+   <i>
+    Where the Buffalo Roam
+   </i>
+  </a>
+ </li>
+ <li>
+  <a class="external text" href="http://www.imdb.com/title/tt0120669/" rel="nofollow">
+   IMDB site for
+   <i>
+    Fear and Loathing in Las Vegas
+   </i>
+  </a>
+ </li>
+ <li>
+  <a class="external text" href="http://www.imdb.com/title/tt0376136/" rel="nofollow">
+   IMDB site for
+   <i>
+    The Rum Diary
+   </i>
+  </a>
+ </li>
+</ul>
+<p>
+ <ins>
+  <a class="new" href="/w/index.php?title=Template:Lived&amp;action=edit&amp;redlink=1" title="Template:Lived (page does not exist)">
+   Template:Lived
+  </a>
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Movies_2">
+   Movies
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  The film
+  <i>
+   Where The Buffalo Roam
+  </i>
+  (
+  <a href="/wiki/1980_in_film" title="1980 in film">
+   1980
+  </a>
+  ) depicts Thompson's attempts at writing stories for both the Super Bowl and the 1972 Presidential Election. It stars
+  <a href="/wiki/Bill_Murray" title="Bill Murray">
+   Bill Murray
+  </a>
+  as Thompson and
+  <a href="/wiki/Peter_Boyle" title="Peter Boyle">
+   Peter Boyle
+  </a>
+  as Thompson's attorney Oscar Acosta, referred to in the movie as Carl Laslow, Esq.
+ </ins>
+</p>
+<p>
+ <ins>
+  The film
+  <i>
+   <a href="/wiki/Fear_and_Loathing_in_Las_Vegas" title="Fear and Loathing in Las Vegas">
+    Fear and Loathing in Las Vegas
+   </a>
+   : A Savage Journey to the Heart of the American Dream
+  </i>
+  , based on the book was released in
+  <a href="/wiki/1998_in_film" title="1998 in film">
+   1998
+  </a>
+  , starring
+  <a href="/wiki/Johnny_Depp" title="Johnny Depp">
+   Johnny Depp
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Benicio_Del_Toro" title="Benicio Del Toro">
+   Benicio Del Toro
+  </a>
+  as the journalist and the attorney, respectively. The film, directed by
+  <a href="/wiki/Monty_Python" title="Monty Python">
+   Monty Python
+  </a>
+  veteran
+  <a href="/wiki/Terry_Gilliam" title="Terry Gilliam">
+   Terry Gilliam
+  </a>
+  , has achieved something of a
+  <a href="/wiki/Cult_film" title="Cult film">
+   cult following
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Also, a new film is currently (
+  <a href="/wiki/2005_in_film" title="2005 in film">
+   2005
+  </a>
+  ) in production, based upon Thompson's novel
+  <i>
+   The Rum Diary.
+  </i>
+  Both Depp and Del Toro will be starring in this new Thompson film. Del Toro was supposed to have directed, but he withdrew in January of 2004. The director is yet to be announced.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Bibliography_2">
+   Bibliography
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   <i>
+    The Rum Diary: The Long Lost Novel
+   </i>
+   (1959)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Hell's Angels: A Strange and Terrible Saga
+   </i>
+   (New York, Random House, 1966)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Fear and Loathing in Las Vegas, and Other American Stories
+   </i>
+   (New York, Random House, 1971)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Fear and Loathing: On the Campaign Trail '72
+   </i>
+   (San Francisco, Straight Arrow Books, 1973)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 1: The Great Shark Hunt: Strange Tales from a Strange Time
+   </i>
+   (New York, Summit Books, 1979)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    The Curse of Lono,
+   </i>
+   illustrated by
+   <a href="/wiki/Ralph_Steadman" title="Ralph Steadman">
+    Ralph Steadman
+   </a>
+   (Bantam Books, 1983)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 2: Generation of Swine: Tales of Shame and Degradation in the '80s
+   </i>
+   (New York, Summit Books, 1988)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 3: Songs of the Doomed: More Notes on the Death of the American Dream
+   </i>
+   (New York, Summit Books, 1990)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Screwjack and Other Stories.
+   </i>
+   (Santa Barbara, Neville Press, 1991)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 4: Better Than Sex: Confessions of a Political Junkie
+   </i>
+   Trapped Like a Rat in Mr. Bill's Neighborhood (New York, Random House, 1994)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    The Fear and Loathing Letters, Volume I -- The Proud Highway -- Saga of a Desperate Southern Gentleman 1955 - 1967
+   </i>
+   (New York, Random House, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Fear and Loathing in America: The Brutal Odyssey of an Outlaw Journalist 1968-1976
+   </i>
+   (Collection of Papers first appeared in Time magazine, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Kingdom of Fear: Loathsome Secrets of a Star-Crossed Child in the Final Days of the American Century
+   </i>
+   (Simon &amp; Schuster; 1st Simon edition, November 1, 2003)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   --Hey Rube : Blood Sport, the Bush Doctrine, and the Downward Spiral of Dumbness Modern History from the Sports Desk (Simon &amp; Schuster, August 11, 2004)
+  </ins>
+ </li>
+</ul>
+<h2>
+ <ins>
+  <span class="mw-headline" id="External_link_2">
+   External link
+  </span>
+ </ins>
+</h2>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    Wikiquote has quotations related to:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikiquote.org/wiki/Special:Search/Hunter_S._Thompson" title="q:Special:Search/Hunter S. Thompson">
+       Hunter S. Thompson
+      </a>
+     </b>
+    </i>
+   </ins>
+  </td>
+ </tr>
+</table>
+<ul>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.gonzo.org" rel="nofollow">
+    The Great Thompson Hunt
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://sports.espn.go.com/espn/archive?columnist=hunter_s._thompson&amp;root=page2" rel="nofollow">
+    Thompson's ESPN column
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0081748/" rel="nofollow">
+    IMDB site for
+   Where the Buffalo Roam
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0120669/" rel="nofollow">
+    IMDB site for
+   Fear and Loathing in Las Vegas
+   </a>
+  </ins>
@@ -502,0 +1693,98 @@
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0376136/" rel="nofollow">
+    IMDB site for
+   The Rum Diary
+   </a>
+  </ins>
+ </li>
+</ul>
+<p>
+ <ins>
+  <a class="new" href="/w/index.php?title=Template:Lived&amp;action=edit&amp;redlink=1" title="Template:Lived (page does not exist)">
+   Template:Lived
+  </a>
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Later_Years_2">
+   Later Years
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  Thompson's last book,
+  <i>
+   Kingdom of Fear
+  </i>
+  , is an angry commentary on the passing of the
+  <a href="/wiki/American_Century" title="American Century">
+   American Century
+  </a>
+  . Thompson currently writes a Web column, "Hey Rube," for
+  <a href="/wiki/ESPN" title="ESPN">
+   ESPN
+  </a>
+  . He had at times also toured on the lecture circuit, once with
+  <a href="/wiki/John_Belushi" title="John Belushi">
+   John Belushi
+  </a>
+  . Another modern appearance is through
+  <a href="/wiki/Cartoonist" title="Cartoonist">
+   cartoonist
+  </a>
+  <a href="/wiki/Garry_Trudeau" title="Garry Trudeau">
+   Garry Trudeau
+  </a>
+  , who based his
+  <i>
+   <a href="/wiki/Doonesbury" title="Doonesbury">
+    Doonesbury
+   </a>
+  </i>
+  character
+  <a href="/wiki/Uncle_Duke" title="Uncle Duke">
+   Uncle Duke
+  </a>
+  on Thompson, to loud protests from Thompson himself. Similarly, Spider Jerusalem, the gonzo journalist protagonist off
+  <a href="/wiki/Warren_Ellis" title="Warren Ellis">
+   Warren Ellis
+  </a>
+  's
+  <i>
+   <a href="/wiki/Transmetropolitan" title="Transmetropolitan">
+    Transmetropolitan
+   </a>
+   ,
+  </i>
+  is (more lovingly) based on Thompson.
+ </ins>
+</p>
+<p>
+ <ins>
+  Thompson died of a self-inflicted gunshot wound at his home in
+  <a href="/wiki/Woody_Creek,_Colorado" title="Woody Creek, Colorado">
+   Woody Creek, Colorado
+  </a>
+  , on Sunday,
+  <a href="/wiki/February_20" title="February 20">
+   February 20
+  </a>
+  ,
+  <a href="/wiki/2005" title="2005">
+   2005
+  </a>
+  . He was 67.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Movies_3">
+   Movies
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  The film
@@ -504 +1792 @@
-   Gonzo Papers, Vol. 1: The Great Shark Hunt: Strange Tales from a Strange Time
+   Where The Buffalo Roam
@@ -506,3 +1794,18 @@
-  (New York, Summit Books, 1979)
- </li>
- <li>
+  (
+  <a href="/wiki/1980_in_film" title="1980 in film">
+   1980
+  </a>
+  ) depicts Thompson's attempts at writing stories for both the Super Bowl and the 1972 Presidential Election. It stars
+  <a href="/wiki/Bill_Murray" title="Bill Murray">
+   Bill Murray
+  </a>
+  as Thompson and
+  <a href="/wiki/Peter_Boyle" title="Peter Boyle">
+   Peter Boyle
+  </a>
+  as Thompson's attorney Oscar Acosta, referred to in the movie as Carl Laslow, Esq.
+ </ins>
+</p>
+<p>
+ <ins>
+  The film
@@ -510 +1813,4 @@
-   The Curse of Lono,
+   <a href="/wiki/Fear_and_Loathing_in_Las_Vegas" title="Fear and Loathing in Las Vegas">
+    Fear and Loathing in Las Vegas
+   </a>
+   : A Savage Journey to the Heart of the American Dream
@@ -512,3 +1818,3 @@
-  illustrated by
-  <a href="/wiki/Ralph_Steadman" title="Ralph Steadman">
-   Ralph Steadman
+  , based on the book was released in
+  <a href="/wiki/1998_in_film" title="1998 in film">
+   1998
@@ -516,3 +1822,30 @@
-  (Bantam Books, 1983)
- </li>
- <li>
+  , starring
+  <a href="/wiki/Johnny_Depp" title="Johnny Depp">
+   Johnny Depp
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Benicio_Del_Toro" title="Benicio Del Toro">
+   Benicio Del Toro
+  </a>
+  as the journalist and the attorney, respectively. The film, directed by
+  <a href="/wiki/Monty_Python" title="Monty Python">
+   Monty Python
+  </a>
+  veteran
+  <a href="/wiki/Terry_Gilliam" title="Terry Gilliam">
+   Terry Gilliam
+  </a>
+  , has achieved something of a
+  <a href="/wiki/Cult_film" title="Cult film">
+   cult following
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Also, a new film is currently (
+  <a href="/wiki/2005_in_film" title="2005 in film">
+   2005
+  </a>
+  ) in production, based upon Thompson's novel
@@ -520 +1853 @@
-   Gonzo Papers, Vol. 2: Generation of Swine: Tales of Shame and Degradation in the '80s
+   The Rum Diary.
@@ -522 +1855,18 @@
-  (New York, Summit Books, 1988)
+  Both Depp and Del Toro will be starring in this new Thompson film. Del Toro was supposed to have directed, but he withdrew in January of 2004. The director is yet to be announced.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Bibliography_3">
+   Bibliography
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   <i>
+    The Rum Diary: The Long Lost Novel
+   </i>
+   (1959)
+  </ins>
@@ -525,4 +1875,6 @@
-  <i>
-   Gonzo Papers, Vol. 3: Songs of the Doomed: More Notes on the Death of the American Dream
-  </i>
-  (New York, Summit Books, 1990)
+  <ins>
+   <i>
+    Hell's Angels: A Strange and Terrible Saga
+   </i>
+   (New York, Random House, 1966)
+  </ins>
@@ -531,4 +1883,6 @@
-  <i>
-   Screwjack and Other Stories.
-  </i>
-  (Santa Barbara, Neville Press, 1991)
+  <ins>
+   <i>
+    Fear and Loathing in Las Vegas, and Other American Stories
+   </i>
+   (New York, Random House, 1971)
+  </ins>
@@ -537,4 +1891,6 @@
-  <i>
-   Gonzo Papers, Vol. 4: Better Than Sex: Confessions of a Political Junkie
-  </i>
-  Trapped Like a Rat in Mr. Bill's Neighborhood (New York, Random House, 1994)
+  <ins>
+   <i>
+    Fear and Loathing: On the Campaign Trail '72
+   </i>
+   (San Francisco, Straight Arrow Books, 1973)
+  </ins>
@@ -543,4 +1899,6 @@
-  <i>
-   The Fear and Loathing Letters, Volume I -- The Proud Highway -- Saga of a Desperate Southern Gentleman 1955 - 1967
-  </i>
-  (New York, Random House, 1997)
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 1: The Great Shark Hunt: Strange Tales from a Strange Time
+   </i>
+   (New York, Summit Books, 1979)
+  </ins>
@@ -549,4 +1907,10 @@
-  <i>
-   Fear and Loathing in America: The Brutal Odyssey of an Outlaw Journalist 1968-1976
-  </i>
-  (Collection of Papers first appeared in Time magazine, 1997)
+  <ins>
+   <i>
+    The Curse of Lono,
+   </i>
+   illustrated by
+   <a href="/wiki/Ralph_Steadman" title="Ralph Steadman">
+    Ralph Steadman
+   </a>
+   (Bantam Books, 1983)
+  </ins>
@@ -555,4 +1919,6 @@
-  <i>
-   Kingdom of Fear: Loathsome Secrets of a Star-Crossed Child in the Final Days of the American Century
-  </i>
-  (Simon &amp; Schuster; 1st Simon edition, November 1, 2003)
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 2: Generation of Swine: Tales of Shame and Degradation in the '80s
+   </i>
+   (New York, Summit Books, 1988)
+  </ins>
@@ -561 +1927,51 @@
-  --Hey Rube : Blood Sport, the Bush Doctrine, and the Downward Spiral of Dumbness Modern History from the Sports Desk (Simon &amp; Schuster, August 11, 2004)
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 3: Songs of the Doomed: More Notes on the Death of the American Dream
+   </i>
+   (New York, Summit Books, 1990)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Screwjack and Other Stories.
+   </i>
+   (Santa Barbara, Neville Press, 1991)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 4: Better Than Sex: Confessions of a Political Junkie
+   </i>
+   Trapped Like a Rat in Mr. Bill's Neighborhood (New York, Random House, 1994)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    The Fear and Loathing Letters, Volume I -- The Proud Highway -- Saga of a Desperate Southern Gentleman 1955 - 1967
+   </i>
+   (New York, Random House, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Fear and Loathing in America: The Brutal Odyssey of an Outlaw Journalist 1968-1976
+   </i>
+   (Collection of Papers first appeared in Time magazine, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Kingdom of Fear: Loathsome Secrets of a Star-Crossed Child in the Final Days of the American Century
+   </i>
+   (Simon &amp; Schuster; 1st Simon edition, November 1, 2003)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   --Hey Rube : Blood Sport, the Bush Doctrine, and the Downward Spiral of Dumbness Modern History from the Sports Desk (Simon &amp; Schuster, August 11, 2004)
+  </ins>
@@ -565,3 +1981,5 @@
- <span class="mw-headline" id="External_link">
-  External link
- </span>
+ <ins>
+  <span class="mw-headline" id="External_link_3">
+   External link
+  </span>
+ </ins>
@@ -572 +1989,0 @@
-   <img alt="" class="noviewer" data-file-height="355" data-file-width="300" height="40" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/34px-Wikiquote-logo.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/51px-Wikiquote-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Wikiquote-logo.svg/68px-Wikiquote-logo.svg.png 2x" width="34"/>
@@ -575,8 +1992,10 @@
-   Wikiquote has quotations related to:
-   <i>
-    <b>
-     <a class="extiw" href="https://en.wikiquote.org/wiki/Special:Search/Hunter_S._Thompson" title="q:Special:Search/Hunter S. Thompson">
-      Hunter S. Thompson
-     </a>
-    </b>
-   </i>
+   <ins>
+    Wikiquote has quotations related to:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikiquote.org/wiki/Special:Search/Hunter_S._Thompson" title="q:Special:Search/Hunter S. Thompson">
+       Hunter S. Thompson
+      </a>
+     </b>
+    </i>
+   </ins>
@@ -588,3 +2007,5 @@
-  <a class="external text" href="http://www.gonzo.org" rel="nofollow">
-   The Great Thompson Hunt
-  </a>
+  <ins>
+   <a class="external text" href="http://www.gonzo.org" rel="nofollow">
+    The Great Thompson Hunt
+   </a>
+  </ins>
@@ -593,2 +2014,94 @@
-  <a class="external text" href="http://sports.espn.go.com/espn/archive?columnist=hunter_s._thompson&amp;root=page2" rel="nofollow">
-   Thompson's ESPN column
+  <ins>
+   <a class="external text" href="http://sports.espn.go.com/espn/archive?columnist=hunter_s._thompson&amp;root=page2" rel="nofollow">
+    Thompson's ESPN column
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0081748/" rel="nofollow">
+    IMDB site for
+   Where the Buffalo Roam
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0120669/" rel="nofollow">
+    IMDB site for
+   Fear and Loathing in Las Vegas
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0376136/" rel="nofollow">
+    IMDB site for
+   The Rum Diary
+   </a>
+  </ins>
+ </li>
+</ul>
+<p>
+ <ins>
+  <a class="new" href="/w/index.php?title=Template:Lived&amp;action=edit&amp;redlink=1" title="Template:Lived (page does not exist)">
+   Template:Lived
+  </a>
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Movies_4">
+   Movies
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  The film
+  <i>
+   Where The Buffalo Roam
+  </i>
+  (
+  <a href="/wiki/1980_in_film" title="1980 in film">
+   1980
+  </a>
+  ) depicts Thompson's attempts at writing stories for both the Super Bowl and the 1972 Presidential Election. It stars
+  <a href="/wiki/Bill_Murray" title="Bill Murray">
+   Bill Murray
+  </a>
+  as Thompson and
+  <a href="/wiki/Peter_Boyle" title="Peter Boyle">
+   Peter Boyle
+  </a>
+  as Thompson's attorney Oscar Acosta, referred to in the movie as Carl Laslow, Esq.
+ </ins>
+</p>
+<p>
+ <ins>
+  The film
+  <i>
+   <a href="/wiki/Fear_and_Loathing_in_Las_Vegas" title="Fear and Loathing in Las Vegas">
+    Fear and Loathing in Las Vegas
+   </a>
+   : A Savage Journey to the Heart of the American Dream
+  </i>
+  , based on the book was released in
+  <a href="/wiki/1998_in_film" title="1998 in film">
+   1998
+  </a>
+  , starring
+  <a href="/wiki/Johnny_Depp" title="Johnny Depp">
+   Johnny Depp
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Benicio_Del_Toro" title="Benicio Del Toro">
+   Benicio Del Toro
+  </a>
+  as the journalist and the attorney, respectively. The film, directed by
+  <a href="/wiki/Monty_Python" title="Monty Python">
+   Monty Python
+  </a>
+  veteran
+  <a href="/wiki/Terry_Gilliam" title="Terry Gilliam">
+   Terry Gilliam
@@ -595,0 +2109,35 @@
+  , has achieved something of a
+  <a href="/wiki/Cult_film" title="Cult film">
+   cult following
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Also, a new film is currently (
+  <a href="/wiki/2005_in_film" title="2005 in film">
+   2005
+  </a>
+  ) in production, based upon Thompson's novel
+  <i>
+   The Rum Diary.
+  </i>
+  Both Depp and Del Toro will be starring in this new Thompson film. Del Toro was supposed to have directed, but he withdrew in January of 2004. The director is yet to be announced.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Bibliography_4">
+   Bibliography
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   <i>
+    The Rum Diary: The Long Lost Novel
+   </i>
+   (1959)
+  </ins>
@@ -598,2 +2146 @@
-  <a class="external text" href="http://www.imdb.com/title/tt0081748/" rel="nofollow">
-   IMDB site for
+  <ins>
@@ -601 +2148 @@
-    Where the Buffalo Roam
+    Hell's Angels: A Strange and Terrible Saga
@@ -603 +2150,2 @@
-  </a>
+   (New York, Random House, 1966)
+  </ins>
@@ -606,2 +2154 @@
-  <a class="external text" href="http://www.imdb.com/title/tt0120669/" rel="nofollow">
-   IMDB site for
+  <ins>
@@ -609 +2156 @@
-    Fear and Loathing in Las Vegas
+    Fear and Loathing in Las Vegas, and Other American Stories
@@ -611 +2158,2 @@
-  </a>
+   (New York, Random House, 1971)
+  </ins>
@@ -614,2 +2162 @@
-  <a class="external text" href="http://www.imdb.com/title/tt0376136/" rel="nofollow">
-   IMDB site for
+  <ins>
@@ -617 +2164 @@
-    The Rum Diary
+    Fear and Loathing: On the Campaign Trail '72
@@ -619 +2166,148 @@
-  </a>
+   (San Francisco, Straight Arrow Books, 1973)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 1: The Great Shark Hunt: Strange Tales from a Strange Time
+   </i>
+   (New York, Summit Books, 1979)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    The Curse of Lono,
+   </i>
+   illustrated by
+   <a href="/wiki/Ralph_Steadman" title="Ralph Steadman">
+    Ralph Steadman
+   </a>
+   (Bantam Books, 1983)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 2: Generation of Swine: Tales of Shame and Degradation in the '80s
+   </i>
+   (New York, Summit Books, 1988)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 3: Songs of the Doomed: More Notes on the Death of the American Dream
+   </i>
+   (New York, Summit Books, 1990)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Screwjack and Other Stories.
+   </i>
+   (Santa Barbara, Neville Press, 1991)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Gonzo Papers, Vol. 4: Better Than Sex: Confessions of a Political Junkie
+   </i>
+   Trapped Like a Rat in Mr. Bill's Neighborhood (New York, Random House, 1994)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    The Fear and Loathing Letters, Volume I -- The Proud Highway -- Saga of a Desperate Southern Gentleman 1955 - 1967
+   </i>
+   (New York, Random House, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Fear and Loathing in America: The Brutal Odyssey of an Outlaw Journalist 1968-1976
+   </i>
+   (Collection of Papers first appeared in Time magazine, 1997)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <i>
+    Kingdom of Fear: Loathsome Secrets of a Star-Crossed Child in the Final Days of the American Century
+   </i>
+   (Simon &amp; Schuster; 1st Simon edition, November 1, 2003)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   --Hey Rube : Blood Sport, the Bush Doctrine, and the Downward Spiral of Dumbness Modern History from the Sports Desk (Simon &amp; Schuster, August 11, 2004)
+  </ins>
+ </li>
+</ul>
+<h2>
+ <ins>
+  <span class="mw-headline" id="External_link_4">
+   External link
+  </span>
+ </ins>
+</h2>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    Wikiquote has quotations related to:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikiquote.org/wiki/Special:Search/Hunter_S._Thompson" title="q:Special:Search/Hunter S. Thompson">
+       Hunter S. Thompson
+      </a>
+     </b>
+    </i>
+   </ins>
+  </td>
+ </tr>
+</table>
+<ul>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.gonzo.org" rel="nofollow">
+    The Great Thompson Hunt
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://sports.espn.go.com/espn/archive?columnist=hunter_s._thompson&amp;root=page2" rel="nofollow">
+    Thompson's ESPN column
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0081748/" rel="nofollow">
+    IMDB site for
+   Where the Buffalo Roam
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0120669/" rel="nofollow">
+    IMDB site for
+   Fear and Loathing in Las Vegas
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="external text" href="http://www.imdb.com/title/tt0376136/" rel="nofollow">
+    IMDB site for
+   The Rum Diary
+   </a>
+  </ins>
@@ -621,0 +2316,7 @@
+<p>
+ <ins>
+  <a class="new" href="/w/index.php?title=Template:Lived&amp;action=edit&amp;redlink=1" title="Template:Lived (page does not exist)">
+   Template:Lived
+  </a>
+ </ins>
+</p>
\ No newline at end of file
