--- 716414662
+++ 716415567
@@ -2779,2 +2779,8 @@
- <span class="mw-headline" id="2013.E2.80.93present:_3rdeyegirl_and_return_to_Warner_Bros.">
-  2013–present: 3rdeyegirl and return to Warner Bros.
+ <span class="mw-headline" id="2013.E2.80.932016:_3rdeyegirl_and_return_to_Warner_Bros.">
+  <ins>
+   2013–2016:
+  </ins>
+  <del>
+   2013–present:
+  </del>
+  3rdeyegirl and return to Warner Bros.
@@ -2890,0 +2897,13 @@
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  Prince died on 21 April 2016 at the age of 57.
+ He had a medical problem on 15 April and had, had to land with his private plane in Illinois when he was heading to one of his concerts.
+ </ins>
+</p>
+<h2>
@@ -4240 +4259 @@
-</ul>
+</ul>
\ No newline at end of file
