--- 716414662
+++ 716443829
@@ -5,8 +5,40 @@
- (born June 7, 1958), known by his
- <a href="/wiki/Mononymous_person" title="Mononymous person">
-  mononym
- </a>
- <b>
-  Prince
- </b>
- , is an American singer, songwriter, multi-instrumentalist and actor and has been a major figure in popular music for over three decades. Prince is renowned as an innovator and is widely known for his eclectic work, flamboyant stage presence, and wide
+ <ins>
+  (June 7, 1958 – April 21, 2016),
+ </ins>
+ <del>
+  (born June 7, 1958),
+ </del>
+ known by his
+ <ins>
+  <a href="/wiki/Stage_name" title="Stage name">
+   stage name
+  </a>
+  <b>
+   Prince
+  </b>
+  , was
+ </ins>
+ <del>
+  <a href="/wiki/Mononymous_person" title="Mononymous person">
+   mononym
+  </a>
+  <b>
+   Prince
+  </b>
+  , is
+ </del>
+ an American singer, songwriter, multi-instrumentalist and
+ <ins>
+  actor. Prince was
+ </ins>
+ <del>
+  actor and has been a major figure in popular music for over three decades. Prince is
+ </del>
+ renowned as an innovator and
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ widely known for his eclectic work, flamboyant stage presence, and wide
@@ -16 +48,8 @@
- . He is widely regarded as the pioneer of
+ . He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ widely regarded as the pioneer of
@@ -20 +59,7 @@
- . His music combines
+ . His music
+ <ins>
+  combined
+ </ins>
+ <del>
+  combines
+ </del>
@@ -154 +198,0 @@
-  <img alt="Prince logo.svg" data-file-height="153" data-file-width="130" height="15" src="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/13px-Prince_logo.svg.png" srcset="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/20px-Prince_logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/26px-Prince_logo.svg.png 2x" width="13"/>
@@ -164 +208,5 @@
- in 1998. In 2000, he began referring to himself as "Prince" again. He has released 15 albums since then, including his latest,
+ in 1998. In 2000, he began referring to himself as "Prince" again. He
+ <del>
+  has
+ </del>
+ released 15 albums since then, including his latest,
@@ -182 +230,5 @@
- Prince has sold over 100 million records worldwide, making him one of the
+ Prince
+ <del>
+  has
+ </del>
+ sold over 100 million records worldwide, making him one of the
@@ -187 +239,5 @@
- He has won seven
+ He
+ <del>
+  has
+ </del>
+ won seven
@@ -214,0 +271,9 @@
+ <ins>
+  He died at his Paisley Park recording studio and home in Chanhassen, Minnesota, near Minneapolis, on April 21, 2016, after suffering
+  <a href="/wiki/Influenza-like_illness" title="Influenza-like illness">
+   flu-like symptoms
+  </a>
+  for about 2 weeks prior to his death.
+ </ins>
+</p>
+<p>
@@ -224 +289,7 @@
- Prince was born in
+ Prince was born
+ <ins>
+  <b>
+   Prince Rogers Nelson
+  </b>
+ </ins>
+ in
@@ -243,4 +314,4 @@
- singer. Prince was named after his father, whose
- <a href="/wiki/Stage_name" title="Stage name">
-  stage name
- </a>
+ singer. Prince was named after his father, whose stage name
+ <del>
+  Link: /wiki/Stage_name
+ </del>
@@ -257,4 +328,10 @@
- In a
- <a class="mw-redirect" href="/wiki/Public_Broadcasting_Service" title="Public Broadcasting Service">
-  PBS
- </a>
+ In
+ <ins>
+  an
+ </ins>
+ <del>
+  a
+  <a class="mw-redirect" href="/wiki/Public_Broadcasting_Service" title="Public Broadcasting Service">
+   PBS
+  </a>
+ </del>
@@ -360,0 +438,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -379,0 +468,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -383,5 +482,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (October 2009)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -389,5 +488,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -624 +723,8 @@
- . The band released four albums between 1981 and 1990, with Prince writing and performing most of the instrumentation and backing vocals, with lead vocals by
+ . The band released four albums between 1981 and 1990, with Prince writing and performing most of the instrumentation and backing
+ <ins>
+  vocals (sometimes credited under the pseudonyms "Jamie Starr" or "The Starr Company"),
+ </ins>
+ <del>
+  vocals,
+ </del>
+ with lead vocals by
@@ -629,5 +735,5 @@
- <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
-  [
-  <i>
-   <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
-    <span title="This claim needs references to reliable sources. (October 2009)">
+ <del>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
@@ -635,5 +741,5 @@
-    </span>
-   </a>
-  </i>
-  ]
- </sup>
+    </a>
+   </i>
+   ]
+  </sup>
+ </del>
@@ -704 +810,5 @@
- on bass, and Dez Dickerson on guitar. Jill Jones, a backing singer, was also part of The Revolution line up for the
+ on bass, and
+ <a href="/wiki/Dez_Dickerson" title="Dez Dickerson">
+  Dez Dickerson
+ </a>
+ on guitar. Jill Jones, a backing singer, was also part of The Revolution line up for the
@@ -1121 +1231 @@
- <div class="thumbinner" style="width:222px;">
+ <div class="thumbinner" style="width:172px;">
@@ -1123 +1232,0 @@
-   <img alt="" class="thumbimage" data-file-height="1024" data-file-width="774" height="291" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/02/Prince_by_jimieye.jpg/220px-Prince_by_jimieye.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/02/Prince_by_jimieye.jpg/330px-Prince_by_jimieye.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/02/Prince_by_jimieye.jpg/440px-Prince_by_jimieye.jpg 2x" width="220"/>
@@ -1282 +1391 @@
- <div class="thumbinner" style="width:222px;">
+ <div class="thumbinner" style="width:172px;">
@@ -1284 +1392,0 @@
-   <img alt="" class="thumbimage" data-file-height="1918" data-file-width="1326" height="318" src="//upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Yellow_Cloud.jpg/220px-Yellow_Cloud.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Yellow_Cloud.jpg/330px-Yellow_Cloud.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Yellow_Cloud.jpg/440px-Yellow_Cloud.jpg 2x" width="220"/>
@@ -1459 +1566,0 @@
-  <img alt="Prince logo.svg" data-file-height="153" data-file-width="130" height="15" src="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/13px-Prince_logo.svg.png" srcset="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/20px-Prince_logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/26px-Prince_logo.svg.png 2x" width="13"/>
@@ -1781,2 +1888,8 @@
- <span class="mw-headline" id="2000.E2.80.9306:_Turnaround.2C_Musicology.2C_label_change_and_3121">
-  2000–06: Turnaround,
+ <span class="mw-headline" id="2000.E2.80.9307:_Turnaround.2C_Musicology.2C_label_change_and_3121">
+  <ins>
+   2000–07:
+  </ins>
+  <del>
+   2000–06:
+  </del>
+  Turnaround,
@@ -1827,0 +1941,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -1890,0 +2015,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -1957,0 +2093,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -2080,0 +2227,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -2148 +2305 @@
- <a class="mw-redirect" href="/wiki/Hollywood,_Los_Angeles,_California" title="Hollywood, Los Angeles, California">
+ <a class="mw-redirect" href="/wiki/Hollywood,_Los_Angeles" title="Hollywood, Los Angeles">
@@ -2155,0 +2313,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -2168,0 +2337,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -2211,0 +2391,11 @@
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
@@ -2226 +2416,13 @@
- . He performed weekly on Friday and Saturday nights until April 2007, when his contract with the Rio ended. On August 22, 2006, Prince released
+ . He performed weekly on Friday and Saturday nights until April 2007, when his contract with the Rio ended.
+ <ins>
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+ </ins>
+ On August 22, 2006, Prince released
@@ -2361 +2563,16 @@
- . Coincidentally, Miami had rain on the day of the Super Bowl, which was lit purple during the performance of "Purple Rain". He played on a large stage shaped as his symbol. The event was carried to 140 million television viewers, the largest audience of his life. On February 4, 2010, Billboard.com ranked the performance as the greatest Super Bowl performance ever.
+ . Coincidentally, Miami had rain on the day of the Super Bowl, which was lit purple during the performance of "Purple Rain". He played on a large stage shaped as his symbol. The event was carried to 140 million television viewers, the
+ <ins>
+  "biggest
+ </ins>
+ <del>
+  largest
+ </del>
+ audience of his
+ <ins>
+  life."
+ In 2015,
+ </ins>
+ <del>
+  life. On February 4, 2010,
+ </del>
+ Billboard.com ranked the performance as the greatest Super Bowl performance ever.
@@ -2503,3 +2720,11 @@
- <a class="new" href="/w/index.php?title=2008_Coachella_Valley_Music_and_Arts_Festival&amp;action=edit&amp;redlink=1" title="2008 Coachella Valley Music and Arts Festival (page does not exist)">
-  Coachella Festival 2008
- </a>
+ <ins>
+  2008
+  <a href="/wiki/Coachella_Valley_Music_and_Arts_Festival" title="Coachella Valley Music and Arts Festival">
+   Coachella Festival
+  </a>
+ </ins>
+ <del>
+  <a class="new" href="/w/index.php?title=2008_Coachella_Valley_Music_and_Arts_Festival&amp;action=edit&amp;redlink=1" title="2008 Coachella Valley Music and Arts Festival (page does not exist)">
+   Coachella Festival 2008
+  </a>
+ </del>
@@ -2707 +2932,5 @@
- Prince has refused access to the album to digital download services. He also closed his official website, LotusFlow3r.com. In an interview with the
+ Prince
+ <del>
+  has
+ </del>
+ refused access to the album to digital download services. He also closed his official website, LotusFlow3r.com. In an interview with the
@@ -2724 +2953,8 @@
- The second half of the tour has a new band, John Blackwell,
+ The second half of the tour
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ a new band, John Blackwell,
@@ -2779,2 +3015,8 @@
- <span class="mw-headline" id="2013.E2.80.93present:_3rdeyegirl_and_return_to_Warner_Bros.">
-  2013–present: 3rdeyegirl and return to Warner Bros.
+ <span class="mw-headline" id="2013.E2.80.9316:_3rdeyegirl_and_return_to_Warner_Bros.">
+  <ins>
+   2013–16:
+  </ins>
+  <del>
+   2013–present:
+  </del>
+  3rdeyegirl and return to Warner Bros.
@@ -2878 +3120,8 @@
- His latest album
+ His
+ <ins>
+  final
+ </ins>
+ <del>
+  latest
+ </del>
+ album
@@ -2885 +3134 @@
- <a class="mw-redirect" href="/wiki/Tidal" title="Tidal">
+ <a href="/wiki/Tidal_(service)" title="Tidal (service)">
@@ -2888 +3137,7 @@
- before released on CD and download on September 14, 2015.
+ released on CD and download on September
+ <ins>
+  14.
+ </ins>
+ <del>
+  14, 2015.
+ </del>
@@ -2900 +3155 @@
-<table class="plainlinks metadata ambox ambox-content ambox-BLP_unsourced" role="presentation">
+<table class="plainlinks metadata ambox ambox-content ambox-Refimprove" role="presentation">
@@ -2901,0 +3157,8 @@
+  <td class="mbox-text">
+   <span class="mbox-text-span">
+    <b>
+     <a href="/wiki/Wikipedia:Verifiability" title="Wikipedia:Verifiability">
+     </a>
+    </b>
+   </span>
+  </td>
@@ -2904 +3166,0 @@
-    <img alt="" data-file-height="40" data-file-width="40" height="40" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Ambox_important.svg/40px-Ambox_important.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Ambox_important.svg/60px-Ambox_important.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Ambox_important.svg/80px-Ambox_important.svg.png 2x" width="40"/>
@@ -2907,5 +3169,42 @@
-  <td class="mbox-text">
-   <span class="mbox-text-span">
-    This section of a
-    <a href="/wiki/Wikipedia:Biographies_of_living_persons" title="Wikipedia:Biographies of living persons">
-     biography of a living person
+  <ins>
+   This section
+    needs additional citations for
+     verification
+  </ins>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <del>
+   This section of a
+  </del>
+  <del>
+   <a href="/wiki/Wikipedia:Biographies_of_living_persons" title="Wikipedia:Biographies of living persons">
+    biography of a living person
+   </a>
+   does not
+  </del>
+  <del>
+   <a href="/wiki/Wikipedia:Citing_sources" title="Wikipedia:Citing sources">
+    include
+   </a>
+   any
+     references or sources
+  </del>
+  .
+  <span class="hide-when-compact">
+   Please help
+   <ins>
+    <a class="external text" href="//en.wikipedia.org/w/index.php?title=Prince_(musician)&amp;action=edit">
+     improve this article
+    </a>
+    by
+    <a href="/wiki/Help:Introduction_to_referencing_with_Wiki_Markup/1" title="Help:Introduction to referencing with Wiki Markup/1">
+     adding citations to reliable sources
+    </a>
+    . Unsourced material may be challenged and removed.
+   </ins>
+   <del>
+    by adding
+    <a href="/wiki/Wikipedia:Identifying_reliable_sources" title="Wikipedia:Identifying reliable sources">
+     reliable sources
@@ -2912,0 +3212 @@
+    . Contentious material about living people that is unsourced or poorly sourced
@@ -2914,8 +3214 @@
-     does not
-     <a href="/wiki/Wikipedia:Citing_sources" title="Wikipedia:Citing sources">
-      include
-     </a>
-     any
-     <a href="/wiki/Wikipedia:Verifiability" title="Wikipedia:Verifiability">
-      references or sources
-     </a>
+     must be removed immediately
@@ -2924,50 +3217 @@
-    <span class="hide-when-compact">
-     Please help by adding
-     <a href="/wiki/Wikipedia:Identifying_reliable_sources" title="Wikipedia:Identifying reliable sources">
-      reliable sources
-     </a>
-     . Contentious material about living people that is unsourced or poorly sourced
-     <b>
-      must be removed immediately
-     </b>
-     .
-     <br/>
-     <small>
-      <span class="plainlinks">
-       <i>
-        Find sources:
-       </i>
-       <a class="external text" href="//www.google.com/search?as_eq=wikipedia&amp;q=%22Prince%22+musician&amp;num=50" rel="nofollow">
-        "Prince" musician
-       </a>
-       –
-       <a class="external text" href="//www.google.com/search?q=%22Prince%22+musician&amp;tbm=nws" rel="nofollow">
-        news
-       </a>
-       <b>
-        ·
-       </b>
-       <a class="external text" href="//www.google.com/search?&amp;q=%22Prince%22+musician+site:news.google.com/newspapers&amp;source=newspapers" rel="nofollow">
-        newspapers
-       </a>
-       <b>
-        ·
-       </b>
-       <a class="external text" href="//www.google.com/search?tbs=bks:1&amp;q=%22Prince%22+musician" rel="nofollow">
-        books
-       </a>
-       <b>
-        ·
-       </b>
-       <a class="external text" href="//scholar.google.com/scholar?q=%22Prince%22+musician" rel="nofollow">
-        scholar
-       </a>
-       <b>
-        ·
-       </b>
-       <a class="external text" href="https://www.jstor.org/action/doBasicSearch?Query=%22Prince%22+musician&amp;acc=on&amp;wc=on" rel="nofollow">
-        JSTOR
-       </a>
-      </span>
-     </small>
-    </span>
+    <br/>
@@ -2975,3 +3219,36 @@
-     <i>
-      (December 2014)
-     </i>
+     <span class="plainlinks">
+      <i>
+       Find sources:
+      </i>
+      <a class="external text" href="//www.google.com/search?as_eq=wikipedia&amp;q=%22Prince%22+musician&amp;num=50" rel="nofollow">
+       "Prince" musician
+      </a>
+      –
+      <a class="external text" href="//www.google.com/search?q=%22Prince%22+musician&amp;tbm=nws" rel="nofollow">
+       news
+      </a>
+      <b>
+       ·
+      </b>
+      <a class="external text" href="//www.google.com/search?&amp;q=%22Prince%22+musician+site:news.google.com/newspapers&amp;source=newspapers" rel="nofollow">
+       newspapers
+      </a>
+      <b>
+       ·
+      </b>
+      <a class="external text" href="//www.google.com/search?tbs=bks:1&amp;q=%22Prince%22+musician" rel="nofollow">
+       books
+      </a>
+      <b>
+       ·
+      </b>
+      <a class="external text" href="//scholar.google.com/scholar?q=%22Prince%22+musician" rel="nofollow">
+       scholar
+      </a>
+      <b>
+       ·
+      </b>
+      <a class="external text" href="https://www.jstor.org/action/doBasicSearch?Query=%22Prince%22+musician&amp;acc=on&amp;wc=on" rel="nofollow">
+       JSTOR
+      </a>
+     </span>
@@ -2979 +3256 @@
-    <small class="hide-when-compact">
+    <small>
@@ -2981,5 +3258 @@
-      (
-      <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
-       Learn how and when to remove this template message
-      </a>
-      )
+      (December 2014)
@@ -2988,2 +3261,11 @@
-   </span>
-  </td>
+   </del>
+  </span>
+  <small class="hide-when-compact">
+   <i>
+    (
+    <a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">
+     Learn how and when to remove this template message
+    </a>
+    )
+   </i>
+  </small>
@@ -2993,4 +3275,62 @@
- In addition to his singing abilities (which encompass a wide range from falsetto to baritone and rapid, flamboyant shifts of register, as well as diverse characterisation) Prince is one of pop music's most prominent multi-instrumentalists. The precise number of instruments he is capable of playing remains obscure, partly due to the active mythologizing of his abilities and working practices (such as the claim that he had played "all 27 instruments" on his debut album, which appears to have included various different kinds of guitar, keyboard and synthesizer in the total).
-</p>
-<p>
- As a live performer he tends to focus on guitar, piano, lead vocals and occasional bass, drums and harmonica. On recordings he has also played assorted keyboards and synthesizers, bass guitar, drums, various percussion instruments and saxophone, as well as mastering both drum and synthesizer programming. Prince has mentioned learning and performing simple parts on particular instruments that he does not otherwise play (such as concert harp) in order to serve song arrangements on albums.
+ In addition to his singing abilities (which
+ <ins>
+  encompassed
+ </ins>
+ <del>
+  encompass
+ </del>
+ a wide range from falsetto to baritone and rapid,
+ <ins>
+  seemingly effortless shifts of register
+ ) Prince was renowned as a multi-instrumentalist.
+ </ins>
+ <del>
+  flamboyant shifts of register, as well as diverse characterisation) Prince is one of pop music's most prominent multi-instrumentalists.
+ </del>
+ The precise number of instruments he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ capable of playing remains
+ <ins>
+  obscure. On
+ </ins>
+ <del>
+  obscure, partly due to the active mythologizing of his abilities and working practices (such as the claim that he had played "all 27 instruments" on
+ </del>
+ his debut album,
+ <ins>
+  he was credited with having played 27 instruments,
+ which based on the album credits included various types of bass,
+ </ins>
+ <del>
+  which appears to have included various different kinds of guitar,
+ </del>
+ keyboard and synthesizer in the
+ <ins>
+  total.
+ </ins>
+ <del>
+  total).
+ </del>
+</p>
+<p>
+ As a live performer he
+ <ins>
+  generally focused
+ </ins>
+ <del>
+  tends to focus
+ </del>
+ on guitar, piano, lead vocals and occasional bass, drums and harmonica. On recordings he
+ <del>
+  has
+ </del>
+ also played assorted keyboards and synthesizers, bass guitar, drums, various percussion instruments and saxophone, as well as mastering both drum and synthesizer programming. Prince
+ <del>
+  has
+ </del>
+ mentioned learning and performing simple parts on particular instruments that he does not otherwise play (such as concert harp) in order to serve song arrangements on albums.
@@ -3021 +3361,12 @@
- Prince has used pseudonyms to separate himself from the music (either his own or that of others) for which he has had input; "I was just getting tired of seeing my name," he said, "If you give away an idea, you still own that idea. In fact, giving it away strengthens it. Why do people feel they have to take credit for everything they do? Ego, that's the only reason."
+ Prince
+ <del>
+  has
+ </del>
+ used pseudonyms to separate himself from the music (either his own or that of others) for which he
+ <ins>
+  had input during his career;
+ </ins>
+ <del>
+  has had input;
+ </del>
+ "I was just getting tired of seeing my name," he said, "If you give away an idea, you still own that idea. In fact, giving it away strengthens it. Why do people feel they have to take credit for everything they do? Ego, that's the only reason."
@@ -3030 +3381 @@
- (for the songs he wrote for the
+ (for the songs he wrote for
@@ -3031,0 +3383,6 @@
+  <ins>
+   The
+  </ins>
+  <del>
+   the
+  </del>
@@ -3083,4 +3440,4 @@
- On September 14, 2007, Prince announced that he was going to sue
- <a href="/wiki/YouTube" title="YouTube">
-  YouTube
- </a>
+ On September 14, 2007, Prince announced that he was going to sue YouTube
+ <del>
+  Link: /wiki/YouTube
+ </del>
@@ -3170 +3527,8 @@
- Prince resides near
+ Prince
+ <ins>
+  resided
+ </ins>
+ <del>
+  resides
+ </del>
+ near
@@ -3174,2 +3538,14 @@
- , Minnesota.
- Over the years Prince has been romantically linked with many celebrities, including
+ <ins>
+  at the time of his death.
+ </ins>
+ <del>
+  , Minnesota.
+ </del>
+ Over the years Prince
+ <ins>
+  was
+ </ins>
+ <del>
+  has been
+ </del>
+ romantically linked with many celebrities, including
@@ -3226 +3602,7 @@
- He married his backup singer and dancer
+ <ins>
+  When he was 37, he married his 22-year-old
+ </ins>
+ <del>
+  He married his
+ </del>
+ backup singer and dancer
@@ -3235,2 +3617,15 @@
- Prince and Mayte divorced in 1999. In 2001, Prince married Manuela Testolini in a private ceremony. Testolini filed for divorce in May 2006.
- He also had a short-term relationship with protégée
+ Prince and Mayte divorced in 1999. In 2001, Prince married Manuela Testolini in a private
+ <ins>
+  ceremony; she
+ </ins>
+ <del>
+  ceremony. Testolini
+ </del>
+ filed for divorce in May 2006.
+ <ins>
+  Prince
+ </ins>
+ <del>
+  He
+ </del>
+ also had a short-term relationship with protégée
@@ -3243 +3638,7 @@
- Prince became one of
+ Prince
+ <ins>
+  joined the
+ </ins>
+ <del>
+  became one of
+ </del>
@@ -3265 +3666,8 @@
- ," he explained. He attends meetings at a local
+ ," he explained. He
+ <ins>
+  attended
+ </ins>
+ <del>
+  attends
+ </del>
+ meetings at a local
@@ -3269,2 +3677,27 @@
- and occasionally knocks on people's doors to discuss his faith.
- Prince has reportedly needed double hip-replacement surgery since 2005 but won't undergo the operation unless it is a
+ and occasionally
+ <ins>
+  knocked
+ </ins>
+ <del>
+  knocks
+ </del>
+ on people's doors to discuss his faith.
+ Prince
+ <del>
+  has
+ </del>
+ reportedly needed double hip-replacement surgery since 2005 but
+ <ins>
+  wouldn't
+ </ins>
+ <del>
+  won't
+ </del>
+ undergo the operation unless it
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a
@@ -3279 +3712,8 @@
- The condition is rumored to be aggravated by repeated onstage dancing in high-heeled boots.
+ The condition
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ rumored to be aggravated by repeated onstage dancing in high-heeled boots.
@@ -3283,5 +3723,11 @@
- Prince is a
- <a class="mw-redirect" href="/wiki/Vegan" title="Vegan">
-  vegan
- </a>
- .
+ Prince
+ <ins>
+  was a vegetarian.
+ </ins>
+ <del>
+  is a
+  <a class="mw-redirect" href="/wiki/Vegan" title="Vegan">
+   vegan
+  </a>
+  .
+ </del>
@@ -3305,6 +3751,19 @@
- Since 2008, Prince has been managed by UK-based Kiran Sharma.
-</p>
-<p>
- <a class="mw-redirect" href="/wiki/Sinead_O%27Connor" title="Sinead O'Connor">
-  Sinead O'Connor
- </a>
+ <ins>
+  Beginning in 2008, Prince was
+ </ins>
+ <del>
+  Since 2008, Prince has been
+ </del>
+ managed by UK-based Kiran Sharma.
+</p>
+<p>
+ <ins>
+  <a href="/wiki/Sin%C3%A9ad_O%27Connor" title="Sinéad O'Connor">
+   Sinéad O'Connor
+  </a>
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/Sinead_O%27Connor" title="Sinead O'Connor">
+   Sinead O'Connor
+  </a>
+ </del>
@@ -3319,4 +3778,25 @@
- ", a song that Prince wrote which became a worldwide hit for O'Connor in 1990. O'Connor said, "I made it without him. I'd never met him. He summoned me to his house—and it's foolish to do this to an Irish woman—he said he didn't like me saying bad words in interviews. So I told him to fuck off." O'Connor alleged that the row became physical. "He got quite violent. I had to escape out of his house at 5 in the morning. He packed a bigger punch than mine."
-</p>
-<p>
- In 2016, Prince announced that he is writing a memoir, tentatively titled
+ ", a song that Prince
+ <ins>
+  wrote,
+ </ins>
+ <del>
+  wrote which
+ </del>
+ became a worldwide hit for O'Connor in 1990. O'Connor said, "I made it without him. I'd never met him. He summoned me to his house—and it's foolish to do this to an Irish woman—he said he didn't like me saying bad words in interviews. So I told him to fuck off." O'Connor
+ <ins>
+  alleged,
+ </ins>
+ <del>
+  alleged that the row became physical.
+ </del>
+ "He got quite violent. I had to escape out of his house at 5 in the morning. He packed a bigger punch than mine."
+</p>
+<p>
+ In 2016, Prince announced that he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ writing a memoir, tentatively titled
@@ -3328,0 +3809,51 @@
+ <ins>
+  <span class="mw-headline" id="Illness_and_death">
+   Illness and death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  Prince postponed two performances from his Piano &amp; A Microphone Tour on April 7, 2016, at the
+  <a href="/wiki/Fox_Theatre_(Atlanta)" title="Fox Theatre (Atlanta)">
+   Fox Theatre
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/Atlanta,_Georgia" title="Atlanta, Georgia">
+   Atlanta, Georgia
+  </a>
+  , stating that he was "battling the flu."
+ On April 15, the artist's private jet was forced to land in
+  <a href="/wiki/Illinois" title="Illinois">
+   Illinois
+  </a>
+  so that he could seek medical treatment for
+  <a class="mw-redirect" href="/wiki/Flu" title="Flu">
+   flu
+  </a>
+  -like symptoms, after performing the rescheduled Atlanta concert the day before.
+ He was seen in public the following evening in
+  <a href="/wiki/Minneapolis" title="Minneapolis">
+   Minneapolis
+  </a>
+  , when he stopped at the
+  <a href="/wiki/Electric_Fetus" title="Electric Fetus">
+   Electric Fetus
+  </a>
+  on
+  <a href="/wiki/Record_Store_Day" title="Record Store Day">
+   Record Store Day
+  </a>
+  , and made a brief appearance at a dance party at his
+  <a class="mw-redirect" href="/wiki/Paisley_Park" title="Paisley Park">
+   Paisley Park
+  </a>
+  recording studio complex in
+  <a href="/wiki/Chanhassen,_Minnesota" title="Chanhassen, Minnesota">
+   Chanhassen, Minnesota
+  </a>
+  , stating that he was feeling okay.
+ Prince died on April 21, 2016 10:05 am, at the age of 57, having been found unresponsive in an elevator that morning at his Paisley Park complex.
+ </ins>
+</p>
+<h2>
@@ -3356 +3887 @@
-<div class="div-col columns column-count column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2;">
+<div class="div-col columns column-width" style="-moz-column-width: 20em; -webkit-column-width: 20em; column-width: 20em;">
@@ -3464 +3994,0 @@
-    <img alt="Prince logo.svg" data-file-height="153" data-file-width="130" height="18" src="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/15px-Prince_logo.svg.png" srcset="//upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/23px-Prince_logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/a/af/Prince_logo.svg/30px-Prince_logo.svg.png 2x" width="15"/>
@@ -3660 +4190 @@
-    <a class="mw-redirect" href="/wiki/HITnRUN_Phase_One" title="HITnRUN Phase One">
+    <a href="/wiki/Hit_n_Run_Phase_One" title="Hit n Run Phase One">
@@ -3668 +4198 @@
-    <a class="mw-redirect" href="/wiki/HITnRUN_Phase_Two" title="HITnRUN Phase Two">
+    <a href="/wiki/Hit_n_Run_Phase_Two" title="Hit n Run Phase Two">
@@ -4240 +4770 @@
-</ul>
+</ul>
\ No newline at end of file
