--- 298631851
+++ 298648459
@@ -5 +5,8 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958 – June 25, 2009
+ ) was
+ </ins>
+ <del>
+  (born August 29, 1958) is
+ </del>
+ an American recording artist, entertainer, and businessman. The seventh child of the
@@ -119 +126,5 @@
- Jackson has donated and raised millions of dollars for beneficial causes through his
+ Jackson
+ <del>
+  has
+ </del>
+ donated and raised millions of dollars for beneficial causes through his
@@ -135 +146,5 @@
- , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
+ , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer
+ <del>
+  has
+ </del>
+ experienced
@@ -139 +154,21 @@
- since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
+ <ins>
+  from
+ </ins>
+ <del>
+  since
+ </del>
+ the early 1990s
+ <ins>
+  until his death, as well as
+ </ins>
+ <del>
+  and
+ </del>
+ conflicting reports regarding the state of his
+ <ins>
+  finances.
+ </ins>
+ <del>
+  finances since the late 1990s.
+ </del>
+ Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
@@ -172 +207,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -178,0 +218,40 @@
+ <ins>
+  On June 25, 2009, he collapsed at his home in Los Angeles.
+ After being taken to the hospital in a coma,
+  <sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">
+   [
+   <i>
+    <a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">
+     citation needed
+    </a>
+   </i>
+   ]
+  </sup>
+  Jackson was reportedly pronounced dead.
+ ref&gt;
+  <cite class="citation web">
+   <a class="external text" href="http://news.yahoo.com/s/ap/us_obit_michael_jackson" rel="nofollow">
+    "AP Source: Michael Jackson dies in LA hospital"
+   </a>
+   .
+   <i>
+    <a href="/wiki/Associated_Press" title="Associated Press">
+     Associated Press
+    </a>
+   </i>
+   <span class="reference-accessdate">
+    . Retrieved
+    <span class="nowrap">
+     2009-06-25
+    </span>
+   </span>
+   .
+  </cite>
+  <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMichael+Jackson&amp;rft.atitle=AP+Source%3A+Michael+Jackson+dies+in+LA+hospital&amp;rft.genre=unknown&amp;rft_id=http%3A%2F%2Fnews.yahoo.com%2Fs%2Fap%2Fus_obit_michael_jackson&amp;rft.jtitle=Associated+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal">
+   <span style="display:none;">
+   </span>
+  </span>
+  &lt;/ref&gt;
+ </ins>
+</p>
+<p>
@@ -221 +300,8 @@
- he is the seventh of nine children. His siblings are
+ he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the seventh of nine children. His siblings are
@@ -926 +1012,8 @@
- ; the latter is in
+ ; the latter
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ in
@@ -930 +1023,8 @@
- in Jackson's case, and both illnesses make him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
+ in Jackson's case, and both illnesses
+ <ins>
+  made
+ </ins>
+ <del>
+  make
+ </del>
+ him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
@@ -1950,2 +2050,12 @@
- <span class="mw-headline" id="2008_onward:_Milestones.2C_real_estate_and_return_to_live_performance">
-  2008 onward: Milestones, real estate and return to live performance
+ <span class="mw-headline" id="2008.E2.80.932009:_Milestones.2C_real_estate_and_planned_return_to_live_performance">
+  <ins>
+   2008–2009:
+  </ins>
+  <del>
+   2008 onward:
+  </del>
+  Milestones, real estate and
+  <ins>
+   planned
+  </ins>
+  return to live performance
@@ -2043 +2153,8 @@
- From July 13, 2009 to March 6, 2010, Jackson is scheduled to perform
+ From July 13, 2009 to March 6, 2010, Jackson
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ scheduled to perform
@@ -2063 +2180,19 @@
- On June 25, 2009,
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<ins>
+ In the afternoon of
+</ins>
+<del>
+ On
+</del>
+ June 25, 2009, <ins>
+ paramedics responded to a call to Jackson's house. He reportedly was not breathing at the time that paramedics arrived, and CPR was quickly performed.
+ He died shortly after being
+</ins>
+<del>
@@ -2067,4 +2202,12 @@
- reported that Jackson has been rushed to the hospital and has suffered
- <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
-  cardiac arrest
- </a>
+ reported that Jackson has been
+</del>
+ rushed to the <ins>
+ hospital.
+ The cause of death is suspected to be
+</ins>
+<del>
+ hospital and has suffered
+</del>
+<a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+ cardiac arrest
+</a>
@@ -2071,0 +2215 @@
+ <del>
@@ -2084 +2228 @@
-</p>
+</del>
@@ -2100 +2244,5 @@
- asserts that throughout his solo career, Jackson's versatility has allowed him to experiment with various themes and genres.
+ asserts that throughout his solo career, Jackson's versatility
+ <del>
+  has
+ </del>
+ allowed him to experiment with various themes and genres.
@@ -2881 +3029,8 @@
- He is a double-inductee of the
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a double-inductee of the
@@ -2925 +3080,8 @@
- He is characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
@@ -2961 +3123,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -3120 +3286 @@
-</ul>
+</ul>
\ No newline at end of file
