--- 298631851
+++ 298688671
@@ -5 +5,8 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958 – June 25, 2009
+ ) was
+ </ins>
+ <del>
+  (born August 29, 1958) is
+ </del>
+ an American recording artist, entertainer, and businessman. The seventh child of the
@@ -21 +28,7 @@
- have become some of the world's best-selling records:
+ <ins>
+  are among
+ </ins>
+ <del>
+  have become some of
+ </del>
+ the world's best-selling records:
@@ -104,0 +118,3 @@
+ <ins>
+  many
+ </ins>
@@ -119,25 +135,27 @@
- Jackson has donated and raised millions of dollars for beneficial causes through his
- <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
-  foundation
- </a>
- , charity
- <a href="/wiki/Single_(music)" title="Single (music)">
-  singles
- </a>
- and support of 39 charities. Other aspects of his personal life, including his
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  changing appearance
- </a>
- and behavior, generated significant controversy, damaging his public image. Though he was accused of
- <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
-  child sexual abuse in 1993
- </a>
- , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  health concerns
- </a>
- since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  tried and acquitted
- </a>
- of further sexual abuse allegations and several other charges.
+ <del>
+  Jackson has donated and raised millions of dollars for beneficial causes through his
+  <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
+   foundation
+  </a>
+  , charity
+  <a href="/wiki/Single_(music)" title="Single (music)">
+   singles
+  </a>
+  and support of 39 charities. Other aspects of his personal life, including his
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   changing appearance
+  </a>
+  and behavior, generated significant controversy, damaging his public image. Though he was accused of
+  <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
+   child sexual abuse in 1993
+  </a>
+  , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   health concerns
+  </a>
+  since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   tried and acquitted
+  </a>
+  of further sexual abuse allegations and several other charges.
+ </del>
@@ -172 +190,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -176,0 +199,7 @@
+ <ins>
+  Jackson lived to the age of 50; he passed away on June 25, 2009 due to
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  .
+ </ins>
@@ -221 +250,8 @@
- he is the seventh of nine children. His siblings are
+ he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the seventh of nine children. His siblings are
@@ -265 +301,8 @@
- One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years afterward, Jackson suffered nightmares about being kidnapped from his bedroom.
+ One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson suffered nightmares about being kidnapped from his bedroom.
@@ -273 +316 @@
- <a class="mw-redirect mw-disambig" href="/wiki/Regurgitate" title="Regurgitate">
+ <a href="/wiki/Vomiting" title="Vomiting">
@@ -608 +651,6 @@
- ".
+ <ins>
+  " being on the charts at the same time.
+ </ins>
+ <del>
+  ".
+ </del>
@@ -753 +801,7 @@
-   , 1984.
+   ,
+   <ins>
+    1984
+   </ins>
+   <del>
+    1984.
+   </del>
@@ -764 +818,8 @@
- Jackson had his third rhinoplasty shortly afterward and grew self conscious about his appearance.
+ Jackson had his third rhinoplasty shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
+ and grew self conscious about his appearance.
@@ -776 +837,8 @@
- Jackson won eight awards during the 1984 Grammys. Unlike later albums,
+ Jackson won eight awards during the
+ <a class="mw-redirect" href="/wiki/1984_Grammy_Awards" title="1984 Grammy Awards">
+  1984 Grammy Awards
+ </a>
+ <del>
+  Grammys.
+ </del>
+ Unlike later albums,
@@ -811 +879,7 @@
- ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly afterward,
+ ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
@@ -899 +973,14 @@
-   Jackson two years after he was diagnosed with vitiligo, pictured in the early stages of the disease.
+   Jackson two years after he was diagnosed with vitiligo,
+   <ins>
+    here
+   </ins>
+   <del>
+    pictured in
+   </del>
+   the early stages of the
+   <ins>
+    disease
+   </ins>
+   <del>
+    disease.
+   </del>
@@ -908 +995,15 @@
- , an act which extended his eccentric persona. In 2003 the singer claimed that Bubbles shared his toilet, and cleaned his bedroom.
+ , an act which extended his eccentric persona. In
+ <ins>
+  2003,
+ </ins>
+ <del>
+  2003
+ </del>
+ the singer claimed that Bubbles shared his
+ <ins>
+  toilet
+ </ins>
+ <del>
+  toilet,
+ </del>
+ and cleaned his bedroom.
@@ -926 +1027,8 @@
- ; the latter is in
+ ; the latter
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ in
@@ -930,2 +1038,27 @@
- in Jackson's case, and both illnesses make him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
- The structure of his face has changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
+ in Jackson's case, and both illnesses
+ <ins>
+  made
+ </ins>
+ <del>
+  make
+ </del>
+ him sensitive to sunlight. The treatments he
+ <ins>
+  used
+ </ins>
+ <del>
+  uses
+ </del>
+ for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he
+ <ins>
+  could
+ </ins>
+ <del>
+  can
+ </del>
+ appear very pale.
+ The structure of his face
+ <del>
+  has
+ </del>
+ changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
@@ -939 +1072,7 @@
- Some medical professionals have publicly stated their belief that the singer has
+ Some medical professionals have publicly stated their belief that the singer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
@@ -947 +1086,7 @@
-  Why not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
+  <ins>
+   "Why
+  </ins>
+  <del>
+   Why
+  </del>
+  not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
@@ -955 +1100,22 @@
-  . But if I, Michael Jackson, were to say, "I'm an alien from Mars and I eat live chickens and do a voodoo dance at midnight", people would say, "Oh, man, that Michael Jackson is
+  . But if I, Michael Jackson, were to say,
+  <ins>
+   'I'm
+  </ins>
+  <del>
+   "I'm
+  </del>
+  an alien from Mars and I eat live chickens and do a voodoo dance at
+  <ins>
+   midnight,'
+  </ins>
+  <del>
+   midnight",
+  </del>
+  people would say,
+  <ins>
+   'Oh,
+  </ins>
+  <del>
+   "Oh,
+  </del>
+  man, that Michael Jackson is
@@ -959 +1125,7 @@
-  . He's cracked up. You can't believe a damn word that comes out of his mouth".
+  . He's cracked up. You can't believe a damn word that comes out of his
+  <ins>
+   mouth.'"
+  </ins>
+  <del>
+   mouth".
+  </del>
@@ -1019 +1191,14 @@
-   A gold plated military style jacket with belt, worn by Jackson in the
+   <ins>
+    Jackson wore a gold-plated
+   </ins>
+   <del>
+    A gold plated
+   </del>
+   military style jacket with
+   <ins>
+    belt
+   </ins>
+   <del>
+    belt, worn by Jackson
+   </del>
+   in the
@@ -1135 +1320,8 @@
- Shortly afterward, Jackson became the first Westerner to appear in a television advert for Russia.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson became the first Westerner to appear in a television advert for Russia.
@@ -1175 +1367,15 @@
- In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the time.
+ In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the
+ <ins>
+  time, displacing
+  <a href="/wiki/Neil_Diamond" title="Neil Diamond">
+   Neil Diamond
+  </a>
+  's renewal contract with
+  <a href="/wiki/Columbia_Records" title="Columbia Records">
+   Columbia Records
+  </a>
+  .
+ </ins>
+ <del>
+  time.
+ </del>
@@ -1186 +1392,8 @@
- has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album off all time.
+ has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album
+ <ins>
+  of
+ </ins>
+ <del>
+  off
+ </del>
+ all time.
@@ -1313 +1526,8 @@
- The friendship between Jackson and Evan Chandler broke down. Sometime afterward, Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
+ The friendship between Jackson and Evan Chandler broke down. Sometime
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
@@ -1391 +1611,8 @@
- Shortly afterward, she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
@@ -1484 +1711,8 @@
- Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration With Vocals".
+ Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration
+ <ins>
+  with
+ </ins>
+ <del>
+  With
+ </del>
+ Vocals".
@@ -1534,2 +1768,17 @@
- Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them to.
- The couple divorced in 1999, with Rowe giving full custody rights of the children to Jackson; they still remain friends.
+ Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them
+ <ins>
+  to do so.
+ After the
+ </ins>
+ <del>
+  to.
+ The
+ </del>
+ couple divorced in 1999, with Rowe giving full custody rights of the children to
+ <ins>
+  Jackson, they remained
+ </ins>
+ <del>
+  Jackson; they still remain
+ </del>
+ friends.
@@ -1726 +1975,4 @@
- . The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
+ <del>
+  .
+ </del>
+ The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
@@ -1806 +2058,5 @@
- Shortly after the documentary aired, Jackson was charged with seven counts of child sexual abuse and two counts of
+ Shortly after the documentary aired, Jackson was charged with seven counts of
+ <a href="/wiki/Child_sexual_abuse" title="Child sexual abuse">
+  child sexual abuse
+ </a>
+ and two counts of
@@ -1810 +2066,11 @@
- in order to commit that felony; all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
+ in order to commit that
+ <ins>
+  <a href="/wiki/Felony" title="Felony">
+   felony
+  </a>
+  ;
+ </ins>
+ <del>
+  felony;
+ </del>
+ all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
@@ -1818 +2084,14 @@
-   Fans show their support for Jackson after the entertainer is accused of child molestation in 2003.
+   Fans
+   <ins>
+    showing
+   </ins>
+   <del>
+    show
+   </del>
+   their support for Jackson after the entertainer is accused of child molestation in
+   <ins>
+    2003
+   </ins>
+   <del>
+    2003.
+   </del>
@@ -1823 +2102,5 @@
- Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend Elizabeth Taylor defended him on
+ Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend
+ <a href="/wiki/Elizabeth_Taylor" title="Elizabeth Taylor">
+  Elizabeth Taylor
+ </a>
+ defended him on
@@ -1829,2 +2112,26 @@
- , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of Walt Disney. There was nothing odd about it."
- During the investigation, Jackson's profile was examined by a mental health professional called Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a pedophile.
+ , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of
+ <a class="mw-redirect" href="/wiki/Walt_Disney_Motion_Pictures_Group" title="Walt Disney Motion Pictures Group">
+  Walt Disney
+ </a>
+ <del>
+  Disney.
+ </del>
+ There was nothing odd about it."
+ During the investigation, Jackson's profile was examined by
+ <del>
+  a
+ </del>
+ mental health professional
+ <del>
+  called
+ </del>
+ Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a
+ <ins>
+  <a class="mw-redirect" href="/wiki/Pedophile" title="Pedophile">
+   pedophile
+  </a>
+  .
+ </ins>
+ <del>
+  pedophile.
+ </del>
@@ -1834,3 +2141,5 @@
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  People v. Jackson
- </a>
+ <i>
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   People v. Jackson
+  </a>
+ </i>
@@ -1879 +2188,7 @@
- format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set afterward.
+ format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set
+ <ins>
+  afterwards.
+ </ins>
+ <del>
+  afterward.
+ </del>
@@ -1950,2 +2265,8 @@
- <span class="mw-headline" id="2008_onward:_Milestones.2C_real_estate_and_return_to_live_performance">
-  2008 onward: Milestones, real estate and return to live performance
+ <span class="mw-headline" id="2008.E2.80.932009:_Milestones.2C_real_estate.2C_planned_return_to_live_performance">
+  <ins>
+   2008–2009: Milestones, real estate, planned
+  </ins>
+  <del>
+   2008 onward: Milestones, real estate and
+  </del>
+  return to live performance
@@ -1971 +2292 @@
-  <a class="mw-disambig" href="/wiki/Thriller" title="Thriller">
+  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
@@ -1981 +2302,8 @@
- , comprising of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
+ ,
+ <ins>
+  comprised
+ </ins>
+ <del>
+  comprising
+ </del>
+ of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
@@ -2015 +2343,8 @@
- stood at 688,000 copies, making it the best selling catalog album of 2008.
+ stood at 688,000 copies, making it the
+ <ins>
+  best-selling
+ </ins>
+ <del>
+  best selling
+ </del>
+ catalog album of 2008.
@@ -2018 +2353 @@
- To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
+ To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
@@ -2043 +2378,7 @@
- From July 13, 2009 to March 6, 2010, Jackson is scheduled to perform
+ <ins>
+  Prior to his death, Jackson had been
+ </ins>
+ <del>
+  From July 13, 2009 to March 6, 2010, Jackson is
+ </del>
+ scheduled to perform
@@ -2045 +2386,8 @@
-  50 sell out concerts
+  50
+  <ins>
+   sold-out
+  </ins>
+  <del>
+   sell out
+  </del>
+  concerts
@@ -2055 +2403,7 @@
- . According to Jackson's website, tickets sales for the concerts broke several records. During a prior press conference, Jackson made suggestions of possible retirement.
+ <ins>
+  , from July 13, 2009 to March 6, 2010. During a publicity
+ </ins>
+ <del>
+  . According to Jackson's website, tickets sales for the concerts broke several records. During a prior
+ </del>
+ press conference, Jackson made suggestions of possible retirement.
@@ -2060 +2414,11 @@
- stated that the first 10 dates alone would earn the singer approximately £50 million.
+ <ins>
+  , had
+ </ins>
+ stated that the first 10 dates alone would
+ <ins>
+  have earned
+ </ins>
+ <del>
+  earn
+ </del>
+ the singer approximately £50 million.
@@ -2063 +2427,71 @@
- On June 25, 2009,
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h3>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    Wikinews has related news:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikinews.org/wiki/Singer_and_songwriter_Michael_Jackson_dies" title="wikinews:Singer and songwriter Michael Jackson dies">
+       Singer and songwriter Michael Jackson dies
+      </a>
+     </b>
+    </i>
+   </ins>
+  </td>
+ </tr>
+</table>
+<ins>
+ On the morning of
+</ins>
+<del>
+ On
+</del>
+ June 25, 2009, <ins>
+ Jackson reportedly collapsed at a rented home in
+ <a class="mw-redirect" href="/wiki/Holmby_Hills" title="Holmby Hills">
+  Holmby Hills
+ </a>
+ in
+ <a href="/wiki/Los_Angeles" title="Los Angeles">
+  Los Angeles
+ </a>
+ .
+ <a href="/wiki/Los_Angeles_Fire_Department" title="Los Angeles Fire Department">
+  Los Angeles Fire Department
+ </a>
+ paramedics responded to a
+ <a href="/wiki/9-1-1" title="9-1-1">
+  911
+ </a>
+ call at 12:21 pm local (
+ <a class="mw-redirect" href="/wiki/Pacific_time" title="Pacific time">
+  Pacific
+ </a>
+ ) time
+ and arrived at 12:30 pm. Jackson was reportedly not breathing and
+ <a href="/wiki/Cardiopulmonary_resuscitation" title="Cardiopulmonary resuscitation">
+  CPR
+ </a>
+ was quickly performed.
+ He fell into a
+ <a href="/wiki/Coma" title="Coma">
+  coma
+ </a>
+ and died shortly after being rushed to
+ <a href="/wiki/Ronald_Reagan_UCLA_Medical_Center" title="Ronald Reagan UCLA Medical Center">
+  UCLA Medical Center
+ </a>
+ .
+ The cause of death is reported to be
+</ins>
+<del>
@@ -2068,4 +2502,22 @@
- <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
-  cardiac arrest
- </a>
- .
+</del>
+<a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+ cardiac arrest
+</a>
+ .
+ <ins>
+ Jackson was pronounced dead at about 2:26 pm local time.
+ <a href="/wiki/Los_Angeles_Police_Department" title="Los Angeles Police Department">
+  Los Angeles Police Department
+ </a>
+ homicide and robbery division
+ <a href="/wiki/Detective" title="Detective">
+  detectives
+ </a>
+ attended the scene as a matter of routine in high profile death cases.
+ An
+ <a href="/wiki/Autopsy" title="Autopsy">
+  autopsy
+ </a>
+ is said to be scheduled for Friday, June 26, 2009.
+</ins>
+<del>
@@ -2084 +2536 @@
-</p>
+</del>
@@ -2100,2 +2552,10 @@
- asserts that throughout his solo career, Jackson's versatility has allowed him to experiment with various themes and genres.
- As a musician, he has ranged from
+ asserts that throughout his solo career, Jackson's versatility
+ <del>
+  has
+ </del>
+ allowed him to experiment with various themes and genres.
+ As a musician, he
+ <del>
+  has
+ </del>
+ ranged from
@@ -2177 +2637,8 @@
- , Christoper Connelly of
+ ,
+ <ins>
+  Christopher
+ </ins>
+ <del>
+  Christoper
+ </del>
+ Connelly of
@@ -2318 +2785,8 @@
- presents Jackson as a stark paradoxal individual.
+ presents Jackson as a stark
+ <ins>
+  paradoxical
+ </ins>
+ <del>
+  paradoxal
+ </del>
+ individual.
@@ -2430 +2904,22 @@
- Jackson has been singing since a child, and over time his voice and vocal style have notably changed, either through puberty or a personal preference to align his vocal interpretation to the themes and genres he chooses to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
+ Jackson
+ <ins>
+  sang since he was
+ </ins>
+ <del>
+  has been singing since
+ </del>
+ a child, and over time his voice and vocal style
+ <ins>
+  changed noticeably,
+ </ins>
+ <del>
+  have notably changed,
+ </del>
+ either through puberty or a personal preference to align his vocal interpretation to the themes and genres he
+ <ins>
+  chose
+ </ins>
+ <del>
+  chooses
+ </del>
+ to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
@@ -2435 +2930,8 @@
- ". The purpose of the hiccup—somewhat like a gulping for air or gasping—is to help promote a certain emotion, be it excitement, sadness or fear.
+ ". The purpose of the hiccup—somewhat like a gulping for air or
+ <ins>
+  gasping—was
+ </ins>
+ <del>
+  gasping—is
+ </del>
+ to help promote a certain emotion, be it excitement, sadness or fear.
@@ -2449 +2951,8 @@
- . Their analysis was also that "Jackson's feathery-timbered tenor is extraordinary beautiful. It slides smoothly into a startling
+ . Their analysis was also that "Jackson's feathery-timbered tenor is
+ <ins>
+  extraordinarily
+ </ins>
+ <del>
+  extraordinary
+ </del>
+ beautiful. It slides smoothly into a startling
@@ -2521 +3030,5 @@
- The turn of the 1990s saw the release of the paradoxical, introspective album
+ The turn of the 1990s saw the release of the
+ <del>
+  paradoxical,
+ </del>
+ introspective album
@@ -2525 +3038,7 @@
- , here Jackson used his vocals to intensify the split themes and genres described earlier.
+ <ins>
+  ;
+ </ins>
+ <del>
+  ,
+ </del>
+ here Jackson used his vocals to intensify the split themes and genres described earlier.
@@ -2541 +3060,8 @@
- Nelson George summed by Jackson's vocals as, "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
+ Nelson George summed
+ <ins>
+  up Jackson's vocals by stating
+ </ins>
+ <del>
+  by Jackson's vocals as,
+ </del>
+ "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
@@ -2554 +3080,3 @@
-   Image of
+   <del>
+    Image of
+   </del>
@@ -2560,2 +3088,4 @@
-   <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
-    <span style="display: none;">
+   <ins>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
@@ -2563,2 +3093,10 @@
-   </span>
-   filed by Jackson describing the anti-gravity lean used in the music video for "Smooth Criminal".
+    , filed by Jackson, described
+   </ins>
+   <del>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
+    </span>
+    filed by Jackson describing
+   </del>
+   the anti-gravity lean used in the music video for "Smooth Criminal".
@@ -2725 +3263 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -2754 +3292,7 @@
- A year later, it won a Grammy for Best Music Video, Short Form; shortly afterward
+ A year later, it won a Grammy for Best Music Video, Short Form; shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
@@ -2823 +3367,7 @@
-   Jackson's star on the Hollywood walk of fame, set in 1984.
+   Jackson's star on the Hollywood walk of fame, set in
+   <ins>
+    1984
+   </ins>
+   <del>
+    1984.
+   </del>
@@ -2881 +3431,8 @@
- He is a double-inductee of the
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a double-inductee of the
@@ -2920 +3477,11 @@
-   in June 1988.
+   in June
+   <p>
+    <ins>
+     1988
+  He was
+    </ins>
+    <del>
+     1988.
+  He is
+    </del>
+   </p>
@@ -2924,2 +3491 @@
-<p>
- He is characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
+ characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
@@ -2928,2 +3494,2 @@
-  Time
- </i>
+ Time
+</i>
@@ -2932,2 +3498,2 @@
-  Elvis Presley
- </a>
+ Elvis Presley
+</a>
@@ -2937,4 +3503,4 @@
-  <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
-   Vanity Fair
-  </a>
- </i>
+ <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
+  Vanity Fair
+ </a>
+</i>
@@ -2943,4 +3509,4 @@
-  <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
-   Daily Telegraph
-  </a>
- </i>
+ <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
+  Daily Telegraph
+ </a>
+</i>
@@ -2949,2 +3515,2 @@
-  Tom Utley
- </a>
+ Tom Utley
+</a>
@@ -2954,2 +3520,2 @@
-  royalties
- </a>
+ royalties
+</a>
@@ -2959,3 +3525,6 @@
-  successful career
- </a>
- , has made him a part of
+ successful career
+</a>
+ , <del>
+ has
+</del>
+ made him a part of
@@ -2963,2 +3532,2 @@
-  popular culture
- </a>
+ popular culture
+</a>
@@ -2966,2 +3535 @@
-</p>
-<h2>
+ <h2>
@@ -3120 +3688 @@
-</ul>
+</ul>
\ No newline at end of file
