--- 298631851
+++ 298632300
@@ -5 +5,7 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958 - June 25, 2009)
+ </ins>
+ <del>
+  (born August 29, 1958)
+ </del>
+ is an American recording artist, entertainer, and businessman. The seventh child of the
@@ -2064,4 +2070,10 @@
- <a href="/wiki/TMZ" title="TMZ">
-  TMZ
- </a>
- reported that Jackson has been rushed to the hospital and has suffered
+ <ins>
+  Jackson was
+ </ins>
+ <del>
+  <a href="/wiki/TMZ" title="TMZ">
+   TMZ
+  </a>
+  reported that Jackson has been
+ </del>
+ rushed to the hospital and has suffered
@@ -3120 +3132 @@
-</ul>
+</ul>
\ No newline at end of file
