--- 298631851
+++ 298655072
@@ -5 +5,8 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958 – June 25, 2009)
+ was
+ </ins>
+ <del>
+  (born August 29, 1958) is
+ </del>
+ an American recording artist, entertainer, and businessman. The seventh child of the
@@ -21 +28,7 @@
- have become some of the world's best-selling records:
+ <ins>
+  are among
+ </ins>
+ <del>
+  have become some of
+ </del>
+ the world's best-selling records:
@@ -104,0 +118,3 @@
+ <ins>
+  many
+ </ins>
@@ -119,25 +135,27 @@
- Jackson has donated and raised millions of dollars for beneficial causes through his
- <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
-  foundation
- </a>
- , charity
- <a href="/wiki/Single_(music)" title="Single (music)">
-  singles
- </a>
- and support of 39 charities. Other aspects of his personal life, including his
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  changing appearance
- </a>
- and behavior, generated significant controversy, damaging his public image. Though he was accused of
- <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
-  child sexual abuse in 1993
- </a>
- , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  health concerns
- </a>
- since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  tried and acquitted
- </a>
- of further sexual abuse allegations and several other charges.
+ <del>
+  Jackson has donated and raised millions of dollars for beneficial causes through his
+  <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
+   foundation
+  </a>
+  , charity
+  <a href="/wiki/Single_(music)" title="Single (music)">
+   singles
+  </a>
+  and support of 39 charities. Other aspects of his personal life, including his
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   changing appearance
+  </a>
+  and behavior, generated significant controversy, damaging his public image. Though he was accused of
+  <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
+   child sexual abuse in 1993
+  </a>
+  , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   health concerns
+  </a>
+  since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   tried and acquitted
+  </a>
+  of further sexual abuse allegations and several other charges.
+ </del>
@@ -172 +190,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -178,0 +201,6 @@
+ <ins>
+  On June 25, 2009, he collapsed at his home in Los Angeles.
+ After being taken to the hospital in a coma, Jackson was pronounced dead at 2:26pm.
+ </ins>
+</p>
+<p>
@@ -221 +249,8 @@
- he is the seventh of nine children. His siblings are
+ he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the seventh of nine children. His siblings are
@@ -926 +961,8 @@
- ; the latter is in
+ ; the latter
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ in
@@ -930,2 +972,20 @@
- in Jackson's case, and both illnesses make him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
- The structure of his face has changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
+ in Jackson's case, and both illnesses
+ <ins>
+  made
+ </ins>
+ <del>
+  make
+ </del>
+ him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he
+ <ins>
+  could
+ </ins>
+ <del>
+  can
+ </del>
+ appear very pale.
+ The structure of his face
+ <del>
+  has
+ </del>
+ changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
@@ -939 +999,7 @@
- Some medical professionals have publicly stated their belief that the singer has
+ Some medical professionals have publicly stated their belief that the singer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
@@ -947 +1013,7 @@
-  Why not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
+  <ins>
+   "Why
+  </ins>
+  <del>
+   Why
+  </del>
+  not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
@@ -955 +1027,22 @@
-  . But if I, Michael Jackson, were to say, "I'm an alien from Mars and I eat live chickens and do a voodoo dance at midnight", people would say, "Oh, man, that Michael Jackson is
+  . But if I, Michael Jackson, were to say,
+  <ins>
+   'I'm
+  </ins>
+  <del>
+   "I'm
+  </del>
+  an alien from Mars and I eat live chickens and do a voodoo dance at
+  <ins>
+   midnight,'
+  </ins>
+  <del>
+   midnight",
+  </del>
+  people would say,
+  <ins>
+   'Oh,
+  </ins>
+  <del>
+   "Oh,
+  </del>
+  man, that Michael Jackson is
@@ -959 +1052,7 @@
-  . He's cracked up. You can't believe a damn word that comes out of his mouth".
+  . He's cracked up. You can't believe a damn word that comes out of his
+  <ins>
+   mouth.'"
+  </ins>
+  <del>
+   mouth".
+  </del>
@@ -1186 +1285,8 @@
- has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album off all time.
+ has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album
+ <ins>
+  of
+ </ins>
+ <del>
+  off
+ </del>
+ all time.
@@ -1806 +1912,5 @@
- Shortly after the documentary aired, Jackson was charged with seven counts of child sexual abuse and two counts of
+ Shortly after the documentary aired, Jackson was charged with seven counts of
+ <a href="/wiki/Child_sexual_abuse" title="Child sexual abuse">
+  child sexual abuse
+ </a>
+ and two counts of
@@ -1810 +1920,11 @@
- in order to commit that felony; all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
+ in order to commit that
+ <ins>
+  <a href="/wiki/Felony" title="Felony">
+   felony
+  </a>
+  ;
+ </ins>
+ <del>
+  felony;
+ </del>
+ all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
@@ -1823 +1943,5 @@
- Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend Elizabeth Taylor defended him on
+ Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend
+ <a href="/wiki/Elizabeth_Taylor" title="Elizabeth Taylor">
+  Elizabeth Taylor
+ </a>
+ defended him on
@@ -1829,2 +1953,26 @@
- , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of Walt Disney. There was nothing odd about it."
- During the investigation, Jackson's profile was examined by a mental health professional called Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a pedophile.
+ , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of
+ <a href="/wiki/Walt_Disney" title="Walt Disney">
+  Walt Disney
+ </a>
+ <del>
+  Disney.
+ </del>
+ There was nothing odd about it."
+ During the investigation, Jackson's profile was examined by
+ <del>
+  a
+ </del>
+ mental health professional
+ <del>
+  called
+ </del>
+ Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a
+ <ins>
+  <a class="mw-redirect" href="/wiki/Pedophile" title="Pedophile">
+   pedophile
+  </a>
+  .
+ </ins>
+ <del>
+  pedophile.
+ </del>
@@ -1834,3 +1982,5 @@
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  People v. Jackson
- </a>
+ <i>
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   People v. Jackson
+  </a>
+ </i>
@@ -1950,2 +2100,14 @@
- <span class="mw-headline" id="2008_onward:_Milestones.2C_real_estate_and_return_to_live_performance">
-  2008 onward: Milestones, real estate and return to live performance
+ <span class="mw-headline" id="2008.E2.80.932009:_Milestones.2C_real_estate.2C_planned_return_to_live_performance.2C_and_death">
+  <ins>
+   2008–2009: Milestones, real estate, planned
+  </ins>
+  <del>
+   2008 onward: Milestones, real estate and
+  </del>
+  return to live
+  <ins>
+   performance, and death
+  </ins>
+  <del>
+   performance
+  </del>
@@ -1971 +2133 @@
-  <a class="mw-disambig" href="/wiki/Thriller" title="Thriller">
+  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
@@ -1981 +2143,8 @@
- , comprising of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
+ ,
+ <ins>
+  comprised
+ </ins>
+ <del>
+  comprising
+ </del>
+ of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
@@ -2015 +2184,8 @@
- stood at 688,000 copies, making it the best selling catalog album of 2008.
+ stood at 688,000 copies, making it the
+ <ins>
+  best-selling
+ </ins>
+ <del>
+  best selling
+ </del>
+ catalog album of 2008.
@@ -2043 +2219,8 @@
- From July 13, 2009 to March 6, 2010, Jackson is scheduled to perform
+ From July 13, 2009 to March 6, 2010, Jackson
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ scheduled to perform
@@ -2045 +2228,8 @@
-  50 sell out concerts
+  50
+  <ins>
+   sold-out
+  </ins>
+  <del>
+   sell out
+  </del>
+  concerts
@@ -2055 +2245,8 @@
- . According to Jackson's website, tickets sales for the concerts broke several records. During a prior press conference, Jackson made suggestions of possible retirement.
+ . According to Jackson's website,
+ <ins>
+  ticket
+ </ins>
+ <del>
+  tickets
+ </del>
+ sales for the concerts broke several records. During a prior press conference, Jackson made suggestions of possible retirement.
@@ -2063,5 +2260,39 @@
- On June 25, 2009,
- <a href="/wiki/TMZ" title="TMZ">
-  TMZ
- </a>
- reported that Jackson has been rushed to the hospital and has suffered
+ On
+ <ins>
+  the morning of
+ </ins>
+ June 25, 2009,
+ <ins>
+  Jackson collapsed at a home he was renting in west
+  <a href="/wiki/Los_Angeles" title="Los Angeles">
+   Los Angeles
+  </a>
+  . Paramedics responded to a
+  <a href="/wiki/9-1-1" title="9-1-1">
+   911
+  </a>
+  call at 12:12 pm local time.
+ He was not breathing at the time that paramedics arrived, and CPR was quickly performed.
+ He fell into a
+  <a href="/wiki/Coma" title="Coma">
+   coma
+  </a>
+  and died shortly after being
+ </ins>
+ <del>
+  <a href="/wiki/TMZ" title="TMZ">
+   TMZ
+  </a>
+  reported that Jackson has been
+ </del>
+ rushed to the
+ <ins>
+  <a href="/wiki/Ronald_Reagan_UCLA_Medical_Center" title="Ronald Reagan UCLA Medical Center">
+   UCLA hospital
+  </a>
+  .
+ The cause of death is suspected to be
+ </ins>
+ <del>
+  hospital and has suffered
+ </del>
@@ -2072,12 +2303,33 @@
- <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
-  Paramedics
- </a>
- administered
- <a class="mw-redirect" href="/wiki/CPR" title="CPR">
-  CPR
- </a>
- in the
- <a href="/wiki/Ambulance" title="Ambulance">
-  ambulance
- </a>
- and a family member says that he is in "really bad shape.".
+ <ins>
+  At 4:36 pm local time, the Los Angeles
+  <a href="/wiki/Coroner" title="Coroner">
+   coroner
+  </a>
+  confirmed Jackson's death.
+ Although there is no evidence of
+  <a href="/wiki/Crime" title="Crime">
+   criminality
+  </a>
+  ,
+  <a href="/wiki/Los_Angeles_Police_Department" title="Los Angeles Police Department">
+   Los Angeles Police Department
+  </a>
+  <a href="/wiki/Detective" title="Detective">
+   detectives
+  </a>
+  have opened an investigation into his death.
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
+   Paramedics
+  </a>
+  administered
+  <a class="mw-redirect" href="/wiki/CPR" title="CPR">
+   CPR
+  </a>
+  in the
+  <a href="/wiki/Ambulance" title="Ambulance">
+   ambulance
+  </a>
+  and a family member says that he is in "really bad shape.".
+ </del>
@@ -2100,2 +2352,10 @@
- asserts that throughout his solo career, Jackson's versatility has allowed him to experiment with various themes and genres.
- As a musician, he has ranged from
+ asserts that throughout his solo career, Jackson's versatility
+ <del>
+  has
+ </del>
+ allowed him to experiment with various themes and genres.
+ As a musician, he
+ <del>
+  has
+ </del>
+ ranged from
@@ -2430 +2690,15 @@
- Jackson has been singing since a child, and over time his voice and vocal style have notably changed, either through puberty or a personal preference to align his vocal interpretation to the themes and genres he chooses to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
+ Jackson
+ <ins>
+  sang since he was
+ </ins>
+ <del>
+  has been singing since
+ </del>
+ a child, and over time his voice and vocal style
+ <ins>
+  changed noticably,
+ </ins>
+ <del>
+  have notably changed,
+ </del>
+ either through puberty or a personal preference to align his vocal interpretation to the themes and genres he chooses to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
@@ -2521 +2795,5 @@
- The turn of the 1990s saw the release of the paradoxical, introspective album
+ The turn of the 1990s saw the release of the
+ <del>
+  paradoxical,
+ </del>
+ introspective album
@@ -2525 +2803,7 @@
- , here Jackson used his vocals to intensify the split themes and genres described earlier.
+ <ins>
+  ;
+ </ins>
+ <del>
+  ,
+ </del>
+ here Jackson used his vocals to intensify the split themes and genres described earlier.
@@ -2881 +3165,8 @@
- He is a double-inductee of the
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a double-inductee of the
@@ -2925 +3216,8 @@
- He is characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
@@ -2961 +3259,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -3120 +3422 @@
-</ul>
+</ul>
\ No newline at end of file
