--- 298631851
+++ 298701746
@@ -5 +5,15 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958 – June 25, 2009)
+ was
+ </ins>
+ <del>
+  (born August 29, 1958) is
+ </del>
+ an American
+ <a class="mw-redirect" href="/wiki/Recording_artist" title="Recording artist">
+  recording artist
+ </a>
+ <del>
+  artist, entertainer, and businessman.
+ </del>
+ The seventh child of the
@@ -17 +31,8 @@
- " in subsequent years, five of his solo
+ " in subsequent years,
+ <ins>
+  four
+ </ins>
+ <del>
+  five
+ </del>
+ of his solo
@@ -21 +42,7 @@
- have become some of the world's best-selling records:
+ <ins>
+  are among
+ </ins>
+ <del>
+  have become some of
+ </del>
+ the world's best-selling records:
@@ -28,6 +55,8 @@
- <i>
-  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
-   Thriller
-  </a>
- </i>
- (1982),
+ <del>
+  <i>
+   <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
+    Thriller
+   </a>
+  </i>
+  (1982),
+ </del>
@@ -51 +80,12 @@
- (1995).
+ <ins>
+  (1995), while his 1982
+  <i>
+   <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
+    Thriller
+   </a>
+  </i>
+  is the world's best-selling record of all time.
+ </ins>
+ <del>
+  (1995).
+ </del>
@@ -104,0 +145,3 @@
+ <ins>
+  many
+ </ins>
@@ -119,25 +162,27 @@
- Jackson has donated and raised millions of dollars for beneficial causes through his
- <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
-  foundation
- </a>
- , charity
- <a href="/wiki/Single_(music)" title="Single (music)">
-  singles
- </a>
- and support of 39 charities. Other aspects of his personal life, including his
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  changing appearance
- </a>
- and behavior, generated significant controversy, damaging his public image. Though he was accused of
- <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
-  child sexual abuse in 1993
- </a>
- , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  health concerns
- </a>
- since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  tried and acquitted
- </a>
- of further sexual abuse allegations and several other charges.
+ <del>
+  Jackson has donated and raised millions of dollars for beneficial causes through his
+  <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
+   foundation
+  </a>
+  , charity
+  <a href="/wiki/Single_(music)" title="Single (music)">
+   singles
+  </a>
+  and support of 39 charities. Other aspects of his personal life, including his
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   changing appearance
+  </a>
+  and behavior, generated significant controversy, damaging his public image. Though he was accused of
+  <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
+   child sexual abuse in 1993
+  </a>
+  , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   health concerns
+  </a>
+  since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   tried and acquitted
+  </a>
+  of further sexual abuse allegations and several other charges.
+ </del>
@@ -168 +213,7 @@
- —and the sales of over 750 million albums worldwide. Cited as one of the world's most famous men, Jackson's highly publicized personal life, coupled with his
+ <ins>
+  .
+ </ins>
+ <del>
+  —and the sales of over 750 million albums worldwide. Cited as one of the world's most famous men,
+ </del>
+ Jackson's highly publicized personal life, coupled with his
@@ -172 +223,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -176,0 +232,7 @@
+ <ins>
+  Michael Jackson died of
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  on June 25, 2009, aged 50.
+ </ins>
@@ -221 +283,8 @@
- he is the seventh of nine children. His siblings are
+ he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the seventh of nine children. His siblings are
@@ -255 +324 @@
- Jackson was raised as one of
+ Jackson was raised as
@@ -257 +326,8 @@
-  Jehovah's Witnesses
+  <ins>
+   a
+ Jehovah's Witness
+  </ins>
+  <del>
+   one of
+ Jehovah's Witnesses
+  </del>
@@ -265 +341,8 @@
- One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years afterward, Jackson suffered nightmares about being kidnapped from his bedroom.
+ One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson suffered nightmares about being kidnapped from his bedroom.
@@ -273 +356 @@
- <a class="mw-redirect mw-disambig" href="/wiki/Regurgitate" title="Regurgitate">
+ <a href="/wiki/Vomiting" title="Vomiting">
@@ -608,14 +691,25 @@
- ".
- <i>
-  Thriller
- </i>
- was certified for 28 million shipments by the
- <a href="/wiki/Recording_Industry_Association_of_America" title="Recording Industry Association of America">
-  RIAA
- </a>
- , giving it
- <a class="mw-redirect" href="/wiki/List_of_music_recording_sales_certifications" title="List of music recording sales certifications">
-  Double Diamond
- </a>
- status in the US.
- It is often cited as the
+ <ins>
+  " being on the charts at the same time.
+ The apex of Jackson's career,
+  <i>
+   Thriller
+  </i>
+  sold upwards of 109 million copies, making it
+ </ins>
+ <del>
+  ".
+  <i>
+   Thriller
+  </i>
+  was certified for 28 million shipments by the
+  <a href="/wiki/Recording_Industry_Association_of_America" title="Recording Industry Association of America">
+   RIAA
+  </a>
+  , giving it
+  <a class="mw-redirect" href="/wiki/List_of_music_recording_sales_certifications" title="List of music recording sales certifications">
+   Double Diamond
+  </a>
+  status in the US.
+ It is often cited as
+ </del>
+ the
@@ -625 +719,6 @@
- , with worldwide sales between 47 million and 109 million copies.
+ <ins>
+  .
+ </ins>
+ <del>
+  , with worldwide sales between 47 million and 109 million copies.
+ </del>
@@ -753 +852,7 @@
-   , 1984.
+   ,
+   <ins>
+    1984
+   </ins>
+   <del>
+    1984.
+   </del>
@@ -764 +869,8 @@
- Jackson had his third rhinoplasty shortly afterward and grew self conscious about his appearance.
+ Jackson had his third rhinoplasty shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
+ and grew self conscious about his appearance.
@@ -776 +888,8 @@
- Jackson won eight awards during the 1984 Grammys. Unlike later albums,
+ Jackson won eight awards during the
+ <a class="mw-redirect" href="/wiki/Grammy_Awards_of_1984" title="Grammy Awards of 1984">
+  1984 Grammys
+ </a>
+ <del>
+  Grammys.
+ </del>
+ Unlike later albums,
@@ -811 +930,7 @@
- ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly afterward,
+ ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
@@ -899 +1024,14 @@
-   Jackson two years after he was diagnosed with vitiligo, pictured in the early stages of the disease.
+   Jackson two years after he was diagnosed with vitiligo,
+   <ins>
+    here
+   </ins>
+   <del>
+    pictured in
+   </del>
+   the early stages of the
+   <ins>
+    disease
+   </ins>
+   <del>
+    disease.
+   </del>
@@ -908 +1046,15 @@
- , an act which extended his eccentric persona. In 2003 the singer claimed that Bubbles shared his toilet, and cleaned his bedroom.
+ , an act which extended his eccentric persona. In
+ <ins>
+  2003,
+ </ins>
+ <del>
+  2003
+ </del>
+ the singer claimed that Bubbles shared his
+ <ins>
+  toilet
+ </ins>
+ <del>
+  toilet,
+ </del>
+ and cleaned his bedroom.
@@ -926 +1078,8 @@
- ; the latter is in
+ ; the latter
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ in
@@ -930,2 +1089,34 @@
- in Jackson's case, and both illnesses make him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
- The structure of his face has changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
+ in Jackson's case, and both illnesses
+ <ins>
+  made
+ </ins>
+ <del>
+  make
+ </del>
+ him sensitive to sunlight. The treatments he
+ <ins>
+  used
+ </ins>
+ <del>
+  uses
+ </del>
+ for his condition further
+ <ins>
+  lightened
+ </ins>
+ <del>
+  lighten
+ </del>
+ his skin tone, and, with the application of pancake makeup to even out blotches, he
+ <ins>
+  could
+ </ins>
+ <del>
+  can
+ </del>
+ appear very pale.
+ The structure of his face
+ <del>
+  has
+ </del>
+ changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
@@ -939 +1130,7 @@
- Some medical professionals have publicly stated their belief that the singer has
+ Some medical professionals have publicly stated their belief that the singer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
@@ -947 +1144,7 @@
-  Why not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
+  <ins>
+   "Why
+  </ins>
+  <del>
+   Why
+  </del>
+  not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
@@ -955 +1158,22 @@
-  . But if I, Michael Jackson, were to say, "I'm an alien from Mars and I eat live chickens and do a voodoo dance at midnight", people would say, "Oh, man, that Michael Jackson is
+  . But if I, Michael Jackson, were to say,
+  <ins>
+   'I'm
+  </ins>
+  <del>
+   "I'm
+  </del>
+  an alien from Mars and I eat live chickens and do a voodoo dance at
+  <ins>
+   midnight,'
+  </ins>
+  <del>
+   midnight",
+  </del>
+  people would say,
+  <ins>
+   'Oh,
+  </ins>
+  <del>
+   "Oh,
+  </del>
+  man, that Michael Jackson is
@@ -959 +1183,7 @@
-  . He's cracked up. You can't believe a damn word that comes out of his mouth".
+  . He's cracked up. You can't believe a damn word that comes out of his
+  <ins>
+   mouth.'"
+  </ins>
+  <del>
+   mouth".
+  </del>
@@ -1019 +1249,14 @@
-   A gold plated military style jacket with belt, worn by Jackson in the
+   <ins>
+    Jackson wore a gold-plated
+   </ins>
+   <del>
+    A gold plated
+   </del>
+   military style jacket with
+   <ins>
+    belt
+   </ins>
+   <del>
+    belt, worn by Jackson
+   </del>
+   in the
@@ -1135 +1378,8 @@
- Shortly afterward, Jackson became the first Westerner to appear in a television advert for Russia.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson became the first Westerner to appear in a television advert for Russia.
@@ -1175 +1425,15 @@
- In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the time.
+ In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the
+ <ins>
+  time, displacing
+  <a href="/wiki/Neil_Diamond" title="Neil Diamond">
+   Neil Diamond
+  </a>
+  's renewal contract with
+  <a href="/wiki/Columbia_Records" title="Columbia Records">
+   Columbia Records
+  </a>
+  .
+ </ins>
+ <del>
+  time.
+ </del>
@@ -1186 +1450,8 @@
- has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album off all time.
+ has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album
+ <ins>
+  of
+ </ins>
+ <del>
+  off
+ </del>
+ all time.
@@ -1313 +1584,8 @@
- The friendship between Jackson and Evan Chandler broke down. Sometime afterward, Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
+ The friendship between Jackson and Evan Chandler broke down. Sometime
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
@@ -1391 +1669,8 @@
- Shortly afterward, she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
@@ -1484 +1769,8 @@
- Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration With Vocals".
+ Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration
+ <ins>
+  with
+ </ins>
+ <del>
+  With
+ </del>
+ Vocals".
@@ -1518,0 +1811,23 @@
+ <ins>
+  In early 1996, the
+  <a href="/wiki/Anti-Defamation_League" title="Anti-Defamation League">
+   Anti-Defamation League
+  </a>
+  (ADL) issued a press release charging Jackson with
+  <a href="/wiki/Antisemitism" title="Antisemitism">
+   antisemitism
+  </a>
+  regarding lyrics in the song "
+  <a href="/wiki/They_Don%27t_Care_About_Us" title="They Don't Care About Us">
+   They Don't Care About Us
+  </a>
+  ", the fourth single from HIStory.
+ The song had originally been recorded with lyrics that included the phrase "Jew me, sue me", and "Kick me, kike me". The ADL complained and Jackson responded by saying he would re-record the lyrics before the album went into production. However the ADL's press release charged that Jackson had performed the song live and included the lyrics in question during the live performance.
+ The dispute over the lyrics upset long-time Jackson friend
+  <a href="/wiki/Steven_Spielberg" title="Steven Spielberg">
+   Steven Spielberg
+  </a>
+  , who considered the song anti-semitic
+ </ins>
+</p>
+<p>
@@ -1534,2 +1849,17 @@
- Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them to.
- The couple divorced in 1999, with Rowe giving full custody rights of the children to Jackson; they still remain friends.
+ Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them
+ <ins>
+  to do so.
+ After the
+ </ins>
+ <del>
+  to.
+ The
+ </del>
+ couple divorced in 1999, with Rowe giving full custody rights of the children to
+ <ins>
+  Jackson, they remained
+ </ins>
+ <del>
+  Jackson; they still remain
+ </del>
+ friends.
@@ -1726 +2056,4 @@
- . The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
+ <del>
+  .
+ </del>
+ The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
@@ -1756 +2089,8 @@
- The mother's identity has not been released by Jackson, but he has said the child was the result of
+ The mother's identity
+ <ins>
+  was never
+ </ins>
+ <del>
+  has not been
+ </del>
+ released by Jackson, but he has said the child was the result of
@@ -1806 +2146,5 @@
- Shortly after the documentary aired, Jackson was charged with seven counts of child sexual abuse and two counts of
+ Shortly after the documentary aired, Jackson was charged with seven counts of
+ <a href="/wiki/Child_sexual_abuse" title="Child sexual abuse">
+  child sexual abuse
+ </a>
+ and two counts of
@@ -1810 +2154,11 @@
- in order to commit that felony; all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
+ in order to commit that
+ <ins>
+  <a href="/wiki/Felony" title="Felony">
+   felony
+  </a>
+  ;
+ </ins>
+ <del>
+  felony;
+ </del>
+ all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
@@ -1818 +2172,14 @@
-   Fans show their support for Jackson after the entertainer is accused of child molestation in 2003.
+   Fans
+   <ins>
+    showing
+   </ins>
+   <del>
+    show
+   </del>
+   their support for Jackson after the entertainer is accused of child molestation in
+   <ins>
+    2003
+   </ins>
+   <del>
+    2003.
+   </del>
@@ -1823 +2190,5 @@
- Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend Elizabeth Taylor defended him on
+ Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend
+ <a href="/wiki/Elizabeth_Taylor" title="Elizabeth Taylor">
+  Elizabeth Taylor
+ </a>
+ defended him on
@@ -1829,2 +2200,26 @@
- , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of Walt Disney. There was nothing odd about it."
- During the investigation, Jackson's profile was examined by a mental health professional called Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a pedophile.
+ , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of
+ <a class="mw-redirect" href="/wiki/Walt_Disney_Motion_Pictures_Group" title="Walt Disney Motion Pictures Group">
+  Walt Disney
+ </a>
+ <del>
+  Disney.
+ </del>
+ There was nothing odd about it."
+ During the investigation, Jackson's profile was examined by
+ <del>
+  a
+ </del>
+ mental health professional
+ <del>
+  called
+ </del>
+ Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a
+ <ins>
+  <a class="mw-redirect" href="/wiki/Pedophile" title="Pedophile">
+   pedophile
+  </a>
+  .
+ </ins>
+ <del>
+  pedophile.
+ </del>
@@ -1834,3 +2229,5 @@
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  People v. Jackson
- </a>
+ <i>
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   People v. Jackson
+  </a>
+ </i>
@@ -1879 +2276,7 @@
- format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set afterward.
+ format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set
+ <ins>
+  afterwards.
+ </ins>
+ <del>
+  afterward.
+ </del>
@@ -1950,2 +2353,8 @@
- <span class="mw-headline" id="2008_onward:_Milestones.2C_real_estate_and_return_to_live_performance">
-  2008 onward: Milestones, real estate and return to live performance
+ <span class="mw-headline" id="2008.E2.80.932009:_Milestones.2C_real_estate.2C_planned_return_to_live_performance">
+  <ins>
+   2008–2009: Milestones, real estate, planned
+  </ins>
+  <del>
+   2008 onward: Milestones, real estate and
+  </del>
+  return to live performance
@@ -1964 +2373,8 @@
-   Neverland Ranch was Jackson's permanent residency from 1988 to 2005. Pictured is an aerial shot of the theme park area of the property.
+   Neverland Ranch was Jackson's permanent
+   <ins>
+    residence
+   </ins>
+   <del>
+    residency
+   </del>
+   from 1988 to 2005. Pictured is an aerial shot of the theme park area of the property.
@@ -1971 +2387 @@
-  <a class="mw-disambig" href="/wiki/Thriller" title="Thriller">
+  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
@@ -1981 +2397,8 @@
- , comprising of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
+ ,
+ <ins>
+  comprised
+ </ins>
+ <del>
+  comprising
+ </del>
+ of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
@@ -2015 +2438,8 @@
- stood at 688,000 copies, making it the best selling catalog album of 2008.
+ stood at 688,000 copies, making it the
+ <ins>
+  best-selling
+ </ins>
+ <del>
+  best selling
+ </del>
+ catalog album of 2008.
@@ -2018 +2448 @@
- To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
+ To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
@@ -2043 +2473,102 @@
- From July 13, 2009 to March 6, 2010, Jackson is scheduled to perform
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h3>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    Wikinews has related news:
+    <i>
+     <b>
+      <a class="extiw" href="https://en.wikinews.org/wiki/Singer_and_songwriter_Michael_Jackson_dies" title="wikinews:Singer and songwriter Michael Jackson dies">
+       Singer and songwriter Michael Jackson dies
+      </a>
+     </b>
+    </i>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  On the morning of June 25, 2009, Jackson reportedly collapsed at a rented home in
+  <a class="mw-redirect" href="/wiki/Holmby_Hills" title="Holmby Hills">
+   Holmby Hills
+  </a>
+  in
+  <a href="/wiki/Los_Angeles" title="Los Angeles">
+   Los Angeles
+  </a>
+  . After he collapsed, Jackson's personal physician, who was with him at the time, attempted to resuscitate him.
+  <a href="/wiki/Los_Angeles_Fire_Department" title="Los Angeles Fire Department">
+   Los Angeles Fire Department
+  </a>
+  paramedics responded to a
+  <a href="/wiki/9-1-1" title="9-1-1">
+   911
+  </a>
+  call at 12:21 pm (
+  <a class="mw-redirect" href="/wiki/Pacific_time" title="Pacific time">
+   PST
+  </a>
+  )
+ and arrived nine minutes later. Jackson was reportedly not breathing and
+  <a href="/wiki/Cardiopulmonary_resuscitation" title="Cardiopulmonary resuscitation">
+   CPR
+  </a>
+  was quickly performed.
+ Resuscitation efforts continued both en route to the
+  <a href="/wiki/Ronald_Reagan_UCLA_Medical_Center" title="Ronald Reagan UCLA Medical Center">
+   UCLA Medical Center
+  </a>
+  , and after arrival at approximately 1:14 pm (20:14 UTC), for a further hour.
+ He fell into a
+  <a href="/wiki/Coma" title="Coma">
+   coma
+  </a>
+  and died shortly after arrival.
+ The cause of death is reported to be
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  .
+ Jackson was pronounced dead at about 2:26 pm local time (21:26 UTC).
+ </ins>
+</p>
+<p>
+ <ins>
+  <a href="/wiki/Los_Angeles_Police_Department" title="Los Angeles Police Department">
+   Los Angeles Police Department
+  </a>
+  homicide and robbery division
+  <a href="/wiki/Detective" title="Detective">
+   detectives
+  </a>
+  attended the scene as a matter of routine in high profile death cases.
+ The case was transferred to the Los Angeles coroner for investigation as there was no doctor in attendance to sign the
+  <a href="/wiki/Death_certificate" title="Death certificate">
+   death certificate
+  </a>
+  .
+ Jackson's body was transported by helicopter from UCLA to the LA Coroners offices.
+ The
+  <a href="/wiki/Autopsy" title="Autopsy">
+   autopsy
+  </a>
+  is said to be scheduled for Friday, June 26, 2009.
+ </ins>
+</p>
+<ins>
+ Prior to his death, Jackson had been
+</ins>
+<del>
+ From July 13, 2009 to March 6, 2010, Jackson is
+</del>
+ scheduled to perform
@@ -2045,2 +2576,9 @@
-  50 sell out concerts
- </a>
+ 50
+ <ins>
+  sold-out
+ </ins>
+ <del>
+  sell out
+ </del>
+ concerts
+</a>
@@ -2049,7 +2587,13 @@
-  O
-  <sub>
-   2
-  </sub>
-  arena
- </a>
- . According to Jackson's website, tickets sales for the concerts broke several records. During a prior press conference, Jackson made suggestions of possible retirement.
+ O
+ <sub>
+  2
+ </sub>
+ arena
+</a>
+<ins>
+ , from July 13, 2009 to March 6, 2010. During a publicity
+</ins>
+<del>
+ . According to Jackson's website, tickets sales for the concerts broke several records. During a prior
+</del>
+ press conference, Jackson made suggestions of possible retirement.
@@ -2058,26 +2602,36 @@
-  AEG Live
- </a>
- stated that the first 10 dates alone would earn the singer approximately £50 million.
-</p>
-<p>
- On June 25, 2009,
- <a href="/wiki/TMZ" title="TMZ">
-  TMZ
- </a>
- reported that Jackson has been rushed to the hospital and has suffered
- <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
-  cardiac arrest
- </a>
- .
- <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
-  Paramedics
- </a>
- administered
- <a class="mw-redirect" href="/wiki/CPR" title="CPR">
-  CPR
- </a>
- in the
- <a href="/wiki/Ambulance" title="Ambulance">
-  ambulance
- </a>
- and a family member says that he is in "really bad shape.".
+ AEG Live
+</a>
+<ins>
+ , had
+</ins>
+ stated that the first 10 dates alone would <ins>
+ have earned
+</ins>
+<del>
+ earn
+</del>
+ the singer approximately £50 million.
+ <p>
+ <del>
+  On June 25, 2009,
+  <a href="/wiki/TMZ" title="TMZ">
+   TMZ
+  </a>
+  reported that Jackson has been rushed to the hospital and has suffered
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  .
+  <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
+   Paramedics
+  </a>
+  administered
+  <a class="mw-redirect" href="/wiki/CPR" title="CPR">
+   CPR
+  </a>
+  in the
+  <a href="/wiki/Ambulance" title="Ambulance">
+   ambulance
+  </a>
+  and a family member says that he is in "really bad shape.".
+ </del>
@@ -2100,2 +2654,10 @@
- asserts that throughout his solo career, Jackson's versatility has allowed him to experiment with various themes and genres.
- As a musician, he has ranged from
+ asserts that throughout his solo career, Jackson's versatility
+ <del>
+  has
+ </del>
+ allowed him to experiment with various themes and genres.
+ As a musician, he
+ <del>
+  has
+ </del>
+ ranged from
@@ -2177 +2739,8 @@
- , Christoper Connelly of
+ ,
+ <ins>
+  Christopher
+ </ins>
+ <del>
+  Christoper
+ </del>
+ Connelly of
@@ -2318 +2887,8 @@
- presents Jackson as a stark paradoxal individual.
+ presents Jackson as a stark
+ <ins>
+  paradoxical
+ </ins>
+ <del>
+  paradoxal
+ </del>
+ individual.
@@ -2430 +3006,22 @@
- Jackson has been singing since a child, and over time his voice and vocal style have notably changed, either through puberty or a personal preference to align his vocal interpretation to the themes and genres he chooses to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
+ Jackson
+ <ins>
+  sang since he was
+ </ins>
+ <del>
+  has been singing since
+ </del>
+ a child, and over time his voice and vocal style
+ <ins>
+  changed noticeably,
+ </ins>
+ <del>
+  have notably changed,
+ </del>
+ either through puberty or a personal preference to align his vocal interpretation to the themes and genres he
+ <ins>
+  chose
+ </ins>
+ <del>
+  chooses
+ </del>
+ to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
@@ -2435 +3032,8 @@
- ". The purpose of the hiccup—somewhat like a gulping for air or gasping—is to help promote a certain emotion, be it excitement, sadness or fear.
+ ". The purpose of the hiccup—somewhat like a gulping for air or
+ <ins>
+  gasping—was
+ </ins>
+ <del>
+  gasping—is
+ </del>
+ to help promote a certain emotion, be it excitement, sadness or fear.
@@ -2449 +3053,8 @@
- . Their analysis was also that "Jackson's feathery-timbered tenor is extraordinary beautiful. It slides smoothly into a startling
+ . Their analysis was also that "Jackson's feathery-timbered tenor is
+ <ins>
+  extraordinarily
+ </ins>
+ <del>
+  extraordinary
+ </del>
+ beautiful. It slides smoothly into a startling
@@ -2521 +3132,5 @@
- The turn of the 1990s saw the release of the paradoxical, introspective album
+ The turn of the 1990s saw the release of the
+ <del>
+  paradoxical,
+ </del>
+ introspective album
@@ -2525 +3140,7 @@
- , here Jackson used his vocals to intensify the split themes and genres described earlier.
+ <ins>
+  ;
+ </ins>
+ <del>
+  ,
+ </del>
+ here Jackson used his vocals to intensify the split themes and genres described earlier.
@@ -2541 +3162,8 @@
- Nelson George summed by Jackson's vocals as, "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
+ Nelson George summed
+ <ins>
+  up Jackson's vocals by stating
+ </ins>
+ <del>
+  by Jackson's vocals as,
+ </del>
+ "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
@@ -2554 +3182,3 @@
-   Image of
+   <del>
+    Image of
+   </del>
@@ -2560,2 +3190,4 @@
-   <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
-    <span style="display: none;">
+   <ins>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
@@ -2563,2 +3195,10 @@
-   </span>
-   filed by Jackson describing the anti-gravity lean used in the music video for "Smooth Criminal".
+    , filed by Jackson, described
+   </ins>
+   <del>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
+    </span>
+    filed by Jackson describing
+   </del>
+   the anti-gravity lean used in the music video for "Smooth Criminal".
@@ -2725 +3365 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -2754 +3394,7 @@
- A year later, it won a Grammy for Best Music Video, Short Form; shortly afterward
+ A year later, it won a Grammy for Best Music Video, Short Form; shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
@@ -2823 +3469,7 @@
-   Jackson's star on the Hollywood walk of fame, set in 1984.
+   Jackson's star on the Hollywood walk of fame, set in
+   <ins>
+    1984
+   </ins>
+   <del>
+    1984.
+   </del>
@@ -2832 +3484,5 @@
- in 1984, Jackson has had a notable impact on music and culture throughout the world. He broke down racial barriers, transformed the art of the music video and paved the way for modern pop music in his own country. Jackson's work, distinctive musical sound and vocal style have influenced
+ in 1984, Jackson
+ <del>
+  has
+ </del>
+ had a notable impact on music and culture throughout the world. He broke down racial barriers, transformed the art of the music video and paved the way for modern pop music in his own country. Jackson's work, distinctive musical sound and vocal style have influenced
@@ -2881 +3537,8 @@
- He is a double-inductee of the
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a double-inductee of the
@@ -2920 +3583,11 @@
-   in June 1988.
+   in June
+   <p>
+    <ins>
+     1988
+  He was
+    </ins>
+    <del>
+     1988.
+  He is
+    </del>
+   </p>
@@ -2924,2 +3597 @@
-<p>
- He is characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
+ characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
@@ -2928,2 +3600,2 @@
-  Time
- </i>
+ Time
+</i>
@@ -2932,2 +3604,2 @@
-  Elvis Presley
- </a>
+ Elvis Presley
+</a>
@@ -2937,4 +3609,4 @@
-  <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
-   Vanity Fair
-  </a>
- </i>
+ <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
+  Vanity Fair
+ </a>
+</i>
@@ -2943,4 +3615,4 @@
-  <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
-   Daily Telegraph
-  </a>
- </i>
+ <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
+  Daily Telegraph
+ </a>
+</i>
@@ -2949,2 +3621,2 @@
-  Tom Utley
- </a>
+ Tom Utley
+</a>
@@ -2954,2 +3626,2 @@
-  royalties
- </a>
+ royalties
+</a>
@@ -2959,3 +3631,6 @@
-  successful career
- </a>
- , has made him a part of
+ successful career
+</a>
+ , <del>
+ has
+</del>
+ made him a part of
@@ -2963,2 +3638,2 @@
-  popular culture
- </a>
+ popular culture
+</a>
@@ -2966,2 +3641 @@
-</p>
-<h2>
+ <h2>
@@ -3120 +3794 @@
-</ul>
+</ul>
\ No newline at end of file
