--- 298631851
+++ 298739528
@@ -5 +5,14 @@
- (born August 29, 1958) is an American recording artist, entertainer, and businessman. The seventh child of the
+ <ins>
+  (August 29, 1958–June 25, 2009) was
+ </ins>
+ <del>
+  (born August 29, 1958) is
+ </del>
+ an American
+ <a class="mw-redirect" href="/wiki/Recording_artist" title="Recording artist">
+  recording artist
+ </a>
+ <del>
+  artist, entertainer, and businessman.
+ </del>
+ The seventh child of the
@@ -17 +30,16 @@
- " in subsequent years, five of his solo
+ "
+ in subsequent years,
+ <ins>
+  his 1982
+  <i>
+   <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
+    Thriller
+   </a>
+  </i>
+  is the world's best-selling record of all time
+ and four other
+ </ins>
+ <del>
+  five of his
+ </del>
+ solo
@@ -21 +49,7 @@
- have become some of the world's best-selling records:
+ <ins>
+  are also among
+ </ins>
+ <del>
+  have become some of
+ </del>
+ the world's best-selling records:
@@ -28,6 +62,8 @@
- <i>
-  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
-   Thriller
-  </a>
- </i>
- (1982),
+ <del>
+  <i>
+   <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
+    Thriller
+   </a>
+  </i>
+  (1982),
+ </del>
@@ -104,0 +141,3 @@
+ <ins>
+  many
+ </ins>
@@ -119,25 +158,27 @@
- Jackson has donated and raised millions of dollars for beneficial causes through his
- <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
-  foundation
- </a>
- , charity
- <a href="/wiki/Single_(music)" title="Single (music)">
-  singles
- </a>
- and support of 39 charities. Other aspects of his personal life, including his
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  changing appearance
- </a>
- and behavior, generated significant controversy, damaging his public image. Though he was accused of
- <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
-  child sexual abuse in 1993
- </a>
- , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
- <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
-  health concerns
- </a>
- since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  tried and acquitted
- </a>
- of further sexual abuse allegations and several other charges.
+ <del>
+  Jackson has donated and raised millions of dollars for beneficial causes through his
+  <a href="/wiki/Heal_the_World_Foundation" title="Heal the World Foundation">
+   foundation
+  </a>
+  , charity
+  <a href="/wiki/Single_(music)" title="Single (music)">
+   singles
+  </a>
+  and support of 39 charities. Other aspects of his personal life, including his
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   changing appearance
+  </a>
+  and behavior, generated significant controversy, damaging his public image. Though he was accused of
+  <a href="/wiki/1993_child_sexual_abuse_accusations_against_Michael_Jackson" title="1993 child sexual abuse accusations against Michael Jackson">
+   child sexual abuse in 1993
+  </a>
+  , the criminal investigation was closed due to lack of evidence and Jackson was not charged. The singer has experienced
+  <a href="/wiki/Michael_Jackson%27s_health_and_appearance" title="Michael Jackson's health and appearance">
+   health concerns
+  </a>
+  since the early 1990s and conflicting reports regarding the state of his finances since the late 1990s. Jackson married twice and fathered three children, all of which caused further controversy. In 2005, Jackson was
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   tried and acquitted
+  </a>
+  of further sexual abuse allegations and several other charges.
+ </del>
@@ -168 +209,7 @@
- —and the sales of over 750 million albums worldwide. Cited as one of the world's most famous men, Jackson's highly publicized personal life, coupled with his
+ <ins>
+  .
+ </ins>
+ <del>
+  —and the sales of over 750 million albums worldwide. Cited as one of the world's most famous men,
+ </del>
+ Jackson's highly publicized personal life, coupled with his
@@ -172 +219,5 @@
- , has made him a part of
+ ,
+ <del>
+  has
+ </del>
+ made him a part of
@@ -176,0 +228,4 @@
+ <ins>
+  He died unexpectedly at 2:26 pm
+ on June 25, 2009, aged 50. The specific cause of death has yet to be determined.
+ </ins>
@@ -221 +276,8 @@
- he is the seventh of nine children. His siblings are
+ he
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the seventh of nine children. His siblings are
@@ -255 +317 @@
- Jackson was raised as one of
+ Jackson was raised as
@@ -257,7 +319,43 @@
-  Jehovah's Witnesses
- </a>
- by his devout mother.
-</p>
-<p>
- From a young age Jackson was physically and mentally abused by his father, enduring incessant rehearsals, whippings and name-calling. Jackson's abuse as a child affected him throughout his grown life.
- In one altercation—later recalled by Marlon Jackson—Joseph held Michael upside down by one leg and "pummeled him over and over again with his hand, hitting him on his back and buttocks".
+  <ins>
+   a
+ Jehovah's Witness
+  </ins>
+  <del>
+   one of
+ Jehovah's Witnesses
+  </del>
+ </a>
+ by his devout
+ <ins>
+  mother, but was later "disfellowshipped" for unbiblical actions.
+ </ins>
+ <del>
+  mother.
+ </del>
+</p>
+<p>
+ From a young age Jackson was
+ <a href="/wiki/Corporal_punishment" title="Corporal punishment">
+  physically
+  Link: /wiki/Corporal_punishment and
+ emotionally abused
+ </a>
+ <del>
+  and mentally abused
+ </del>
+ by his father, enduring incessant rehearsals, whippings and name-calling. Jackson's abuse as a child affected him throughout his grown life.
+ In one
+ <ins>
+  altercation — later
+ </ins>
+ <del>
+  altercation—later
+ </del>
+ recalled by Marlon
+ <ins>
+  Jackson — Joseph
+ </ins>
+ <del>
+  Jackson—Joseph
+ </del>
+ held Michael upside down by one leg and "pummeled him over and over again with his hand, hitting him on his back and buttocks".
@@ -265 +363,8 @@
- One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years afterward, Jackson suffered nightmares about being kidnapped from his bedroom.
+ One night while Jackson was asleep, Joseph climbed into his room through the bedroom window. Wearing a fright mask, he entered the room screaming and shouting. Joseph said he wanted to teach his children not to leave the window open when they went to sleep. For years
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson suffered nightmares about being kidnapped from his bedroom.
@@ -273,3 +378,10 @@
- <a class="mw-redirect mw-disambig" href="/wiki/Regurgitate" title="Regurgitate">
-  regurgitate
- </a>
+ <ins>
+  <a href="/wiki/Vomiting" title="Vomiting">
+   vomit
+  </a>
+ </ins>
+ <del>
+  <a class="mw-redirect mw-disambig" href="/wiki/Regurgitate" title="Regurgitate">
+   regurgitate
+  </a>
+ </del>
@@ -481 +593,7 @@
- , who formed a partnership with Jackson during the film's production and agreed to produce the singer's next solo album
+ , who formed a partnership with Jackson during the film's production and agreed to produce the singer's next solo
+ <ins>
+  album,
+ </ins>
+ <del>
+  album
+ </del>
@@ -572 +690,14 @@
- In 1982, Jackson contributed the song "Someone In the Dark" to the storybook for the film
+ <ins>
+  After Jackson's early 1982 contribution,
+ </ins>
+ <del>
+  In 1982, Jackson contributed the song
+ </del>
+ "Someone In the
+ <ins>
+  Dark", to the blockbuster
+ </ins>
+ <del>
+  Dark" to the storybook for the
+ </del>
+ film
@@ -578,2 +709,20 @@
- ; the record won a Grammy for Best Album for Children.
- That year Jackson issued his second Epic album,
+ <ins>
+  , which nabbed him
+ </ins>
+ <del>
+  ; the record won
+ </del>
+ a Grammy for Best Album for
+ <ins>
+  Children,
+ Epic
+ </ins>
+ <del>
+  Children.
+ That year Jackson
+ </del>
+ issued his second
+ <del>
+  Epic
+ </del>
+ album,
@@ -585 +734,8 @@
- . The album remained in the top 10 of the
+ .
+ <ins>
+  In what would turn out to be the apex of Jackson's career, the
+ </ins>
+ <del>
+  The
+ </del>
+ album remained in the top 10 of the
@@ -592 +748,11 @@
- for 80 consecutive weeks and 37 of those weeks at the peak position. It was the first album to have seven
+ for 80 consecutive
+ <ins>
+  weeks, 37 at the peak. Seven singles from
+  <i>
+   Thriller
+  </i>
+  concurrently hit the
+ </ins>
+ <del>
+  weeks and 37 of those weeks at the peak position. It was the first album to have seven
+ </del>
@@ -596 +762,8 @@
- Hot 100 top 10 singles, including "
+ Hot 100 top
+ <ins>
+  10,
+ </ins>
+ <del>
+  10 singles,
+ </del>
+ including "
@@ -612,10 +785,16 @@
- was certified for 28 million shipments by the
- <a href="/wiki/Recording_Industry_Association_of_America" title="Recording Industry Association of America">
-  RIAA
- </a>
- , giving it
- <a class="mw-redirect" href="/wiki/List_of_music_recording_sales_certifications" title="List of music recording sales certifications">
-  Double Diamond
- </a>
- status in the US.
- It is often cited as the
+ <ins>
+  went on to sell upwards of 109 million copies, making it
+ </ins>
+ <del>
+  was certified for 28 million shipments by the
+  <a href="/wiki/Recording_Industry_Association_of_America" title="Recording Industry Association of America">
+   RIAA
+  </a>
+  , giving it
+  <a class="mw-redirect" href="/wiki/List_of_music_recording_sales_certifications" title="List of music recording sales certifications">
+   Double Diamond
+  </a>
+  status in the US.
+ It is often cited as
+ </del>
+ the
@@ -625,24 +804,28 @@
- , with worldwide sales between 47 million and 109 million copies.
-</p>
-<p>
- Jackson's attorney
- <a href="/wiki/John_Branca" title="John Branca">
-  John Branca
- </a>
- noted that Jackson had the highest royalty rate in the music industry at that point; approximately $2 for every album sold. He was also making record breaking profit from compact discs or the sale of
- <i>
-  The Making of Michael Jackson's Thriller
- </i>
- ; a documentary produced by Jackson and
- <a href="/wiki/John_Landis" title="John Landis">
-  John Landis
- </a>
- . Funded by MTV, the documentary sold over 350,000 copies in a few months of sale. The era saw the arrival of novelties like dolls modeled after Michael Jackson, that appeared in stores in May 1984 at a price of $12.
- <i>
-  Thriller
- </i>
- retains a position in
- <a href="/wiki/Culture_of_the_United_States" title="Culture of the United States">
-  American culture
- </a>
- . Biographer
+ ,
+ <ins>
+  causing Jackson biographer
+ </ins>
+ <del>
+  with worldwide sales between 47 million and 109 million copies.
+Jackson's attorney
+  <a href="/wiki/John_Branca" title="John Branca">
+   John Branca
+  </a>
+  noted that Jackson had the highest royalty rate in the music industry at that point; approximately $2 for every album sold. He was also making record breaking profit from compact discs or the sale of
+  <i>
+   The Making of Michael Jackson's Thriller
+  </i>
+  ; a documentary produced by Jackson and
+  <a href="/wiki/John_Landis" title="John Landis">
+   John Landis
+  </a>
+  . Funded by MTV, the documentary sold over 350,000 copies in a few months of sale. The era saw the arrival of novelties like dolls modeled after Michael Jackson, that appeared in stores in May 1984 at a price of $12.
+  <i>
+   Thriller
+  </i>
+  retains a position in
+  <a href="/wiki/Culture_of_the_United_States" title="Culture of the United States">
+   American culture
+  </a>
+  . Biographer
+ </del>
@@ -652 +835,7 @@
- explains, "At some point,
+ <ins>
+  to muse that "at
+ </ins>
+ <del>
+  explains, "At
+ </del>
+ some point,
@@ -658,0 +848,32 @@
+</p>
+<p>
+ <ins>
+  The period of
+  <i>
+   Thriller
+  </i>
+  was an extraordinarily lucrative one for Jackson, whose lawyer
+  <a href="/wiki/John_Branca" title="John Branca">
+   John Branca
+  </a>
+  had negotiated what he boasted then as the highest royalty rate ever in the music industry, approximately $2 per album. Meanwhile, Jackson raked in profits from
+  <i>
+   The Making of Michael Jackson's Thriller
+  </i>
+  , a documentary by Jackson and
+  <a href="/wiki/John_Landis" title="John Landis">
+   John Landis
+  </a>
+  which quickly sold over 350,000 copies. In addition, Jackson began profiting from his image in earnest, as Michael Jackson dolls and other novelties hit the market.
+ </ins>
+</p>
+<ins>
+ Beyond its record-breaking success among fans,
+</ins>
+<ins>
+ <i>
+  Thriller
+ </i>
+ instituted multiple changes within the music industry. One, it
+</ins>
+<del>
@@ -667 +888,7 @@
- raised the importance of albums, but multiple hits also changed notions about the number of singles to release.
+</del>
+ raised the importance of albums, <ins>
+ while challenging notions about how many prospective hits an album should contain.
+ Two, it restored to the industry a sense of confidence in its ability to release high-level artistry during a time when profits had been sinking due to what one industry analyst called
+</ins>
+<del>
+ but multiple hits also changed notions about the number of singles to release.
@@ -685 +912,12 @@
- s impact as a "restoration of confidence" for an industry bordering on "the ruins of punk and the chic regions of synthesizer pop". The publication described Jackson's influence at that point as "Star of records, radio, rock video. A one-man rescue team for the music business. A songwriter who sets the beat for a decade. A dancer with the fanciest feet on the street. A singer who cuts across all boundaries of taste and style and color too".
+ s impact as a "restoration of confidence" for an industry bordering on
+</del>
+ "the ruins of punk and the chic regions of synthesizer pop".
+ <ins>
+ Three, it helped bring MTV into its heyday, even as MTV helped fuel
+ <i>
+  Thriller's
+ </i>
+ success. Four,
+</ins>
+<del>
+ The publication described Jackson's influence at that point as "Star of records, radio, rock video. A one-man rescue team for the music business. A songwriter who sets the beat for a decade. A dancer with the fanciest feet on the street. A singer who cuts across all boundaries of taste and style and color too".
@@ -698,0 +937,15 @@
+</del>
+<i>
+ Thriller
+</i>
+ paved the way for other <ins>
+ well-profitable
+</ins>
+ acts such as
+ <a href="/wiki/Prince_(musician)" title="Prince (musician)">
+ Prince
+</a>
+<ins>
+ .
+ In the end, in many ways, Jackson had become a one-man rescue team for the music business.
+ At its 25th anniversary,
@@ -702,3 +955,3 @@
- paved the way for other acts such as
- <a href="/wiki/Prince_(musician)" title="Prince (musician)">
-  Prince
+ retained important influence over the music industry, artists, and
+ <a href="/wiki/Culture_of_the_United_States" title="Culture of the United States">
+  American culture
@@ -705,0 +959 @@
+</ins>
@@ -707,2 +961 @@
-</p>
-<p>
+ <p>
@@ -740 +993 @@
- <div class="thumbinner" style="width:232px;">
+ <div class="thumbinner" style="width:222px;">
@@ -742 +994,0 @@
-   <img alt="" class="thumbimage" data-file-height="2928" data-file-width="2918" height="231" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/de/Michael_Jackson_with_the_Reagans.jpg/230px-Michael_Jackson_with_the_Reagans.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/de/Michael_Jackson_with_the_Reagans.jpg/345px-Michael_Jackson_with_the_Reagans.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/de/Michael_Jackson_with_the_Reagans.jpg/460px-Michael_Jackson_with_the_Reagans.jpg 2x" width="230"/>
@@ -753 +1005,7 @@
-   , 1984.
+   ,
+   <ins>
+    1984
+   </ins>
+   <del>
+    1984.
+   </del>
@@ -762 +1020,5 @@
- commercial at the Shrine Auditorium in Los Angeles, Jackson suffered second degree burns to his scalp after pyrotechnics accidentally set his hair on fire. Happening in front of a full house of fans during a simulated concert, the incident was the subject of heavy media scrutiny and elicited an outpouring of sympathy.
+ commercial at the
+ <a href="/wiki/Shrine_Auditorium" title="Shrine Auditorium">
+  Shrine Auditorium
+ </a>
+ in Los Angeles, Jackson suffered second degree burns to his scalp after pyrotechnics accidentally set his hair on fire. Happening in front of a full house of fans during a simulated concert, the incident was the subject of heavy media scrutiny and elicited an outpouring of sympathy.
@@ -764 +1026,8 @@
- Jackson had his third rhinoplasty shortly afterward and grew self conscious about his appearance.
+ Jackson had his third rhinoplasty shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
+ and grew self conscious about his appearance.
@@ -771 +1040,9 @@
- to receive an award presented by American President
+ to receive an award presented by
+ <ins>
+  <a class="mw-redirect" href="/wiki/U.S._President" title="U.S. President">
+   U.S. President
+  </a>
+ </ins>
+ <del>
+  American President
+ </del>
@@ -775,2 +1052,18 @@
- . The award was given for Jackson's support of charities that helped people overcome alcohol and drug abuse.
- Jackson won eight awards during the 1984 Grammys. Unlike later albums,
+ . The award was given for Jackson's support of charities that helped people overcome
+ <a href="/wiki/Alcoholism" title="Alcoholism">
+  alcohol
+  Link: /wiki/Alcoholism and
+ drug abuse
+  Link: /wiki/Drug_abuse  .
+ </a>
+ <del>
+  and drug abuse.
+ </del>
+ Jackson won eight awards during the
+ <a class="mw-redirect" href="/wiki/Grammy_Awards_of_1984" title="Grammy Awards of 1984">
+  1984 Grammys
+ </a>
+ <del>
+  Grammys.
+ </del>
+ Unlike later albums,
@@ -811 +1104,7 @@
- ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly afterward,
+ ", the pair became friendly, occasionally visiting one another. In one discussion, McCartney told Jackson about the millions of dollars he had made from music catalogs; he was earning approximately $40 million a year from other people's songs. Jackson then began a business career buying, selling and distributing publishing rights to music from numerous artists. Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
@@ -890 +1189 @@
- <div class="thumbinner" style="width:202px;">
+ <div class="thumbinner" style="width:222px;">
@@ -892 +1190,0 @@
-   <img alt="" class="thumbimage" data-file-height="775" data-file-width="682" height="227" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Michael_Jackson_gives_autographCropped.jpg/200px-Michael_Jackson_gives_autographCropped.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Michael_Jackson_gives_autographCropped.jpg/300px-Michael_Jackson_gives_autographCropped.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Michael_Jackson_gives_autographCropped.jpg/400px-Michael_Jackson_gives_autographCropped.jpg 2x" width="200"/>
@@ -899 +1197,14 @@
-   Jackson two years after he was diagnosed with vitiligo, pictured in the early stages of the disease.
+   Jackson two years after he was diagnosed with vitiligo,
+   <ins>
+    here
+   </ins>
+   <del>
+    pictured in
+   </del>
+   the early stages of the
+   <ins>
+    disease
+   </ins>
+   <del>
+    disease.
+   </del>
@@ -908 +1219,15 @@
- , an act which extended his eccentric persona. In 2003 the singer claimed that Bubbles shared his toilet, and cleaned his bedroom.
+ , an act which extended his eccentric persona. In
+ <ins>
+  2003,
+ </ins>
+ <del>
+  2003
+ </del>
+ the singer claimed that Bubbles shared his
+ <ins>
+  toilet
+ </ins>
+ <del>
+  toilet,
+ </del>
+ and cleaned his bedroom.
@@ -926 +1251,8 @@
- ; the latter is in
+ ; the latter
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ in
@@ -930,2 +1262,34 @@
- in Jackson's case, and both illnesses make him sensitive to sunlight. The treatments he uses for his condition further lighten his skin tone, and, with the application of pancake makeup to even out blotches, he can appear very pale.
- The structure of his face has changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
+ in Jackson's case, and both illnesses
+ <ins>
+  made
+ </ins>
+ <del>
+  make
+ </del>
+ him sensitive to sunlight. The treatments he
+ <ins>
+  used
+ </ins>
+ <del>
+  uses
+ </del>
+ for his condition further
+ <ins>
+  lightened
+ </ins>
+ <del>
+  lighten
+ </del>
+ his skin tone, and, with the application of pancake makeup to even out blotches, he
+ <ins>
+  could
+ </ins>
+ <del>
+  can
+ </del>
+ appear very pale.
+ The structure of his face
+ <del>
+  has
+ </del>
+ changed as well; several surgeons have speculated that Jackson had undergone multiple nasal surgeries, a forehead lift, thinned lips and a cheekbone surgery.
@@ -939 +1303,7 @@
- Some medical professionals have publicly stated their belief that the singer has
+ Some medical professionals have publicly stated their belief that the singer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
@@ -947 +1317,7 @@
-  Why not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
+  <ins>
+   "Why
+  </ins>
+  <del>
+   Why
+  </del>
+  not just tell people I'm an alien from Mars. Tell them I eat live chickens and do a voodoo dance at midnight. They'll believe anything
@@ -955 +1331,22 @@
-  . But if I, Michael Jackson, were to say, "I'm an alien from Mars and I eat live chickens and do a voodoo dance at midnight", people would say, "Oh, man, that Michael Jackson is
+  . But if I, Michael Jackson, were to say,
+  <ins>
+   'I'm
+  </ins>
+  <del>
+   "I'm
+  </del>
+  an alien from Mars and I eat live chickens and do a voodoo dance at
+  <ins>
+   midnight,'
+  </ins>
+  <del>
+   midnight",
+  </del>
+  people would say,
+  <ins>
+   'Oh,
+  </ins>
+  <del>
+   "Oh,
+  </del>
+  man, that Michael Jackson is
@@ -959 +1356,7 @@
-  . He's cracked up. You can't believe a damn word that comes out of his mouth".
+  . He's cracked up. You can't believe a damn word that comes out of his
+  <ins>
+   mouth.'"
+  </ins>
+  <del>
+   mouth".
+  </del>
@@ -1010,4 +1413 @@
- <div class="thumbinner" style="width:202px;">
-  <a class="image" href="/wiki/File:Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg">
-   <img alt="" class="thumbimage" data-file-height="2318" data-file-width="2047" height="226" src="//upload.wikimedia.org/wikipedia/commons/thumb/e/e5/Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg/200px-Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e5/Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg/300px-Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e5/Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg/400px-Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg 2x" width="200"/>
-  </a>
+ <div class="thumbinner" style="width:222px;">
@@ -1016,2 +1415,0 @@
-    <a class="internal" href="/wiki/File:Michael_Jackson%27s_%22Bad%22_Jacket_and_Belt.jpg" title="Enlarge">
-    </a>
@@ -1019 +1417,16 @@
-   A gold plated military style jacket with belt, worn by Jackson in the
+   <ins>
+    Jackson wore a gold-plated
+   </ins>
+   <div class="magnify">
+   </div>
+   <del>
+    A gold plated
+   </del>
+   military style jacket with
+   <ins>
+    belt
+   </ins>
+   <del>
+    belt, worn by Jackson
+   </del>
+   in the
@@ -1023 +1436,6 @@
-   era.
+   <ins>
+    era
+   </ins>
+   <del>
+    era.
+   </del>
@@ -1135 +1553,8 @@
- Shortly afterward, Jackson became the first Westerner to appear in a television advert for Russia.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Jackson became the first Westerner to appear in a television advert for Russia.
@@ -1175 +1600,15 @@
- In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the time.
+ In March 1991, Jackson renewed his contract with Sony for $65 million; a record breaking deal at the
+ <ins>
+  time, displacing
+  <a href="/wiki/Neil_Diamond" title="Neil Diamond">
+   Neil Diamond
+  </a>
+  's renewal contract with
+  <a href="/wiki/Columbia_Records" title="Columbia Records">
+   Columbia Records
+  </a>
+  .
+ </ins>
+ <del>
+  time.
+ </del>
@@ -1186 +1625,8 @@
- has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful New Jack Swing album off all time.
+ has shipped 7 million copies in the US and has sold 32 million copies worldwide; it is the most successful
+ <a class="mw-redirect" href="/wiki/New_Jack_Swing" title="New Jack Swing">
+  New Jack Swing
+ </a>
+ <del>
+  album off
+ </del>
+ all time.
@@ -1258 +1704,10 @@
- In a high-profile visit to Africa, Jackson visited several countries, among them Gabon and Egypt.
+ In a high-profile visit to Africa, Jackson visited several countries, among them
+ <a href="/wiki/Gabon" title="Gabon">
+  Gabon
+  Link: /wiki/Gabon and
+ Egypt
+  Link: /wiki/Egypt  .
+ </a>
+ <del>
+  and Egypt.
+ </del>
@@ -1297 +1752,9 @@
- in February 1993, his first television interview since 1979. He grimaced when speaking of his childhood abuse at the hands of his father; he believed he had missed out on much of his childhood years, admitting that he often cried from loneliness. He denied previous tabloid rumors that he bought the bones of the Elephant Man or slept in a hyperbaric oxygen chamber. The entertainer went on to dispel suggestions that he bleached his skin, admitting for the first time that he had
+ in February 1993, his first television interview since 1979. He grimaced when speaking of his childhood abuse at the hands of his father; he believed he had missed out on much of his childhood years, admitting that he often cried from loneliness. He denied previous tabloid rumors that he bought the bones of
+ <a class="mw-redirect" href="/wiki/The_Elephant_Man" title="The Elephant Man">
+  the Elephant Man
+ </a>
+ or slept in a
+ <a class="mw-redirect" href="/wiki/Hyperbaric" title="Hyperbaric">
+  hyperbaric
+ </a>
+ oxygen chamber. The entertainer went on to dispel suggestions that he bleached his skin, admitting for the first time that he had
@@ -1313 +1776,8 @@
- The friendship between Jackson and Evan Chandler broke down. Sometime afterward, Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
+ The friendship between Jackson and Evan Chandler broke down. Sometime
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ Evan Chandler was tape-recorded saying amongst other things, "If I go through with this, I win big-time. There's no way I lose. I will get everything I want and they will be destroyed forever...Michael's career will be over".
@@ -1391 +1861,8 @@
- Shortly afterward, she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
+ Shortly
+ <ins>
+  afterwards,
+ </ins>
+ <del>
+  afterward,
+ </del>
+ she tried to persuade Jackson to settle the allegations out of court and go into rehabilitation to recover—he subsequently did both.
@@ -1412 +1889 @@
- <div class="thumbinner" style="width:222px;">
+ <div class="thumbinner" style="width:172px;">
@@ -1414 +1890,0 @@
-   <img alt="" class="thumbimage" data-file-height="2242" data-file-width="1403" height="352" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Michael_Jackson_sculpture.jpg/220px-Michael_Jackson_sculpture.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Michael_Jackson_sculpture.jpg/330px-Michael_Jackson_sculpture.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Michael_Jackson_sculpture.jpg/440px-Michael_Jackson_sculpture.jpg 2x" width="220"/>
@@ -1484 +1960,8 @@
- Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration With Vocals".
+ Hot 100 at number five, and received a Grammy nomination for "Best Pop Collaboration
+ <ins>
+  with
+ </ins>
+ <del>
+  With
+ </del>
+ Vocals".
@@ -1518,0 +2002,23 @@
+ <ins>
+  In early 1996, the
+  <a href="/wiki/Anti-Defamation_League" title="Anti-Defamation League">
+   Anti-Defamation League
+  </a>
+  (ADL) issued a press release charging Jackson with
+  <a href="/wiki/Antisemitism" title="Antisemitism">
+   antisemitism
+  </a>
+  regarding lyrics in the song "
+  <a href="/wiki/They_Don%27t_Care_About_Us" title="They Don't Care About Us">
+   They Don't Care About Us
+  </a>
+  ", the fourth single from HIStory.
+ The song had originally been recorded with lyrics that included the phrase "Jew me, sue me", and "Kick me, kike me". The ADL complained and Jackson responded by saying he would re-record the lyrics before the album went into production. However the ADL's press release charged that Jackson had performed the song live and included the lyrics in question during the live performance.
+ The dispute over the lyrics upset long-time Jackson friend
+  <a href="/wiki/Steven_Spielberg" title="Steven Spielberg">
+   Steven Spielberg
+  </a>
+  , who considered the song anti-semitic
+ </ins>
+</p>
+<p>
@@ -1523 +2029,7 @@
- began on September 7, 1996, and finished on October 15, 1997. Jackson performed 82 concerts in 58 cities to over 4.5 million fans. The show, which visited 5 continents and 35 countries, became Jackson's most successful in terms of audience figures; he has not toured the world since.
+ began on September 7, 1996, and finished on October 15, 1997. Jackson performed 82 concerts in 58 cities to over 4.5 million fans. The show, which visited 5 continents and 35 countries, became Jackson's most successful in terms of audience
+ <ins>
+  figures.
+ </ins>
+ <del>
+  figures; he has not toured the world since.
+ </del>
@@ -1534,2 +2046,17 @@
- Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them to.
- The couple divorced in 1999, with Rowe giving full custody rights of the children to Jackson; they still remain friends.
+ Originally there were no plans to marry, but following Rowe's first pregnancy, Jackson's mother intervened and persuaded them
+ <ins>
+  to do so.
+ After the
+ </ins>
+ <del>
+  to.
+ The
+ </del>
+ couple divorced in 1999, with Rowe giving full custody rights of the children to
+ <ins>
+  Jackson, they remained
+ </ins>
+ <del>
+  Jackson; they still remain
+ </del>
+ friends.
@@ -1642 +2169,5 @@
- At the time, Jackson was waiting for the licenses to the masters of his albums to revert back to him; this allowed him to promote his old material how he liked and prevented Sony from getting a cut of the profit. Jackson expected this to occur early in the new millennium, however, due to the fine print and various clauses in the contract, this revert date is still many years away. Jackson began an investigation, and it emerged that the attorney who represented the singer in the deal was also representing Sony, creating a
+ At the time, Jackson was waiting for the licenses to the masters of his albums to revert
+ <del>
+  back
+ </del>
+ to him; this allowed him to promote his old material how he liked and prevented Sony from getting a cut of the profit. Jackson expected this to occur early in the new millennium, however, due to the fine print and various clauses in the contract, this revert date is still many years away. Jackson began an investigation, and it emerged that the attorney who represented the singer in the deal was also representing Sony, creating a
@@ -1726 +2257,4 @@
- . The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
+ <del>
+  .
+ </del>
+ The concert was aired on October 21, 2001, and included performances from dozens of major artists, including Jackson, who performed his song "
@@ -1756 +2290,8 @@
- The mother's identity has not been released by Jackson, but he has said the child was the result of
+ The mother's identity
+ <ins>
+  was never
+ </ins>
+ <del>
+  has not been
+ </del>
+ released by Jackson, but he has said the child was the result of
@@ -1785 +2326,8 @@
- . In the US, the album peaked at number 13 and was certified platinum by the RIAA, in the UK it was certified for shipments of at least 1.2 million units.
+ . In the US, the album peaked at number 13 and was certified platinum by the
+ <ins>
+  RIAA;
+ </ins>
+ <del>
+  RIAA,
+ </del>
+ in the UK it was certified for shipments of at least 1.2 million units.
@@ -1806 +2354,5 @@
- Shortly after the documentary aired, Jackson was charged with seven counts of child sexual abuse and two counts of
+ Shortly after the documentary aired, Jackson was charged with seven counts of
+ <a href="/wiki/Child_sexual_abuse" title="Child sexual abuse">
+  child sexual abuse
+ </a>
+ and two counts of
@@ -1810 +2362,11 @@
- in order to commit that felony; all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
+ in order to commit that
+ <ins>
+  <a href="/wiki/Felony" title="Felony">
+   felony
+  </a>
+  ;
+ </ins>
+ <del>
+  felony;
+ </del>
+ all charges regarded the same boy, Gavin Arvizo, who was under 14 at the time of the alleged crime.
@@ -1818 +2380,14 @@
-   Fans show their support for Jackson after the entertainer is accused of child molestation in 2003.
+   Fans
+   <ins>
+    showing
+   </ins>
+   <del>
+    show
+   </del>
+   their support for Jackson after the entertainer is accused of child molestation in
+   <ins>
+    2003
+   </ins>
+   <del>
+    2003.
+   </del>
@@ -1823 +2398,5 @@
- Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend Elizabeth Taylor defended him on
+ Jackson denied the sexual abuse allegations, saying that the sleepovers were in no way sexual in nature. Jackson's friend
+ <a href="/wiki/Elizabeth_Taylor" title="Elizabeth Taylor">
+  Elizabeth Taylor
+ </a>
+ defended him on
@@ -1829,2 +2408,33 @@
- , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of Walt Disney. There was nothing odd about it."
- During the investigation, Jackson's profile was examined by a mental health professional called Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by Katz, was that Jackson had become a regressed 10-year-old and did not fit the profile of a pedophile.
+ , saying that she had been there when they "were in the bed, watching television. There was nothing abnormal about it. There was no touchy-feely going on. We laughed like children and we watched a lot of
+ <a class="mw-redirect" href="/wiki/Walt_Disney_Motion_Pictures_Group" title="Walt Disney Motion Pictures Group">
+  Walt Disney
+ </a>
+ <del>
+  Disney.
+ </del>
+ There was nothing odd about it."
+ During the investigation, Jackson's profile was examined by
+ <del>
+  a
+ </del>
+ mental health professional
+ <del>
+  called
+ </del>
+ Dr. Stan Katz; the doctor spent several hours with the accuser too. The assessment made by
+ <ins>
+  Katz
+ </ins>
+ <del>
+  Katz,
+ </del>
+ was that Jackson had become a regressed 10-year-old and did not fit the profile of a
+ <ins>
+  <a class="mw-redirect" href="/wiki/Pedophile" title="Pedophile">
+   pedophile
+  </a>
+  .
+ </ins>
+ <del>
+  pedophile.
+ </del>
@@ -1834,3 +2444,5 @@
- <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
-  People v. Jackson
- </a>
+ <i>
+  <a class="mw-redirect" href="/wiki/People_v._Jackson" title="People v. Jackson">
+   People v. Jackson
+  </a>
+ </i>
@@ -1879 +2491,7 @@
- format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set afterward.
+ format (DVD video on one side, CD audio on the other), and the whole group of discs was made available as a boxed set
+ <ins>
+  afterwards.
+ </ins>
+ <del>
+  afterward.
+ </del>
@@ -1897 +2515,5 @@
- One of Jackson's first documented public appearances since his trial was in November 2006, when he visited the London office of the
+ One of Jackson's first documented public appearances since his trial was in November 2006, when he visited the
+ <a href="/wiki/London" title="London">
+  London
+ </a>
+ office of the
@@ -1916 +2538,8 @@
- In late 2006, Jackson agreed to share joint custody of his first two children with ex-wife Debbie Rowe.
+ In late 2006, Jackson agreed to share joint custody of his first two children with ex-wife
+ <a href="/wiki/Debbie_Rowe" title="Debbie Rowe">
+  Debbie Rowe
+  Link: /wiki/Debbie_Rowe .
+ </a>
+ <del>
+  Rowe.
+ </del>
@@ -1950,2 +2579,8 @@
- <span class="mw-headline" id="2008_onward:_Milestones.2C_real_estate_and_return_to_live_performance">
-  2008 onward: Milestones, real estate and return to live performance
+ <span class="mw-headline" id="2008.E2.80.932009:_Milestones.2C_real_estate.2C_planned_return_to_live_performance">
+  <ins>
+   2008–2009: Milestones, real estate, planned
+  </ins>
+  <del>
+   2008 onward: Milestones, real estate and
+  </del>
+  return to live performance
@@ -1955 +2590 @@
- <div class="thumbinner" style="width:202px;">
+ <div class="thumbinner" style="width:222px;">
@@ -1957 +2591,0 @@
-   <img alt="" class="thumbimage" data-file-height="322" data-file-width="550" height="117" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f5/NeverlandRides.jpg/200px-NeverlandRides.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f5/NeverlandRides.jpg/300px-NeverlandRides.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f5/NeverlandRides.jpg/400px-NeverlandRides.jpg 2x" width="200"/>
@@ -1964 +2598,8 @@
-   Neverland Ranch was Jackson's permanent residency from 1988 to 2005. Pictured is an aerial shot of the theme park area of the property.
+   Neverland Ranch was Jackson's permanent
+   <ins>
+    residence
+   </ins>
+   <del>
+    residency
+   </del>
+   from 1988 to 2005. Pictured is an aerial shot of the theme park area of the property.
@@ -1971 +2612 @@
-  <a class="mw-disambig" href="/wiki/Thriller" title="Thriller">
+  <a class="mw-redirect" href="/wiki/Thriller_(album)" title="Thriller (album)">
@@ -1981 +2622,8 @@
- , comprising of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
+ ,
+ <ins>
+  comprised
+ </ins>
+ <del>
+  comprising
+ </del>
+ of original material from the album, re-mixes, the previously unreleased song "For All Time" and a DVD. Two singles were released to moderate success: "
@@ -2015 +2663,8 @@
- stood at 688,000 copies, making it the best selling catalog album of 2008.
+ stood at 688,000 copies, making it the
+ <ins>
+  best-selling
+ </ins>
+ <del>
+  best selling
+ </del>
+ catalog album of 2008.
@@ -2018 +2673 @@
- To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
+ To celebrate Jackson's 50th birthday, Sony BMG released a compilation album called
@@ -2036 +2691,15 @@
- . In November, Jackson transferred the title of Neverland Ranch to Sycamore Valley Ranch Company LLC. Jackson still owns an unknown stake in the property—Sycamore Valley Ranch is a
+ . In November, Jackson transferred the title of Neverland Ranch to Sycamore Valley Ranch Company LLC.
+ <ins>
+  At the time of his death, Jackson still owned
+ </ins>
+ <del>
+  Jackson still owns
+ </del>
+ an unknown stake in the property—Sycamore Valley Ranch
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a
@@ -2043 +2712,126 @@
- From July 13, 2009 to March 6, 2010, Jackson is scheduled to perform
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="2009:_Death">
+   2009: Death
+  </span>
+ </ins>
+</h3>
+<table class="mbox-small plainlinks sistersitebox" role="presentation" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
+ <tr>
+  <td class="mbox-image">
+  </td>
+  <td class="mbox-text plainlist">
+   <ins>
+    <i>
+     <a href="/wiki/Wikinews" title="Wikinews">
+      Wikinews
+     </a>
+    </i>
+    has related news:
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <td class="mbox-text" colspan="2">
+   <div style="margin-left: 5px; clear: left;">
+    <ul>
+     <li>
+      <ins>
+       <a class="extiw" href="https://en.wikinews.org/wiki/Singer_and_songwriter_Michael_Jackson_dies_at_age_50" title="n:Singer and songwriter Michael Jackson dies at age 50">
+        Singer and songwriter Michael Jackson dies at age 50
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a class="extiw" href="https://en.wikinews.org/wiki/News_of_Michael_Jackson%27s_death_overloads_Internet_sites_and_sparks_hoaxes" title="n:News of Michael Jackson's death overloads Internet sites and sparks hoaxes">
+        News of Michael Jackson's death overloads Internet sites and sparks hoaxes
+       </a>
+      </ins>
+     </li>
+     <li>
+      <ins>
+       <a class="extiw" href="https://en.wikinews.org/wiki/Fans_mourn_and_celebrate_death_of_Michael_Jackson_around_the_world" title="n:Fans mourn and celebrate death of Michael Jackson around the world">
+        Fans mourn and celebrate death of Michael Jackson around the world
+       </a>
+      </ins>
+     </li>
+    </ul>
+   </div>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  On June 25, 2009, Jackson collapsed at a rented home in
+  <a class="mw-redirect" href="/wiki/Holmby_Hills,_Los_Angeles,_California" title="Holmby Hills, Los Angeles, California">
+   Holmby Hills
+  </a>
+  in
+  <a href="/wiki/Los_Angeles" title="Los Angeles">
+   Los Angeles
+  </a>
+  . Jackson's personal physician attempted to resuscitate him.
+  <a href="/wiki/Los_Angeles_Fire_Department" title="Los Angeles Fire Department">
+   Los Angeles Fire Department
+  </a>
+  paramedics responded to a
+  <a href="/wiki/9-1-1" title="9-1-1">
+   911
+  </a>
+  call at 12:21 pm (
+  <a class="mw-redirect" href="/wiki/Pacific_time" title="Pacific time">
+   PST
+  </a>
+  ),
+ arriving nine minutes later. Jackson was reportedly not breathing and
+  <a href="/wiki/Cardiopulmonary_resuscitation" title="Cardiopulmonary resuscitation">
+   CPR
+  </a>
+  was performed.
+ Resuscitation efforts continued both en route to the
+  <a href="/wiki/Ronald_Reagan_UCLA_Medical_Center" title="Ronald Reagan UCLA Medical Center">
+   UCLA Medical Center
+  </a>
+  , and after arrival at approximately 1:14 pm (20:14 UTC), for a further hour.
+ Jackson remained in a
+  <a href="/wiki/Coma" title="Coma">
+   coma
+  </a>
+  and died shortly after arrival.
+ He was noted to have already been in
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  by the paramedics who attended his house.
+ Jackson was pronounced dead at about 2:26 pm local time (21:26 UTC).
+ The cause of death is currently unknown.
+ </ins>
+</p>
+<p>
+ <ins>
+  The case was transferred to the Los Angeles
+  <a href="/wiki/Coroner" title="Coroner">
+   coroner
+  </a>
+  for investigation.
+ Jackson's body was transported by helicopter from UCLA to the LA Coroners offices in
+  <a class="mw-redirect" href="/wiki/Boyle_Heights" title="Boyle Heights">
+   Boyle Heights
+  </a>
+  .
+ The
+  <a href="/wiki/Autopsy" title="Autopsy">
+   autopsy
+  </a>
+  is scheduled for Friday, June 26, 2009.
+ </ins>
+</p>
+<ins>
+ At the time of his death, Jackson was scheduled
+</ins>
+<del>
+ From July 13, 2009 to March 6, 2010, Jackson is scheduled
+</del>
+ to perform
@@ -2045,2 +2839,9 @@
-  50 sell out concerts
- </a>
+ 50
+ <ins>
+  sold-out
+ </ins>
+ <del>
+  sell out
+ </del>
+ concerts
+</a>
@@ -2049,7 +2850,13 @@
-  O
-  <sub>
-   2
-  </sub>
-  arena
- </a>
- . According to Jackson's website, tickets sales for the concerts broke several records. During a prior press conference, Jackson made suggestions of possible retirement.
+ O
+ <sub>
+  2
+ </sub>
+ arena
+</a>
+<ins>
+ , from July 13, 2009, to March 6, 2010. During a publicity
+</ins>
+<del>
+ . According to Jackson's website, tickets sales for the concerts broke several records. During a prior
+</del>
+ press conference, Jackson made suggestions of possible retirement.
@@ -2058,3 +2865,41 @@
-  AEG Live
- </a>
- stated that the first 10 dates alone would earn the singer approximately £50 million.
+ AEG Live
+</a>
+<ins>
+ , had
+</ins>
+ stated that the first 10 dates alone would <ins>
+ have earned
+</ins>
+<del>
+ earn
+</del>
+ the singer approximately £50 million.
+ <p>
+ <ins>
+  Jackson's death caused an outpouring of grief among fans not seen since the deaths of
+  <a href="/wiki/Elvis_Presley" title="Elvis Presley">
+   Elvis Presley
+  </a>
+  and
+  <a href="/wiki/John_Lennon" title="John Lennon">
+   John Lennon
+  </a>
+  as fans gathered outside the UCLA Medical Center and his Holmby Hills home.
+ Fans also gathered in
+  <a href="/wiki/New_York_City" title="New York City">
+   New York City
+  </a>
+  outside the
+  <a href="/wiki/Apollo_Theater" title="Apollo Theater">
+   Apollo Theater
+  </a>
+  and in
+  <a href="/wiki/Detroit" title="Detroit">
+   Detroit
+  </a>
+  outside
+  <a href="/wiki/Hitsville_U.S.A." title="Hitsville U.S.A.">
+   Hitsville U.S.A.
+  </a>
+  , the old Motown headquarters now the Motown Museum, where fans created a shrine.
+ </ins>
@@ -2063,21 +2908,37 @@
- On June 25, 2009,
- <a href="/wiki/TMZ" title="TMZ">
-  TMZ
- </a>
- reported that Jackson has been rushed to the hospital and has suffered
- <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
-  cardiac arrest
- </a>
- .
- <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
-  Paramedics
- </a>
- administered
- <a class="mw-redirect" href="/wiki/CPR" title="CPR">
-  CPR
- </a>
- in the
- <a href="/wiki/Ambulance" title="Ambulance">
-  ambulance
- </a>
- and a family member says that he is in "really bad shape.".
+ <ins>
+  In the hours following Jackson's death, his record sales increased dramatically. His seminal album
+  <i>
+   Thriller
+  </i>
+  climbed to number one on the American iTunes music chart, while another eight have made it into the top 40.
+ In the UK, where Jackson would have performed less than three weeks after his death, fifteen of his albums occupied the top 15 spots on the
+  <a href="/wiki/Amazon.com" title="Amazon.com">
+   Amazon
+  </a>
+  music chart.
+ </ins>
+</p>
+<p>
+ <del>
+  On June 25, 2009,
+  <a href="/wiki/TMZ" title="TMZ">
+   TMZ
+  </a>
+  reported that Jackson has been rushed to the hospital and has suffered
+  <a href="/wiki/Cardiac_arrest" title="Cardiac arrest">
+   cardiac arrest
+  </a>
+  .
+  <a class="mw-redirect" href="/wiki/Paramedics" title="Paramedics">
+   Paramedics
+  </a>
+  administered
+  <a class="mw-redirect" href="/wiki/CPR" title="CPR">
+   CPR
+  </a>
+  in the
+  <a href="/wiki/Ambulance" title="Ambulance">
+   ambulance
+  </a>
+  and a family member says that he is in "really bad shape.".
+ </del>
@@ -2100,2 +2961,10 @@
- asserts that throughout his solo career, Jackson's versatility has allowed him to experiment with various themes and genres.
- As a musician, he has ranged from
+ asserts that throughout his solo career, Jackson's versatility
+ <del>
+  has
+ </del>
+ allowed him to experiment with various themes and genres.
+ As a musician, he
+ <del>
+  has
+ </del>
+ ranged from
@@ -2177 +3046,8 @@
- , Christoper Connelly of
+ ,
+ <ins>
+  Christopher
+ </ins>
+ <del>
+  Christoper
+ </del>
+ Connelly of
@@ -2318 +3194,8 @@
- presents Jackson as a stark paradoxal individual.
+ presents Jackson as a stark
+ <ins>
+  paradoxical
+ </ins>
+ <del>
+  paradoxal
+ </del>
+ individual.
@@ -2430 +3313,22 @@
- Jackson has been singing since a child, and over time his voice and vocal style have notably changed, either through puberty or a personal preference to align his vocal interpretation to the themes and genres he chooses to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
+ Jackson
+ <ins>
+  sang since he was
+ </ins>
+ <del>
+  has been singing since
+ </del>
+ a child, and over time his voice and vocal style
+ <ins>
+  changed noticeably,
+ </ins>
+ <del>
+  have notably changed,
+ </del>
+ either through puberty or a personal preference to align his vocal interpretation to the themes and genres he
+ <ins>
+  chose
+ </ins>
+ <del>
+  chooses
+ </del>
+ to express. Between 1971 and 1975, Jackson's voice "descended ever so slightly from boy soprano to his current androgynous high tenor".
@@ -2435 +3339,8 @@
- ". The purpose of the hiccup—somewhat like a gulping for air or gasping—is to help promote a certain emotion, be it excitement, sadness or fear.
+ ". The purpose of the hiccup—somewhat like a gulping for air or
+ <ins>
+  gasping—was
+ </ins>
+ <del>
+  gasping—is
+ </del>
+ to help promote a certain emotion, be it excitement, sadness or fear.
@@ -2449 +3360,8 @@
- . Their analysis was also that "Jackson's feathery-timbered tenor is extraordinary beautiful. It slides smoothly into a startling
+ . Their analysis was also that "Jackson's feathery-timbered tenor is
+ <ins>
+  extraordinarily
+ </ins>
+ <del>
+  extraordinary
+ </del>
+ beautiful. It slides smoothly into a startling
@@ -2521 +3439,5 @@
- The turn of the 1990s saw the release of the paradoxical, introspective album
+ The turn of the 1990s saw the release of the
+ <del>
+  paradoxical,
+ </del>
+ introspective album
@@ -2525 +3447,7 @@
- , here Jackson used his vocals to intensify the split themes and genres described earlier.
+ <ins>
+  ;
+ </ins>
+ <del>
+  ,
+ </del>
+ here Jackson used his vocals to intensify the split themes and genres described earlier.
@@ -2541 +3469,8 @@
- Nelson George summed by Jackson's vocals as, "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
+ Nelson George summed
+ <ins>
+  up Jackson's vocals by stating
+ </ins>
+ <del>
+  by Jackson's vocals as,
+ </del>
+ "The grace, the aggression, the growling, the natural boyishness, the falsetto, the smoothness—that combination of elements mark him as a major vocalist".
@@ -2554 +3489,3 @@
-   Image of
+   <del>
+    Image of
+   </del>
@@ -2560,2 +3497,4 @@
-   <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
-    <span style="display: none;">
+   <ins>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
@@ -2563,2 +3502,22 @@
-   </span>
-   filed by Jackson describing the anti-gravity lean used in the music video for "Smooth Criminal".
+    , filed by Jackson, described
+   </ins>
+   <del>
+    <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=5255452&amp;rft.cc=US&amp;rft.title=">
+     <span style="display: none;">
+     </span>
+    </span>
+    filed by Jackson describing
+   </del>
+   the
+   <a href="/wiki/Smooth_Criminal#Anti-gravity_lean" title="Smooth Criminal">
+    anti-gravity lean
+   </a>
+   used in the music video for
+   <ins>
+    <a href="/wiki/Smooth_Criminal" title="Smooth Criminal">
+     Smooth Criminal
+    </a>
+   </ins>
+   <del>
+    "Smooth Criminal".
+   </del>
@@ -2725 +3684 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -2754 +3713,7 @@
- A year later, it won a Grammy for Best Music Video, Short Form; shortly afterward
+ A year later, it won a Grammy for Best Music Video, Short Form; shortly
+ <ins>
+  afterwards
+ </ins>
+ <del>
+  afterward
+ </del>
@@ -2814 +3779 @@
- <div class="thumbinner" style="width:192px;">
+ <div class="thumbinner" style="width:222px;">
@@ -2816 +3780,0 @@
-   <img alt="" class="thumbimage" data-file-height="415" data-file-width="575" height="137" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/95/MJ_Star.jpg/190px-MJ_Star.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/95/MJ_Star.jpg/285px-MJ_Star.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/95/MJ_Star.jpg/380px-MJ_Star.jpg 2x" width="190"/>
@@ -2823 +3787,11 @@
-   Jackson's star on the Hollywood walk of fame, set in 1984.
+   Jackson's star on the Hollywood walk of fame, set in
+   <p>
+    <ins>
+     1984
+  Michael Jackson was inducted
+    </ins>
+    <del>
+     1984.
+  Inducted
+    </del>
+   </p>
@@ -2827,2 +3801 @@
-<p>
- Inducted into the
+ into the
@@ -2830,3 +3803,13 @@
-  Hollywood Walk of Fame
- </a>
- in 1984, Jackson has had a notable impact on music and culture throughout the world. He broke down racial barriers, transformed the art of the music video and paved the way for modern pop music in his own country. Jackson's work, distinctive musical sound and vocal style have influenced
+ Hollywood Walk of Fame
+</a>
+ in <ins>
+ 1984. There are actually two stars bearing the name Michael Jackson in the walk, the other being that of
+ <a href="/wiki/Michael_Jackson_(radio_commentator)" title="Michael Jackson (radio commentator)">
+  the Los Angeles talk radio show host
+ </a>
+ of the same name; while Jackson's name is marked with a record icon, the radio Jackson is denoted with a microphone icon. Jackson
+</ins>
+<del>
+ 1984, Jackson has
+</del>
+ had a notable impact on music and culture throughout the world. He broke down racial barriers, transformed the art of the music video and paved the way for modern pop music in his own country. Jackson's work, distinctive musical sound and vocal style have influenced
@@ -2834,2 +3817,2 @@
-  hip hop
- </a>
+ hip hop
+</a>
@@ -2838,2 +3821,2 @@
-  pop
- </a>
+ pop
+</a>
@@ -2842,2 +3825,2 @@
-  R&amp;B
- </a>
+ R&amp;B
+</a>
@@ -2846,2 +3829,2 @@
-  Mariah Carey
- </a>
+ Mariah Carey
+</a>
@@ -2850,2 +3833,2 @@
-  Usher
- </a>
+ Usher
+</a>
@@ -2854,2 +3837,2 @@
-  Britney Spears
- </a>
+ Britney Spears
+</a>
@@ -2858,2 +3841,2 @@
-  Justin Timberlake
- </a>
+ Justin Timberlake
+</a>
@@ -2862,2 +3845,2 @@
-  R. Kelly
- </a>
+ R. Kelly
+</a>
@@ -2866,2 +3849 @@
-</p>
-<p>
+ <p>
@@ -2881 +3863,8 @@
- He is a double-inductee of the
+ He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a double-inductee of the
@@ -2904 +3893,12 @@
- in his solo career—more than any other male artist in the Hot 100 era—and the sale of over 750 million albums worldwide, making him the world's best selling male pop artist.
+ in his solo career—more than any other male artist in the Hot 100 era—and the sale of over 750 million
+ <ins>
+  records
+ </ins>
+ <del>
+  albums
+ </del>
+ worldwide, making him the world's best selling male
+ <ins>
+  solo
+ </ins>
+ pop artist.
@@ -2920 +3920,11 @@
-   in June 1988.
+   in June
+   <p>
+    <ins>
+     1988
+  He was
+    </ins>
+    <del>
+     1988.
+  He is
+    </del>
+   </p>
@@ -2924,2 +3934 @@
-<p>
- He is characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
+ characterized as "an unstoppable juggernaut, possessed of all the tools to dominate the charts seemingly at will: an instantly identifiable voice, eye-popping dance moves, stunning musical versatility and loads of sheer star power".
@@ -2928,2 +3937,2 @@
-  Time
- </i>
+ Time
+</i>
@@ -2932,2 +3941,2 @@
-  Elvis Presley
- </a>
+ Elvis Presley
+</a>
@@ -2937,4 +3946,4 @@
-  <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
-   Vanity Fair
-  </a>
- </i>
+ <a href="/wiki/Vanity_Fair_(magazine)" title="Vanity Fair (magazine)">
+  Vanity Fair
+ </a>
+</i>
@@ -2943,4 +3952,4 @@
-  <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
-   Daily Telegraph
-  </a>
- </i>
+ <a class="mw-redirect" href="/wiki/Daily_Telegraph" title="Daily Telegraph">
+  Daily Telegraph
+ </a>
+</i>
@@ -2949,2 +3958,2 @@
-  Tom Utley
- </a>
+ Tom Utley
+</a>
@@ -2954,2 +3963,2 @@
-  royalties
- </a>
+ royalties
+</a>
@@ -2959,3 +3968,6 @@
-  successful career
- </a>
- , has made him a part of
+ successful career
+</a>
+ , <del>
+ has
+</del>
+ made him a part of
@@ -2963,2 +3975,2 @@
-  popular culture
- </a>
+ popular culture
+</a>
@@ -2966,2 +3978 @@
-</p>
-<h2>
+ <h2>
@@ -3049 +4060 @@
-   <a href="/wiki/HIStory:_Past,_Present_and_Future,_Book_I" title="HIStory: Past, Present and Future, Book I">
+   <a class="mw-redirect" href="/wiki/HIStory" title="HIStory">
@@ -3050,0 +4062,2 @@
+    Link: /wiki/HIStory (1995) (one disc greatest hits album, one disc new studio album)
+  Link: /wiki/HIStory:_Past,_Present_and_Future,_Book_I (1995)
@@ -3053 +4065,0 @@
-  (1995)
@@ -3071,2 +4083,3 @@
-  <a class="mw-redirect" href="/wiki/List_of_best-selling_albums_worldwide" title="List of best-selling albums worldwide">
-   List of best-selling albums worldwide
+  <a class="mw-redirect" href="/wiki/List_of_awards_received_by_Michael_Jackson" title="List of awards received by Michael Jackson">
+   List of  awards received by Michael Jackson
+   Link: /wiki/List_of_awards_received_by_Michael_Jackson List of   best-selling albums worldwide
@@ -3080,0 +4094,14 @@
+  <ins>
+   <a class="mw-redirect" href="/wiki/List_of_honorific_titles_in_popular_music" title="List of honorific titles in popular music">
+    List of honorific titles in popular music
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a class="mw-redirect" href="/wiki/List_of_Michael_Jackson_tours" title="List of Michael Jackson tours">
+    List of Michael Jackson tours
+   </a>
+  </ins>
+ </li>
+ <li>
@@ -3091,3 +4118,5 @@
-  <a class="mw-redirect" href="/wiki/List_of_Michael_Jackson%27s_awards" title="List of Michael Jackson's awards">
-   List of Michael Jackson's awards
-  </a>
+  <ins>
+   <a href="/wiki/Michael_Jackson_videography" title="Michael Jackson videography">
+    Michael Jackson videography
+   </a>
+  </ins>
@@ -3096,3 +4125,5 @@
-  <a class="mw-redirect" href="/wiki/List_of_Michael_Jackson_tours" title="List of Michael Jackson tours">
-   List of Michael Jackson tours
-  </a>
+  <del>
+   <a class="mw-redirect" href="/wiki/List_of_Michael_Jackson%27s_awards" title="List of Michael Jackson's awards">
+    List of Michael Jackson's awards
+   </a>
+  </del>
@@ -3101,3 +4132,5 @@
-  <a class="mw-redirect" href="/wiki/Michael_Jackson_filmography" title="Michael Jackson filmography">
-   Michael Jackson filmography
-  </a>
+  <del>
+   <a class="mw-redirect" href="/wiki/List_of_Michael_Jackson_tours" title="List of Michael Jackson tours">
+    List of Michael Jackson tours
+   </a>
+  </del>
@@ -3106,3 +4139,12 @@
-  <a class="mw-redirect" href="/wiki/List_of_honorific_titles_in_popular_music" title="List of honorific titles in popular music">
-   List of honorific titles in popular music
-  </a>
+  <del>
+   <a class="mw-redirect" href="/wiki/Michael_Jackson_filmography" title="Michael Jackson filmography">
+    Michael Jackson filmography
+   </a>
+  </del>
+ </li>
+ <li>
+  <del>
+   <a class="mw-redirect" href="/wiki/List_of_honorific_titles_in_popular_music" title="List of honorific titles in popular music">
+    List of honorific titles in popular music
+   </a>
+  </del>
@@ -3120 +4162 @@
-</ul>
+</ul>
\ No newline at end of file
