--- 493040889
+++ 493040993
@@ -5,2 +5,10 @@
- (born LaDonna Adrian Gaines on December 31, 1948)
- is an American singer/songwriter who gained prominence during the
+ (born LaDonna Adrian Gaines on December 31,
+ <ins>
+  1948 - May 17, 2012)
+ was
+ </ins>
+ <del>
+  1948)
+ is
+ </del>
+ an American singer/songwriter who gained prominence during the
@@ -2391 +2399 @@
-</ul>
+</ul>
\ No newline at end of file
