--- 493040889
+++ 493051348
@@ -1,0 +2,6 @@
+ <del>
+  <b>
+   Donna Summer
+  </b>
+  (born
+ </del>
@@ -3 +9 @@
-  Donna Summer
+  LaDonna Adrian Gaines
@@ -5,2 +11,13 @@
- (born LaDonna Adrian Gaines on December 31, 1948)
- is an American singer/songwriter who gained prominence during the
+ <ins>
+  (December 31, 1948 – May 17, 2012)
+ , known by the stage name
+  <b>
+   Donna Summer
+  </b>
+  , was
+ </ins>
+ <del>
+  on December 31, 1948)
+ is
+ </del>
+ an American singer/songwriter who gained prominence during the
@@ -10 +27,8 @@
- era of the 1970s. She has a
+ era of the 1970s. She
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ a
@@ -14 +38,8 @@
- vocal range, and is a five-time
+ vocal range, and
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a five-time
@@ -33 +64,4 @@
- within a thirteen-month period.
+ within a
+ <ins>
+  13-month period.
+ </ins>
@@ -35,0 +70,8 @@
+ <ins>
+  Summer died on May 17, 2012, after losing her battle with cancer.
+ </ins>
+</p>
+<del>
+ thirteen-month period.
+</del>
+<p>
@@ -61 +103,7 @@
- , she was one of seven children raised by devout
+ <ins>
+  . She
+ </ins>
+ <del>
+  , she
+ </del>
+ was one of seven children raised by devout
@@ -88 +136,5 @@
- after listening to her albums as member of
+ after listening to her albums as
+ <ins>
+  a
+ </ins>
+ member of
@@ -92,5 +144,20 @@
- , and joined the
- <a href="/wiki/Psychedelic_rock" title="Psychedelic rock">
-  psychedelic rock
- </a>
- group the Crow as lead singer. Beforehand, Summer dropped out of school convinced that music was her way out of Boston, where she had always felt herself to be an outsider, even among her own family who ridiculed her for her voice and her looks. The group was short-lived, as they split upon their arrival in
+ , and
+ <del>
+  joined the
+  <a href="/wiki/Psychedelic_rock" title="Psychedelic rock">
+   psychedelic rock
+  </a>
+  group the Crow as lead singer. Beforehand, Summer
+ </del>
+ dropped out of school convinced that music was her way out of Boston, where she had always felt herself to be an outsider, even among her own family who ridiculed her for her voice and her looks.
+ <ins>
+  She joined the
+  <a href="/wiki/Psychedelic_rock" title="Psychedelic rock">
+   psychedelic rock
+  </a>
+  group the Crow as lead singer, but the group was short-lived
+ </ins>
+ <del>
+  The group was short-lived,
+ </del>
+ as they split upon their arrival in
@@ -106 +173,7 @@
- . She lost the part of Sheila to
+ <ins>
+  , but she
+ </ins>
+ <del>
+  . She
+ </del>
+ lost the part of Sheila to
@@ -118,5 +191,22 @@
- for several years. While in Germany, where she learned to speak German fluently, she participated in the musicals "
- <a class="new" href="/w/index.php?title=Ich_Bin_Ich,_the_German_version_of_The_Me_Nobody_Knows,&amp;action=edit&amp;redlink=1" title="Ich Bin Ich, the German version of The Me Nobody Knows, (page does not exist)">
-  Ich Bin Ich, the German version of The Me Nobody Knows,
- </a>
- "
+ for several years. While in Germany, where she learned to speak German fluently, she participated in the musicals
+ <ins>
+  <i>
+   Ich Bin Ich
+  </i>
+  (the
+ </ins>
+ <del>
+  "
+ Ich Bin Ich, the
+ </del>
+ German version of
+ <i>
+  <a href="/wiki/The_Me_Nobody_Knows" title="The Me Nobody Knows">
+   The Me Nobody Knows
+   Link: /wiki/The_Me_Nobody_Knows ),
+  </a>
+ </i>
+ <del>
+  Knows,
+  Link: /w/index.php?title=Ich_Bin_Ich,_the_German_version_of_The_Me_Nobody_Knows,&amp;action=edit&amp;redlink=1 "
+ </del>
@@ -146 +236,15 @@
- ). She came to the group in 1973 and toured with the group throughout Europe.
+ ). She
+ <ins>
+  joined
+ </ins>
+ <del>
+  came to
+ </del>
+ the group in 1973 and toured with
+ <ins>
+  them
+ </ins>
+ <del>
+  the group
+ </del>
+ throughout Europe.
@@ -168 +272,8 @@
- (where she was credited under the pseudonym Gayn Pierre). Several single releases over the years have included a young Donna performing with the group, even though she often denied ever singing on any of the Marvos releases. It should also be noted that the name 'Gayn Pierre' was also used by Donna whilst performing in 'Godspell' with Helmuth Sommer during 1972.
+ (where she was credited under the pseudonym Gayn Pierre). Several single releases over the years have included a young Donna performing with the group, even though she often denied ever singing on any of the Marvos releases.
+ <ins>
+  The
+ </ins>
+ <del>
+  It should also be noted that the
+ </del>
+ name 'Gayn Pierre' was also used by Donna whilst performing in 'Godspell' with Helmuth Sommer during 1972.
@@ -741 +852,7 @@
- In the mid 1980s, Summer was embroiled in a controversy. She had allegedly made anti-gay remarks regarding the then-relatively new disease
+ In the mid 1980s, Summer was embroiled in a controversy. She had allegedly made anti-gay remarks regarding the then-relatively new
+ <ins>
+  disease,
+ </ins>
+ <del>
+  disease
+ </del>
@@ -757 +874,8 @@
-   "a terrible misunderstanding. I was unknowingly protected by those around me from the bad press and hate letters....If I have caused you pain, forgive me."
+   "a terrible misunderstanding. I was unknowingly protected by those around me from the bad press and hate
+   <ins>
+    letters ... If
+   </ins>
+   <del>
+    letters....If
+   </del>
+   I have caused you pain, forgive me."
@@ -791,7 +915,26 @@
- According to an
- <i>
-  <a class="mw-redirect" href="/wiki/A%26E_Biography" title="A&amp;E Biography">
-   A&amp;E Biography
-  </a>
- </i>
- program dedicated to Summer in which she participated in 1995, the lawsuit was settled out of court though neither side were able to divulge any details.
+ According to
+ <ins>
+  a
+  <i>
+   <a href="/wiki/Biography_(TV_series)" title="Biography (TV series)">
+    Biography
+   </a>
+  </i>
+  television
+ </ins>
+ <del>
+  an
+  <i>
+   <a class="mw-redirect" href="/wiki/A%26E_Biography" title="A&amp;E Biography">
+    A&amp;E Biography
+   </a>
+  </i>
+ </del>
+ program dedicated to Summer in which she participated in 1995, the lawsuit was settled out of court though neither side
+ <ins>
+  was
+ </ins>
+ <del>
+  were
+ </del>
+ able to divulge any details.
@@ -1090 +1233,8 @@
- , in honor of US President
+ , in honor of
+ <ins>
+  United States
+ </ins>
+ <del>
+  US
+ </del>
+ President
@@ -1134 +1284,8 @@
-  On The Radio
+  On
+  <ins>
+   the
+  </ins>
+  <del>
+   The
+  </del>
+  Radio
@@ -1181 +1338 @@
- launched in 2011 in which 12 singer-songwriters compete through innovative songwriting challenges that will test their creativity, patience and drive. Every episode features a different topic from a dance track to a love ballad, that require the contestants to write and perform lyrics from a multiple of genres, for a cash prize of $100,000, a publishing deal with songwriting collective The Writing Camp, and a recording deal with RCA/Jive label
+ launched in 2011 in which 12 singer-songwriters compete through innovative songwriting challenges that will test their creativity, patience and drive. Every episode features a different topic from a dance track to a love ballad, that require the contestants to write and perform lyrics from a multiple of genres, for a cash prize of $100,000, a publishing deal with songwriting collective The Writing Camp, and a recording deal with RCA/Jive
@@ -1184 +1341,10 @@
- In July, 2011 Summer was working at Paramount Recording Studios in Los Angeles with her nephew, the rapper and producer
+ <ins>
+  label.
+In July 2011,
+ </ins>
+ <del>
+  label
+In July, 2011
+ </del>
+</p>
+ Summer was working at Paramount Recording Studios in Los Angeles with her nephew, the rapper and producer
@@ -1186,2 +1352,2 @@
-  O'Mega Red
- </a>
+ O'Mega Red
+</a>
@@ -1189,2 +1355 @@
-</p>
-<h2>
+ <h2>
@@ -1298 +1463,15 @@
- . In 1973, the couple married and that year Summer gave birth to her first child, daughter Mimi Sommer. However, the marriage crumbled and by 1975, they had formally divorced. Summer took her husband's last name (though slightly altering the spelling) as her
+ . In 1973, the couple married and that year Summer gave birth to her first child, daughter Mimi Sommer. However, the marriage crumbled and by 1975, they had formally divorced. Summer took her husband's last
+ <ins>
+  name,
+ </ins>
+ <del>
+  name (though
+ </del>
+ slightly altering the
+ <ins>
+  spelling,
+ </ins>
+ <del>
+  spelling)
+ </del>
+ as her
@@ -1302 +1481,33 @@
- . After she had divorced from Helmuth, she moved into her Los Angeles house with lover Peter Mühldorfer, who was a respected surrealist painter. As her star continued to rise, Mühldorfer resented all the press and public attention and it drove a huge wedge between them. She has stated that he became violent and he was eventually (with the help of Casablanca Records mogul Neil Bogart) shipped back to Germany after having his visa revoked.
+ . After she had divorced from Helmuth, she moved into her Los Angeles house with lover Peter Mühldorfer, who was a respected surrealist painter. As her
+ <ins>
+  fame increased,
+ </ins>
+ <del>
+  star continued to rise,
+ </del>
+ Mühldorfer resented all the press and public attention and it drove a
+ <del>
+  huge
+ </del>
+ wedge between them. She has stated that he became violent and
+ <ins>
+  with
+ </ins>
+ <del>
+  he was eventually (with
+ </del>
+ the help of Casablanca Records mogul Neil
+ <ins>
+  Bogart he was eventually forced to return
+ </ins>
+ <del>
+  Bogart) shipped back
+ </del>
+ to Germany after
+ <ins>
+  his visa was
+ </ins>
+ <del>
+  having his visa
+ </del>
+ revoked.
@@ -1321,4 +1532,18 @@
- . Within a few months, Summer and Sudano became an item. The couple married on July 16, 1980. A year later, Summer gave birth to another daughter (her first child with Sudano), Brooklyn (named after Sudano's group). A year after that, Summer and Sudano had their second child, Amanda.
-</p>
-<p>
- Summer has often talked about her early successful years as a period of confusion and anxiety. By mid-1977, struggling with the media's titles of her as
+ . Within a few months, Summer and Sudano became an item. The couple married on July 16, 1980. A year later, Summer gave birth to another daughter (her first child with Sudano),
+ <ins>
+  Brooklyn, named after Sudano's group.
+ </ins>
+ <del>
+  Brooklyn (named after Sudano's group).
+ </del>
+ A year after that, Summer and Sudano had their second child, Amanda.
+</p>
+<p>
+ Summer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ often talked about her early successful years as a period of confusion and anxiety. By mid-1977, struggling with the media's titles of her as
@@ -1328 +1553,4 @@
- , she began suffering from depression and started having
+ , she began suffering from depression and
+ <del>
+  started having
+ </del>
@@ -1332 +1560,8 @@
- . Summer has written in her memoirs that she had attempted
+ . Summer
+ <ins>
+  wrote
+ </ins>
+ <del>
+  has written
+ </del>
+ in her memoirs that she had attempted
@@ -1336 +1571,7 @@
- a few times. Her rapid rise to success, combined with some serious regrets about mistakes in her personal life, began to take its toll. During this time, she self-medicated on
+ <ins>
+  several
+ </ins>
+ <del>
+  a few
+ </del>
+ times. Her rapid rise to success, combined with some serious regrets about mistakes in her personal life, began to take its toll. During this time, she self-medicated on
@@ -1339,0 +1581,3 @@
+ <ins>
+  ,
+ </ins>
@@ -1348 +1592,8 @@
- ", would no longer be performed. A quarter of a century later, however, she began performing the song again live. As recently as 2011, she even rerecorded the track, complete with racy sighs and moans, for the 'Loverdose' fragrance advertisement by Diesel.
+ ", would no longer be performed. A quarter of a century later, however, she began performing the song again live. As recently as 2011, she even rerecorded the track, complete with racy sighs and moans, for the
+ <ins>
+  "Loverdose"
+ </ins>
+ <del>
+  'Loverdose'
+ </del>
+ fragrance advertisement by Diesel.
@@ -1359 +1610,11 @@
- where she took time out from showbusiness to focus on painting, a hobby she began in 1985. In 1995, Summer's mother died. Today, Summer still lives in Nashville with her husband and is a grandmother of four.
+ where she took time out from
+ <ins>
+  show business
+ </ins>
+ <del>
+  showbusiness
+ </del>
+ to focus on painting, a hobby she began in 1985. In 1995, Summer's mother died.
+ <del>
+  Today, Summer still lives in Nashville with her husband and is a grandmother of four.
+ </del>
@@ -1368,22 +1629,72 @@
-  One NAACP Image Award.
- </li>
- <li>
-  One time Juno Award nomination for Best Selling International Single,"I feel Love".
- </li>
- <li>
-  Three Multi-Platinum albums in the US.
- </li>
- <li>
-  Eleven of her albums went Gold in the US.
- </li>
- <li>
-  Twelve Gold singles.
- </li>
- <li>
-  Six American Music Awards.
- </li>
- <li>
-  She was the first female African American to receive an MTV Video Music Awards nomination. ("Best Female Video" and "Best Choreography" for "She Works Hard For The Money")
- </li>
- <li>
-  Two Golden Globe Award nominations (one win for "Last Dance" Song and one nominated for "The Deep" John Barry's Song).
+  One NAACP Image
+  <ins>
+   Award
+  </ins>
+  <del>
+   Award.
+  </del>
+ </li>
+ <li>
+  One time Juno Award nomination for Best Selling International Single,"I
+  <ins>
+   Feel Love"
+  </ins>
+  <del>
+   feel Love".
+  </del>
+ </li>
+ <li>
+  Three Multi-Platinum albums in the
+  <ins>
+   United States
+  </ins>
+  <del>
+   US.
+  </del>
+ </li>
+ <li>
+  Eleven of her albums went Gold in the
+  <ins>
+   United States
+  </ins>
+ </li>
+ <li>
+  <ins>
+   Twelve Gold singles
+  </ins>
+ </li>
+ <del>
+  US.
+ </del>
+ <li>
+  <del>
+   Twelve Gold singles.
+  </del>
+ </li>
+ <li>
+  Six American Music
+  <ins>
+   Awards
+  </ins>
+  <del>
+   Awards.
+  </del>
+ </li>
+ <li>
+  She was the first female African American to receive an MTV Video Music Awards
+  <ins>
+   nomination
+  </ins>
+  <del>
+   nomination.
+  </del>
+  ("Best Female Video" and "Best Choreography" for "She Works Hard For The Money")
+ </li>
+ <li>
+  Two Golden Globe Award nominations (one win for "Last Dance" Song and one nominated for "The Deep" John Barry's
+  <ins>
+   Song)
+  </ins>
+  <del>
+   Song).
+  </del>
@@ -2060,2 +2371,8 @@
- <span class="mw-headline" id="Donna_Summer_concert_tours">
-  Donna Summer concert tours
+ <span class="mw-headline" id="Concert_tours">
+  <ins>
+   Concert
+  </ins>
+  <del>
+   Donna Summer concert
+  </del>
+  tours
@@ -2233,2 +2550,3 @@
-   <a class="new" href="/w/index.php?title=Thank_God_It%27s_Friday_(sountrack)&amp;action=edit&amp;redlink=1" title="Thank God It's Friday (sountrack) (page does not exist)">
-    Thank God It's Friday (sountrack)
+   <a class="mw-redirect" href="/wiki/Thank_God_It%27s_Friday_(soundtrack)" title="Thank God It's Friday (soundtrack)">
+    Thank God It's Friday (soundtrack)
+    Link: /wiki/Thank_God_It%27s_Friday_(soundtrack) (sountrack)
@@ -2248,5 +2566,14 @@
-  <i>
-   <a class="new" href="/w/index.php?title=Donna_Summer_Greatest_Hits/On_The_Radio&amp;action=edit&amp;redlink=1" title="Donna Summer Greatest Hits/On The Radio (page does not exist)">
-    Donna Summer Greatest Hits/On The Radio
-   </a>
-  </i>
+  <ins>
+   <i>
+    <a href="/wiki/On_the_Radio:_Greatest_Hits_Volumes_I_%26_II" title="On the Radio: Greatest Hits Volumes I &amp; II">
+     On the Radio: Greatest Hits Volumes I &amp; II
+    </a>
+   </i>
+  </ins>
+  <del>
+   <i>
+    <a class="new" href="/w/index.php?title=Donna_Summer_Greatest_Hits/On_The_Radio&amp;action=edit&amp;redlink=1" title="Donna Summer Greatest Hits/On The Radio (page does not exist)">
+     Donna Summer Greatest Hits/On The Radio
+    </a>
+   </i>
+  </del>
@@ -2321,2 +2648,2 @@
-   <a class="new" href="/w/index.php?title=Donna_Summer_Anthology&amp;action=edit&amp;redlink=1" title="Donna Summer Anthology (page does not exist)">
-    Donna Summer Anthology
+   <a href="/wiki/The_Donna_Summer_Anthology" title="The Donna Summer Anthology">
+    The Donna Summer Anthology
@@ -2337,10 +2664,9 @@
-   <a class="new" href="/w/index.php?title=Endless_Summer/Greatest_Hits&amp;action=edit&amp;redlink=1" title="Endless Summer/Greatest Hits (page does not exist)">
-    Endless Summer/Greatest Hits
-   </a>
-  </i>
- </li>
- <li>
-  1999:
-  <i>
-   <a class="new" href="/w/index.php?title=Live_and_More_Encore&amp;action=edit&amp;redlink=1" title="Live and More Encore (page does not exist)">
-    Live and More Encore
+   <a class="mw-redirect" href="/wiki/Endless_Summer:_Greatest_Hits" title="Endless Summer: Greatest Hits">
+    Endless Summer: Greatest Hits
+    Link: /wiki/Endless_Summer:_Greatest_Hits 1999:
+  Live &amp; More Encore
+    Link: /wiki/Live_%26_More_Encore    2003:
+  The Journey: The Very Best of Donna Summer
+    Link: /wiki/The_Journey:_The_Very_Best_of_Donna_Summer    Summer/Greatest Hits
+    Link: /w/index.php?title=Endless_Summer/Greatest_Hits&amp;action=edit&amp;redlink=1 1999:
+  Live and More Encore
@@ -2360,3 +2686,5 @@
- <span class="mw-headline" id="See_also">
-  See also
- </span>
+ <del>
+  <span class="mw-headline" id="See_also">
+   See also
+  </span>
+ </del>
@@ -2367 +2694,0 @@
-   <img alt="Book icon" data-file-height="48" data-file-width="48" height="30" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/30px-Office-book.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/45px-Office-book.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/60px-Office-book.svg.png 2x" width="30"/>
@@ -2372,3 +2699,5 @@
-     <a href="/wiki/Book:Donna_Summer" title="Book:Donna Summer">
-      Book: Donna Summer
-     </a>
+     <del>
+      <a href="/wiki/Book:Donna_Summer" title="Book:Donna Summer">
+       Book: Donna Summer
+      </a>
+     </del>
@@ -2382,3 +2711,5 @@
-  <a href="/wiki/Brooklyn_Dreams_(group)" title="Brooklyn Dreams (group)">
-   Brooklyn Dreams
-  </a>
+  <del>
+   <a href="/wiki/Brooklyn_Dreams_(group)" title="Brooklyn Dreams (group)">
+    Brooklyn Dreams
+   </a>
+  </del>
@@ -2387,3 +2718,5 @@
-  <a class="mw-redirect" href="/wiki/Best_selling_music_artists" title="Best selling music artists">
-   Best selling music artists
-  </a>
+  <del>
+   <a class="mw-redirect" href="/wiki/Best_selling_music_artists" title="Best selling music artists">
+    Best selling music artists
+   </a>
+  </del>
@@ -2391 +2724 @@
-</ul>
+</ul>
\ No newline at end of file
