--- 493040889
+++ 493047051
@@ -1,0 +2,6 @@
+ <del>
+  <b>
+   Donna Summer
+  </b>
+  (born
+ </del>
@@ -3 +9 @@
-  Donna Summer
+  LaDonna Adrian Gaines
@@ -5,2 +11,13 @@
- (born LaDonna Adrian Gaines on December 31, 1948)
- is an American singer/songwriter who gained prominence during the
+ <ins>
+  (December 31, 1948 – May 17, 2012)
+ , known by the stage name
+  <b>
+   Donna Summer
+  </b>
+  , was
+ </ins>
+ <del>
+  on December 31, 1948)
+ is
+ </del>
+ an American singer/songwriter who gained prominence during the
@@ -10 +27,8 @@
- era of the 1970s. She has a
+ era of the 1970s. She
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ a
@@ -14 +38,8 @@
- vocal range, and is a five-time
+ vocal range, and
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ a five-time
@@ -35,0 +67,5 @@
+ <ins>
+  Donna Summer died on Thursday, May 17,2012 from cancer.
+ </ins>
+</p>
+<p>
@@ -603,0 +640,5 @@
+<p>
+ <ins>
+  OH MY GOSH! HAHA
+ </ins>
+</p>
@@ -1324 +1365,8 @@
- Summer has often talked about her early successful years as a period of confusion and anxiety. By mid-1977, struggling with the media's titles of her as
+ Summer
+ <ins>
+  had
+ </ins>
+ <del>
+  has
+ </del>
+ often talked about her early successful years as a period of confusion and anxiety. By mid-1977, struggling with the media's titles of her as
@@ -1359 +1407 @@
- where she took time out from showbusiness to focus on painting, a hobby she began in 1985. In 1995, Summer's mother died. Today, Summer still lives in Nashville with her husband and is a grandmother of four.
+ where she took time out from showbusiness to focus on painting, a hobby she began in 1985. In 1995, Summer's mother died.
@@ -1361,0 +1410,15 @@
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  On May 17, 2012, Summer died after a long battle with anal cancer.
+ </ins>
+</p>
+<del>
+ Today, Summer still lives in Nashville with her husband and is a grandmother of four.
+</del>
+<h2>
@@ -2060,2 +2123,8 @@
- <span class="mw-headline" id="Donna_Summer_concert_tours">
-  Donna Summer concert tours
+ <span class="mw-headline" id="Concert_tours">
+  <ins>
+   Concert
+  </ins>
+  <del>
+   Donna Summer concert
+  </del>
+  tours
@@ -2360,3 +2429,5 @@
- <span class="mw-headline" id="See_also">
-  See also
- </span>
+ <del>
+  <span class="mw-headline" id="See_also">
+   See also
+  </span>
+ </del>
@@ -2367 +2437,0 @@
-   <img alt="Book icon" data-file-height="48" data-file-width="48" height="30" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/30px-Office-book.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/45px-Office-book.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Office-book.svg/60px-Office-book.svg.png 2x" width="30"/>
@@ -2372,3 +2442,5 @@
-     <a href="/wiki/Book:Donna_Summer" title="Book:Donna Summer">
-      Book: Donna Summer
-     </a>
+     <del>
+      <a href="/wiki/Book:Donna_Summer" title="Book:Donna Summer">
+       Book: Donna Summer
+      </a>
+     </del>
@@ -2382,3 +2454,5 @@
-  <a href="/wiki/Brooklyn_Dreams_(group)" title="Brooklyn Dreams (group)">
-   Brooklyn Dreams
-  </a>
+  <del>
+   <a href="/wiki/Brooklyn_Dreams_(group)" title="Brooklyn Dreams (group)">
+    Brooklyn Dreams
+   </a>
+  </del>
@@ -2387,3 +2461,5 @@
-  <a class="mw-redirect" href="/wiki/Best_selling_music_artists" title="Best selling music artists">
-   Best selling music artists
-  </a>
+  <del>
+   <a class="mw-redirect" href="/wiki/Best_selling_music_artists" title="Best selling music artists">
+    Best selling music artists
+   </a>
+  </del>
@@ -2391 +2467 @@
-</ul>
+</ul>
\ No newline at end of file
