--- 3955334
+++ 3957981
@@ -2 +1,0 @@
- <br/>
@@ -6 +5,6 @@
- (born
+ <ins>
+  (
+ </ins>
+ <del>
+  (born
+ </del>
@@ -13,0 +18,10 @@
+ <ins>
+  -
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+ </ins>
@@ -46,5 +60,15 @@
- before entering politics. He is the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days). He is the only
- <a href="/wiki/Divorce" title="Divorce">
-  divorced
- </a>
- person to be elected President, and the only President to have married a pregnant woman.
+ before entering politics. He
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days).
+ <del>
+  He is the only
+  <a href="/wiki/Divorce" title="Divorce">
+   divorced
+  </a>
+  person to be elected President, and the only President to have married a pregnant woman.
+ </del>
@@ -106 +130 @@
- <a class="mw-disambig" href="/wiki/Rock_River" title="Rock River">
+ <a class="mw-redirect" href="/wiki/Rock_River_(Illinois)" title="Rock River (Illinois)">
@@ -128 +152,8 @@
- . Child of an
+ .
+ <ins>
+  The child
+ </ins>
+ <del>
+  Child
+ </del>
+ of an
@@ -192 +223,5 @@
- he played the role of George "The Gipper" Gipp in the film
+ he played the role of
+ <a href="/wiki/George_Gipp" title="George Gipp">
+  George "The Gipper" Gipp
+ </a>
+ in the film
@@ -244 +279,5 @@
- officer in the U.S. Army in
+ officer in the
+ <a class="mw-redirect" href="/wiki/U.S._Army" title="U.S. Army">
+  U.S. Army
+ </a>
+ in
@@ -252 +291,5 @@
- he was activated and was assigned to the First Motion Picture Unit in the
+ he was activated and was assigned to the
+ <a href="/wiki/First_Motion_Picture_Unit" title="First Motion Picture Unit">
+  First Motion Picture Unit
+ </a>
+ in the
@@ -262,0 +306,13 @@
+ <ins>
+  In 1940, Reagan married actress
+  <a href="/wiki/Jane_Wyman" title="Jane Wyman">
+   Jane Wyman
+  </a>
+  and had a daughter with her. After divorcing in 1948, Reagan remarried in 1952 to actress
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy Davis
+  </a>
+  . Nancy was pregnant with a child at the time and Reagan is the only President to have married a pregnant woman.
+ </ins>
+</p>
+<p>
@@ -324,5 +380,9 @@
- (SAG), aligning himself with Senator
- <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
-  Joseph McCarthy
- </a>
- and cooperating with the
+ (SAG),
+ <del>
+  aligning himself with Senator
+  <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
+   Joseph McCarthy
+  </a>
+  and
+ </del>
+ cooperating with the
@@ -596 +656,7 @@
-   struck
+   <ins>
+    went on
+  strike
+   </ins>
+   <del>
+    struck
+   </del>
@@ -1034,0 +1101,279 @@
+<table align="right" border="1" cellpadding="2" cellspacing="0" style="margin-left:1em">
+ <ins>
+  <caption>
+   <font size="+1">
+    <b>
+     Ronald Reagan
+    </b>
+   </font>
+  </caption>
+ </ins>
+ <tr>
+  <td align="center" colspan="2" style="background:#efefef;">
+   <ins>
+    <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan.jpg" title="File:Reagan.jpg">
+     Ronald Reagan
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Order:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    40th President
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Term of Office:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/January_20" title="January 20">
+     January 20
+    </a>
+    ,
+    <a href="/wiki/1981" title="1981">
+     1981
+    </a>
+    -
+    <a href="/wiki/January_20" title="January 20">
+     January 20
+    </a>
+    ,
+    <a href="/wiki/1989" title="1989">
+     1989
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Predecessor:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+     Jimmy Carter
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Successor:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+     George H. W. Bush
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Date of Birth:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Monday" title="Monday">
+     Monday
+    </a>
+    ,
+    <a href="/wiki/February_6" title="February 6">
+     February 6
+    </a>
+    ,
+    <a href="/wiki/1911" title="1911">
+     1911
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Place of Birth:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Tampico,_Illinois" title="Tampico, Illinois">
+     Tampico, Illinois
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Date of Death:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Saturday" title="Saturday">
+     Saturday
+    </a>
+    ,
+    <a href="/wiki/June_5" title="June 5">
+     June 5
+    </a>
+    ,
+    <a href="/wiki/2004" title="2004">
+     2004
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Place of Death:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a class="mw-redirect" href="/wiki/Bel_Air,_California" title="Bel Air, California">
+     Bel Air, California
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/First_Lady_of_the_United_States" title="First Lady of the United States">
+     First Lady
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+     Nancy Reagan
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Profession:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Actor" title="Actor">
+     Actor
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/List_of_political_parties_in_the_United_States" title="List of political parties in the United States">
+     Political Party
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a class="mw-redirect" href="/wiki/United_States_Republican_Party" title="United States Republican Party">
+     Republican
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/Vice_President_of_the_United_States" title="Vice President of the United States">
+     Vice President
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+     George H. W. Bush
+    </a>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  <b>
+   Ronald Wilson Reagan
+  </b>
+  (
+  <a href="/wiki/February_6" title="February 6">
+   February 6
+  </a>
+  ,
+  <a href="/wiki/1911" title="1911">
+   1911
+  </a>
+  -
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+  ) was the 40th (
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  -
+  <a href="/wiki/1989" title="1989">
+   1989
+  </a>
+  )
+  <a href="/wiki/President_of_the_United_States" title="President of the United States">
+   President
+  </a>
+  of the
+  <a href="/wiki/United_States" title="United States">
+   United States
+  </a>
+  and the 33rd
+  <a href="/wiki/Governor_of_California" title="Governor of California">
+   Governor
+  </a>
+  of
+  <a href="/wiki/California" title="California">
+   California
+  </a>
+  . Reagan was also an
+  <a href="/wiki/Actor" title="Actor">
+   actor
+  </a>
+  in
+  <a href="/wiki/Film" title="Film">
+   films
+  </a>
+  before entering politics. He was the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days).
+ </ins>
+</p>
@@ -1036,3 +1381,5 @@
- <span class="mw-headline" id="Legacy_and_retirement_from_public_life">
-  Legacy and retirement from public life
- </span>
+ <ins>
+  <span class="mw-headline" id="Early_life_and_career_2">
+   Early life and career
+  </span>
+ </ins>
@@ -1041,33 +1388,35 @@
- Reagan was in many ways the founder of the modern Republican Party. His redefinition of fiscal conservatism as being focused on tax cuts without significant regard to a balanced budget ("
- <a href="/wiki/Reaganomics" title="Reaganomics">
-  Reaganomics
- </a>
- "); his opposition to
- <a class="mw-redirect" href="/wiki/Progressive_taxation" title="Progressive taxation">
-  progressive taxation
- </a>
- , greater environmental protection and regulation, and
- <a href="/wiki/Abortion" title="Abortion">
-  abortion
- </a>
- ; the importance of the
- <a href="/wiki/Moral_Majority" title="Moral Majority">
-  Moral Majority
- </a>
- and its supporters in his governing coalition; and even his support of
- <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
-  missile defense systems
- </a>
- have all become trademarks of subsequent Republican leaders, including
- <a href="/wiki/George_W._Bush" title="George W. Bush">
-  George W. Bush
- </a>
- . Reagan's immediate predecessors such as
- <a href="/wiki/Richard_Nixon" title="Richard Nixon">
-  Richard Nixon
- </a>
- and
- <a class="mw-redirect" href="/wiki/Dwight_Eisenhower" title="Dwight Eisenhower">
-  Dwight Eisenhower
- </a>
- would not have recognized any of these as part of the Republican platform.
+ <ins>
+  Reagan was born in
+  <a href="/wiki/Tampico,_Illinois" title="Tampico, Illinois">
+   Tampico
+  </a>
+  ,
+  <a href="/wiki/Illinois" title="Illinois">
+   Illinois
+  </a>
+  , the second of two sons to John (Jack) Reagan and Nelle Wilson. In
+  <a href="/wiki/1920" title="1920">
+   1920
+  </a>
+  , after years of moving from town to town, the family settled in the Illinois town of
+  <a href="/wiki/Dixon,_Illinois" title="Dixon, Illinois">
+   Dixon
+  </a>
+  . In
+  <a href="/wiki/1921" title="1921">
+   1921
+  </a>
+  , at the age of 11, Reagan was
+  <a href="/wiki/Baptism" title="Baptism">
+   baptized
+  </a>
+  in his mother's
+  <a class="mw-redirect" href="/wiki/Disciples_of_Christ" title="Disciples of Christ">
+   Disciples of Christ
+  </a>
+  church in Dixon, and in
+  <a href="/wiki/1924" title="1924">
+   1924
+  </a>
+  he began attending Dixon's Northside High School.
+ </ins>
@@ -1076,21 +1425,15 @@
- In
- <a href="/wiki/1992" title="1992">
-  1992
- </a>
- , four years after leaving office, Reagan was diagnosed with
- <a href="/wiki/Alzheimer%27s_disease" title="Alzheimer's disease">
-  Alzheimer's disease
- </a>
- . As the years went on, the disease began to slowly take over the former President's brain and body, forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
- <a href="/wiki/November_5" title="November 5">
-  November 5
- </a>
- ,
- <a href="/wiki/1994" title="1994">
-  1994
- </a>
- in the form of a personal letter. His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
- <a href="/wiki/2004" title="2004">
-  2004
- </a>
- Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was rumoured, although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
+ <ins>
+  In
+  <a href="/wiki/1926" title="1926">
+   1926
+  </a>
+  , at age 15, Reagan took a summer job as
+  <a href="/wiki/Lifeguard" title="Lifeguard">
+   lifeguard
+  </a>
+  in Lowell Park, two miles away from Dixon on the nearby
+  <a class="mw-redirect" href="/wiki/Rock_River_(Illinois)" title="Rock River (Illinois)">
+   Rock River
+  </a>
+  . He continued to work as a lifeguard on the Rock for the next seven years, reportedly saving 77 people from drowning.
+ </ins>
@@ -1099,22 +1442,70 @@
- On
- <a href="/wiki/February_6" title="February 6">
-  February 6
- </a>
- ,
- <a href="/wiki/1998" title="1998">
-  1998
- </a>
- , Washington National Airport in
- <a class="mw-redirect" href="/wiki/Washington,_DC" title="Washington, DC">
-  Washington, DC
- </a>
- was renamed
- <a href="/wiki/Ronald_Reagan_Washington_National_Airport" title="Ronald Reagan Washington National Airport">
-  Ronald Reagan Washington National Airport
- </a>
- . Also, the
- <a href="/wiki/Aircraft_carrier" title="Aircraft carrier">
-  aircraft carrier
- </a>
- <a class="mw-redirect" href="/wiki/USS_Ronald_Reagan_(CVN-76)" title="USS Ronald Reagan (CVN-76)">
-  USS
+ <ins>
+  In
+  <a href="/wiki/1928" title="1928">
+   1928
+  </a>
+  , Reagan entered
+  <a href="/wiki/Eureka_College" title="Eureka College">
+   Eureka College
+  </a>
+  in
+  <a href="/wiki/Eureka,_Illinois" title="Eureka, Illinois">
+   Eureka, Illinois
+  </a>
+  , graduating in
+  <a href="/wiki/1932" title="1932">
+   1932
+  </a>
+  . The child of an
+  <a class="mw-redirect" href="/wiki/Alcoholic" title="Alcoholic">
+   alcoholic
+  </a>
+  father, Reagan developed an early gift for
+  <a href="/wiki/Storytelling" title="Storytelling">
+   storytelling
+  </a>
+  and
+  <a href="/wiki/Acting" title="Acting">
+   acting
+  </a>
+  . He was a
+  <a href="/wiki/Radio" title="Radio">
+   radio
+  </a>
+  <a href="/wiki/Announcer" title="Announcer">
+   announcer
+  </a>
+  of
+  <a href="/wiki/Chicago_Cubs" title="Chicago Cubs">
+   Chicago Cubs
+  </a>
+  games, getting only the bare outlines of the game from a ticker and relying on his
+  <a href="/wiki/Imagination" title="Imagination">
+   imagination
+  </a>
+  and storytelling gifts to flesh out the game. Once in
+  <a href="/wiki/1934" title="1934">
+   1934
+  </a>
+  , during the ninth inning of a Cubs -
+  <a href="/wiki/St._Louis_Cardinals" title="St. Louis Cardinals">
+   St. Louis Cardinals
+  </a>
+  game, the wire went dead. Reagan smoothly
+  <a class="mw-redirect" href="/wiki/Improvise" title="Improvise">
+   improvised
+  </a>
+  a fictional play-by-play (in which hitters on both teams gained a superhuman ability to foul off pitches) until the wire was restored.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan had a successful career in
+  <a href="/wiki/Hollywood" title="Hollywood">
+   Hollywood
+  </a>
+  as a second-rank leading man, as his face and body were as handsome as his voice. His first screen credit was the starring role the
+  <a href="/wiki/1937" title="1937">
+   1937
+  </a>
+  movie
@@ -1122 +1513,3 @@
-   Ronald Reagan
+   <a class="new" href="/w/index.php?title=Love_is_On_the_Air&amp;action=edit&amp;redlink=1" title="Love is On the Air (page does not exist)">
+    Love is On the Air
+   </a>
@@ -1124,18 +1517,59 @@
-  (CVN-76)
- </a>
- was christened on
- <a href="/wiki/March_4" title="March 4">
-  March 4
- </a>
- ,
- <a href="/wiki/2001" title="2001">
-  2001
- </a>
- , making it one of the
- <a class="mw-redirect" href="/wiki/List_of_military_vessels_named_after_living_Americans" title="List of military vessels named after living Americans">
-  very few
- </a>
- <a href="/wiki/United_States_Navy" title="United States Navy">
-  United States Navy
- </a>
- ships to be named after a living person.
+  . By the end of
+  <a href="/wiki/1939" title="1939">
+   1939
+  </a>
+  , he had appeared in 19 films. In
+  <a href="/wiki/1940" title="1940">
+   1940
+  </a>
+  he played the role of
+  <a href="/wiki/George_Gipp" title="George Gipp">
+   George "The Gipper" Gipp
+  </a>
+  in the film
+  <i>
+   <a class="mw-redirect" href="/wiki/Knute_Rockne_All_American" title="Knute Rockne All American">
+    Knute Rockne All American
+   </a>
+  </i>
+  , from which he acquired the nickname
+  <b>
+   the Gipper
+  </b>
+  , which he retained the rest of his life. Reagan himself considered that his best acting work was in
+  <i>
+   <a href="/wiki/Kings_Row" title="Kings Row">
+    Kings Row
+   </a>
+  </i>
+  (
+  <a href="/wiki/1942" title="1942">
+   1942
+  </a>
+  ). Other notable Reagan films include
+  <i>
+   <a href="/wiki/Hellcats_of_the_Navy" title="Hellcats of the Navy">
+    Hellcats of the Navy
+   </a>
+  </i>
+  ,
+  <i>
+   <a href="/wiki/This_Is_the_Army" title="This Is the Army">
+    This Is the Army
+   </a>
+  </i>
+  , and the
+  <a class="mw-disambig" href="/wiki/Camp" title="Camp">
+   campy
+  </a>
+  <i>
+   <a href="/wiki/Bedtime_for_Bonzo" title="Bedtime for Bonzo">
+    Bedtime for Bonzo
+   </a>
+  </i>
+  . He has a star on the
+  <a href="/wiki/Hollywood_Walk_of_Fame" title="Hollywood Walk of Fame">
+   Hollywood Walk of Fame
+  </a>
+  at 6374 Hollywood Blvd.
+ </ins>
@@ -1144,8 +1578,28 @@
- In November
- <a href="/wiki/2003" title="2003">
-  2003
- </a>
- , Reagan and his family were the subject of a controversial television miniseries,
- <i>
-  <a href="/wiki/The_Reagans" title="The Reagans">
-   The Reagans
+ <ins>
+  Reagan was commissioned as a reserve
+  <a href="/wiki/Cavalry" title="Cavalry">
+   cavalry
+  </a>
+  officer in the
+  <a class="mw-redirect" href="/wiki/U.S._Army" title="U.S. Army">
+   U.S. Army
+  </a>
+  in
+  <a href="/wiki/1935" title="1935">
+   1935
+  </a>
+  . After
+  <a href="/wiki/Pearl_Harbor" title="Pearl Harbor">
+   Pearl Harbor
+  </a>
+  he was activated and was assigned to the
+  <a href="/wiki/First_Motion_Picture_Unit" title="First Motion Picture Unit">
+   First Motion Picture Unit
+  </a>
+  in the
+  <a class="mw-disambig" href="/wiki/Army_Air_Corps" title="Army Air Corps">
+   Army Air Corps
+  </a>
+  , which made training and education films. He remained in
+  <a href="/wiki/Hollywood" title="Hollywood">
+   Hollywood
@@ -1153,10 +1607,2 @@
- </i>
- . In response to the unflattering docu-drama, a number of Congressional Republicans introduced the "Ronald Reagan Dime Act" (HR 3633), a bill that would replace
- <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
-  Franklin Delano Roosevelt
- </a>
- 's portrait on the
- <a class="mw-redirect" href="/wiki/Dime_(U.S._coin)" title="Dime (U.S. coin)">
-  United States dime
- </a>
- with Reagan's. The bill did not have widespread support and appeared unlikely to be put up for a vote.
+  for the duration of the war.
+ </ins>
@@ -1165,21 +1611,11 @@
- In
- <a href="/wiki/2004" title="2004">
-  2004
- </a>
- , Reagan turned 93, continuing his record as the longest-living president in American history. The previous record had been 90 years and 247 days, held by
- <a href="/wiki/John_Adams" title="John Adams">
-  John Adams
- </a>
- , whom Reagan surpassed on
- <a href="/wiki/October_12" title="October 12">
-  October 12
- </a>
- ,
- <a href="/wiki/2001" title="2001">
-  2001
- </a>
- . In June of 2004 the
- <a href="/wiki/White_House" title="White House">
-  White House
- </a>
- was informed by family doctors that Reagan's health had "significantly deteriorated" and that he was not expected to live more than a year.
+ <ins>
+  In 1940, Reagan married actress
+  <a href="/wiki/Jane_Wyman" title="Jane Wyman">
+   Jane Wyman
+  </a>
+  and had a daughter with her. After divorcing in 1948, Reagan remarried in 1952 to actress
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy Davis
+  </a>
+  . Nancy was pregnant with a child at the time and Reagan is the only President to have married a pregnant woman.
+ </ins>
@@ -1187,5 +1622,0 @@
-<div class="floatright">
- <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
-  Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
- </a>
-</div>
@@ -1193,5 +1624,23 @@
- On
- <a href="/wiki/June_5" title="June 5">
-  June 5
- </a>
- , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+ <ins>
+  As Reagan's film roles became fewer in the late
+  <a href="/wiki/1950s" title="1950s">
+   1950s
+  </a>
+  , he moved into
+  <a href="/wiki/Television" title="Television">
+   television
+  </a>
+  as a host and frequent performer for
+  <i>
+   <a href="/wiki/General_Electric_Theater" title="General Electric Theater">
+    General Electric Theater
+   </a>
+  </i>
+  . His final regular acting job was as host and performer on
+  <i>
+   <a href="/wiki/Death_Valley_Days" title="Death Valley Days">
+    Death Valley Days
+   </a>
+  </i>
+  .
+ </ins>
@@ -1200,6 +1649,5 @@
- <span class="mw-headline" id="Supreme_Court_appointments">
-  <a href="/wiki/Supreme_Court_of_the_United_States" title="Supreme Court of the United States">
-   Supreme Court
-  </a>
-  appointments
- </span>
+ <ins>
+  <span class="mw-headline" id="Early_political_career_2">
+   Early political career
+  </span>
+ </ins>
@@ -1207,4 +1655,2454 @@
-<ul>
- <li>
-  <a href="/wiki/Sandra_Day_O%27Connor" title="Sandra Day O'Connor">
-   Sandra Day O'Connor
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Ronald Reagan began his political life as a
+  <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
+   liberal
+  </a>
+  <a class="mw-redirect" href="/wiki/United_States_Democratic_Party" title="United States Democratic Party">
+   Democrat
+  </a>
+  , supporting
+  <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
+   Franklin Delano Roosevelt
+  </a>
+  and his
+  <a href="/wiki/New_Deal" title="New Deal">
+   New Deal
+  </a>
+  . He gradually became a staunch social and fiscal
+  <a class="mw-redirect" href="/wiki/Conservative" title="Conservative">
+   conservative
+  </a>
+  . He embarked upon the path that led him to a career in politics during his tenure as president of the
+  <a href="/wiki/Screen_Actors_Guild" title="Screen Actors Guild">
+   Screen Actors Guild
+  </a>
+  (SAG), aligning himself with Senator
+  <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
+   Joseph McCarthy
+  </a>
+  and cooperating with the
+  <a href="/wiki/House_Un-American_Activities_Committee" title="House Un-American Activities Committee">
+   House Un-American Activities Committee
+  </a>
+  to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
+  <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
+   American Communist Party
+  </a>
+  . His employment by the
+  <a href="/wiki/General_Electric" title="General Electric">
+   General Electric
+  </a>
+  company, delivering
+  <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+   anti-communist
+  </a>
+  speeches on radio broadcasts and speaking tours, further enhanced his political image in the anti-Soviet climate of 1950s America. By the
+  <a class="mw-redirect" href="/wiki/U.S._presidential_election,_1964" title="U.S. presidential election, 1964">
+   1964 election
+  </a>
+  , Reagan was an outspoken supporter of conservative Republican
+  <a href="/wiki/Barry_Goldwater" title="Barry Goldwater">
+   Barry Goldwater
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1966" title="1966">
+   1966
+  </a>
+  , he was elected the
+  <a href="/wiki/Governor_of_California" title="Governor of California">
+   33rd Governor of California
+  </a>
+  . Reagan tried to gain the Republican presidential nomination in
+  <a href="/wiki/1968" title="1968">
+   1968
+  </a>
+  , and again in
+  <a href="/wiki/1976" title="1976">
+   1976
+  </a>
+  over the incumbent
+  <a href="/wiki/Gerald_Ford" title="Gerald Ford">
+   Gerald Ford
+  </a>
+  but was defeated at the Republican Convention. He succeeded in gaining the Republican nomination in 1980. The campaign was marked by the
+  <a href="/wiki/Iran_hostage_crisis" title="Iran hostage crisis">
+   Iran hostage crisis
+  </a>
+  . Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election. Most analysts believe President
+  <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+   Jimmy Carter
+  </a>
+  's inability to solve the hostage crisis played a large role in his defeat and Reagan's victory. Reagan went on to be elected President that year. In
+  <a href="/wiki/1984" title="1984">
+   1984
+  </a>
+  , he was re-elected in a landslide, winning in 49 of 50 states and receiving nearly 60 percent of the popular vote.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Presidency_2">
+   Presidency
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Then-Vice President Bush meets with President Reagan in 1984.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  On
+  <a href="/wiki/March_30" title="March 30">
+   March 30
+  </a>
+  ,
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  , just 69 days into his Presidency, while leaving the
+  <a class="mw-redirect" href="/wiki/Hilton_Hotel" title="Hilton Hotel">
+   Hilton Hotel
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/Washington,_DC" title="Washington, DC">
+   Washington, DC
+  </a>
+  , President Reagan,
+  <a class="mw-redirect" href="/wiki/Press_Secretary" title="Press Secretary">
+   Press Secretary
+  </a>
+  <a href="/wiki/James_Brady" title="James Brady">
+   James Brady
+  </a>
+  , a
+  <a class="mw-disambig" href="/wiki/Secret_Service" title="Secret Service">
+   Secret Service
+  </a>
+  agent, and a
+  <a class="mw-redirect" href="/wiki/District_of_Columbia" title="District of Columbia">
+   District of Columbia
+  </a>
+  police officer were shot by
+  <a class="mw-redirect" href="/wiki/John_Hinckley,_Jr." title="John Hinckley, Jr.">
+   John Hinckley, Jr.
+  </a>
+  Shortly before
+  <a href="/wiki/Surgery" title="Surgery">
+   surgery
+  </a>
+  to remove the bullet from his chest (which barely missed his heart) he remarked to his surgeons, "I hope you're all Republicans,"
+ and to his wife Nancy he jokingly commented, "Honey, I forgot to duck."
+ </ins>
+</p>
+<p>
+ <ins>
+  As a politician and as President, he portrayed himself as being:
+ </ins>
+</p>
+<ul>
+ <li>
+  <ins>
+   <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+    anti-communist
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of
+   <a href="/wiki/Tax" title="Tax">
+    tax
+   </a>
+   cuts
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of smaller government (with the exclusion of the
+   <a href="/wiki/Military" title="Military">
+    military
+   </a>
+   )
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of removing regulations on
+   <a href="/wiki/Corporation" title="Corporation">
+    corporations
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   supportive of business interests, both large and small
+  </ins>
+ </li>
+ <li>
+  <ins>
+   supportive of some individual liberties
+  </ins>
+ </li>
+ <li>
+  <ins>
+   tough on
+   <a href="/wiki/Crime" title="Crime">
+    crime
+   </a>
+  </ins>
+ </li>
+</ul>
+<p>
+ <ins>
+  He is credited with:
+ </ins>
+</p>
+<ul>
+ <li>
+  <ins>
+   increasing military spending
+  </ins>
+ </li>
+ <li>
+  <ins>
+   deploying U.S.
+   <a href="/wiki/Pershing_II" title="Pershing II">
+    Pershing II
+   </a>
+   missiles in Germany in response to the Soviet stationing of
+   <a class="mw-redirect" href="/wiki/SS-20" title="SS-20">
+    SS-20
+   </a>
+   missiles near Europe
+  </ins>
+ </li>
+ <li>
+  <ins>
+   pushing for the deployment of the
+   <a class="mw-redirect" href="/wiki/Peacekeeper_missile" title="Peacekeeper missile">
+    Peacekeeper missile
+   </a>
+   system
+  </ins>
+ </li>
+ <li>
+  <ins>
+   negotiating nuclear arms reduction treaties
+  </ins>
+ </li>
+ <li>
+  <ins>
+   proposing the
+   <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
+    Strategic Defense Initiative
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   arming and training anti-communist groups like the
+   <a href="/wiki/Contras" title="Contras">
+    Contras
+   </a>
+   and the
+   <a href="/wiki/Mujahideen" title="Mujahideen">
+    mujahideen
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   selling arms to foreign allies such
+   <a href="/wiki/Taiwan" title="Taiwan">
+    Taiwan
+   </a>
+   ,
+   <a href="/wiki/Israel" title="Israel">
+    Israel
+   </a>
+   ,
+   <a href="/wiki/Saudi_Arabia" title="Saudi Arabia">
+    Saudi Arabia
+   </a>
+   and, ironically,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   lowering taxes significantly, reversing a historic trend towards higher taxes
+  </ins>
+ </li>
+ <li>
+  <ins>
+   increasing the federal deficit
+  </ins>
+ </li>
+ <li>
+  <ins>
+   greatly escalating the "
+   <a class="mw-redirect" href="/wiki/War_on_drugs" title="War on drugs">
+    war on drugs
+   </a>
+   "
+  </ins>
+ </li>
+ <li>
+  <ins>
+   ending the high
+   <a href="/wiki/Inflation" title="Inflation">
+    inflation
+   </a>
+   that damaged the economy under his predecessors,
+   <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+    Jimmy Carter
+   </a>
+   and
+   <a href="/wiki/Gerald_Ford" title="Gerald Ford">
+    Gerald Ford
+   </a>
+   .
+  </ins>
+ </li>
+ <li>
+  <ins>
+   helping to 'win' the
+   <a href="/wiki/Cold_War" title="Cold War">
+    Cold War
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   firing
+   <a href="/wiki/Air_traffic_controller" title="Air traffic controller">
+    air traffic controllers
+   </a>
+   when they illegally went on
+   <a href="/wiki/Strike_action" title="Strike action">
+    strike
+   </a>
+  </ins>
+ </li>
+</ul>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Reaganomics_2">
+   Reaganomics
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+    File:Reaganhiro.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Emperor
+    <a href="/wiki/Hirohito" title="Hirohito">
+     Hirohito
+    </a>
+    meeting with President Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting stagflation, or both
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  and
+  <a href="/wiki/Unemployment" title="Unemployment">
+   unemployment
+  </a>
+  . Reagan's policies, partially based on
+  <a href="/wiki/Supply-side_economics" title="Supply-side economics">
+   supply-side economics
+  </a>
+  sought to stimulate the economy with large across-the-board
+  <a href="/wiki/Tax_cut" title="Tax cut">
+   tax cuts
+  </a>
+  . Most economists agree that tax cuts stimulate the economy, but supply-siders say that they have a much greater effect than most economists say they would.
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  had called Reagan's economic ideas "
+  <a class="mw-redirect" href="/wiki/Voodoo_economics" title="Voodoo economics">
+   voodoo economics
+  </a>
+  " prior to becoming his vice-president. Reagan's policies soon became known as "
+  <a href="/wiki/Reaganomics" title="Reaganomics">
+   Reaganomics
+  </a>
+  ", a nickname used by both his supporters and detractors. These tax cuts combined with heavy military spending increases led to enormous
+  <a href="/wiki/Deficit_spending" title="Deficit spending">
+   deficit spending
+  </a>
+  and a dramatic increase in the
+  <a class="mw-redirect" href="/wiki/National_debt" title="National debt">
+   national debt
+  </a>
+  . The debt increased by approximately 200% (tripled) between when Reagan took office and when his successor,
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  , took office. Reagan's supporters note that although the President writes an annual budget proposal, the actual spending bill originates in the
+  <a href="/wiki/United_States_House_of_Representatives" title="United States House of Representatives">
+   House of Representatives
+  </a>
+  , so Reagan's policies are not the exclusive cause of the debt increase.
+ </ins>
+</p>
+<p>
+ <ins>
+  On the other hand, this spending was slightly offset by increased tax revenues, and some supporters of Reagan attribute this to the successful use of supply-side economics tax policies. Critics of President Reagan argue that despite his frequent pronouncements that he advocated smaller and less intrusive government, federal spending and bureaucracy increased in size during his administration. Not surprisingly, there is disagreement over how much Reagan's policies contributed to the severe
+  <a href="/wiki/Recession" title="Recession">
+   recession
+  </a>
+  that took place in
+  <a href="/wiki/1982" title="1982">
+   1982
+  </a>
+  , the strong economic expansion that began late in his first term and ran throughout his second term, and the fall in the average inflation-adjusted hourly wage for American workers that happened between his 1981 swearing-in and his successor's 1989 swearing-in.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="The_Soviet_Union_and_the_Cold_War_2">
+   The Soviet Union and the Cold War
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Like many successful American politicians, Reagan had great stage presence, as well as great instincts for cultivating positive responses from the public. His calm speaking voice and forceful language earned him the nickname "the Great Communicator." On
+  <a href="/wiki/March_8" title="March 8">
+   March 8
+  </a>
+  ,
+  <a href="/wiki/1983" title="1983">
+   1983
+  </a>
+  he called the
+  <a href="/wiki/Soviet_Union" title="Soviet Union">
+   Soviet Union
+  </a>
+  an "Evil Empire" and later in his presidency while speaking in front of the
+  <a href="/wiki/Berlin_Wall" title="Berlin Wall">
+   Berlin Wall
+  </a>
+  he challenged Soviet leader
+  <a href="/wiki/Mikhail_Gorbachev" title="Mikhail Gorbachev">
+   Mikhail Gorbachev
+  </a>
+  to "tear down this wall". Some historians believe that all of those traits were greatly supported by his enthusiasm for America and strong personal belief in rights of the individual.
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Many Reagan supporters credit him with winning the
+  <a href="/wiki/Cold_War" title="Cold War">
+   Cold War
+  </a>
+  , while others attribute the collapse of
+  <a href="/wiki/Communism" title="Communism">
+   communism
+  </a>
+  in
+  <a href="/wiki/1989" title="1989">
+   1989
+  </a>
+  in Eastern Europe and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
+  <a href="/wiki/Mikhail_Gorbachev" title="Mikhail Gorbachev">
+   Mikhail Gorbachev
+  </a>
+  . Reagan's policies included strong support of the U.S. military and the doctrine of "peace through strength." One of his more controversial proposals was the
+  <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
+   Strategic Defense Initiative
+  </a>
+  (SDI), a missile defense system which he hoped would make the U.S. invulnerable to nuclear missile attack by the
+  <a href="/wiki/Soviet_Union" title="Soviet Union">
+   Soviet Union
+  </a>
+  . SDI was dubbed "Star Wars" by opponents, connoting the fantastic or impractical.
+ </ins>
+</p>
+<p>
+ <ins>
+  Critics of SDI argued that the technological objective was unattainable in practical terms, and that the attempt would be likely to increase the
+  <a class="mw-redirect" href="/wiki/Arms_Race" title="Arms Race">
+   Arms Race
+  </a>
+  , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application the
+  <a class="mw-redirect" href="/wiki/Strategy_of_technology" title="Strategy of technology">
+   strategy of technology
+  </a>
+  . They saw SDI as an attempt to convince the Soviets that their nuclear missile arsenal was obsolescent, burdening the Soviets with additional spending on new technology to maintain a credible nuclear deterrent.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan had a close friendship with many other conservative political leaders across the globe, especially
+  <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+   Margaret Thatcher
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/UK" title="UK">
+   Britain
+  </a>
+  , and
+  <a href="/wiki/Brian_Mulroney" title="Brian Mulroney">
+   Brian Mulroney
+  </a>
+  in
+  <a href="/wiki/Canada" title="Canada">
+   Canada
+  </a>
+  . Reagan had a great desire for establishing personal relationships with other heads of state, often inviting them to his ranch or
+  <a href="/wiki/Camp_David" title="Camp David">
+   Camp David
+  </a>
+  for casual retreats.
+ </ins>
+</p>
+<p>
+ <ins>
+  As part of the policies that became known as the
+  <a href="/wiki/Reagan_Doctrine" title="Reagan Doctrine">
+   Reagan Doctrine
+  </a>
+  , the United States also took an increasingly hard line against Communist influences in Latin America, which often involved the controversial support of anti-Communist
+  <a href="/wiki/Military_dictatorship" title="Military dictatorship">
+   military dictatorships
+  </a>
+  with poor
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  records. This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
+  <a href="/wiki/1983" title="1983">
+   1983
+  </a>
+  Reagan ordered a formal military invasion of the small island nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  after it underwent a Communist coup. Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
+  <a href="/wiki/United_States_Agency_for_International_Development" title="United States Agency for International Development">
+   foreign aid
+  </a>
+  packages to states who held free elections.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Iran-Contra_Affair_2">
+   Iran-Contra Affair
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  During his administration, there was a major scandal and investigation of his administration's covert support of wars in
+  <a href="/wiki/Iran" title="Iran">
+   Iran
+  </a>
+  and
+  <a href="/wiki/Nicaragua" title="Nicaragua">
+   Nicaragua
+  </a>
+  in what came to be known as the
+  <a class="mw-redirect" href="/wiki/Iran-Contra_Affair" title="Iran-Contra Affair">
+   Iran-Contra Affair
+  </a>
+  . Two members of administration,
+  <a class="mw-redirect" href="/wiki/National_Security_Advisor" title="National Security Advisor">
+   National Security Advisor
+  </a>
+  <a href="/wiki/John_Poindexter" title="John Poindexter">
+   John Poindexter
+  </a>
+  and Col.
+  <a href="/wiki/Oliver_North" title="Oliver North">
+   Oliver North
+  </a>
+  had hatched an elaborate plot to sell arms to the Iranian government and give the profits to the anti-Communist
+  <a class="mw-disambig" href="/wiki/Contra" title="Contra">
+   Contras
+  </a>
+  guerillas in Nicaragua, who were engaged in a bloody civil war. Both actions were contrary to acts of
+  <a href="/wiki/United_States_Congress" title="United States Congress">
+   Congress
+  </a>
+  . Reagan professed ignorance of the plot, but admitted that he had supported the initial sale of arms to Iran, on the grounds that such sales were supposed to help secure the release of Americans being held hostage by the Iranian-backed
+  <a href="/wiki/Hezbollah" title="Hezbollah">
+   Hezbollah
+  </a>
+  terrorist group in
+  <a href="/wiki/Lebanon" title="Lebanon">
+   Lebanon
+  </a>
+  .
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Reagan quickly called for the appointment of an
+  <a class="mw-redirect" href="/wiki/Office_of_the_Independent_Counsel" title="Office of the Independent Counsel">
+   Independent Counsel
+  </a>
+  to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led Congressman Patricia Schroeder to dub him the "
+  <a class="mw-redirect" href="/wiki/Teflon" title="Teflon">
+   Teflon
+  </a>
+  President", a term that has been occasionally attached to later Presidents and their scandals.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id=".22War_on_Drugs.22_2">
+   "War on Drugs"
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Reagan's policies in the "
+  <a href="/wiki/War_on_Drugs" title="War on Drugs">
+   War on Drugs
+  </a>
+  " emphasized
+  <a href="/wiki/Imprisonment" title="Imprisonment">
+   imprisonment
+  </a>
+  for drug offenders while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's prison population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless, surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Miscellaneous_2">
+   Miscellaneous
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  On
+  <a href="/wiki/August_5" title="August 5">
+   August 5
+  </a>
+  ,
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  , Reagan fired 11,359
+  <a href="/wiki/Strike_action" title="Strike action">
+   striking
+  </a>
+  <a href="/wiki/Air_traffic_controller" title="Air traffic controller">
+   air traffic controllers
+  </a>
+  who had ignored his order to return to work. Ironically,
+  <a class="mw-disambig" href="/wiki/Professional_Air_Traffic_Controllers_Organization" title="Professional Air Traffic Controllers Organization">
+   PATCO
+  </a>
+  , the air traffic controller's union, had been one of the few unions that had supported Reagan over Carter in the election nine months previously.
+ </ins>
+</p>
+<p>
+ <ins>
+  In the spring of 1983, Reagan sent
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   US Marines
+  </a>
+  into
+  <a href="/wiki/Lebanon" title="Lebanon">
+   Lebanon
+  </a>
+  . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines. Two days later Reagan invaded the tiny
+  <a href="/wiki/Caribbean" title="Caribbean">
+   Caribbean
+  </a>
+  nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  . Three months later, Reagan withdrew the Marines from Lebanon.
+ </ins>
+</p>
+<p>
+ <ins>
+  On
+  <a href="/wiki/July_13" title="July 13">
+   July 13
+  </a>
+  ,
+  <a href="/wiki/1985" title="1985">
+   1985
+  </a>
+  , Reagan underwent surgery to remove cancerous
+  <a href="/wiki/Polyp_(medicine)" title="Polyp (medicine)">
+   polyps
+  </a>
+  from his
+  <a class="mw-redirect" href="/wiki/Colon_(anatomy)" title="Colon (anatomy)">
+   colon
+  </a>
+  , causing the first-ever invocation of the
+  <a href="/wiki/Acting_President_of_the_United_States" title="Acting President of the United States">
+   Acting President
+  </a>
+  clause of the
+  <a href="/wiki/Twenty-fifth_Amendment_to_the_United_States_Constitution" title="Twenty-fifth Amendment to the United States Constitution">
+   25th Amendment
+  </a>
+  . On
+  <a href="/wiki/January_5" title="January 5">
+   January 5
+  </a>
+  ,
+  <a href="/wiki/1987" title="1987">
+   1987
+  </a>
+  , Reagan underwent
+  <a href="/wiki/Prostate" title="Prostate">
+   prostate
+  </a>
+  surgery which caused further worries about his health.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan was widely criticized in 1985 for an incident related to an official visit to
+  <a href="/wiki/West_Germany" title="West Germany">
+   West Germany
+  </a>
+  . On
+  <a href="/wiki/April_11" title="April 11">
+   April 11
+  </a>
+  , the White House announced that Reagan would be visiting the
+  <a class="mw-redirect" href="/wiki/Bitburg_military_cemetery" title="Bitburg military cemetery">
+   Bitburg military cemetery
+  </a>
+  together with Chancellor
+  <a href="/wiki/Helmut_Kohl" title="Helmut Kohl">
+   Helmut Kohl
+  </a>
+  , to lay a wreath in honor of German soldiers who died in both World Wars. This became controversial when it came to public attention that a small number (variously reported as 49 or 56) of gravesites contained remains of soldiers who had served in
+  <a href="/wiki/Waffen-SS" title="Waffen-SS">
+   Waffen-SS
+  </a>
+  units. Despite protests from various quarters, most notably
+  <a href="/wiki/Elie_Wiesel" title="Elie Wiesel">
+   Elie Wiesel
+  </a>
+  , Reagan carried out the visit on
+  <a href="/wiki/May_5" title="May 5">
+   May 5
+  </a>
+  on the grounds that it would promote reconciliation between the former adversaries.
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1991" title="1991">
+   1991
+  </a>
+  , former
+  <a class="mw-disambig" href="/wiki/Starlet" title="Starlet">
+   starlet
+  </a>
+  <a class="new" href="/w/index.php?title=Selene_Walters&amp;action=edit&amp;redlink=1" title="Selene Walters (page does not exist)">
+   Selene Walters
+  </a>
+  claimed that Reagan had raped her in the early 1950s.
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres37-41.jpg" title="File:Pres37-41.jpg">
+    File:Pres37-41.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential Library.
+   </ins>
+  </div>
+ </div>
+</div>
+<table align="right" border="1" cellpadding="2" cellspacing="0" style="margin-left:1em">
+ <ins>
+  <caption>
+   <font size="+1">
+    <b>
+     Ronald Reagan
+    </b>
+   </font>
+  </caption>
+ </ins>
+ <tr>
+  <td align="center" colspan="2" style="background:#efefef;">
+   <ins>
+    <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan.jpg" title="File:Reagan.jpg">
+     Ronald Reagan
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Order:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    40th President
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Term of Office:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/January_20" title="January 20">
+     January 20
+    </a>
+    ,
+    <a href="/wiki/1981" title="1981">
+     1981
+    </a>
+    -
+    <a href="/wiki/January_20" title="January 20">
+     January 20
+    </a>
+    ,
+    <a href="/wiki/1989" title="1989">
+     1989
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Predecessor:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+     Jimmy Carter
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Successor:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+     George H. W. Bush
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Date of Birth:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Monday" title="Monday">
+     Monday
+    </a>
+    ,
+    <a href="/wiki/February_6" title="February 6">
+     February 6
+    </a>
+    ,
+    <a href="/wiki/1911" title="1911">
+     1911
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Place of Birth:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Tampico,_Illinois" title="Tampico, Illinois">
+     Tampico, Illinois
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Date of Death:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Saturday" title="Saturday">
+     Saturday
+    </a>
+    ,
+    <a href="/wiki/June_5" title="June 5">
+     June 5
+    </a>
+    ,
+    <a href="/wiki/2004" title="2004">
+     2004
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Place of Death:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a class="mw-redirect" href="/wiki/Bel_Air,_California" title="Bel Air, California">
+     Bel Air, California
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/First_Lady_of_the_United_States" title="First Lady of the United States">
+     First Lady
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+     Nancy Reagan
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    Profession:
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/Actor" title="Actor">
+     Actor
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/List_of_political_parties_in_the_United_States" title="List of political parties in the United States">
+     Political Party
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a class="mw-redirect" href="/wiki/United_States_Republican_Party" title="United States Republican Party">
+     Republican
+    </a>
+   </ins>
+  </td>
+ </tr>
+ <tr>
+  <th align="left">
+   <ins>
+    <a href="/wiki/Vice_President_of_the_United_States" title="Vice President of the United States">
+     Vice President
+    </a>
+    :
+   </ins>
+  </th>
+  <td>
+   <ins>
+    <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+     George H. W. Bush
+    </a>
+   </ins>
+  </td>
+ </tr>
+</table>
+<p>
+ <ins>
+  <b>
+   Ronald Wilson Reagan
+  </b>
+  (
+  <a href="/wiki/February_6" title="February 6">
+   February 6
+  </a>
+  ,
+  <a href="/wiki/1911" title="1911">
+   1911
+  </a>
+  -
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+  ) was the 40th (
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  -
+  <a href="/wiki/1989" title="1989">
+   1989
+  </a>
+  )
+  <a href="/wiki/President_of_the_United_States" title="President of the United States">
+   President
+  </a>
+  of the
+  <a href="/wiki/United_States" title="United States">
+   United States
+  </a>
+  and the 33rd
+  <a href="/wiki/Governor_of_California" title="Governor of California">
+   Governor
+  </a>
+  of
+  <a href="/wiki/California" title="California">
+   California
+  </a>
+  . Reagan was also an
+  <a href="/wiki/Actor" title="Actor">
+   actor
+  </a>
+  in
+  <a href="/wiki/Film" title="Film">
+   films
+  </a>
+  before entering politics. He was the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days).
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Early_life_and_career_3">
+   Early life and career
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  Reagan was born in
+  <a href="/wiki/Tampico,_Illinois" title="Tampico, Illinois">
+   Tampico
+  </a>
+  ,
+  <a href="/wiki/Illinois" title="Illinois">
+   Illinois
+  </a>
+  , the second of two sons to John (Jack) Reagan and Nelle Wilson. In
+  <a href="/wiki/1920" title="1920">
+   1920
+  </a>
+  , after years of moving from town to town, the family settled in the Illinois town of
+  <a href="/wiki/Dixon,_Illinois" title="Dixon, Illinois">
+   Dixon
+  </a>
+  . In
+  <a href="/wiki/1921" title="1921">
+   1921
+  </a>
+  , at the age of 11, Reagan was
+  <a href="/wiki/Baptism" title="Baptism">
+   baptized
+  </a>
+  in his mother's
+  <a class="mw-redirect" href="/wiki/Disciples_of_Christ" title="Disciples of Christ">
+   Disciples of Christ
+  </a>
+  church in Dixon, and in
+  <a href="/wiki/1924" title="1924">
+   1924
+  </a>
+  he began attending Dixon's Northside High School.
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1926" title="1926">
+   1926
+  </a>
+  , at age 15, Reagan took a summer job as
+  <a href="/wiki/Lifeguard" title="Lifeguard">
+   lifeguard
+  </a>
+  in Lowell Park, two miles away from Dixon on the nearby
+  <a class="mw-redirect" href="/wiki/Rock_River_(Illinois)" title="Rock River (Illinois)">
+   Rock River
+  </a>
+  . He continued to work as a lifeguard on the Rock for the next seven years, reportedly saving 77 people from drowning.
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1928" title="1928">
+   1928
+  </a>
+  , Reagan entered
+  <a href="/wiki/Eureka_College" title="Eureka College">
+   Eureka College
+  </a>
+  in
+  <a href="/wiki/Eureka,_Illinois" title="Eureka, Illinois">
+   Eureka, Illinois
+  </a>
+  , graduating in
+  <a href="/wiki/1932" title="1932">
+   1932
+  </a>
+  . The child of an
+  <a class="mw-redirect" href="/wiki/Alcoholic" title="Alcoholic">
+   alcoholic
+  </a>
+  father, Reagan developed an early gift for
+  <a href="/wiki/Storytelling" title="Storytelling">
+   storytelling
+  </a>
+  and
+  <a href="/wiki/Acting" title="Acting">
+   acting
+  </a>
+  . He was a
+  <a href="/wiki/Radio" title="Radio">
+   radio
+  </a>
+  <a href="/wiki/Announcer" title="Announcer">
+   announcer
+  </a>
+  of
+  <a href="/wiki/Chicago_Cubs" title="Chicago Cubs">
+   Chicago Cubs
+  </a>
+  games, getting only the bare outlines of the game from a ticker and relying on his
+  <a href="/wiki/Imagination" title="Imagination">
+   imagination
+  </a>
+  and storytelling gifts to flesh out the game. Once in
+  <a href="/wiki/1934" title="1934">
+   1934
+  </a>
+  , during the ninth inning of a Cubs -
+  <a href="/wiki/St._Louis_Cardinals" title="St. Louis Cardinals">
+   St. Louis Cardinals
+  </a>
+  game, the wire went dead. Reagan smoothly
+  <a class="mw-redirect" href="/wiki/Improvise" title="Improvise">
+   improvised
+  </a>
+  a fictional play-by-play (in which hitters on both teams gained a superhuman ability to foul off pitches) until the wire was restored.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan had a successful career in
+  <a href="/wiki/Hollywood" title="Hollywood">
+   Hollywood
+  </a>
+  as a second-rank leading man, as his face and body were as handsome as his voice. His first screen credit was the starring role the
+  <a href="/wiki/1937" title="1937">
+   1937
+  </a>
+  movie
+  <i>
+   <a class="new" href="/w/index.php?title=Love_is_On_the_Air&amp;action=edit&amp;redlink=1" title="Love is On the Air (page does not exist)">
+    Love is On the Air
+   </a>
+  </i>
+  . By the end of
+  <a href="/wiki/1939" title="1939">
+   1939
+  </a>
+  , he had appeared in 19 films. In
+  <a href="/wiki/1940" title="1940">
+   1940
+  </a>
+  he played the role of
+  <a href="/wiki/George_Gipp" title="George Gipp">
+   George "The Gipper" Gipp
+  </a>
+  in the film
+  <i>
+   <a class="mw-redirect" href="/wiki/Knute_Rockne_All_American" title="Knute Rockne All American">
+    Knute Rockne All American
+   </a>
+  </i>
+  , from which he acquired the nickname
+  <b>
+   the Gipper
+  </b>
+  , which he retained the rest of his life. Reagan himself considered that his best acting work was in
+  <i>
+   <a href="/wiki/Kings_Row" title="Kings Row">
+    Kings Row
+   </a>
+  </i>
+  (
+  <a href="/wiki/1942" title="1942">
+   1942
+  </a>
+  ). Other notable Reagan films include
+  <i>
+   <a href="/wiki/Hellcats_of_the_Navy" title="Hellcats of the Navy">
+    Hellcats of the Navy
+   </a>
+  </i>
+  ,
+  <i>
+   <a href="/wiki/This_Is_the_Army" title="This Is the Army">
+    This Is the Army
+   </a>
+  </i>
+  , and the
+  <a class="mw-disambig" href="/wiki/Camp" title="Camp">
+   campy
+  </a>
+  <i>
+   <a href="/wiki/Bedtime_for_Bonzo" title="Bedtime for Bonzo">
+    Bedtime for Bonzo
+   </a>
+  </i>
+  . He has a star on the
+  <a href="/wiki/Hollywood_Walk_of_Fame" title="Hollywood Walk of Fame">
+   Hollywood Walk of Fame
+  </a>
+  at 6374 Hollywood Blvd.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan was commissioned as a reserve
+  <a href="/wiki/Cavalry" title="Cavalry">
+   cavalry
+  </a>
+  officer in the
+  <a class="mw-redirect" href="/wiki/U.S._Army" title="U.S. Army">
+   U.S. Army
+  </a>
+  in
+  <a href="/wiki/1935" title="1935">
+   1935
+  </a>
+  . After
+  <a href="/wiki/Pearl_Harbor" title="Pearl Harbor">
+   Pearl Harbor
+  </a>
+  he was activated and was assigned to the
+  <a href="/wiki/First_Motion_Picture_Unit" title="First Motion Picture Unit">
+   First Motion Picture Unit
+  </a>
+  in the
+  <a class="mw-disambig" href="/wiki/Army_Air_Corps" title="Army Air Corps">
+   Army Air Corps
+  </a>
+  , which made training and education films. He remained in
+  <a href="/wiki/Hollywood" title="Hollywood">
+   Hollywood
+  </a>
+  for the duration of the war.
+ </ins>
+</p>
+<p>
+ <ins>
+  In 1940, Reagan married actress
+  <a href="/wiki/Jane_Wyman" title="Jane Wyman">
+   Jane Wyman
+  </a>
+  and had a daughter with her. After divorcing in 1948, Reagan remarried in 1952 to actress
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy Davis
+  </a>
+  . Nancy was pregnant with a child at the time and Reagan is the only President to have married a pregnant woman.
+ </ins>
+</p>
+<p>
+ <ins>
+  As Reagan's film roles became fewer in the late
+  <a href="/wiki/1950s" title="1950s">
+   1950s
+  </a>
+  , he moved into
+  <a href="/wiki/Television" title="Television">
+   television
+  </a>
+  as a host and frequent performer for
+  <i>
+   <a href="/wiki/General_Electric_Theater" title="General Electric Theater">
+    General Electric Theater
+   </a>
+  </i>
+  . His final regular acting job was as host and performer on
+  <i>
+   <a href="/wiki/Death_Valley_Days" title="Death Valley Days">
+    Death Valley Days
+   </a>
+  </i>
+  .
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Early_political_career_3">
+   Early political career
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Ronald Reagan began his political life as a
+  <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
+   liberal
+  </a>
+  <a class="mw-redirect" href="/wiki/United_States_Democratic_Party" title="United States Democratic Party">
+   Democrat
+  </a>
+  , supporting
+  <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
+   Franklin Delano Roosevelt
+  </a>
+  and his
+  <a href="/wiki/New_Deal" title="New Deal">
+   New Deal
+  </a>
+  . He gradually became a staunch social and fiscal
+  <a class="mw-redirect" href="/wiki/Conservative" title="Conservative">
+   conservative
+  </a>
+  . He embarked upon the path that led him to a career in politics during his tenure as president of the
+  <a href="/wiki/Screen_Actors_Guild" title="Screen Actors Guild">
+   Screen Actors Guild
+  </a>
+  (SAG), aligning himself with Senator
+  <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
+   Joseph McCarthy
+  </a>
+  and cooperating with the
+  <a href="/wiki/House_Un-American_Activities_Committee" title="House Un-American Activities Committee">
+   House Un-American Activities Committee
+  </a>
+  to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
+  <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
+   American Communist Party
+  </a>
+  . His employment by the
+  <a href="/wiki/General_Electric" title="General Electric">
+   General Electric
+  </a>
+  company, delivering
+  <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+   anti-communist
+  </a>
+  speeches on radio broadcasts and speaking tours, further enhanced his political image in the anti-Soviet climate of 1950s America. By the
+  <a class="mw-redirect" href="/wiki/U.S._presidential_election,_1964" title="U.S. presidential election, 1964">
+   1964 election
+  </a>
+  , Reagan was an outspoken supporter of conservative Republican
+  <a href="/wiki/Barry_Goldwater" title="Barry Goldwater">
+   Barry Goldwater
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1966" title="1966">
+   1966
+  </a>
+  , he was elected the
+  <a href="/wiki/Governor_of_California" title="Governor of California">
+   33rd Governor of California
+  </a>
+  . Reagan tried to gain the Republican presidential nomination in
+  <a href="/wiki/1968" title="1968">
+   1968
+  </a>
+  , and again in
+  <a href="/wiki/1976" title="1976">
+   1976
+  </a>
+  over the incumbent
+  <a href="/wiki/Gerald_Ford" title="Gerald Ford">
+   Gerald Ford
+  </a>
+  but was defeated at the Republican Convention. He succeeded in gaining the Republican nomination in 1980. The campaign was marked by the
+  <a href="/wiki/Iran_hostage_crisis" title="Iran hostage crisis">
+   Iran hostage crisis
+  </a>
+  . Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election. Most analysts believe President
+  <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+   Jimmy Carter
+  </a>
+  's inability to solve the hostage crisis played a large role in his defeat and Reagan's victory. Reagan went on to be elected President that year. In
+  <a href="/wiki/1984" title="1984">
+   1984
+  </a>
+  , he was re-elected in a landslide, winning in 49 of 50 states and receiving nearly 60 percent of the popular vote.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Presidency_3">
+   Presidency
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Then-Vice President Bush meets with President Reagan in 1984.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  On
+  <a href="/wiki/March_30" title="March 30">
+   March 30
+  </a>
+  ,
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  , just 69 days into his Presidency, while leaving the
+  <a class="mw-redirect" href="/wiki/Hilton_Hotel" title="Hilton Hotel">
+   Hilton Hotel
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/Washington,_DC" title="Washington, DC">
+   Washington, DC
+  </a>
+  , President Reagan,
+  <a class="mw-redirect" href="/wiki/Press_Secretary" title="Press Secretary">
+   Press Secretary
+  </a>
+  <a href="/wiki/James_Brady" title="James Brady">
+   James Brady
+  </a>
+  , a
+  <a class="mw-disambig" href="/wiki/Secret_Service" title="Secret Service">
+   Secret Service
+  </a>
+  agent, and a
+  <a class="mw-redirect" href="/wiki/District_of_Columbia" title="District of Columbia">
+   District of Columbia
+  </a>
+  police officer were shot by
+  <a class="mw-redirect" href="/wiki/John_Hinckley,_Jr." title="John Hinckley, Jr.">
+   John Hinckley, Jr.
+  </a>
+  Shortly before
+  <a href="/wiki/Surgery" title="Surgery">
+   surgery
+  </a>
+  to remove the bullet from his chest (which barely missed his heart) he remarked to his surgeons, "I hope you're all Republicans,"
+ and to his wife Nancy he jokingly commented, "Honey, I forgot to duck."
+ </ins>
+</p>
+<p>
+ <ins>
+  As a politician and as President, he portrayed himself as being:
+ </ins>
+</p>
+<ul>
+ <li>
+  <ins>
+   <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+    anti-communist
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of
+   <a href="/wiki/Tax" title="Tax">
+    tax
+   </a>
+   cuts
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of smaller government (with the exclusion of the
+   <a href="/wiki/Military" title="Military">
+    military
+   </a>
+   )
+  </ins>
+ </li>
+ <li>
+  <ins>
+   in favor of removing regulations on
+   <a href="/wiki/Corporation" title="Corporation">
+    corporations
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   supportive of business interests, both large and small
+  </ins>
+ </li>
+ <li>
+  <ins>
+   supportive of some individual liberties
+  </ins>
+ </li>
+ <li>
+  <ins>
+   tough on
+   <a href="/wiki/Crime" title="Crime">
+    crime
+   </a>
+  </ins>
+ </li>
+</ul>
+<p>
+ <ins>
+  He is credited with:
+ </ins>
+</p>
+<ul>
+ <li>
+  <ins>
+   increasing military spending
+  </ins>
+ </li>
+ <li>
+  <ins>
+   deploying U.S.
+   <a href="/wiki/Pershing_II" title="Pershing II">
+    Pershing II
+   </a>
+   missiles in Germany in response to the Soviet stationing of
+   <a class="mw-redirect" href="/wiki/SS-20" title="SS-20">
+    SS-20
+   </a>
+   missiles near Europe
+  </ins>
+ </li>
+ <li>
+  <ins>
+   pushing for the deployment of the
+   <a class="mw-redirect" href="/wiki/Peacekeeper_missile" title="Peacekeeper missile">
+    Peacekeeper missile
+   </a>
+   system
+  </ins>
+ </li>
+ <li>
+  <ins>
+   negotiating nuclear arms reduction treaties
+  </ins>
+ </li>
+ <li>
+  <ins>
+   proposing the
+   <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
+    Strategic Defense Initiative
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   arming and training anti-communist groups like the
+   <a href="/wiki/Contras" title="Contras">
+    Contras
+   </a>
+   and the
+   <a href="/wiki/Mujahideen" title="Mujahideen">
+    mujahideen
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   selling arms to foreign allies such
+   <a href="/wiki/Taiwan" title="Taiwan">
+    Taiwan
+   </a>
+   ,
+   <a href="/wiki/Israel" title="Israel">
+    Israel
+   </a>
+   ,
+   <a href="/wiki/Saudi_Arabia" title="Saudi Arabia">
+    Saudi Arabia
+   </a>
+   and, ironically,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   lowering taxes significantly, reversing a historic trend towards higher taxes
+  </ins>
+ </li>
+ <li>
+  <ins>
+   increasing the federal deficit
+  </ins>
+ </li>
+ <li>
+  <ins>
+   greatly escalating the "
+   <a class="mw-redirect" href="/wiki/War_on_drugs" title="War on drugs">
+    war on drugs
+   </a>
+   "
+  </ins>
+ </li>
+ <li>
+  <ins>
+   ending the high
+   <a href="/wiki/Inflation" title="Inflation">
+    inflation
+   </a>
+   that damaged the economy under his predecessors,
+   <a href="/wiki/Jimmy_Carter" title="Jimmy Carter">
+    Jimmy Carter
+   </a>
+   and
+   <a href="/wiki/Gerald_Ford" title="Gerald Ford">
+    Gerald Ford
+   </a>
+   .
+  </ins>
+ </li>
+ <li>
+  <ins>
+   helping to 'win' the
+   <a href="/wiki/Cold_War" title="Cold War">
+    Cold War
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   firing
+   <a href="/wiki/Air_traffic_controller" title="Air traffic controller">
+    air traffic controllers
+   </a>
+   when they illegally went on
+   <a href="/wiki/Strike_action" title="Strike action">
+    strike
+   </a>
+  </ins>
+ </li>
+</ul>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Reaganomics_3">
+   Reaganomics
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+    File:Reaganhiro.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Emperor
+    <a href="/wiki/Hirohito" title="Hirohito">
+     Hirohito
+    </a>
+    meeting with President Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting stagflation, or both
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  and
+  <a href="/wiki/Unemployment" title="Unemployment">
+   unemployment
+  </a>
+  . Reagan's policies, partially based on
+  <a href="/wiki/Supply-side_economics" title="Supply-side economics">
+   supply-side economics
+  </a>
+  sought to stimulate the economy with large across-the-board
+  <a href="/wiki/Tax_cut" title="Tax cut">
+   tax cuts
+  </a>
+  . Most economists agree that tax cuts stimulate the economy, but supply-siders say that they have a much greater effect than most economists say they would.
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  had called Reagan's economic ideas "
+  <a class="mw-redirect" href="/wiki/Voodoo_economics" title="Voodoo economics">
+   voodoo economics
+  </a>
+  " prior to becoming his vice-president. Reagan's policies soon became known as "
+  <a href="/wiki/Reaganomics" title="Reaganomics">
+   Reaganomics
+  </a>
+  ", a nickname used by both his supporters and detractors. These tax cuts combined with heavy military spending increases led to enormous
+  <a href="/wiki/Deficit_spending" title="Deficit spending">
+   deficit spending
+  </a>
+  and a dramatic increase in the
+  <a class="mw-redirect" href="/wiki/National_debt" title="National debt">
+   national debt
+  </a>
+  . The debt increased by approximately 200% (tripled) between when Reagan took office and when his successor,
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  , took office. Reagan's supporters note that although the President writes an annual budget proposal, the actual spending bill originates in the
+  <a href="/wiki/United_States_House_of_Representatives" title="United States House of Representatives">
+   House of Representatives
+  </a>
+  , so Reagan's policies are not the exclusive cause of the debt increase.
+ </ins>
+</p>
+<p>
+ <ins>
+  On the other hand, this spending was slightly offset by increased tax revenues, and some supporters of Reagan attribute this to the successful use of supply-side economics tax policies. Critics of President Reagan argue that despite his frequent pronouncements that he advocated smaller and less intrusive government, federal spending and bureaucracy increased in size during his administration. Not surprisingly, there is disagreement over how much Reagan's policies contributed to the severe
+  <a href="/wiki/Recession" title="Recession">
+   recession
+  </a>
+  that took place in
+  <a href="/wiki/1982" title="1982">
+   1982
+  </a>
+  , the strong economic expansion that began late in his first term and ran throughout his second term, and the fall in the average inflation-adjusted hourly wage for American workers that happened between his 1981 swearing-in and his successor's 1989 swearing-in.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="The_Soviet_Union_and_the_Cold_War_3">
+   The Soviet Union and the Cold War
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Like many successful American politicians, Reagan had great stage presence, as well as great instincts for cultivating positive responses from the public. His calm speaking voice and forceful language earned him the nickname "the Great Communicator." On
+  <a href="/wiki/March_8" title="March 8">
+   March 8
+  </a>
+  ,
+  <a href="/wiki/1983" title="1983">
+   1983
+  </a>
+  he called the
+  <a href="/wiki/Soviet_Union" title="Soviet Union">
+   Soviet Union
+  </a>
+  an "Evil Empire" and later in his presidency while speaking in front of the
+  <a href="/wiki/Berlin_Wall" title="Berlin Wall">
+   Berlin Wall
+  </a>
+  he challenged Soviet leader
+  <a href="/wiki/Mikhail_Gorbachev" title="Mikhail Gorbachev">
+   Mikhail Gorbachev
+  </a>
+  to "tear down this wall". Some historians believe that all of those traits were greatly supported by his enthusiasm for America and strong personal belief in rights of the individual.
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Many Reagan supporters credit him with winning the
+  <a href="/wiki/Cold_War" title="Cold War">
+   Cold War
+  </a>
+  , while others attribute the collapse of
+  <a href="/wiki/Communism" title="Communism">
+   communism
+  </a>
+  in
+  <a href="/wiki/1989" title="1989">
+   1989
+  </a>
+  in Eastern Europe and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
+  <a href="/wiki/Mikhail_Gorbachev" title="Mikhail Gorbachev">
+   Mikhail Gorbachev
+  </a>
+  . Reagan's policies included strong support of the U.S. military and the doctrine of "peace through strength." One of his more controversial proposals was the
+  <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
+   Strategic Defense Initiative
+  </a>
+  (SDI), a missile defense system which he hoped would make the U.S. invulnerable to nuclear missile attack by the
+  <a href="/wiki/Soviet_Union" title="Soviet Union">
+   Soviet Union
+  </a>
+  . SDI was dubbed "Star Wars" by opponents, connoting the fantastic or impractical.
+ </ins>
+</p>
+<p>
+ <ins>
+  Critics of SDI argued that the technological objective was unattainable in practical terms, and that the attempt would be likely to increase the
+  <a class="mw-redirect" href="/wiki/Arms_Race" title="Arms Race">
+   Arms Race
+  </a>
+  , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application the
+  <a class="mw-redirect" href="/wiki/Strategy_of_technology" title="Strategy of technology">
+   strategy of technology
+  </a>
+  . They saw SDI as an attempt to convince the Soviets that their nuclear missile arsenal was obsolescent, burdening the Soviets with additional spending on new technology to maintain a credible nuclear deterrent.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan had a close friendship with many other conservative political leaders across the globe, especially
+  <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+   Margaret Thatcher
+  </a>
+  in
+  <a class="mw-redirect" href="/wiki/UK" title="UK">
+   Britain
+  </a>
+  , and
+  <a href="/wiki/Brian_Mulroney" title="Brian Mulroney">
+   Brian Mulroney
+  </a>
+  in
+  <a href="/wiki/Canada" title="Canada">
+   Canada
+  </a>
+  . Reagan had a great desire for establishing personal relationships with other heads of state, often inviting them to his ranch or
+  <a href="/wiki/Camp_David" title="Camp David">
+   Camp David
+  </a>
+  for casual retreats.
+ </ins>
+</p>
+<p>
+ <ins>
+  As part of the policies that became known as the
+  <a href="/wiki/Reagan_Doctrine" title="Reagan Doctrine">
+   Reagan Doctrine
+  </a>
+  , the United States also took an increasingly hard line against Communist influences in Latin America, which often involved the controversial support of anti-Communist
+  <a href="/wiki/Military_dictatorship" title="Military dictatorship">
+   military dictatorships
+  </a>
+  with poor
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  records. This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
+  <a href="/wiki/1983" title="1983">
+   1983
+  </a>
+  Reagan ordered a formal military invasion of the small island nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  after it underwent a Communist coup. Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
+  <a href="/wiki/United_States_Agency_for_International_Development" title="United States Agency for International Development">
+   foreign aid
+  </a>
+  packages to states who held free elections.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Iran-Contra_Affair_3">
+   Iran-Contra Affair
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  During his administration, there was a major scandal and investigation of his administration's covert support of wars in
+  <a href="/wiki/Iran" title="Iran">
+   Iran
+  </a>
+  and
+  <a href="/wiki/Nicaragua" title="Nicaragua">
+   Nicaragua
+  </a>
+  in what came to be known as the
+  <a class="mw-redirect" href="/wiki/Iran-Contra_Affair" title="Iran-Contra Affair">
+   Iran-Contra Affair
+  </a>
+  . Two members of administration,
+  <a class="mw-redirect" href="/wiki/National_Security_Advisor" title="National Security Advisor">
+   National Security Advisor
+  </a>
+  <a href="/wiki/John_Poindexter" title="John Poindexter">
+   John Poindexter
+  </a>
+  and Col.
+  <a href="/wiki/Oliver_North" title="Oliver North">
+   Oliver North
+  </a>
+  had hatched an elaborate plot to sell arms to the Iranian government and give the profits to the anti-Communist
+  <a class="mw-disambig" href="/wiki/Contra" title="Contra">
+   Contras
+  </a>
+  guerillas in Nicaragua, who were engaged in a bloody civil war. Both actions were contrary to acts of
+  <a href="/wiki/United_States_Congress" title="United States Congress">
+   Congress
+  </a>
+  . Reagan professed ignorance of the plot, but admitted that he had supported the initial sale of arms to Iran, on the grounds that such sales were supposed to help secure the release of Americans being held hostage by the Iranian-backed
+  <a href="/wiki/Hezbollah" title="Hezbollah">
+   Hezbollah
+  </a>
+  terrorist group in
+  <a href="/wiki/Lebanon" title="Lebanon">
+   Lebanon
+  </a>
+  .
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Reagan quickly called for the appointment of an
+  <a class="mw-redirect" href="/wiki/Office_of_the_Independent_Counsel" title="Office of the Independent Counsel">
+   Independent Counsel
+  </a>
+  to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led Congressman Patricia Schroeder to dub him the "
+  <a class="mw-redirect" href="/wiki/Teflon" title="Teflon">
+   Teflon
+  </a>
+  President", a term that has been occasionally attached to later Presidents and their scandals.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id=".22War_on_Drugs.22_3">
+   "War on Drugs"
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Reagan's policies in the "
+  <a href="/wiki/War_on_Drugs" title="War on Drugs">
+   War on Drugs
+  </a>
+  " emphasized
+  <a href="/wiki/Imprisonment" title="Imprisonment">
+   imprisonment
+  </a>
+  for drug offenders while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's prison population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless, surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Miscellaneous_3">
+   Miscellaneous
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  On
+  <a href="/wiki/August_5" title="August 5">
+   August 5
+  </a>
+  ,
+  <a href="/wiki/1981" title="1981">
+   1981
+  </a>
+  , Reagan fired 11,359
+  <a href="/wiki/Strike_action" title="Strike action">
+   striking
+  </a>
+  <a href="/wiki/Air_traffic_controller" title="Air traffic controller">
+   air traffic controllers
+  </a>
+  who had ignored his order to return to work. Ironically,
+  <a class="mw-disambig" href="/wiki/Professional_Air_Traffic_Controllers_Organization" title="Professional Air Traffic Controllers Organization">
+   PATCO
+  </a>
+  , the air traffic controller's union, had been one of the few unions that had supported Reagan over Carter in the election nine months previously.
+ </ins>
+</p>
+<p>
+ <ins>
+  In the spring of 1983, Reagan sent
+  <a href="/wiki/United_States_Marine_Corps" title="United States Marine Corps">
+   US Marines
+  </a>
+  into
+  <a href="/wiki/Lebanon" title="Lebanon">
+   Lebanon
+  </a>
+  . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines. Two days later Reagan invaded the tiny
+  <a href="/wiki/Caribbean" title="Caribbean">
+   Caribbean
+  </a>
+  nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  . Three months later, Reagan withdrew the Marines from Lebanon.
+ </ins>
+</p>
+<p>
+ <ins>
+  On
+  <a href="/wiki/July_13" title="July 13">
+   July 13
+  </a>
+  ,
+  <a href="/wiki/1985" title="1985">
+   1985
+  </a>
+  , Reagan underwent surgery to remove cancerous
+  <a href="/wiki/Polyp_(medicine)" title="Polyp (medicine)">
+   polyps
+  </a>
+  from his
+  <a class="mw-redirect" href="/wiki/Colon_(anatomy)" title="Colon (anatomy)">
+   colon
+  </a>
+  , causing the first-ever invocation of the
+  <a href="/wiki/Acting_President_of_the_United_States" title="Acting President of the United States">
+   Acting President
+  </a>
+  clause of the
+  <a href="/wiki/Twenty-fifth_Amendment_to_the_United_States_Constitution" title="Twenty-fifth Amendment to the United States Constitution">
+   25th Amendment
+  </a>
+  . On
+  <a href="/wiki/January_5" title="January 5">
+   January 5
+  </a>
+  ,
+  <a href="/wiki/1987" title="1987">
+   1987
+  </a>
+  , Reagan underwent
+  <a href="/wiki/Prostate" title="Prostate">
+   prostate
+  </a>
+  surgery which caused further worries about his health.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan was widely criticized in 1985 for an incident related to an official visit to
+  <a href="/wiki/West_Germany" title="West Germany">
+   West Germany
+  </a>
+  . On
+  <a href="/wiki/April_11" title="April 11">
+   April 11
+  </a>
+  , the White House announced that Reagan would be visiting the
+  <a class="mw-redirect" href="/wiki/Bitburg_military_cemetery" title="Bitburg military cemetery">
+   Bitburg military cemetery
+  </a>
+  together with Chancellor
+  <a href="/wiki/Helmut_Kohl" title="Helmut Kohl">
+   Helmut Kohl
+  </a>
+  , to lay a wreath in honor of German soldiers who died in both World Wars. This became controversial when it came to public attention that a small number (variously reported as 49 or 56) of gravesites contained remains of soldiers who had served in
+  <a href="/wiki/Waffen-SS" title="Waffen-SS">
+   Waffen-SS
+  </a>
+  units. Despite protests from various quarters, most notably
+  <a href="/wiki/Elie_Wiesel" title="Elie Wiesel">
+   Elie Wiesel
+  </a>
+  , Reagan carried out the visit on
+  <a href="/wiki/May_5" title="May 5">
+   May 5
+  </a>
+  on the grounds that it would promote reconciliation between the former adversaries.
+ </ins>
+</p>
+<p>
+ <ins>
+  In
+  <a href="/wiki/1991" title="1991">
+   1991
+  </a>
+  , former
+  <a class="mw-disambig" href="/wiki/Starlet" title="Starlet">
+   starlet
+  </a>
+  <a class="new" href="/w/index.php?title=Selene_Walters&amp;action=edit&amp;redlink=1" title="Selene Walters (page does not exist)">
+   Selene Walters
+  </a>
+  claimed that Reagan had raped her in the early 1950s.
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres37-41.jpg" title="File:Pres37-41.jpg">
+    File:Pres37-41.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential Library.
+   </ins>
+  </div>
+ </div>
+</div>
+<h2>
+ <span class="mw-headline" id="Legacy_and_retirement_from_public_life">
+  Legacy and retirement from public life
+ </span>
+</h2>
+<p>
+ Reagan was in many ways the founder of the modern Republican Party. His redefinition of fiscal conservatism as being focused on tax cuts without significant regard to a balanced budget ("
+ <a href="/wiki/Reaganomics" title="Reaganomics">
+  Reaganomics
+ </a>
+ "); his opposition to
+ <a class="mw-redirect" href="/wiki/Progressive_taxation" title="Progressive taxation">
+  progressive taxation
+ </a>
+ , greater environmental protection and regulation, and
+ <a href="/wiki/Abortion" title="Abortion">
+  abortion
+ </a>
+ ; the importance of the
+ <a href="/wiki/Moral_Majority" title="Moral Majority">
+  Moral Majority
+ </a>
+ and its supporters in his governing coalition; and even his support of
+ <a href="/wiki/Strategic_Defense_Initiative" title="Strategic Defense Initiative">
+  missile defense systems
+ </a>
+ have all become trademarks of subsequent Republican leaders, including
+ <a href="/wiki/George_W._Bush" title="George W. Bush">
+  George W. Bush
+ </a>
+ . Reagan's immediate predecessors such as
+ <a href="/wiki/Richard_Nixon" title="Richard Nixon">
+  Richard Nixon
+ </a>
+ and
+ <a class="mw-redirect" href="/wiki/Dwight_Eisenhower" title="Dwight Eisenhower">
+  Dwight Eisenhower
+ </a>
+ would not have recognized any of these as part of the Republican platform.
+</p>
+<p>
+ In
+ <a href="/wiki/1992" title="1992">
+  1992
+ </a>
+ , four years after leaving office, Reagan was diagnosed with
+ <a href="/wiki/Alzheimer%27s_disease" title="Alzheimer's disease">
+  Alzheimer's disease
+ </a>
+ . As the years went on, the disease began to slowly take over the former President's brain and body, forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
+ <a href="/wiki/November_5" title="November 5">
+  November 5
+ </a>
+ ,
+ <a href="/wiki/1994" title="1994">
+  1994
+ </a>
+ in the form of a personal letter. His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
+ <a href="/wiki/2004" title="2004">
+  2004
+ </a>
+ Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was rumoured, although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
+</p>
+<p>
+ On
+ <a href="/wiki/February_6" title="February 6">
+  February 6
+ </a>
+ ,
+ <a href="/wiki/1998" title="1998">
+  1998
+ </a>
+ , Washington National Airport in
+ <a class="mw-redirect" href="/wiki/Washington,_DC" title="Washington, DC">
+  Washington, DC
+ </a>
+ was renamed
+ <a href="/wiki/Ronald_Reagan_Washington_National_Airport" title="Ronald Reagan Washington National Airport">
+  Ronald Reagan Washington National Airport
+ </a>
+ . Also, the
+ <a href="/wiki/Aircraft_carrier" title="Aircraft carrier">
+  aircraft carrier
+ </a>
+ <a class="mw-redirect" href="/wiki/USS_Ronald_Reagan_(CVN-76)" title="USS Ronald Reagan (CVN-76)">
+  USS
+  <i>
+   Ronald Reagan
+  </i>
+  (CVN-76)
+ </a>
+ was christened on
+ <a href="/wiki/March_4" title="March 4">
+  March 4
+ </a>
+ ,
+ <a href="/wiki/2001" title="2001">
+  2001
+ </a>
+ , making it one of the
+ <a class="mw-redirect" href="/wiki/List_of_military_vessels_named_after_living_Americans" title="List of military vessels named after living Americans">
+  very few
+ </a>
+ <a href="/wiki/United_States_Navy" title="United States Navy">
+  United States Navy
+ </a>
+ ships to be named after a living person.
+</p>
+<p>
+ In November
+ <a href="/wiki/2003" title="2003">
+  2003
+ </a>
+ , Reagan and his family were the subject of a controversial television miniseries,
+ <i>
+  <a href="/wiki/The_Reagans" title="The Reagans">
+   The Reagans
+  </a>
+ </i>
+ . In response to the unflattering docu-drama, a number of Congressional Republicans introduced the "Ronald Reagan Dime Act" (HR 3633), a bill that would replace
+ <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
+  Franklin Delano Roosevelt
+ </a>
+ 's portrait on the
+ <a class="mw-redirect" href="/wiki/Dime_(U.S._coin)" title="Dime (U.S. coin)">
+  United States dime
+ </a>
+ with Reagan's. The bill did not have widespread support and appeared unlikely to be put up for a vote.
+</p>
+<p>
+ In
+ <a href="/wiki/2004" title="2004">
+  2004
+ </a>
+ , Reagan turned 93, continuing his record as the longest-living president in American history. The previous record had been 90 years and 247 days, held by
+ <a href="/wiki/John_Adams" title="John Adams">
+  John Adams
+ </a>
+ , whom Reagan surpassed on
+ <a href="/wiki/October_12" title="October 12">
+  October 12
+ </a>
+ ,
+ <a href="/wiki/2001" title="2001">
+  2001
+ </a>
+ . In June of 2004 the
+ <a href="/wiki/White_House" title="White House">
+  White House
+ </a>
+ was informed by family doctors that Reagan's health had "significantly deteriorated" and that he was not expected to live
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   much longer. He passed away at his home in
+   <a class="mw-redirect" href="/wiki/Los_Angeles,_California" title="Los Angeles, California">
+    Los Angeles
+   </a>
+   on
+   <a href="/wiki/June_5" title="June 5">
+    June 5
+   </a>
+   at 1:09 PM local time. He died of
+   <a href="/wiki/Pneumonia" title="Pneumonia">
+    pneumonia
+   </a>
+   , the
+   <a href="/wiki/White_House" title="White House">
+    White House
+   </a>
+   said. He was survived by his wife
+   <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+    Nancy Reagan
+   </a>
+   and his children: Michael, Patti Davis and Ron. He was 40th president of the USA and was with responsible for the Fall of the
+   <a href="/wiki/Berlin_Wall" title="Berlin Wall">
+    Berlin Wall
+   </a>
+   and the fall of the
+   <a href="/wiki/Soviet_Union" title="Soviet Union">
+    Soviet Union
+   </a>
+   . His family stood beside on his bed when he died. The White House lowers the flag to half-staff.
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
+    File:Pres38-42.jpg
+   </a>
+  </ins>
+  <del>
+   more than a year.
+  </del>
+  <div class="thumbcaption">
+   Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
+  </div>
+ </div>
+ <del>
+  Link: //en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg
+ </del>
+ <p>
+  <del>
+   On
+   <a href="/wiki/June_5" title="June 5">
+    June 5
+   </a>
+   , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+  </del>
+ </p>
+</div>
+<h2>
+ <span class="mw-headline" id="Supreme_Court_appointments">
+  <a href="/wiki/Supreme_Court_of_the_United_States" title="Supreme Court of the United States">
+   Supreme Court
+  </a>
+  appointments
+ </span>
+</h2>
+<ul>
+ <li>
+  <a href="/wiki/Sandra_Day_O%27Connor" title="Sandra Day O'Connor">
+   Sandra Day O'Connor
@@ -1249 +4147 @@
-</ul>
+</ul>
\ No newline at end of file
