--- 3955334
+++ 3986827
@@ -2 +1,0 @@
- <br/>
@@ -6 +5,6 @@
- (born
+ <ins>
+  (
+ </ins>
+ <del>
+  (born
+ </del>
@@ -13,0 +18,10 @@
+ <ins>
+  -
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+ </ins>
@@ -46,5 +60,15 @@
- before entering politics. He is the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days). He is the only
- <a href="/wiki/Divorce" title="Divorce">
-  divorced
- </a>
- person to be elected President, and the only President to have married a pregnant woman.
+ before entering politics. He
+ <ins>
+  lived longer than any other President (93 years, 119 days) and was
+ </ins>
+ <del>
+  is the longest-lived person to have served as President, as well as
+ </del>
+ the oldest elected President (69 years, 349 days).
+ <del>
+  He is the only
+  <a href="/wiki/Divorce" title="Divorce">
+   divorced
+  </a>
+  person to be elected President, and the only President to have married a pregnant woman.
+ </del>
@@ -106 +130 @@
- <a class="mw-disambig" href="/wiki/Rock_River" title="Rock River">
+ <a class="mw-redirect" href="/wiki/Rock_River_(Illinois)" title="Rock River (Illinois)">
@@ -124 +148,7 @@
- , graduating in
+ <ins>
+  majoring in economics and sociology,
+ </ins>
+ <del>
+  ,
+ </del>
+ graduating in
@@ -128 +158,8 @@
- . Child of an
+ .
+ <ins>
+  The child
+ </ins>
+ <del>
+  Child
+ </del>
+ of an
@@ -151,4 +188,4 @@
- games, getting only the bare outlines of the game from a ticker and relying on his
- <a href="/wiki/Imagination" title="Imagination">
-  imagination
- </a>
+ games, getting only the bare outlines of the game from a ticker and relying on his imagination
+ <del>
+  Link: /wiki/Imagination
+ </del>
@@ -163,4 +200,4 @@
- game, the wire went dead. Reagan smoothly
- <a class="mw-redirect" href="/wiki/Improvise" title="Improvise">
-  improvised
- </a>
+ game, the wire went dead. Reagan smoothly improvised
+ <del>
+  Link: /wiki/Improvise
+ </del>
@@ -168,0 +206,14 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -174 +225,8 @@
- as a second-rank leading man, as his face and body were as handsome as his voice. His first screen credit was the starring role the
+ as a
+ <ins>
+  eading
+ </ins>
+ <del>
+  second-rank leading
+ </del>
+ man, as his face and body were as handsome as his voice. His first screen credit was the starring role the
@@ -192 +250,5 @@
- he played the role of George "The Gipper" Gipp in the film
+ he played the role of
+ <a href="/wiki/George_Gipp" title="George Gipp">
+  George "The Gipper" Gipp
+ </a>
+ in the film
@@ -244 +306,5 @@
- officer in the U.S. Army in
+ officer in the
+ <a class="mw-redirect" href="/wiki/U.S._Army" title="U.S. Army">
+  U.S. Army
+ </a>
+ in
@@ -252,8 +318,20 @@
- he was activated and was assigned to the First Motion Picture Unit in the
- <a class="mw-disambig" href="/wiki/Army_Air_Corps" title="Army Air Corps">
-  Army Air Corps
- </a>
- , which made training and education films. He remained in
- <a href="/wiki/Hollywood" title="Hollywood">
-  Hollywood
- </a>
+ he was
+ <ins>
+  volunteered for activation
+ </ins>
+ <del>
+  activated
+ </del>
+ and was assigned to the
+ <a href="/wiki/First_Motion_Picture_Unit" title="First Motion Picture Unit">
+  First Motion Picture Unit
+ </a>
+ in the
+ <a href="/wiki/United_States_Air_Force" title="United States Air Force">
+  Army Air Force
+  Link: /wiki/United_States_Air_Force Corps
+ </a>
+ , which made training and education films. He remained in Hollywood
+ <del>
+  Link: /wiki/Hollywood
+ </del>
@@ -262,0 +341,64 @@
+ <ins>
+  Reagan married actress
+  <a href="/wiki/Jane_Wyman" title="Jane Wyman">
+   Jane Wyman
+  </a>
+  in 1940. They had a daughter,
+  <a href="/wiki/Maureen_Reagan" title="Maureen Reagan">
+   Maureen
+  </a>
+  (born
+  <a href="/wiki/January_4" title="January 4">
+   January 4
+  </a>
+  <a href="/wiki/1941" title="1941">
+   1941
+  </a>
+  ), adopted a son
+  <a href="/wiki/Michael_Reagan" title="Michael Reagan">
+   Michael
+  </a>
+  (born
+  <a href="/wiki/March_18" title="March 18">
+   March 18
+  </a>
+  <a href="/wiki/1946" title="1946">
+   1946
+  </a>
+  ), and had a daughter born four months prematurely in 1947 who lived but one day. They divorced in 1948, making Reagan the only President who has been divorced. Reagan remarried actress
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy Davis
+  </a>
+  on
+  <a href="/wiki/March_4" title="March 4">
+   March 4
+  </a>
+  <a href="/wiki/1952" title="1952">
+   1952
+  </a>
+  ; they had two children
+  <a href="/wiki/Ron_Reagan" title="Ron Reagan">
+   Ron
+  </a>
+  (born
+  <a href="/wiki/May_20" title="May 20">
+   May 20
+  </a>
+  <a href="/wiki/1958" title="1958">
+   1958
+  </a>
+  ) and
+  <a class="mw-redirect" href="/wiki/Patti_Reagan" title="Patti Reagan">
+   Patti
+  </a>
+  (born
+  <a href="/wiki/October_21" title="October 21">
+   October 21
+  </a>
+  <a href="/wiki/1952" title="1952">
+   1952
+  </a>
+  ). Nancy was pregnant during the wedding, making Reagan the only President to have married a pregnant woman.
+ </ins>
+</p>
+<p>
@@ -292,3 +434,5 @@
-  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
-   File:Nancy and Ronald Reagan.jpg
-  </a>
+  <del>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </del>
@@ -296 +440,3 @@
-   Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   <del>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </del>
@@ -302,3 +448,5 @@
- <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
-  liberal
- </a>
+ <del>
+  <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
+   liberal
+  </a>
+ </del>
@@ -324,5 +472,12 @@
- (SAG), aligning himself with Senator
- <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
-  Joseph McCarthy
- </a>
- and cooperating with the
+ (SAG),
+ <ins>
+  during which time he did not cooperate
+ </ins>
+ <del>
+  aligning himself with Senator
+  <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
+   Joseph McCarthy
+  </a>
+  and cooperating
+ </del>
+ with the
@@ -332,5 +487,11 @@
- to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
- <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
-  American Communist Party
- </a>
- . His employment by the
+ <ins>
+  , instead using his post at SAG to help Hollywood police its own ranks.
+ </ins>
+ <del>
+  to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
+  <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
+   American Communist Party
+  </a>
+  .
+ </del>
+ His employment by the
@@ -340,5 +501,18 @@
- company, delivering
- <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
-  anti-communist
- </a>
- speeches on radio broadcasts and speaking tours, further enhanced his political image in the anti-Soviet climate of 1950s America. By the
+ <ins>
+  company
+ </ins>
+ <del>
+  company, delivering
+  <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+   anti-communist
+  </a>
+  speeches on radio broadcasts and speaking tours,
+ </del>
+ further enhanced his political
+ <ins>
+  image.
+ </ins>
+ <del>
+  image in the anti-Soviet climate of 1950s America.
+ </del>
+ By the
@@ -379 +553,5 @@
- . Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election. Most analysts believe President
+ .
+ <del>
+  Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election.
+ </del>
+ Most analysts believe President
@@ -389,5 +566,0 @@
-<h2>
- <span class="mw-headline" id="Presidency">
-  Presidency
- </span>
-</h2>
@@ -395 +568 @@
- <div class="thumbinner" style="width:302px;">
+ <div class="thumbinner" style="width:202px;">
@@ -397 +570,5 @@
-   <img alt="" class="thumbimage" data-file-height="332" data-file-width="500" height="199" src="//upload.wikimedia.org/wikipedia/commons/thumb/c/ce/Bush_reagan.jpg/300px-Bush_reagan.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/c/ce/Bush_reagan.jpg/450px-Bush_reagan.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/c/ce/Bush_reagan.jpg 2x" width="300"/>
+   Much of his first election and this second term landslide is attributed to the then named "
+ Reagan Democrats
+  Link: /wiki/Reagan_Democrat  ", a newly emerged but mostly unorganized political force.
+Presidency
+    Presidency
@@ -404 +581,15 @@
-   Then-Vice President Bush meets with President Reagan in 1984.
+   Then-Vice President
+   <ins>
+    Bush, right,
+   </ins>
+   <del>
+    Bush
+   </del>
+   meets with President
+   <ins>
+    Reagan, left,
+   </ins>
+   <del>
+    Reagan
+   </del>
+   in 1984.
@@ -480,0 +672,5 @@
+  <ins>
+   in favor of the use violence to protect U.S. interests
+  </ins>
+ </li>
+ <li>
@@ -484 +680,3 @@
-  supportive of some individual liberties
+  <del>
+   supportive of some individual liberties
+  </del>
@@ -493,0 +692,24 @@
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+    File:Reaganhiro.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Emperor
+    <a href="/wiki/Hirohito" title="Hirohito">
+     Hirohito
+    </a>
+    of
+    <a href="/wiki/Japan" title="Japan">
+     Japan
+    </a>
+    meeting with Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
@@ -528 +750,5 @@
-  arming and training anti-communist groups like the
+  arming
+  <del>
+   and training anti-communist
+  </del>
+  groups like the
@@ -539 +765 @@
-  <a href="/wiki/Taiwan" title="Taiwan">
+  <a class="mw-redirect" href="/wiki/Republic_of_China" title="Republic of China">
@@ -550,4 +776,16 @@
-  and, ironically,
-  <a href="/wiki/Iraq" title="Iraq">
-   Iraq
-  </a>
+  <ins>
+   ,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+   , and
+   <a href="/wiki/Iran" title="Iran">
+    Iran
+   </a>
+  </ins>
+  <del>
+   and, ironically,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+  </del>
@@ -584,4 +822,6 @@
-  helping to 'win' the
-  <a href="/wiki/Cold_War" title="Cold War">
-   Cold War
-  </a>
+  <del>
+   helping to 'win' the
+   <a href="/wiki/Cold_War" title="Cold War">
+    Cold War
+   </a>
+  </del>
@@ -594,20 +834,7 @@
-  when they illegally
-  <a href="/wiki/Strike_action" title="Strike action">
-   struck
-  </a>
- </li>
-</ul>
-<h3>
- <span class="mw-headline" id="Reaganomics">
-  Reaganomics
- </span>
-</h3>
-<div class="thumb tright">
- <div class="thumbinner" style="width:182px;">
-  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
-   File:Reaganhiro.jpg
-  </a>
-  <div class="thumbcaption">
-   Emperor
-   <a href="/wiki/Hirohito" title="Hirohito">
-    Hirohito
+  when they
+  <ins>
+   went on
+  </ins>
+  <ins>
+   <a href="/wiki/Strike_action" title="Strike action">
+    strike
@@ -615 +842,24 @@
-   meeting with President Reagan.
+  </ins>
+  <h3>
+   <ins>
+    <span class="mw-headline" id="Reaganomics">
+     Reaganomics
+    </span>
+   </ins>
+  </h3>
+  <div class="thumb tright">
+   <div class="thumbinner" style="width:302px;">
+    <div class="thumbcaption">
+     <div class="magnify">
+     </div>
+     <ins>
+      President Reagan signs the
+     </ins>
+     <ins>
+      <a href="/wiki/Tax_Reform_Act_of_1986" title="Tax Reform Act of 1986">
+       Tax Reform Act of 1986
+      </a>
+      on the South Lawn.
+     </ins>
+    </div>
+   </div>
@@ -617,2 +867,35 @@
- </div>
-</div>
+  <del>
+   illegally
+  </del>
+  <del>
+   <a href="/wiki/Strike_action" title="Strike action">
+    struck
+   </a>
+  </del>
+  <h3>
+   <del>
+    <span class="mw-headline" id="Reaganomics">
+     Reaganomics
+    </span>
+   </del>
+  </h3>
+  <div class="thumb tright">
+   <div class="thumbinner" style="width:182px;">
+    <del>
+     <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+      File:Reaganhiro.jpg
+     </a>
+    </del>
+    <div class="thumbcaption">
+     <del>
+      Emperor
+      <a href="/wiki/Hirohito" title="Hirohito">
+       Hirohito
+      </a>
+      meeting with President Reagan.
+     </del>
+    </div>
+   </div>
+  </div>
+ </li>
+</ul>
@@ -620,8 +903,24 @@
- Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting stagflation, or both
- <a href="/wiki/Inflation" title="Inflation">
-  inflation
- </a>
- and
- <a href="/wiki/Unemployment" title="Unemployment">
-  unemployment
- </a>
+ Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting
+ <ins>
+  <a href="/wiki/Stagflation" title="Stagflation">
+   stagflation
+  </a>
+  , a high rate of
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  combined with an economic
+  <a href="/wiki/Recession" title="Recession">
+   recession
+  </a>
+ </ins>
+ <del>
+  stagflation, or both
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  and
+  <a href="/wiki/Unemployment" title="Unemployment">
+   unemployment
+  </a>
+ </del>
@@ -656 +955,8 @@
- . The debt increased by approximately 200% (tripled) between when Reagan took office and when his successor,
+ . The debt
+ <ins>
+  approximately tripled
+ </ins>
+ <del>
+  increased by approximately 200% (tripled)
+ </del>
+ between when Reagan took office and when his successor,
@@ -677,0 +984,79 @@
+ <ins>
+  <span class="mw-headline" id="Response_to_AIDS">
+   Response to AIDS
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Reagan's presidency saw the advent of
+  <a href="/wiki/HIV" title="HIV">
+   HIV
+  </a>
+  -
+  <a class="mw-redirect" href="/wiki/AIDS" title="AIDS">
+   AIDS
+  </a>
+  as a widespread epidemic in the US. Although AIDS was first reported in 1981, Reagan did not mention it publicly until
+  <a href="/wiki/1987" title="1987">
+   1987
+  </a>
+  . His administration approached the epidemic as a series of local and state issues rather than with a national strategy, and politicians for the Department of Health and Human Services pled behind the scenes for adequate funding.
+ </ins>
+</p>
+<p>
+ <ins>
+  In deference to the views of the powerful religious right, who saw AIDS as a disease limited to the
+  <a href="/wiki/Gay" title="Gay">
+   gay
+  </a>
+  male community and spread by immoral behaviour, Reagan prevented his surgeon general,
+  <a href="/wiki/C._Everett_Koop" title="C. Everett Koop">
+   C. Everett Koop
+  </a>
+  , from speaking out about the epidemic. When in
+  <a href="/wiki/1986" title="1986">
+   1986
+  </a>
+  Reagan finally authorized Koop to issue a report on the epidemic, he expected it to be in line with conservative policies; instead, Koop's
+  <i>
+   Surgeon General's Report on Acquired Immune Deficiency Syndrome
+  </i>
+  greatly emphasized the importance of a comprehensive AIDS education strategy, including widespread distribution of condoms, and rejected mandatory testing. This approach brought Koop into conflict with other administration officials such as
+  <a href="/wiki/William_Bennett" title="William Bennett">
+   William Bennett
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan appointed the
+  <a class="mw-disambig" href="/wiki/Watkins_Commission" title="Watkins Commission">
+   Watkins Commission
+  </a>
+  on AIDS in
+  <a href="/wiki/1987" title="1987">
+   1987
+  </a>
+  , but its recommendations for increased funding went largely ignored by the Reagan and the subsequent Bush administration.
+ </ins>
+</p>
+<p>
+ <ins>
+  Many socially conservative commentators see Reagan's approach to AIDS as a common sense approach to a problem of social immorality whose scope was not yet fully appreciated. Many in the
+  <a href="/wiki/LGBT" title="LGBT">
+   gay and lesbian
+  </a>
+  communities and many
+  <a class="mw-redirect" href="/wiki/People_with_AIDS" title="People with AIDS">
+   people with AIDS
+  </a>
+  , on the other hand, see in his policies anything from politically motivated willful blindness in the face of a devastating illness to an atrocious disregard for the value of the lives of people in sexual and racial minorities. The group
+  <a href="/wiki/ACT_UP" title="ACT UP">
+   ACT UP
+  </a>
+  worked, through civil disobedience and direct action, to raise awareness on the issue of AIDS.
+ </ins>
+</p>
+<h3>
@@ -681,0 +1067,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan, left, in one-on-one discussions with Mikhail Gorbachev, the General Secretary of the Communist Party of the USSR from 1985 to 1991.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -690,0 +1087,3 @@
+ <ins>
+  ,
+ </ins>
@@ -695 +1094,12 @@
- an "Evil Empire" and later in his presidency while speaking in front of the
+ an
+ <ins>
+  "
+  <a href="/wiki/Evil_empire" title="Evil empire">
+   evil empire
+  </a>
+  ". Later in his presidency,
+ </ins>
+ <del>
+  "Evil Empire" and later in his presidency
+ </del>
+ while speaking in front of the
@@ -698,0 +1109,3 @@
+ <ins>
+  ,
+ </ins>
@@ -707,3 +1119,0 @@
-  <a class="image" href="/wiki/File:Reagan_and_Gorbachev_hold_discussions.jpg">
-   <img alt="" class="thumbimage" data-file-height="505" data-file-width="750" height="202" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/300px-Reagan_and_Gorbachev_hold_discussions.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/450px-Reagan_and_Gorbachev_hold_discussions.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/600px-Reagan_and_Gorbachev_hold_discussions.jpg 2x" width="300"/>
-  </a>
@@ -712,2 +1121,0 @@
-    <a class="internal" href="/wiki/File:Reagan_and_Gorbachev_hold_discussions.jpg" title="Enlarge">
-    </a>
@@ -715 +1123,3 @@
-   President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   <del>
+    President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   </del>
@@ -732 +1142,10 @@
- in Eastern Europe and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
+ in
+ <ins>
+  <a href="/wiki/Central_Europe" title="Central Europe">
+   Central Europe
+  </a>
+ </ins>
+ <del>
+  Eastern Europe
+ </del>
+ and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
@@ -751 +1170,5 @@
- , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application the
+ , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application
+ <ins>
+  of
+ </ins>
+ the
@@ -758 +1181,34 @@
- Reagan had a close friendship with many other conservative political leaders across the globe, especially
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan_thatcher.jpg" title="File:Reagan thatcher.jpg">
+    File:Reagan thatcher.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Reagan and
+    <a href="/wiki/United_Kingdom" title="United Kingdom">
+     United Kingdom
+    </a>
+    Prime Minister
+    <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+     Margaret Thatcher
+    </a>
+    in
+    <a href="/wiki/Camp_David" title="Camp David">
+     Camp David
+    </a>
+    .
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ Reagan had close friendships
+</ins>
+<del>
+ Reagan had a close friendship
+</del>
+ with many other conservative political leaders across the globe, especially
@@ -760,2 +1216,2 @@
-  Margaret Thatcher
- </a>
+ Margaret Thatcher
+</a>
@@ -764,2 +1220,2 @@
-  Britain
- </a>
+ Britain
+</a>
@@ -768,2 +1224,2 @@
-  Brian Mulroney
- </a>
+ Brian Mulroney
+</a>
@@ -772,2 +1228,2 @@
-  Canada
- </a>
+ Canada
+</a>
@@ -776,2 +1232,2 @@
-  Camp David
- </a>
+ Camp David
+</a>
@@ -779,2 +1235 @@
-</p>
-<p>
+ <p>
@@ -781,0 +1237,3 @@
+ <ins>
+  "
+ </ins>
@@ -785,9 +1243,53 @@
- , the United States also took an increasingly hard line against Communist influences in Latin America, which often involved the controversial support of anti-Communist
- <a href="/wiki/Military_dictatorship" title="Military dictatorship">
-  military dictatorships
- </a>
- with poor
- <a href="/wiki/Human_rights" title="Human rights">
-  human rights
- </a>
- records. This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
+ <ins>
+  ,"
+ </ins>
+ <del>
+  ,
+ </del>
+ the United States also
+ <ins>
+  offerd financial and logistics support to the anti-communist opposition in central Europe (most notably the
+  <a href="/wiki/Poland" title="Poland">
+   Polish
+  </a>
+  <i>
+   <b>
+    <a href="/wiki/Solidarity" title="Solidarity">
+     Solidarity
+    </a>
+   </b>
+  </i>
+  movement) and
+ </ins>
+ took an increasingly hard line against Communist
+ <ins>
+  regimes in
+  <a href="/wiki/Afghanistan" title="Afghanistan">
+   Afghanistan
+  </a>
+  ,
+  <a href="/wiki/Angola" title="Angola">
+   Angola
+  </a>
+  ,
+  <a href="/wiki/Cambodia" title="Cambodia">
+   Cambodia
+  </a>
+  , and
+  <a href="/wiki/Nicaragua" title="Nicaragua">
+   Nicaragua
+  </a>
+  .
+ </ins>
+ <del>
+  influences in Latin America, which often involved the controversial support of anti-Communist
+  <a href="/wiki/Military_dictatorship" title="Military dictatorship">
+   military dictatorships
+  </a>
+  with poor
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  records.
+ </del>
+ This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
@@ -796,0 +1299,3 @@
+ <ins>
+  ,
+ </ins>
@@ -801 +1306,23 @@
- after it underwent a Communist coup. Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
+ after it underwent a Communist coup.
+ <ins>
+  In
+  <a href="/wiki/1986" title="1986">
+   1986
+  </a>
+  , representing the global promise he felt was inherent in the success of the Reagan Doctrine, Reagan invited anti-communist Angolan resistance leader
+  <a href="/wiki/Jonas_Savimbi" title="Jonas Savimbi">
+   Jonas Savimbi
+  </a>
+  to The White House, where he spoke of Savimbi winning "a victory that electrifies the world." The Reagan Doctrine was vigorously supported by influential foreign policy analysts at The
+  <a class="mw-redirect" href="/wiki/Heritage_Foundation" title="Heritage Foundation">
+   Heritage Foundation
+  </a>
+  and other
+  <a class="mw-redirect" href="/wiki/Conservatives" title="Conservatives">
+   conservatives
+  </a>
+  , leading to the flow of American weapons to anti-communist resistance movements on several continents.
+ </ins>
+</p>
+<p>
+ Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
@@ -808,2 +1335,8 @@
- <span class="mw-headline" id="Iran-Contra_Affair">
-  Iran-Contra Affair
+ <span class="mw-headline" id="Iran-Contra_affair">
+  Iran-Contra
+  <ins>
+   affair
+  </ins>
+  <del>
+   Affair
+  </del>
@@ -848 +1381,4 @@
- terrorist group in
+ <del>
+  terrorist group
+ </del>
+ in
@@ -856,3 +1391,0 @@
-  <a class="image" href="/wiki/File:Pres_Ronald_Reagan_on_white_horse.jpg">
-   <img alt="" class="thumbimage" data-file-height="835" data-file-width="550" height="334" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/220px-Pres_Ronald_Reagan_on_white_horse.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/330px-Pres_Ronald_Reagan_on_white_horse.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/440px-Pres_Ronald_Reagan_on_white_horse.jpg 2x" width="220"/>
-  </a>
@@ -861,2 +1393,0 @@
-    <a class="internal" href="/wiki/File:Pres_Ronald_Reagan_on_white_horse.jpg" title="Enlarge">
-    </a>
@@ -864 +1395,3 @@
-   President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   <del>
+    President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   </del>
@@ -873 +1406,11 @@
- to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led Congressman Patricia Schroeder to dub him the "
+ to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led
+ <ins>
+  Representative
+  <a href="/wiki/Patricia_Schroeder" title="Patricia Schroeder">
+   Patricia Schroeder
+  </a>
+ </ins>
+ <del>
+  Congressman Patricia Schroeder
+ </del>
+ to dub him the "
@@ -893 +1436,13 @@
- for drug offenders while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's prison population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless, surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
+ for
+ <a href="/wiki/Illegal_drug_trade" title="Illegal drug trade">
+  drug offenders
+ </a>
+ while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's
+ <a href="/wiki/Prison" title="Prison">
+  prison
+ </a>
+ population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless,
+ <ins>
+  some
+ </ins>
+ surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
@@ -895 +1450 @@
-<h3>
+<h2>
@@ -899 +1454,25 @@
-</h3>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan was a Chicago Cubs radio broadcaster in the mid-1930s for an Iowa station. He made several guest trips back to the radio booth, both during and after his presidency.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  During his California governorship, Reagan actively dismantled the public
+  <a href="/wiki/Psychiatric_hospital" title="Psychiatric hospital">
+   psychiatric hospital
+  </a>
+  system, proposing that it be replaced by supportive community-based housing and treatment. The first objective was effectively accomplished, but the community replacement facilities were never adequately funded, neither by Reagan nor his successors, contributing to current California problems with
+  <a href="/wiki/Homelessness" title="Homelessness">
+   homeless
+  </a>
+  people, and an overfilling of jails and penitentiaries by people who would be better served with the earlier hospital system.
+ </ins>
+</p>
@@ -925 +1504,7 @@
-  US Marines
+  <ins>
+   U.S.
+  </ins>
+  <del>
+   US
+  </del>
+  Marines
@@ -931,9 +1516,13 @@
- . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines. Two days later Reagan invaded the tiny
- <a href="/wiki/Caribbean" title="Caribbean">
-  Caribbean
- </a>
- nation of
- <a href="/wiki/Grenada" title="Grenada">
-  Grenada
- </a>
- . Three months later, Reagan withdrew the Marines from Lebanon.
+ . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines.
+ <del>
+  Two days later Reagan invaded the tiny
+  <a href="/wiki/Caribbean" title="Caribbean">
+   Caribbean
+  </a>
+  nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  .
+ </del>
+ Three months later, Reagan withdrew the Marines from Lebanon.
@@ -975,4 +1564,19 @@
- <a href="/wiki/Prostate" title="Prostate">
-  prostate
- </a>
- surgery which caused further worries about his health.
+ <ins>
+  surgery for
+  <a href="/wiki/Prostate_cancer" title="Prostate cancer">
+   prostate cancer
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/Prostate" title="Prostate">
+   prostate
+  </a>
+  surgery
+ </del>
+ which caused further worries about his
+ <ins>
+  health, but which significantly raised the public awareness of this "silent killer."
+ </ins>
+ <del>
+  health.
+ </del>
@@ -1024,0 +1629,7 @@
+<h2>
+ <ins>
+  <span class="mw-headline" id="Legacy_and_retirement_from_public_life">
+   Legacy and retirement from public life
+  </span>
+ </ins>
+</h2>
@@ -1031 +1642,14 @@
-   (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential Library.
+   (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential
+   <ins>
+    Library, 1991.
+   </ins>
+   <del>
+    Library.
+   </del>
+   <h2>
+    <del>
+     <span class="mw-headline" id="Legacy_and_retirement_from_public_life">
+      Legacy and retirement from public life
+     </span>
+    </del>
+   </h2>
@@ -1035,5 +1658,0 @@
-<h2>
- <span class="mw-headline" id="Legacy_and_retirement_from_public_life">
-  Legacy and retirement from public life
- </span>
-</h2>
@@ -1084 +1703,15 @@
- . As the years went on, the disease began to slowly take over the former President's brain and body, forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
+ . As the years went on, the disease
+ <ins>
+  slowly destroyed
+ </ins>
+ <del>
+  began to slowly take over
+ </del>
+ the former President's
+ <ins>
+  mental capacity,
+ </ins>
+ <del>
+  brain and body,
+ </del>
+ forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
@@ -1092 +1725,9 @@
- in the form of a personal letter. His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
+ in the form of a personal letter.
+ <ins>
+  A tragic anecdote told of this time is of his removing from a friend's aquarium a ceramic model of the
+  <a href="/wiki/White_House" title="White House">
+   White House
+  </a>
+  . It is said that he said "I know this is important, but I don't know why."
+ </ins>
+ His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
@@ -1096 +1737,8 @@
- Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was rumoured, although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
+ Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was
+ <ins>
+  rumored,
+ </ins>
+ <del>
+  rumoured,
+ </del>
+ although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
@@ -1164,0 +1813,74 @@
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:202px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Ronald Reagan died at his home in
+  <a class="mw-redirect" href="/wiki/Bel_Air,_California" title="Bel Air, California">
+   Bel Air, California
+  </a>
+  on
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+  at 1:09 PM local (Pacific) time. He died of
+  <a href="/wiki/Pneumonia" title="Pneumonia">
+   pneumonia
+  </a>
+  with his wife
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy
+  </a>
+  and their children
+  <a href="/wiki/Patti_Davis" title="Patti Davis">
+   Patti
+  </a>
+  and
+  <a href="/wiki/Ron_Reagan" title="Ron Reagan">
+   Ron
+  </a>
+  , present. He is survived also by his son
+  <a href="/wiki/Michael_Reagan" title="Michael Reagan">
+   Michael
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  He will be given a full presidential
+  <a href="/wiki/State_funeral" title="State funeral">
+   state funeral
+  </a>
+  . Services at the
+  <a class="mw-redirect" href="/wiki/National_Cathedral" title="National Cathedral">
+   National Cathedral
+  </a>
+  on
+  <a href="/wiki/June_11" title="June 11">
+   June 11
+  </a>
+  will include eulogies by President
+  <a href="/wiki/George_W._Bush" title="George W. Bush">
+   George W. Bush
+  </a>
+  , former President George H.W. Bush, former British Prime Minister Margaret Thatcher and former Canadian Prime Minister Brian Mulroney.
+ </ins>
+</p>
+<ins>
+ Reagan holds the record for the longest-living President in American history, the previous record being
+</ins>
+<del>
@@ -1169 +1891,3 @@
- , Reagan turned 93, continuing his record as the longest-living president in American history. The previous record had been 90 years and 247 days, held by
+ , Reagan turned 93, continuing his record as the longest-living president in American history. The previous record had been
+</del>
+ 90 years and 247 days, held by
@@ -1171,2 +1895,2 @@
-  John Adams
- </a>
+ John Adams
+</a>
@@ -1174,3 +1898,4 @@
- <a href="/wiki/October_12" title="October 12">
-  October 12
- </a>
+ <a href="/wiki/October_11" title="October 11">
+ October 11
+  Link: /wiki/October_11 12
+</a>
@@ -1179,3 +1904,656 @@
-  2001
- </a>
- . In June of 2004 the
+ 2001
+</a>
+ .
+ <p>
+ <ins>
+  Both opponents and supporters noted his "sunny optimism," which was welcomed by many in comparison to his somewhat dour and serious (although often smiling) presidential predecessor.
+ </ins>
+</p>
+<p>
+ <ins>
+  His style of relating to others had often been described as avuncular (in the demeanor of an uncle, one not responsible for discipline but who can provide well meaning guidance).
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Quotations">
+   Quotations
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   "Mister Gorbachev, tear down this
+   <a href="/wiki/Berlin_Wall" title="Berlin Wall">
+    wall
+   </a>
+   !"
+  </ins>
+ </li>
+</ul>
+<ul>
+ <li>
+  <ins>
+   "My fellow Americans, I am pleased to tell you today that I have signed legislation that will outlaw Russia forever. We begin bombing in five minutes." (joke overheard during a microphone test)
+  </ins>
+ </li>
+</ul>
+<ul>
+ <li>
+  <ins>
+   "How do you tell a communist? Well, it's someone who reads
+   <a class="mw-redirect" href="/wiki/Marx" title="Marx">
+    Marx
+   </a>
+   and
+   <a class="mw-redirect" href="/wiki/Lenin" title="Lenin">
+    Lenin
+   </a>
+   . And how do you tell an anti-communist? It's someone who understands Marx and Lenin." From www.ronaldreagan.com
+  </ins>
+ </li>
+</ul>
+<ul>
+ <li>
+  <ins>
+   "One hundred nations in the UN have not agreed with us on just about everything that's come before them, where we're involved, and it didn't upset my breakfast at all." (reacting to international criticism of the
+   <a class="mw-redirect" href="/wiki/Operation_Urgent_Fury" title="Operation Urgent Fury">
+    Operation Urgent Fury
+   </a>
+   , 1983, quoted in
+   <i>
+    Killing Hope
+   </i>
+   by William Blum)
+  </ins>
+ </li>
+</ul>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Criticism_of_Reagan">
+   Criticism of Reagan
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagancoffin.jpg" title="File:Reagancoffin.jpg">
+    File:Reagancoffin.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy Reagan lays her head upon the coffin of President Reagan at the Reagan Presidential Library.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Many people criticise Reagan's record in office, not just on political grounds but also on moral and legal grounds. For example,
+  <a href="/wiki/Noam_Chomsky" title="Noam Chomsky">
+   Noam Chomsky
+  </a>
+  in works including
+  <i>
+   <a class="new" href="/w/index.php?title=Pirates_and_Emperors,_Old_and_New&amp;action=edit&amp;redlink=1" title="Pirates and Emperors, Old and New (page does not exist)">
+    Pirates and Emperors, Old and New
+   </a>
+  </i>
+  . During the 1980s the Reagan administration sponsored a campaign of political violence by
+  <a href="/wiki/Contras" title="Contras">
+   Contras
+  </a>
+  (based in
+  <a href="/wiki/Honduras" title="Honduras">
+   Honduras
+  </a>
+  ) against the left-wing
+  <a class="mw-redirect" href="/wiki/Sandinista" title="Sandinista">
+   Sandinista
+  </a>
+  government in
+  <a href="/wiki/Nicaragua" title="Nicaragua">
+   Nicaragua
+  </a>
+  (who won that country first democratic elections in 1984). This insurgency killed thousands of people. The Sandinstas had previously overthrown the US-backed right-wing
+  <a href="/wiki/Military" title="Military">
+   military
+  </a>
+  <a href="/wiki/Dictatorship" title="Dictatorship">
+   dictatorship
+  </a>
+  of the
+  <a class="mw-redirect" href="/wiki/Somoza" title="Somoza">
+   Somoza
+  </a>
+  family. The Reagan administration is also accused of sponsoring right-wing and fascist
+  <a class="mw-redirect" href="/wiki/Military_dictatorships" title="Military dictatorships">
+   military dictatorships
+  </a>
+  throughout Latin America, and the
+  <a class="mw-redirect" href="/wiki/CIA" title="CIA">
+   CIA
+  </a>
+  and US-based
+  <a class="mw-redirect" href="/wiki/School_of_the_Americas" title="School of the Americas">
+   School of the Americas
+  </a>
+  are accused of training
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  abusers such as Honduran death squads in assassination and torture techniques.
+ </ins>
+</p>
+<p>
+ <ins>
+  Opponents of Reagan argue that this constituted state sponsorship of
+  <a href="/wiki/Terrorism" title="Terrorism">
+   terrorism
+  </a>
+  and an attempt to overthrow a democratically elected government--indeed the
+  <a class="mw-redirect" href="/wiki/World_Court" title="World Court">
+   World Court
+  </a>
+  sanctioned the US for "unlawful use of force" for "sponsoring paramilitary activity in and against Nicaragua", ordering the US government to pay billions of US dollars in compensation (the US refuses to this day).
+ </ins>
+</p>
+<p>
+ <ins>
+  The
+  <a href="/wiki/Contras" title="Contras">
+   Contras
+  </a>
+  had an appalling
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  record, and were notorious for attacking farms and other civilian targets, as well as murder, torture and mutilation of
+  <a class="mw-redirect" href="/wiki/Civilians" title="Civilians">
+   civilians
+  </a>
+  and other
+  <a class="mw-redirect" href="/wiki/War_crimes" title="War crimes">
+   war crimes
+  </a>
+  . The
+  <a href="/wiki/Contras" title="Contras">
+   Contras
+  </a>
+  were also involved in illicit
+  <a href="/wiki/Drug" title="Drug">
+   drug
+  </a>
+  -trafficking. Chomsky and others have accused the US government of inciting the
+  <a href="/wiki/Contras" title="Contras">
+   Contras
+  </a>
+  to attack civilians, and providing them with the positions of the Nicaraguan Army to avoid engagement with the security forces.
+ </ins>
+</p>
+<p>
+ <ins>
+  The Reagan administration also sponsored
+  <a href="/wiki/Iraq" title="Iraq">
+   Iraqi
+  </a>
+  dictator Saddam Hussein's war against the Islamic Republic of
+  <a href="/wiki/Iran" title="Iran">
+   Iran
+  </a>
+  , which the US now considers an "aggressive war". The US and its allies gave weapons and tactical support to Iraq, including illegal
+  <a class="mw-redirect" href="/wiki/Chemical" title="Chemical">
+   chemical
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Biological" title="Biological">
+   biological
+  </a>
+  weapons which were used against Iranian
+  <a class="mw-redirect" href="/wiki/Conscripts" title="Conscripts">
+   conscripts
+  </a>
+  and Kurdish
+  <a class="mw-redirect" href="/wiki/Guerrillas" title="Guerrillas">
+   guerrillas
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Civilians" title="Civilians">
+   civilians
+  </a>
+  , e.g. at
+  <a href="/wiki/Halabja" title="Halabja">
+   Halabja
+  </a>
+  . Several commentators have argued that
+  <a href="/wiki/Iraq" title="Iraq">
+   Iraq
+  </a>
+  could not possibly have invaded
+  <a href="/wiki/Kuwait" title="Kuwait">
+   Kuwait
+  </a>
+  in
+  <a href="/wiki/1991" title="1991">
+   1991
+  </a>
+  if not for the weapons Saddam received from the US and its allies.
+ </ins>
+</p>
+<p>
+ <ins>
+  Some accuse the Reagan administration of indirectly creating
+  <a class="mw-redirect" href="/wiki/Al_Qaeda" title="Al Qaeda">
+   Al Qaeda
+  </a>
+  by sponsoring Islamist
+  <a href="/wiki/Mujahideen" title="Mujahideen">
+   mujahideen
+  </a>
+  guerrillas (including many brutal Islamic extremists who became the
+  <a href="/wiki/Taliban" title="Taliban">
+   Taliban
+  </a>
+  ) who fought the
+  <a class="mw-redirect" href="/wiki/Soviet" title="Soviet">
+   Soviet
+  </a>
+  occupation of Afghanistan, committing many
+  <a class="mw-redirect" href="/wiki/War_crimes" title="War crimes">
+   war crimes
+  </a>
+  . It has been alleged that
+  <a href="/wiki/Osama_bin_Laden" title="Osama bin Laden">
+   Osama bin Laden
+  </a>
+  , the
+  <a class="mw-redirect" href="/wiki/Al_Qaeda" title="Al Qaeda">
+   Al Qaeda
+  </a>
+  leader, was trained by the CIA or an allied intelligence agency, and that while fighting the Soviets bin Laden committed numerous war crimes such as destroying a unisex school full of children for being "un-Islamic".
+ </ins>
+</p>
+<p>
+ <ins>
+  Although the US government initially maintained that support to the
+  <a href="/wiki/Mujahideen" title="Mujahideen">
+   mujahideen
+  </a>
+  after the Soviets invaded, a former CIA officer revealed that support to the
+  <a href="/wiki/Mujahideen" title="Mujahideen">
+   mujahideen
+  </a>
+  began in 1979, when they were fighting the Soviet-backed government. The US government apparently had the intention of provoking a Soviet invasion. Compared to the Soviet occupation and the anarchy and warlords that ruled Afghanistan during the civil war following the Soviet withdrawal, the Soviet-backed government was said by many to be much better. Women had many more rights than under the
+  <a href="/wiki/Taliban" title="Taliban">
+   Taliban
+  </a>
+  , for example.
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan considered the
+  <a href="/wiki/Contras" title="Contras">
+   Contras
+  </a>
+  and Afghan
+  <a href="/wiki/Mujahideen" title="Mujahideen">
+   mujahideen
+  </a>
+  to be
+  <a class="mw-redirect mw-disambig" href="/wiki/Freedom_fighters" title="Freedom fighters">
+   freedom fighters
+  </a>
+  and the "moral equivalent of our [America's] founding fathers" fighting against Communism. In contrast he considered
+  <a class="mw-redirect" href="/wiki/Hezbullah" title="Hezbullah">
+   Hezbullah
+  </a>
+  guerrillas in
+  <a href="/wiki/Lebanon" title="Lebanon">
+   Lebanon
+  </a>
+  and
+  <a class="mw-disambig" href="/wiki/Palestinian" title="Palestinian">
+   Palestinian
+  </a>
+  guerrillas in the
+  <a href="/wiki/West_Bank" title="West Bank">
+   West Bank
+  </a>
+  and
+  <a href="/wiki/Gaza_Strip" title="Gaza Strip">
+   Gaza Strip
+  </a>
+  resisting
+  <a class="mw-redirect" href="/wiki/Israeli" title="Israeli">
+   Israeli
+  </a>
+  military occupation, and left-wing guerrillas fighting a right-wing
+  <a href="/wiki/Military_dictatorship" title="Military dictatorship">
+   military dictatorship
+  </a>
+  in Honduras to be
+  <a class="mw-redirect" href="/wiki/Terrorists" title="Terrorists">
+   terrorists
+  </a>
+  . The Reagan administration also considered guerrillas of the
+  <a class="mw-redirect" href="/wiki/ANC" title="ANC">
+   ANC
+  </a>
+  's armed wing Mkihonto we-Sizwe (MK or Spear of the Nation) and others (e.g. the
+  <a class="mw-disambig" href="/wiki/PAC" title="PAC">
+   PAC
+  </a>
+  ) fighting the
+  <a href="/wiki/Apartheid" title="Apartheid">
+   apartheid
+  </a>
+  government in
+  <a href="/wiki/South_Africa" title="South Africa">
+   South Africa
+  </a>
+  to be
+  <a class="mw-redirect" href="/wiki/Terrorists" title="Terrorists">
+   terrorists
+  </a>
+  , despite many people throughout the world (most likely including the black majority in South Africa) considered the
+  <a class="mw-redirect mw-disambig" href="/wiki/Freedom_fighters" title="Freedom fighters">
+   freedom fighters
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Reagan has also been accused of
+  <a class="mw-redirect" href="/wiki/War_crimes" title="War crimes">
+   war crimes
+  </a>
+  and
+  <a href="/wiki/State_terrorism" title="State terrorism">
+   state terrorism
+  </a>
+  for bombing residential areas in Tripoli, Lebanon. An assassination attempt was also made on Libyan dictator Col.
+  <a class="mw-redirect" href="/wiki/Gaddafi" title="Gaddafi">
+   Gaddafi
+  </a>
+  (also spelt Qaddafi or Kadafi) during the air strikes. Instead his one-year-old daughter was killed. A letter from a seven-year-old female victim of the bombing was found in the ruins of a house belonging to an American-educated family, by ABC correspondent Charles Glass. It read:
+ </ins>
+</p>
+<p>
+ <ins>
+  "Dear Mr Reagan
+ </ins>
+</p>
+<p>
+ <ins>
+  Why did you kill my only sister Rafa and my friend Racha, she is only nine, and my baby doll Strwberry. Is it true you want to kill us all because my father is Palestinian and you want to kill Kadafi because he wants to help us go back to my father's home and land.
+ </ins>
+</p>
+<p>
+ <ins>
+  My name is Kinda."
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Cabinet">
+   Cabinet
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan and his Cabinet in the Oval Office (Feb. 4, 1981)
+   </ins>
+  </div>
+ </div>
+</div>
+<ul>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_State" title="United States Secretary of State">
+    Secretary of State
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Alexander_M._Haig" title="Alexander M. Haig">
+    Alexander M. Haig
+   </a>
+   (1981-82),
+   <a href="/wiki/George_P._Shultz" title="George P. Shultz">
+    George P. Shultz
+   </a>
+   (1982-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Defense" title="United States Secretary of Defense">
+    Secretary of Defense
+   </a>
+   -
+   <a href="/wiki/Caspar_Weinberger" title="Caspar Weinberger">
+    Caspar Weinberger
+   </a>
+   (1981-87),
+   <a class="mw-redirect" href="/wiki/Frank_C._Carlucci" title="Frank C. Carlucci">
+    Frank C. Carlucci
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_the_Treasury" title="United States Secretary of the Treasury">
+    Secretary of the Treasury
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Donald_T._Regan" title="Donald T. Regan">
+    Donald T. Regan
+   </a>
+   (1981-1985),
+   <a href="/wiki/James_Baker" title="James Baker">
+    James Baker
+   </a>
+   (1985-88),
+   <a href="/wiki/Nicholas_F._Brady" title="Nicholas F. Brady">
+    Nicholas F. Brady
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Attorney_General" title="United States Attorney General">
+    Attorney General
+   </a>
+   -
+   <a href="/wiki/William_French_Smith" title="William French Smith">
+    William French Smith
+   </a>
+   (1981-85),
+   <a href="/wiki/Edwin_Meese" title="Edwin Meese">
+    Edwin Meese III
+   </a>
+   (1985-88),
+   <a class="mw-redirect" href="/wiki/Richard_L._Thornburgh" title="Richard L. Thornburgh">
+    Richard L. Thornburgh
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_the_Interior" title="United States Secretary of the Interior">
+    Secretary of the Interior
+   </a>
+   -
+   <a href="/wiki/James_G._Watt" title="James G. Watt">
+    James G. Watt
+   </a>
+   (1981-83),
+   <a class="mw-redirect" href="/wiki/William_P._Clark" title="William P. Clark">
+    William P. Clark
+   </a>
+   (1983-85),
+   <a href="/wiki/Donald_P._Hodel" title="Donald P. Hodel">
+    Donald P. Hodel
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Agriculture" title="United States Secretary of Agriculture">
+    Secretary of Agriculture
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/John_R._Block" title="John R. Block">
+    John R. Block
+   </a>
+   (1981-86),
+   <a class="mw-redirect" href="/wiki/Richard_E._Lyng" title="Richard E. Lyng">
+    Richard E. Lyng
+   </a>
+   (1986-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Commerce" title="United States Secretary of Commerce">
+    Secretary of Commerce
+   </a>
+   -
+   <a class="mw-disambig" href="/wiki/Malcolm_Baldrige" title="Malcolm Baldrige">
+    Malcolm Baldrige
+   </a>
+   (1981-87),
+   <a class="mw-redirect" href="/wiki/C._William_Verity" title="C. William Verity">
+    C. William Verity
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Labor" title="United States Secretary of Labor">
+    Secretary of Labor
+   </a>
+   -
+   <a href="/wiki/Raymond_J._Donovan" title="Raymond J. Donovan">
+    Raymond J. Donovan
+   </a>
+   (1981-85),
+   <a class="mw-redirect" href="/wiki/William_E._Brock" title="William E. Brock">
+    William E. Brock
+   </a>
+   (1985-87),
+   <a class="mw-redirect" href="/wiki/Ann_Dore_McLaughlin" title="Ann Dore McLaughlin">
+    Ann Dore McLaughlin
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Health_and_Human_Services" title="United States Secretary of Health and Human Services">
+    Secretary of Health and Human Services
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Richard_S._Schweiker" title="Richard S. Schweiker">
+    Richard S. Schweiker
+   </a>
+   (1981-83),
+   <a class="mw-redirect" href="/wiki/Margaret_M._Heckler" title="Margaret M. Heckler">
+    Margaret M. Heckler
+   </a>
+   (1983-85),
+   <a href="/wiki/Otis_R._Bowen" title="Otis R. Bowen">
+    Otis R. Bowen
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Housing_and_Urban_Development" title="United States Secretary of Housing and Urban Development">
+    Secretary of Housing and Urban Development
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Samuel_R._Pierce" title="Samuel R. Pierce">
+    Samuel R. Pierce
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Transportation" title="United States Secretary of Transportation">
+    Secretary of Transportation
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Andrew_L._Lewis,_Jr." title="Andrew L. Lewis, Jr.">
+    Andrew L. Lewis, Jr.
+   </a>
+   (1981-83),
+   <a href="/wiki/Elizabeth_Dole" title="Elizabeth Dole">
+    Elizabeth Dole
+   </a>
+   (1983-87),
+   <a href="/wiki/James_H._Burnley_IV" title="James H. Burnley IV">
+    James H. Burnley IV
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Energy" title="United States Secretary of Energy">
+    Secretary of Energy
+   </a>
+   -
+   <a href="/wiki/James_B._Edwards" title="James B. Edwards">
+    James B. Edwards
+   </a>
+   (1981-82),
+   <a href="/wiki/Donald_P._Hodel" title="Donald P. Hodel">
+    Donald P. Hodel
+   </a>
+   (1982-85),
+   <a href="/wiki/John_S._Herrington" title="John S. Herrington">
+    John S. Herrington
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Education" title="United States Secretary of Education">
+    Secretary of Education
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Terrel_H._Bell" title="Terrel H. Bell">
+    Terrel H. Bell
+   </a>
+   (1981-85),
+   <a class="mw-redirect" href="/wiki/William_J._Bennett" title="William J. Bennett">
+    William J. Bennett
+   </a>
+   (1985-88),
+   <a class="mw-redirect" href="/wiki/Lauro_F._Cavazos" title="Lauro F. Cavazos">
+    Lauro F. Cavazos
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+</ul>
+<del>
+ In June of 2004 the
+</del>
+<del>
@@ -1186 +2564 @@
-</p>
+</del>
@@ -1188,3 +2566,5 @@
- <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
-  Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
- </a>
+ <del>
+  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
+   Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
+  </a>
+ </del>
@@ -1193,5 +2573,7 @@
- On
- <a href="/wiki/June_5" title="June 5">
-  June 5
- </a>
- , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+ <del>
+  On
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+ </del>
@@ -1206,0 +2589,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:202px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan and Sandra Day O'Connor at the White House in 1981.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1249 +2642 @@
-</ul>
+</ul>
\ No newline at end of file
