--- 3955334
+++ 4008865
@@ -2 +1,0 @@
- <br/>
@@ -6 +5,6 @@
- (born
+ <ins>
+  (
+ </ins>
+ <del>
+  (born
+ </del>
@@ -13,0 +18,10 @@
+ <ins>
+  –
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+ </ins>
@@ -18 +32,6 @@
- -
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
@@ -46,5 +65,15 @@
- before entering politics. He is the longest-lived person to have served as President, as well as the oldest elected President (69 years, 349 days). He is the only
- <a href="/wiki/Divorce" title="Divorce">
-  divorced
- </a>
- person to be elected President, and the only President to have married a pregnant woman.
+ before entering politics. He
+ <ins>
+  lived longer than any other President (93 years, 119 days) and was
+ </ins>
+ <del>
+  is the longest-lived person to have served as President, as well as
+ </del>
+ the oldest elected President (69 years, 349 days).
+ <del>
+  He is the only
+  <a href="/wiki/Divorce" title="Divorce">
+   divorced
+  </a>
+  person to be elected President, and the only President to have married a pregnant woman.
+ </del>
@@ -95,0 +125,14 @@
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan_family.jpg" title="File:Reagan family.jpg">
+    File:Reagan family.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Ronald and Neil Reagan (front row), Parents Jack and Nelle Reagan (c. 1916-17)
+   </ins>
+  </div>
+ </div>
+</div>
@@ -106 +149 @@
- <a class="mw-disambig" href="/wiki/Rock_River" title="Rock River">
+ <a class="mw-redirect" href="/wiki/Rock_River_(Illinois)" title="Rock River (Illinois)">
@@ -124 +167,7 @@
- , graduating in
+ <ins>
+  majoring in economics and sociology,
+ </ins>
+ <del>
+  ,
+ </del>
+ graduating in
@@ -128 +177,8 @@
- . Child of an
+ .
+ <ins>
+  The child
+ </ins>
+ <del>
+  Child
+ </del>
+ of an
@@ -151,4 +207,4 @@
- games, getting only the bare outlines of the game from a ticker and relying on his
- <a href="/wiki/Imagination" title="Imagination">
-  imagination
- </a>
+ games, getting only the bare outlines of the game from a ticker and relying on his imagination
+ <del>
+  Link: /wiki/Imagination
+ </del>
@@ -163,4 +219,4 @@
- game, the wire went dead. Reagan smoothly
- <a class="mw-redirect" href="/wiki/Improvise" title="Improvise">
-  improvised
- </a>
+ game, the wire went dead. Reagan smoothly improvised
+ <del>
+  Link: /wiki/Improvise
+ </del>
@@ -174 +230,8 @@
- as a second-rank leading man, as his face and body were as handsome as his voice. His first screen credit was the starring role the
+ as a second-rank leading man,
+ <ins>
+  aided by his clear voice and athletic physique.
+ </ins>
+ <del>
+  as his face and body were as handsome as his voice.
+ </del>
+ His first screen credit was the starring role the
@@ -192 +255,5 @@
- he played the role of George "The Gipper" Gipp in the film
+ he played the role of
+ <a href="/wiki/George_Gipp" title="George Gipp">
+  George "The Gipper" Gipp
+ </a>
+ in the film
@@ -244 +311,5 @@
- officer in the U.S. Army in
+ officer in the
+ <a class="mw-redirect" href="/wiki/U.S._Army" title="U.S. Army">
+  U.S. Army
+ </a>
+ in
@@ -252,8 +323,13 @@
- he was activated and was assigned to the First Motion Picture Unit in the
- <a class="mw-disambig" href="/wiki/Army_Air_Corps" title="Army Air Corps">
-  Army Air Corps
- </a>
- , which made training and education films. He remained in
- <a href="/wiki/Hollywood" title="Hollywood">
-  Hollywood
- </a>
+ he was activated and was assigned to the
+ <a href="/wiki/First_Motion_Picture_Unit" title="First Motion Picture Unit">
+  First Motion Picture Unit
+ </a>
+ in the
+ <a href="/wiki/United_States_Air_Force" title="United States Air Force">
+  Army Air Force
+  Link: /wiki/United_States_Air_Force Corps
+ </a>
+ , which made training and education films. He remained in Hollywood
+ <del>
+  Link: /wiki/Hollywood
+ </del>
@@ -261,0 +338,43 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Reagan married actress
+  <a href="/wiki/Jane_Wyman" title="Jane Wyman">
+   Jane Wyman
+  </a>
+  in 1940. They had a daughter,
+  <a href="/wiki/Maureen_Reagan" title="Maureen Reagan">
+   Maureen
+  </a>
+  in 1941, adopted a son
+  <a href="/wiki/Michael_Reagan" title="Michael Reagan">
+   Michael
+  </a>
+  in 1946, and had a daughter born four months prematurely in 1947 who lived but one day. They divorced in 1948, making Reagan the only President who has been divorced. Reagan remarried in 1952 to actress
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy Davis
+  </a>
+  ; they had two children
+  <a href="/wiki/Ron_Reagan" title="Ron Reagan">
+   Ron
+  </a>
+  and
+  <a class="mw-redirect" href="/wiki/Patti_Reagan" title="Patti Reagan">
+   Patti
+  </a>
+  . Nancy was pregnant during the wedding, making Reagan the only President to have married a pregnant woman.
+ </ins>
+</p>
@@ -292,3 +411,5 @@
-  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
-   File:Nancy and Ronald Reagan.jpg
-  </a>
+  <del>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Nancy_and_Ronald_Reagan.jpg" title="File:Nancy and Ronald Reagan.jpg">
+    File:Nancy and Ronald Reagan.jpg
+   </a>
+  </del>
@@ -296 +417,3 @@
-   Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   <del>
+    Nancy and Ronald Reagan married in 1952. Nancy became a powerful background figure in Reagan's rise and roles as governor and president.
+   </del>
@@ -302,3 +425,5 @@
- <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
-  liberal
- </a>
+ <del>
+  <a class="mw-redirect" href="/wiki/Political_liberalism#United_States" title="Political liberalism">
+   liberal
+  </a>
+ </del>
@@ -324,5 +449,11 @@
- (SAG), aligning himself with Senator
- <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
-  Joseph McCarthy
- </a>
- and cooperating with the
+ <ins>
+  (SAG) from 1947 until 1952, and then again from 1959 to 1960. In this position he testified before
+ </ins>
+ <del>
+  (SAG), aligning himself with Senator
+  <a href="/wiki/Joseph_McCarthy" title="Joseph McCarthy">
+   Joseph McCarthy
+  </a>
+  and cooperating with
+ </del>
+ the
@@ -332,5 +463,24 @@
- to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
- <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
-  American Communist Party
- </a>
- . His employment by the
+ <ins>
+  on
+  <a href="/wiki/Communism" title="Communism">
+   Communist
+  </a>
+  influence in
+  <a href="/wiki/Cinema_of_the_United_States" title="Cinema of the United States">
+   Hollywood
+  </a>
+  and later supported
+  <a class="mw-redirect" href="/wiki/Blacklist" title="Blacklist">
+   blacklisting
+  </a>
+ </ins>
+ <del>
+  to "expose Communist influence in Hollywood". He turned in several of his allegedly Communist co-workers, although unlike many anti-Communists of the time he was strongly opposed to the formal banning of the
+  <a class="mw-redirect" href="/wiki/American_Communist_Party" title="American Communist Party">
+   American Communist Party
+  </a>
+ </del>
+ .
+</p>
+<p>
+ His employment by the
@@ -340,5 +490,18 @@
- company, delivering
- <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
-  anti-communist
- </a>
- speeches on radio broadcasts and speaking tours, further enhanced his political image in the anti-Soviet climate of 1950s America. By the
+ <ins>
+  company
+ </ins>
+ <del>
+  company, delivering
+  <a class="mw-redirect" href="/wiki/Anti-communist" title="Anti-communist">
+   anti-communist
+  </a>
+  speeches on radio broadcasts and speaking tours,
+ </del>
+ further enhanced his political
+ <ins>
+  image.
+ </ins>
+ <del>
+  image in the anti-Soviet climate of 1950s America.
+ </del>
+ By the
@@ -351,0 +515,10 @@
+ <ins>
+  . His nationally televised speech
+  <a href="/wiki/A_Time_for_Choosing" title="A Time for Choosing">
+   A Time for Choosing
+  </a>
+  electrified conservatives and led to his being asked to run for
+  <a href="/wiki/Governor_of_California" title="Governor of California">
+   Governor of California
+  </a>
+ </ins>
@@ -379 +552,5 @@
- . Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election. Most analysts believe President
+ .
+ <del>
+  Overseas press charged that the Reagan camp had made a secret deal to keep the hostages imprisoned until after the election.
+ </del>
+ Most analysts believe President
@@ -387,0 +565,9 @@
+ <ins>
+  Much of his first election and this second term landslide is attributed to the then named "
+ </ins>
+ <ins>
+  <a href="/wiki/Reagan_Democrat" title="Reagan Democrat">
+   Reagan Democrats
+  </a>
+  ", a newly emerged but mostly unorganized political force.
+ </ins>
@@ -390,3 +576,26 @@
- <span class="mw-headline" id="Presidency">
-  Presidency
- </span>
+ <ins>
+  <span class="mw-headline" id="Presidency">
+   Presidency
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan_assassination.jpg" title="File:Reagan assassination.jpg">
+    File:Reagan assassination.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Chaos outside the Washington Hilton Hotel after the assassination attempt on President Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
+<h2>
+ <del>
+  <span class="mw-headline" id="Presidency">
+   Presidency
+  </span>
+ </del>
@@ -396,3 +604,0 @@
-  <a class="image" href="/wiki/File:Bush_reagan.jpg">
-   <img alt="" class="thumbimage" data-file-height="332" data-file-width="500" height="199" src="//upload.wikimedia.org/wikipedia/commons/thumb/c/ce/Bush_reagan.jpg/300px-Bush_reagan.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/c/ce/Bush_reagan.jpg/450px-Bush_reagan.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/c/ce/Bush_reagan.jpg 2x" width="300"/>
-  </a>
@@ -401,2 +606,0 @@
-    <a class="internal" href="/wiki/File:Bush_reagan.jpg" title="Enlarge">
-    </a>
@@ -404 +608,3 @@
-   Then-Vice President Bush meets with President Reagan in 1984.
+   <del>
+    Then-Vice President Bush meets with President Reagan in 1984.
+   </del>
@@ -480,0 +687,5 @@
+  <ins>
+   in favor of the use of force to protect U.S. interests
+  </ins>
+ </li>
+ <li>
@@ -484 +695,3 @@
-  supportive of some individual liberties
+  <del>
+   supportive of some individual liberties
+  </del>
@@ -492,0 +706,22 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+    File:Reaganhiro.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Emperor
+    <a href="/wiki/Hirohito" title="Hirohito">
+     Hirohito
+    </a>
+    of
+    <a href="/wiki/Japan" title="Japan">
+     Japan
+    </a>
+    meeting with Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -528 +763,5 @@
-  arming and training anti-communist groups like the
+  arming
+  <del>
+   and training anti-communist
+  </del>
+  groups like the
@@ -539 +778 @@
-  <a href="/wiki/Taiwan" title="Taiwan">
+  <a class="mw-redirect" href="/wiki/Republic_of_China" title="Republic of China">
@@ -550,4 +789,16 @@
-  and, ironically,
-  <a href="/wiki/Iraq" title="Iraq">
-   Iraq
-  </a>
+  <ins>
+   ,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+   , and enemies like
+   <a href="/wiki/Iran" title="Iran">
+    Iran
+   </a>
+  </ins>
+  <del>
+   and, ironically,
+   <a href="/wiki/Iraq" title="Iraq">
+    Iraq
+   </a>
+  </del>
@@ -556 +807,4 @@
-  lowering taxes significantly, reversing a historic trend towards higher taxes
+  lowering taxes significantly,
+  <ins>
+   then raising them significantly.
+  </ins>
@@ -559 +813,15 @@
-  increasing the federal deficit
+  <ins>
+   ending the "
+   <a href="/wiki/Cold_War" title="Cold War">
+    Cold War
+   </a>
+   "
+  </ins>
+ </li>
+ <del>
+  reversing a historic trend towards higher taxes
+ </del>
+ <li>
+  <del>
+   increasing the federal deficit
+  </del>
@@ -584,4 +852,6 @@
-  helping to 'win' the
-  <a href="/wiki/Cold_War" title="Cold War">
-   Cold War
-  </a>
+  <del>
+   helping to 'win' the
+   <a href="/wiki/Cold_War" title="Cold War">
+    Cold War
+   </a>
+  </del>
@@ -594,20 +864,7 @@
-  when they illegally
-  <a href="/wiki/Strike_action" title="Strike action">
-   struck
-  </a>
- </li>
-</ul>
-<h3>
- <span class="mw-headline" id="Reaganomics">
-  Reaganomics
- </span>
-</h3>
-<div class="thumb tright">
- <div class="thumbinner" style="width:182px;">
-  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
-   File:Reaganhiro.jpg
-  </a>
-  <div class="thumbcaption">
-   Emperor
-   <a href="/wiki/Hirohito" title="Hirohito">
-    Hirohito
+  when they
+  <ins>
+   went on
+  </ins>
+  <ins>
+   <a href="/wiki/Strike_action" title="Strike action">
+    strike
@@ -615 +872,34 @@
-   meeting with President Reagan.
+  </ins>
+  <h3>
+   <ins>
+    <span class="mw-headline" id="Reaganomics">
+     Reaganomics
+    </span>
+   </ins>
+  </h3>
+  <p>
+   <ins>
+    <i>
+     Main article:
+     <a href="/wiki/Reaganomics" title="Reaganomics">
+      Reaganomics
+     </a>
+    </i>
+   </ins>
+  </p>
+  <div class="thumb tright">
+   <div class="thumbinner" style="width:302px;">
+    <div class="thumbcaption">
+     <div class="magnify">
+     </div>
+     <ins>
+      President Reagan signs the
+     </ins>
+     <ins>
+      <a href="/wiki/Tax_Reform_Act_of_1986" title="Tax Reform Act of 1986">
+       Tax Reform Act of 1986
+      </a>
+      on the South Lawn.
+     </ins>
+    </div>
+   </div>
@@ -617,2 +907,35 @@
- </div>
-</div>
+  <del>
+   illegally
+  </del>
+  <del>
+   <a href="/wiki/Strike_action" title="Strike action">
+    struck
+   </a>
+  </del>
+  <h3>
+   <del>
+    <span class="mw-headline" id="Reaganomics">
+     Reaganomics
+    </span>
+   </del>
+  </h3>
+  <div class="thumb tright">
+   <div class="thumbinner" style="width:182px;">
+    <del>
+     <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reaganhiro.jpg" title="File:Reaganhiro.jpg">
+      File:Reaganhiro.jpg
+     </a>
+    </del>
+    <div class="thumbcaption">
+     <del>
+      Emperor
+      <a href="/wiki/Hirohito" title="Hirohito">
+       Hirohito
+      </a>
+      meeting with President Reagan.
+     </del>
+    </div>
+   </div>
+  </div>
+ </li>
+</ul>
@@ -620,8 +943,24 @@
- Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting stagflation, or both
- <a href="/wiki/Inflation" title="Inflation">
-  inflation
- </a>
- and
- <a href="/wiki/Unemployment" title="Unemployment">
-  unemployment
- </a>
+ Part of President Reagan's first term in office focused on reviving an inherited economy exhibiting
+ <ins>
+  <a href="/wiki/Stagflation" title="Stagflation">
+   stagflation
+  </a>
+  , a high rate of
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  combined with an economic
+  <a href="/wiki/Recession" title="Recession">
+   recession
+  </a>
+ </ins>
+ <del>
+  stagflation, or both
+  <a href="/wiki/Inflation" title="Inflation">
+   inflation
+  </a>
+  and
+  <a href="/wiki/Unemployment" title="Unemployment">
+   unemployment
+  </a>
+ </del>
@@ -636 +975,4 @@
- . Most economists agree that tax cuts stimulate the economy, but supply-siders say that they have a much greater effect than most economists say they would.
+ .
+ <del>
+  Most economists agree that tax cuts stimulate the economy, but supply-siders say that they have a much greater effect than most economists say they would.
+ </del>
@@ -644 +986,5 @@
- " prior to becoming his vice-president. Reagan's policies soon became known as "
+ " prior to becoming his vice-president.
+ <ins>
+  Less than two years in office, Reagan confronted the fiscal irresponsibility of his 1981 tax cuts with rolling back a large portion of his corporate taxes.
+ </ins>
+ Reagan's policies soon became known as "
@@ -648 +994,7 @@
- ", a nickname used by both his supporters and detractors. These tax cuts combined with heavy military spending increases led to enormous
+ ", a nickname used by both his supporters and detractors.
+ <ins>
+  Although total federal revenues increased after Reagan's tax cuts took effect, these fell short of government spending that included both Reagan's military buildup and Congress's expansion of social programs. The result was greater
+ </ins>
+ <del>
+  These tax cuts combined with heavy military spending increases led to enormous
+ </del>
@@ -656 +1008,8 @@
- . The debt increased by approximately 200% (tripled) between when Reagan took office and when his successor,
+ . The debt
+ <ins>
+  approximately tripled
+ </ins>
+ <del>
+  increased by approximately 200% (tripled)
+ </del>
+ between when Reagan took office and when his successor,
@@ -664 +1023,6 @@
- , so Reagan's policies are not the exclusive cause of the debt increase.
+ <ins>
+  ; thus, they lay blame for the debt increase at the Congress instead of the President.
+ </ins>
+ <del>
+  , so Reagan's policies are not the exclusive cause of the debt increase.
+ </del>
@@ -677,0 +1042,106 @@
+ <ins>
+  <span class="mw-headline" id="Response_to_AIDS">
+   Response to AIDS
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:202px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Then-Vice President Bush, right, meets with President Reagan, left, in 1984.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Reagan's presidency saw the advent of
+  <a href="/wiki/HIV" title="HIV">
+   HIV
+  </a>
+  -
+  <a class="mw-redirect" href="/wiki/AIDS" title="AIDS">
+   AIDS
+  </a>
+  as a widespread epidemic in the US. Although AIDS was first reported in 1981, Reagan did not mention it publicly for several more years; while it is commonly stated that he did not do so until
+  <a href="/wiki/1987" title="1987">
+   1987
+  </a>
+  , this claim appears to be erroneous, with documented instances in late
+  <a href="/wiki/1985" title="1985">
+   1985
+  </a>
+  and early
+  <a href="/wiki/1986" title="1986">
+   1986
+  </a>
+  . His administration approached the epidemic as a series of local and state issues rather than with a national strategy, and politicians for the Department of Health and Human Services pled behind the scenes for adequate funding.
+ </ins>
+</p>
+<p>
+ <ins>
+  In deference to the views of the powerful religious right, who saw AIDS as a disease limited to the
+  <a href="/wiki/Gay" title="Gay">
+   gay
+  </a>
+  male community and spread by immoral behaviour, Reagan prevented his surgeon general,
+  <a href="/wiki/C._Everett_Koop" title="C. Everett Koop">
+   C. Everett Koop
+  </a>
+  , from speaking out about the epidemic. When in
+  <a href="/wiki/1986" title="1986">
+   1986
+  </a>
+  Reagan finally authorized Koop to issue a report on the epidemic, he expected it to be in line with conservative policies; instead, Koop's
+  <i>
+   Surgeon General's Report on Acquired Immune Deficiency Syndrome
+  </i>
+  greatly emphasized the importance of a comprehensive AIDS education strategy, including widespread distribution of
+  <a href="/wiki/Condom" title="Condom">
+   condoms
+  </a>
+  , and rejected mandatory testing. This approach brought Koop into conflict with other administration officials such as
+  <a href="/wiki/William_Bennett" title="William Bennett">
+   William Bennett
+  </a>
+  .
+ </ins>
+</p>
+<p>
+ <ins>
+  Social action groups like
+  <a href="/wiki/ACT_UP" title="ACT UP">
+   ACT UP
+  </a>
+  worked to raise awareness of the AIDS problem. In 1987, Reagan responded by appointing the
+  <a class="mw-disambig" href="/wiki/Watkins_Commission" title="Watkins Commission">
+   Watkins Commission
+  </a>
+  on AIDS, but its recommendations for increased funding went largely ignored by the Reagan and the subsequent Bush administration. The most memorable ACT UP slogan confronting the Reagan/Bush stance was SILENCE= DEATH.
+ </ins>
+</p>
+<p>
+ <ins>
+  Many
+  <a href="/wiki/Social_conservatism" title="Social conservatism">
+   socially conservative
+  </a>
+  commentators saw Reagan's handling of the AIDS crisis as a common sense approach for a problem they believed was caused by social
+  <a href="/wiki/Immorality" title="Immorality">
+   immorality
+  </a>
+  . Many in the
+  <a href="/wiki/LGBT" title="LGBT">
+   gay and lesbian
+  </a>
+  communities and many
+  <a class="mw-redirect" href="/wiki/People_with_AIDS" title="People with AIDS">
+   people with AIDS
+  </a>
+  and those directly effected by the destruction of AIDS, such as health care providers, and family members, on the other hand, saw his policies as a politically motivated willful blindness, to an outright contempt for anyone who didn't fit within the social constructs of the social conservative.
+ </ins>
+</p>
+<h3>
@@ -681,0 +1152,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan, left, in one-on-one discussions with Mikhail Gorbachev, the General Secretary of the Communist Party of the USSR from 1985 to 1991.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -690,0 +1172,3 @@
+ <ins>
+  ,
+ </ins>
@@ -695 +1179,12 @@
- an "Evil Empire" and later in his presidency while speaking in front of the
+ an
+ <ins>
+  "
+  <a href="/wiki/Evil_empire" title="Evil empire">
+   evil empire
+  </a>
+  ". Later in his presidency,
+ </ins>
+ <del>
+  "Evil Empire" and later in his presidency
+ </del>
+ while speaking in front of the
@@ -698,0 +1194,3 @@
+ <ins>
+  ,
+ </ins>
@@ -703 +1201,13 @@
- to "tear down this wall". Some historians believe that all of those traits were greatly supported by his enthusiasm for America and strong personal belief in rights of the individual.
+ to "tear down this wall".
+</p>
+<p>
+ <ins>
+  Many Reagan supporters, including Lady
+  <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+   Margaret Thatcher
+  </a>
+  ,
+ </ins>
+ <del>
+  Some historians believe that all of those traits were greatly supported by his enthusiasm for America and strong personal belief in rights of the individual.
+ </del>
@@ -707,3 +1216,0 @@
-  <a class="image" href="/wiki/File:Reagan_and_Gorbachev_hold_discussions.jpg">
-   <img alt="" class="thumbimage" data-file-height="505" data-file-width="750" height="202" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/300px-Reagan_and_Gorbachev_hold_discussions.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/450px-Reagan_and_Gorbachev_hold_discussions.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/66/Reagan_and_Gorbachev_hold_discussions.jpg/600px-Reagan_and_Gorbachev_hold_discussions.jpg 2x" width="300"/>
-  </a>
@@ -712,2 +1218,0 @@
-    <a class="internal" href="/wiki/File:Reagan_and_Gorbachev_hold_discussions.jpg" title="Enlarge">
-    </a>
@@ -715 +1220,3 @@
-   President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   <del>
+    President Reagan, left, in one-on-one discussions with USSR President Gorbachev, right.
+   </del>
@@ -719,2 +1226,4 @@
-<p>
- Many Reagan supporters credit him with winning the
+<del>
+ Many Reagan supporters
+</del>
+ credit him with winning the
@@ -722,2 +1231,2 @@
-  Cold War
- </a>
+ Cold War
+</a>
@@ -726,2 +1235,2 @@
-  communism
- </a>
+ communism
+</a>
@@ -730,3 +1239,12 @@
-  1989
- </a>
- in Eastern Europe and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
+ 1989
+</a>
+ in
+ <ins>
+ <a href="/wiki/Central_Europe" title="Central Europe">
+  Central Europe
+ </a>
+</ins>
+<del>
+ Eastern Europe
+</del>
+ and the Soviet Union to the mounting Soviet economic crisis and the failure of the economic and political reforms initiated by Soviet President
@@ -734,2 +1252,2 @@
-  Mikhail Gorbachev
- </a>
+ Mikhail Gorbachev
+</a>
@@ -738,2 +1256,2 @@
-  Strategic Defense Initiative
- </a>
+ Strategic Defense Initiative
+</a>
@@ -742,2 +1260,2 @@
-  Soviet Union
- </a>
+ Soviet Union
+</a>
@@ -745,2 +1263 @@
-</p>
-<p>
+ <p>
@@ -751 +1268,5 @@
- , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application the
+ , as well as increasing the instability of future international crises. Other critics saw the extraordinary expenditures involved in the multiple distinct SDI programs as a military-industrial boondoggle. Supporters call SDI the nail in the coffin of the arms race with the Soviet Union through the application
+ <ins>
+  of
+ </ins>
+ the
@@ -758,4 +1279,57 @@
- Reagan had a close friendship with many other conservative political leaders across the globe, especially
- <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
-  Margaret Thatcher
- </a>
+ <ins>
+  At the time, Reagan suggested it would take decades for the program to be carried out. The program was supported by his successor,
+ </ins>
+ <ins>
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  , though not eagerly pursued.
+ </ins>
+ <ins>
+  <a href="/wiki/Bill_Clinton" title="Bill Clinton">
+   Bill Clinton
+  </a>
+  also supported it, though not actively. President
+ </ins>
+ <ins>
+  <a href="/wiki/George_W._Bush" title="George W. Bush">
+   George W. Bush
+  </a>
+  actively supports the program.
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagan_thatcher.jpg" title="File:Reagan thatcher.jpg">
+    File:Reagan thatcher.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Reagan and
+    <a href="/wiki/United_Kingdom" title="United Kingdom">
+     United Kingdom
+    </a>
+    Prime Minister
+    <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+     Margaret Thatcher
+    </a>
+    in
+    <a href="/wiki/Camp_David" title="Camp David">
+     Camp David
+    </a>
+    .
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ Reagan had close friendships
+</ins>
+<del>
+ Reagan had a close friendship
+</del>
+ with many other conservative political leaders across the globe, especially Margaret Thatcher <del>
+ Link: /wiki/Margaret_Thatcher
+</del>
@@ -764,2 +1338,2 @@
-  Britain
- </a>
+ Britain
+</a>
@@ -768,2 +1342,2 @@
-  Brian Mulroney
- </a>
+ Brian Mulroney
+</a>
@@ -772,2 +1346,2 @@
-  Canada
- </a>
+ Canada
+</a>
@@ -776,2 +1350,2 @@
-  Camp David
- </a>
+ Camp David
+</a>
@@ -779,2 +1353 @@
-</p>
-<p>
+ <p>
@@ -781,0 +1355,3 @@
+ <ins>
+  "
+ </ins>
@@ -785,9 +1361,53 @@
- , the United States also took an increasingly hard line against Communist influences in Latin America, which often involved the controversial support of anti-Communist
- <a href="/wiki/Military_dictatorship" title="Military dictatorship">
-  military dictatorships
- </a>
- with poor
- <a href="/wiki/Human_rights" title="Human rights">
-  human rights
- </a>
- records. This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
+ <ins>
+  ,"
+ </ins>
+ <del>
+  ,
+ </del>
+ the United States also
+ <ins>
+  offered financial and logistics support to the anti-communist opposition in central Europe (most notably the
+  <a href="/wiki/Poland" title="Poland">
+   Polish
+  </a>
+  <i>
+   <b>
+    <a href="/wiki/Solidarity" title="Solidarity">
+     Solidarity
+    </a>
+   </b>
+  </i>
+  movement) and
+ </ins>
+ took an increasingly hard line against Communist
+ <ins>
+  regimes in
+  <a href="/wiki/Afghanistan" title="Afghanistan">
+   Afghanistan
+  </a>
+  ,
+  <a href="/wiki/Angola" title="Angola">
+   Angola
+  </a>
+  ,
+  <a href="/wiki/Cambodia" title="Cambodia">
+   Cambodia
+  </a>
+  , and
+  <a href="/wiki/Nicaragua" title="Nicaragua">
+   Nicaragua
+  </a>
+  .
+ </ins>
+ <del>
+  influences in Latin America, which often involved the controversial support of anti-Communist
+  <a href="/wiki/Military_dictatorship" title="Military dictatorship">
+   military dictatorships
+  </a>
+  with poor
+  <a href="/wiki/Human_rights" title="Human rights">
+   human rights
+  </a>
+  records.
+ </del>
+ This has led some to charge that Reagan was undertaking secret and illegal guerilla wars. In
@@ -796,0 +1417,3 @@
+ <ins>
+  ,
+ </ins>
@@ -801 +1424,23 @@
- after it underwent a Communist coup. Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
+ after it underwent a Communist coup.
+ <ins>
+  Prior to the Grenada invasion, on Dec 4, 1982, Reagan openly supported Guatemalan dictator Gen. Efrain Rios Montt's campaign. In
+  <a href="/wiki/1986" title="1986">
+   1986
+  </a>
+  , representing the global promise he felt was inherent in the success of the Reagan Doctrine, Reagan invited anti-communist Angolan resistance leader
+  <a href="/wiki/Jonas_Savimbi" title="Jonas Savimbi">
+   Jonas Savimbi
+  </a>
+  to The White House, where he spoke of Savimbi winning "a victory that electrifies the world." The Reagan Doctrine was vigorously supported by influential foreign policy analysts at The
+  <a class="mw-redirect" href="/wiki/Heritage_Foundation" title="Heritage Foundation">
+   Heritage Foundation
+  </a>
+  and other
+  <a class="mw-redirect" href="/wiki/Conservatives" title="Conservatives">
+   conservatives
+  </a>
+  , leading to the flow of American weapons to anti-communist resistance movements on several continents.
+ </ins>
+</p>
+<p>
+ Near the end of his term, Reagan was also instrumental in supporting the transition of Latin American democracy, giving generous
@@ -808,2 +1453,18 @@
- <span class="mw-headline" id="Iran-Contra_Affair">
-  Iran-Contra Affair
+ <span class="mw-headline" id="Iran-Contra_affair">
+  Iran-Contra
+  <ins>
+   affair
+  </ins>
+  <p>
+   <ins>
+    <i>
+     Main article:
+     <a class="mw-redirect" href="/wiki/Iran-Contra_Affair" title="Iran-Contra Affair">
+      Iran-Contra Affair
+     </a>
+    </i>
+   </ins>
+  </p>
+  <del>
+   Affair
+  </del>
@@ -848 +1509,4 @@
- terrorist group in
+ <del>
+  terrorist group
+ </del>
+ in
@@ -856,3 +1519,0 @@
-  <a class="image" href="/wiki/File:Pres_Ronald_Reagan_on_white_horse.jpg">
-   <img alt="" class="thumbimage" data-file-height="835" data-file-width="550" height="334" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/220px-Pres_Ronald_Reagan_on_white_horse.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/330px-Pres_Ronald_Reagan_on_white_horse.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Pres_Ronald_Reagan_on_white_horse.jpg/440px-Pres_Ronald_Reagan_on_white_horse.jpg 2x" width="220"/>
-  </a>
@@ -861,2 +1521,0 @@
-    <a class="internal" href="/wiki/File:Pres_Ronald_Reagan_on_white_horse.jpg" title="Enlarge">
-    </a>
@@ -864 +1523,3 @@
-   President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   <del>
+    President Reagan had a famous life-long love of horse-riding which added to his all-American image
+   </del>
@@ -873 +1534,11 @@
- to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led Congressman Patricia Schroeder to dub him the "
+ to investigate the wider scandal. His cooperation with counsel helped Iran-Contra from seriously damaging his presidency. It was found that the President was guilty of the scandal only in that his lax control of his own staff resulted in his ignorance of the arms sale. Although Reagan himself was considered personally honest by most Americans, other scandals occurred involving bribery, corruption, and influence peddling among some of Reagan's aides and subordinates, resulting in a significant number of officials in the Reagan Administration either being convicted or forced to resign their posts to avoid prosecution. The failure of these scandals to damage Reagan's reputation led
+ <ins>
+  Representative
+  <a href="/wiki/Patricia_Schroeder" title="Patricia Schroeder">
+   Patricia Schroeder
+  </a>
+ </ins>
+ <del>
+  Congressman Patricia Schroeder
+ </del>
+ to dub him the "
@@ -893 +1564,13 @@
- for drug offenders while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's prison population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless, surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
+ for
+ <a href="/wiki/Illegal_drug_trade" title="Illegal drug trade">
+  drug offenders
+ </a>
+ while cutting funding for addiction treatment. This resulted in a dramatic increase in the USA's
+ <a href="/wiki/Prison" title="Prison">
+  prison
+ </a>
+ population. Critics charged that the policies did little to actually reduce the availability of drugs or crime on the street while resulting in a great financial and human cost for American society. Due to this policy and various cuts in spending for social programs during his Presidency, Reagan was regarded by some critics as indifferent to the needs of poor and minority citizens. Nevertheless,
+ <ins>
+  some
+ </ins>
+ surveys showed that illegal drug use among Americans declined significantly during Reagan's presidency, leading supporters to argue that the policies were successful.
@@ -895 +1578 @@
-<h3>
+<h2>
@@ -899 +1582,25 @@
-</h3>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan was a Chicago Cubs radio broadcaster in the mid-1930s for an Iowa station. He made several guest trips back to the radio booth, both during and after his presidency.
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  During his California governorship, Reagan actively dismantled the public
+  <a href="/wiki/Psychiatric_hospital" title="Psychiatric hospital">
+   psychiatric hospital
+  </a>
+  system, proposing that it be replaced by supportive community-based housing and treatment. According to some Reagan critics, the first objective was effectively accomplished, but the community replacement facilities were never adequately funded, neither by Reagan nor his successors, contributing to current California problems with
+  <a href="/wiki/Homelessness" title="Homelessness">
+   homeless
+  </a>
+  people, and an overfilling of jails and penitentiaries by people who would be better served with the earlier hospital system.
+ </ins>
+</p>
@@ -920,0 +1628,7 @@
+ <ins>
+  Some, including
+  <a href="/wiki/Alan_Greenspan" title="Alan Greenspan">
+   Alan Greenspan
+  </a>
+  , have credited Reagan's action restoring flexibility to the business environment that had prevented American companies from hiring and held back the economy.
+ </ins>
@@ -925 +1639,7 @@
-  US Marines
+  <ins>
+   U.S.
+  </ins>
+  <del>
+   US
+  </del>
+  Marines
@@ -931,9 +1651,13 @@
- . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines. Two days later Reagan invaded the tiny
- <a href="/wiki/Caribbean" title="Caribbean">
-  Caribbean
- </a>
- nation of
- <a href="/wiki/Grenada" title="Grenada">
-  Grenada
- </a>
- . Three months later, Reagan withdrew the Marines from Lebanon.
+ . Following several smaller bombings, a truck bombing of their barracks killed 241 Marines.
+ <del>
+  Two days later Reagan invaded the tiny
+  <a href="/wiki/Caribbean" title="Caribbean">
+   Caribbean
+  </a>
+  nation of
+  <a href="/wiki/Grenada" title="Grenada">
+   Grenada
+  </a>
+  .
+ </del>
+ Three months later, Reagan withdrew the Marines from Lebanon.
@@ -975,4 +1699,19 @@
- <a href="/wiki/Prostate" title="Prostate">
-  prostate
- </a>
- surgery which caused further worries about his health.
+ <ins>
+  surgery for
+  <a href="/wiki/Prostate_cancer" title="Prostate cancer">
+   prostate cancer
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/Prostate" title="Prostate">
+   prostate
+  </a>
+  surgery
+ </del>
+ which caused further worries about his
+ <ins>
+  health, but which significantly raised the public awareness of this "silent killer."
+ </ins>
+ <del>
+  health.
+ </del>
@@ -1012,12 +1751,14 @@
- In
- <a href="/wiki/1991" title="1991">
-  1991
- </a>
- , former
- <a class="mw-disambig" href="/wiki/Starlet" title="Starlet">
-  starlet
- </a>
- <a class="new" href="/w/index.php?title=Selene_Walters&amp;action=edit&amp;redlink=1" title="Selene Walters (page does not exist)">
-  Selene Walters
- </a>
- claimed that Reagan had raped her in the early 1950s.
+ <del>
+  In
+  <a href="/wiki/1991" title="1991">
+   1991
+  </a>
+  , former
+  <a class="mw-disambig" href="/wiki/Starlet" title="Starlet">
+   starlet
+  </a>
+  <a class="new" href="/w/index.php?title=Selene_Walters&amp;action=edit&amp;redlink=1" title="Selene Walters (page does not exist)">
+   Selene Walters
+  </a>
+  claimed that Reagan had raped her in the early 1950s.
+ </del>
@@ -1031 +1772,7 @@
-   (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential Library.
+   (Left to right:) Presidents Gerald Ford, Richard Nixon, George H. W. Bush, Ronald Reagan, and Jimmy Carter at the dedication of the Reagan Presidential
+   <ins>
+    Library, 1991.
+   </ins>
+   <del>
+    Library.
+   </del>
@@ -1041,5 +1788,45 @@
- Reagan was in many ways the founder of the modern Republican Party. His redefinition of fiscal conservatism as being focused on tax cuts without significant regard to a balanced budget ("
- <a href="/wiki/Reaganomics" title="Reaganomics">
-  Reaganomics
- </a>
- "); his opposition to
+ Reagan
+ <ins>
+  is
+ </ins>
+ <del>
+  was
+ </del>
+ in many ways the
+ <ins>
+  father
+ </ins>
+ <del>
+  founder
+ </del>
+ of the modern Republican Party.
+</p>
+<ul>
+ <li>
+  <ins>
+   Among the positions that he propelled forward are the following:
+Fiscal policy focused more
+  </ins>
+  <del>
+   His redefinition of fiscal conservatism as being focused
+  </del>
+  on tax cuts
+ </li>
+ <li>
+  <ins>
+   and increased military spending (
+   <a href="/wiki/Reaganomics" title="Reaganomics">
+    Reaganomics
+   </a>
+   ) than on reducing the national debt.
+ Opposition
+  </ins>
+ </li>
+ <del>
+  without significant regard to a balanced budget ("
+  <a href="/wiki/Reaganomics" title="Reaganomics">
+   Reaganomics
+  </a>
+  "); his opposition
+ </del>
+ to
@@ -1049 +1836,8 @@
- , greater environmental protection and regulation, and
+ ,
+ <ins>
+  government regulation in business and enviromental concerns,
+ </ins>
+ <del>
+  greater environmental protection and regulation,
+ </del>
+ and
@@ -1053,5 +1847,30 @@
- ; the importance of the
- <a href="/wiki/Moral_Majority" title="Moral Majority">
-  Moral Majority
- </a>
- and its supporters in his governing coalition; and even his support of
+ <li>
+  <ins>
+   .
+ The
+  </ins>
+ </li>
+ <del>
+  ; the
+ </del>
+ importance of the
+ <li>
+  <ins>
+   <a class="mw-redirect" href="/wiki/Evangelical_Christians" title="Evangelical Christians">
+    evangelical Christians
+   </a>
+   and
+   <a class="mw-redirect" href="/wiki/Social_conservatives" title="Social conservatives">
+    social conservatives
+   </a>
+   in the electoral coalition that supported him.
+ Support
+  </ins>
+ </li>
+ <del>
+  <a href="/wiki/Moral_Majority" title="Moral Majority">
+   Moral Majority
+  </a>
+  and its supporters in his governing coalition; and even his support
+ </del>
+ of
@@ -1060,0 +1880,38 @@
+ <ins>
+  .
+ </ins>
+ <li>
+  <ins>
+   A relatively
+   <a href="/wiki/Idealism" title="Idealism">
+    idealist
+   </a>
+   view of the role America could play in spreading Western liberalism and freedom.
+  </ins>
+ </li>
+ <li>
+  <ins>
+   Appeal to
+   <a class="mw-redirect" href="/wiki/Blue_collar" title="Blue collar">
+    blue collar
+   </a>
+   workers who traditionally had been
+   <a class="mw-redirect mw-disambig" href="/wiki/Democrats" title="Democrats">
+    Democrats
+   </a>
+   .
+  </ins>
+ </li>
+ <div class="thumb tleft">
+  <div class="thumbinner" style="width:202px;">
+   <div class="thumbcaption">
+    <div class="magnify">
+    </div>
+    <ins>
+     President Reagan had a famous life-long love of horse-riding which added to his all-American image
+    </ins>
+   </div>
+  </div>
+ </div>
+</ul>
+<del>
@@ -1074 +1931 @@
-</p>
+</del>
@@ -1084 +1941,15 @@
- . As the years went on, the disease began to slowly take over the former President's brain and body, forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
+ . As the years went on, the disease
+ <ins>
+  slowly destroyed
+ </ins>
+ <del>
+  began to slowly take over
+ </del>
+ the former President's
+ <ins>
+  mental capacity,
+ </ins>
+ <del>
+  brain and body,
+ </del>
+ forcing him to live his post-presidency in quiet isolation. He informed the nation of his condition himself on
@@ -1092 +1963,13 @@
- in the form of a personal letter. His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
+ in the form of a personal letter.
+ <ins>
+  A tragic
+  <a href="/wiki/Anecdote" title="Anecdote">
+   anecdote
+  </a>
+  told of this time is of his removing from a friend's aquarium a ceramic model of the
+  <a href="/wiki/White_House" title="White House">
+   White House
+  </a>
+  . It is said that he said "I know this is important, but I don't know why."
+ </ins>
+ His ailing health was further destabilized by a fall in 2001, which shattered part of his hip and rendered him virtually immobile. By
@@ -1096 +1979,8 @@
- Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was rumoured, although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
+ Reagan could no longer speak coherently and had trouble with even the most basic tasks and it was
+ <ins>
+  rumored,
+ </ins>
+ <del>
+  rumoured,
+ </del>
+ although not confirmed, that he had begun to enter the final stage of Alzheimer's, commonly described as the "sit and stare" stage, common to many patients in the last stages of the disease.
@@ -1144,8 +2034,10 @@
- In November
- <a href="/wiki/2003" title="2003">
-  2003
- </a>
- , Reagan and his family were the subject of a controversial television miniseries,
- <i>
-  <a href="/wiki/The_Reagans" title="The Reagans">
-   The Reagans
+</p>
+<p>
+ <ins>
+  Ronald Reagan died at his home in
+  <a class="mw-redirect" href="/wiki/Bel_Air,_California" title="Bel Air, California">
+   Bel Air, California
+  </a>
+  on
+  <a href="/wiki/June_5" title="June 5">
+   June 5
@@ -1153,10 +2045,26 @@
- </i>
- . In response to the unflattering docu-drama, a number of Congressional Republicans introduced the "Ronald Reagan Dime Act" (HR 3633), a bill that would replace
- <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
-  Franklin Delano Roosevelt
- </a>
- 's portrait on the
- <a class="mw-redirect" href="/wiki/Dime_(U.S._coin)" title="Dime (U.S. coin)">
-  United States dime
- </a>
- with Reagan's. The bill did not have widespread support and appeared unlikely to be put up for a vote.
+  ,
+  <a href="/wiki/2004" title="2004">
+   2004
+  </a>
+  at 1:09 PM local (Pacific) time. He died of
+  <a href="/wiki/Pneumonia" title="Pneumonia">
+   pneumonia
+  </a>
+  with his wife
+  <a href="/wiki/Nancy_Reagan" title="Nancy Reagan">
+   Nancy
+  </a>
+  and their children
+  <a href="/wiki/Patti_Davis" title="Patti Davis">
+   Patti
+  </a>
+  and
+  <a href="/wiki/Ron_Reagan" title="Ron Reagan">
+   Ron
+  </a>
+  , present. He is survived also by his son
+  <a href="/wiki/Michael_Reagan" title="Michael Reagan">
+   Michael
+  </a>
+  .
+ </ins>
@@ -1164,0 +2073,85 @@
+ <ins>
+  He will be given a full presidential
+  <a href="/wiki/State_funeral" title="State funeral">
+   state funeral
+  </a>
+  . Services at the
+  <a class="mw-redirect" href="/wiki/National_Cathedral" title="National Cathedral">
+   National Cathedral
+  </a>
+  on
+  <a href="/wiki/June_11" title="June 11">
+   June 11
+  </a>
+  will include eulogies by President
+  <a href="/wiki/George_W._Bush" title="George W. Bush">
+   George W. Bush
+  </a>
+  , former President
+  <a href="/wiki/George_H._W._Bush" title="George H. W. Bush">
+   George H. W. Bush
+  </a>
+  , former
+  <a href="/wiki/United_Kingdom" title="United Kingdom">
+   British
+  </a>
+  <a href="/wiki/Prime_Minister_of_the_United_Kingdom" title="Prime Minister of the United Kingdom">
+   Prime Minister
+  </a>
+  <a href="/wiki/Margaret_Thatcher" title="Margaret Thatcher">
+   Margaret Thatcher
+  </a>
+  , and former
+  <a href="/wiki/Canada" title="Canada">
+   Canadian
+  </a>
+  <a href="/wiki/Prime_Minister_of_Canada" title="Prime Minister of Canada">
+   Prime Minister
+  </a>
+  <a href="/wiki/Brian_Mulroney" title="Brian Mulroney">
+   Brian Mulroney
+  </a>
+  .
+ </ins>
+</p>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Reagancoffin.jpg" title="File:Reagancoffin.jpg">
+    File:Reagancoffin.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Nancy Reagan lays her head on the flag draped casket of President Reagan.
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ Reagan holds the record for the longest-living President
+</ins>
+<p>
+ <del>
+  In November
+  <a href="/wiki/2003" title="2003">
+   2003
+  </a>
+  , Reagan and his family were the subject of a controversial television miniseries,
+  <i>
+   <a href="/wiki/The_Reagans" title="The Reagans">
+    The Reagans
+   </a>
+  </i>
+  . In response to the unflattering docu-drama, a number of Congressional Republicans introduced the "Ronald Reagan Dime Act" (HR 3633), a bill that would replace
+  <a class="mw-redirect" href="/wiki/Franklin_Delano_Roosevelt" title="Franklin Delano Roosevelt">
+   Franklin Delano Roosevelt
+  </a>
+  's portrait on the
+  <a class="mw-redirect" href="/wiki/Dime_(U.S._coin)" title="Dime (U.S. coin)">
+   United States dime
+  </a>
+  with Reagan's. The bill did not have widespread support and appeared unlikely to be put up for a vote.
+ </del>
+</p>
+<del>
@@ -1165,0 +2159,2 @@
+</del>
+<del>
@@ -1169 +2164,15 @@
- , Reagan turned 93, continuing his record as the longest-living president in American history. The previous record had been 90 years and 247 days, held by
+ , Reagan turned 93, continuing his record as the longest-living president
+</del>
+ in American history. The previous record <ins>
+ of
+</ins>
+<del>
+ had been
+</del>
+ 90 years and 247 <ins>
+ days was
+</ins>
+<del>
+ days,
+</del>
+ held by
@@ -1171 +2180,6 @@
-  John Adams
+ John Adams
+</a>
+<ins>
+ ; Reagan surpassed it on
+ <a href="/wiki/October_11" title="October 11">
+  October 11
@@ -1172,0 +2187,2 @@
+</ins>
+<del>
@@ -1176,0 +2193 @@
+</del>
@@ -1179,3 +2196,330 @@
-  2001
- </a>
- . In June of 2004 the
+ 2001
+</a>
+ .
+ <p>
+ <ins>
+  Both opponents and supporters noted his "sunny optimism," which was welcomed by many in comparison to his often smiling, but somewhat dour and serious, immediate Presidential predecessor.
+ </ins>
+</p>
+<p>
+ <ins>
+  His style of relating to others had often been described as avuncular - in the demeanor of an uncle, one not responsible for discipline but who can provide well meaning guidance.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Quotations">
+   Quotations
+  </span>
+ </ins>
+</h2>
+<ul>
+ <li>
+  <ins>
+   "This is the issue of this election: Whether we believe in our capacity for self-government or whether we abandon the American Revolution and confess that a little intellectual elite in a far-distant capital can plan our lives for us better than we can plan them ourselves.", nationwide televised speech supporting
+   <a href="/wiki/Barry_Goldwater" title="Barry Goldwater">
+    Barry Goldwater's
+   </a>
+   presidential campaign, October 27, 1964.
+  </ins>
+ </li>
+ <li>
+  <ins>
+   "General Secretary Gorbachev, if you seek peace, if you seek prosperity for the Soviet Union and Eastern Europe, if you seek liberalization: Come here to this gate! Mr. Gorbachev, open this gate! Mr. Gorbachev, tear down this
+   <a href="/wiki/Berlin_Wall" title="Berlin Wall">
+    wall
+   </a>
+   ."
+  </ins>
+ </li>
+ <li>
+  <ins>
+   "My fellow Americans, I am pleased to tell you today that I have signed legislation that will outlaw Russia forever. We begin bombing in five minutes." (joke overheard during a microphone test)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   "How do you tell a communist? Well, it's someone who reads
+   <a class="mw-redirect" href="/wiki/Marx" title="Marx">
+    Marx
+   </a>
+   and
+   <a class="mw-redirect" href="/wiki/Lenin" title="Lenin">
+    Lenin
+   </a>
+   . And how do you tell an anti-communist? It's someone who understands Marx and Lenin."
+  </ins>
+ </li>
+ <li>
+  <ins>
+   "One hundred nations in the UN have not agreed with us on just about everything that's come before them, where we're involved, and it didn't upset my breakfast at all." (reacting to international criticism of the
+   <a class="mw-redirect" href="/wiki/Operation_Urgent_Fury" title="Operation Urgent Fury">
+    Operation Urgent Fury
+   </a>
+   , 1983, quoted in
+   <i>
+    Killing Hope
+   </i>
+   by William Blum)
+  </ins>
+ </li>
+</ul>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Cabinet">
+   Cabinet
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:302px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    President Reagan and his Cabinet in the Oval Office (Feb. 4, 1981)
+   </ins>
+  </div>
+ </div>
+</div>
+<ul>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_State" title="United States Secretary of State">
+    Secretary of State
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Alexander_M._Haig" title="Alexander M. Haig">
+    Alexander M. Haig
+   </a>
+   (1981-82),
+   <a href="/wiki/George_P._Shultz" title="George P. Shultz">
+    George P. Shultz
+   </a>
+   (1982-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Defense" title="United States Secretary of Defense">
+    Secretary of Defense
+   </a>
+   -
+   <a href="/wiki/Caspar_Weinberger" title="Caspar Weinberger">
+    Caspar Weinberger
+   </a>
+   (1981-87),
+   <a class="mw-redirect" href="/wiki/Frank_C._Carlucci" title="Frank C. Carlucci">
+    Frank C. Carlucci
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_the_Treasury" title="United States Secretary of the Treasury">
+    Secretary of the Treasury
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Donald_T._Regan" title="Donald T. Regan">
+    Donald T. Regan
+   </a>
+   (1981-1985),
+   <a href="/wiki/James_Baker" title="James Baker">
+    James Baker
+   </a>
+   (1985-88),
+   <a href="/wiki/Nicholas_F._Brady" title="Nicholas F. Brady">
+    Nicholas F. Brady
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Attorney_General" title="United States Attorney General">
+    Attorney General
+   </a>
+   -
+   <a href="/wiki/William_French_Smith" title="William French Smith">
+    William French Smith
+   </a>
+   (1981-85),
+   <a href="/wiki/Edwin_Meese" title="Edwin Meese">
+    Edwin Meese III
+   </a>
+   (1985-88),
+   <a class="mw-redirect" href="/wiki/Richard_L._Thornburgh" title="Richard L. Thornburgh">
+    Richard L. Thornburgh
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_the_Interior" title="United States Secretary of the Interior">
+    Secretary of the Interior
+   </a>
+   -
+   <a href="/wiki/James_G._Watt" title="James G. Watt">
+    James G. Watt
+   </a>
+   (1981-83),
+   <a class="mw-redirect" href="/wiki/William_P._Clark" title="William P. Clark">
+    William P. Clark
+   </a>
+   (1983-85),
+   <a href="/wiki/Donald_P._Hodel" title="Donald P. Hodel">
+    Donald P. Hodel
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Agriculture" title="United States Secretary of Agriculture">
+    Secretary of Agriculture
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/John_R._Block" title="John R. Block">
+    John R. Block
+   </a>
+   (1981-86),
+   <a class="mw-redirect" href="/wiki/Richard_E._Lyng" title="Richard E. Lyng">
+    Richard E. Lyng
+   </a>
+   (1986-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Commerce" title="United States Secretary of Commerce">
+    Secretary of Commerce
+   </a>
+   -
+   <a class="mw-disambig" href="/wiki/Malcolm_Baldrige" title="Malcolm Baldrige">
+    Malcolm Baldrige
+   </a>
+   (1981-87),
+   <a class="mw-redirect" href="/wiki/C._William_Verity" title="C. William Verity">
+    C. William Verity
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Labor" title="United States Secretary of Labor">
+    Secretary of Labor
+   </a>
+   -
+   <a href="/wiki/Raymond_J._Donovan" title="Raymond J. Donovan">
+    Raymond J. Donovan
+   </a>
+   (1981-85),
+   <a class="mw-redirect" href="/wiki/William_E._Brock" title="William E. Brock">
+    William E. Brock
+   </a>
+   (1985-87),
+   <a class="mw-redirect" href="/wiki/Ann_Dore_McLaughlin" title="Ann Dore McLaughlin">
+    Ann Dore McLaughlin
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Health_and_Human_Services" title="United States Secretary of Health and Human Services">
+    Secretary of Health and Human Services
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Richard_S._Schweiker" title="Richard S. Schweiker">
+    Richard S. Schweiker
+   </a>
+   (1981-83),
+   <a class="mw-redirect" href="/wiki/Margaret_M._Heckler" title="Margaret M. Heckler">
+    Margaret M. Heckler
+   </a>
+   (1983-85),
+   <a href="/wiki/Otis_R._Bowen" title="Otis R. Bowen">
+    Otis R. Bowen
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Housing_and_Urban_Development" title="United States Secretary of Housing and Urban Development">
+    Secretary of Housing and Urban Development
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Samuel_R._Pierce" title="Samuel R. Pierce">
+    Samuel R. Pierce
+   </a>
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Transportation" title="United States Secretary of Transportation">
+    Secretary of Transportation
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Andrew_L._Lewis,_Jr." title="Andrew L. Lewis, Jr.">
+    Andrew L. Lewis, Jr.
+   </a>
+   (1981-83),
+   <a href="/wiki/Elizabeth_Dole" title="Elizabeth Dole">
+    Elizabeth Dole
+   </a>
+   (1983-87),
+   <a href="/wiki/James_H._Burnley_IV" title="James H. Burnley IV">
+    James H. Burnley IV
+   </a>
+   (1987-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Energy" title="United States Secretary of Energy">
+    Secretary of Energy
+   </a>
+   -
+   <a href="/wiki/James_B._Edwards" title="James B. Edwards">
+    James B. Edwards
+   </a>
+   (1981-82),
+   <a href="/wiki/Donald_P._Hodel" title="Donald P. Hodel">
+    Donald P. Hodel
+   </a>
+   (1982-85),
+   <a href="/wiki/John_S._Herrington" title="John S. Herrington">
+    John S. Herrington
+   </a>
+   (1985-89)
+  </ins>
+ </li>
+ <li>
+  <ins>
+   <a href="/wiki/United_States_Secretary_of_Education" title="United States Secretary of Education">
+    Secretary of Education
+   </a>
+   -
+   <a class="mw-redirect" href="/wiki/Terrel_H._Bell" title="Terrel H. Bell">
+    Terrel H. Bell
+   </a>
+   (1981-85),
+   <a class="mw-redirect" href="/wiki/William_J._Bennett" title="William J. Bennett">
+    William J. Bennett
+   </a>
+   (1985-88),
+   <a class="mw-redirect" href="/wiki/Lauro_F._Cavazos" title="Lauro F. Cavazos">
+    Lauro F. Cavazos
+   </a>
+   (1988-89)
+  </ins>
+ </li>
+</ul>
+<del>
+ In June of 2004 the
+</del>
+<del>
@@ -1186 +2530 @@
-</p>
+</del>
@@ -1188,3 +2532,5 @@
- <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
-  Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
- </a>
+ <del>
+  <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Pres38-42.jpg" title="File:Pres38-42.jpg">
+   Presidents Bill Clinton, George Bush, Ronald Reagan, Jimmy Carter, Gerald Ford, and their wives at the funeral of President Richard Nixon on April 27, 1994.
+  </a>
+ </del>
@@ -1193,5 +2539,7 @@
- On
- <a href="/wiki/June_5" title="June 5">
-  June 5
- </a>
- , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+ <del>
+  On
+  <a href="/wiki/June_5" title="June 5">
+   June 5
+  </a>
+  , 2004 it became known that Reagans health had deteriorated and that he is in months or even weeks from death.
+ </del>
@@ -1206,0 +2555,11 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:202px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Reagan and Sandra Day O'Connor at the White House in 1981.
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1249 +2608 @@
-</ul>
+</ul>
\ No newline at end of file
