--- 453910145
+++ 455336051
@@ -9 +9,9 @@
- , born September 9, 1941) is an American
+ ,
+ <ins>
+  September 9, 1941 — October 8/9, 2011
+ ) was
+ </ins>
+ <del>
+  born September 9, 1941) is
+ </del>
+ an American
@@ -122 +130,8 @@
- Ritchie is best known as the creator of the C programming language and a key developer of the Unix operating system, and as co-author of the definitive book on C,
+ Ritchie
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ best known as the creator of the C programming language and a key developer of the Unix operating system, and as co-author of the definitive book on C,
@@ -145 +160,7 @@
- have placed him as an important pioneer of modern computing. The C language is still widely used today in application and
+ <ins>
+  has
+ </ins>
+ <del>
+  have
+ </del>
+ placed him as an important pioneer of modern computing. The C language is still widely used today in application and
@@ -312 +333 @@
-</ul>
+</ul>
\ No newline at end of file
