--- 453910145
+++ 455541717
@@ -5,5 +5,17 @@
- (username:
- <b>
-  dmr
- </b>
- , born September 9, 1941) is an American
+ <ins>
+  (September 9, 1941 – October 8, 2011
+ )
+ , commonly known by his username
+  <tt>
+   dmr
+  </tt>
+  , was
+ </ins>
+ <del>
+  (username:
+  <b>
+   dmr
+  </b>
+  , born September 9, 1941) is
+ </del>
+ an American
@@ -60 +72,7 @@
- Born in
+ <ins>
+  Ritchie was born
+ </ins>
+ <del>
+  Born
+ </del>
+ in
@@ -68 +86,19 @@
- , Ritchie graduated from
+ <ins>
+  . His father was Alistair E. Ritchie, a longtime
+  <a href="/wiki/Bell_Labs" title="Bell Labs">
+   Bell Labs
+  </a>
+  scientist and co-author of
+  <i>
+   The Design of Switching Circuits
+  </i>
+  on
+  <a href="/wiki/Switching_circuit_theory" title="Switching circuit theory">
+   switching circuit theory
+  </a>
+  .
+ </ins>
+ <del>
+  ,
+ </del>
+ Ritchie graduated from
@@ -122 +158,8 @@
- Ritchie is best known as the creator of the C programming language and a key developer of the Unix operating system, and as co-author of the definitive book on C,
+ Ritchie
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ best known as the creator of the C programming language and a key developer of the Unix operating system, and as co-author of the definitive book on C,
@@ -124 +167,3 @@
-  The C Programming Language
+  <i>
+   The C Programming Language
+  </i>
@@ -138,3 +183,10 @@
- <a href="/wiki/Unix" title="Unix">
-  Unix
- </a>
+ <ins>
+  <a class="mw-redirect" href="/wiki/UNIX" title="UNIX">
+   UNIX
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/Unix" title="Unix">
+   Unix
+  </a>
+ </del>
@@ -145 +197,17 @@
- have placed him as an important pioneer of modern computing. The C language is still widely used today in application and
+ <ins>
+  has
+ </ins>
+ <del>
+  have
+ </del>
+ placed him as an important pioneer of modern computing. The C language is
+ <del>
+  still
+ </del>
+ widely used today in
+ <ins>
+  application,
+ </ins>
+ <del>
+  application and
+ </del>
@@ -148,0 +217,6 @@
+ <ins>
+  , and
+  <a href="/wiki/Embedded_system" title="Embedded system">
+   embedded system
+  </a>
+ </ins>
@@ -150,4 +224,15 @@
- <a href="/wiki/Unix" title="Unix">
-  Unix
- </a>
- has also been influential, establishing concepts and principles that are now well-established precepts of computing.
+ <ins>
+  <a class="mw-redirect" href="/wiki/UNIX" title="UNIX">
+   UNIX
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/Unix" title="Unix">
+   Unix
+  </a>
+ </del>
+ has also been influential, establishing concepts and principles that are now
+ <del>
+  well-established
+ </del>
+ precepts of computing.
@@ -173 +258,4 @@
-   Thompson (left) and Ritchie (center) receiving the National Medal of Technology from President
+   <a href="/wiki/Ken_Thompson" title="Ken Thompson">
+    Thompson
+   </a>
+   (left) and Ritchie (center) receiving the National Medal of Technology from President
@@ -177 +265,6 @@
-   .
+   <ins>
+    in 1999.
+   </ins>
+   <del>
+    .
+   </del>
@@ -238 +331,8 @@
- In 2011 Dennis Ritchie, along with
+ In
+ <ins>
+  2011,
+ </ins>
+ <del>
+  2011
+ </del>
+ Dennis Ritchie, along with
@@ -249,3 +349,5 @@
- <span class="mw-headline" id="Nicknames">
-  Nicknames
- </span>
+ <ins>
+  <span class="mw-headline" id="Death_and_legacy">
+   Death and legacy
+  </span>
+ </ins>
@@ -254,5 +356,19 @@
- Dennis Ritchie is often referred to as "
- <b>
-  dmr
- </b>
- " (his Bell Labs email address) in technical discussion groups.
+ <ins>
+  Dennis Ritchie died on October 8, 2011 at the age of 70 after a prolonged illness.
+ </ins>
+</p>
+<h2>
+ <del>
+  <span class="mw-headline" id="Nicknames">
+   Nicknames
+  </span>
+ </del>
+</h2>
+<p>
+ <del>
+  Dennis Ritchie is often referred to as "
+  <b>
+   dmr
+  </b>
+  " (his Bell Labs email address) in technical discussion groups.
+ </del>
@@ -295,0 +412,38 @@
+<table class="plainlinks metadata ambox ambox-move" role="presentation">
+ <tr>
+  <td class="mbox-image">
+   <div style="width:52px">
+   </div>
+  </td>
+  <td class="mbox-text">
+   <ins>
+    <span class="mbox-text-span">
+     This page
+     <b>
+      is a candidate to be
+      <a class="extiw" href="https://meta.wikimedia.org/wiki/Help:Transwiki" title="m:Help:Transwiki">
+       copied
+      </a>
+      to
+      <a class="extiw" href="https://en.wikiquote.org/wiki/Main_Page" title="q:Main Page">
+       Wikiquote
+      </a>
+      using the
+      <a href="/wiki/Wikipedia:Transwiki_log" title="Wikipedia:Transwiki log">
+       Transwiki
+      </a>
+      process
+     </b>
+     .
+     <span class="hide-when-compact">
+      If the page can be
+      <a class="mw-redirect" href="/wiki/Wikipedia:How_to_edit_a_page" title="Wikipedia:How to edit a page">
+       expanded
+      </a>
+      into an encyclopedic article, rather than a list of quotations, please do so and remove this message.
+     </span>
+    </span>
+   </ins>
+  </td>
+ </tr>
+</table>
@@ -310 +464,3 @@
-  "We really didn't buy it thinking we'd have this enormous investment."
+  <del>
+   "We really didn't buy it thinking we'd have this enormous investment."
+  </del>
@@ -312 +468 @@
-</ul>
+</ul>
\ No newline at end of file
