--- 453910145
+++ 455853538
@@ -5,5 +5,16 @@
- (username:
- <b>
-  dmr
- </b>
- , born September 9, 1941) is an American
+ <ins>
+  (September 9, 1941 – October 2011),
+ commonly known by his username
+  <tt>
+   dmr
+  </tt>
+  , was
+ </ins>
+ <del>
+  (username:
+  <b>
+   dmr
+  </b>
+  , born September 9, 1941) is
+ </del>
+ an American
@@ -13,21 +24,56 @@
- notable for developing
- <a href="/wiki/C_(programming_language)" title="C (programming language)">
-  C
- </a>
- and for having influence on other
- <a class="mw-redirect" href="/wiki/Programming_languages" title="Programming languages">
-  programming languages
- </a>
- , as well as
- <a class="mw-redirect" href="/wiki/Operating_systems" title="Operating systems">
-  operating systems
- </a>
- such as
- <a href="/wiki/Multics" title="Multics">
-  Multics
- </a>
- and
- <a href="/wiki/Unix" title="Unix">
-  Unix
- </a>
- . He received the
+ <ins>
+  who "helped shape the digital era."
+ He created the
+  <a href="/wiki/C_(programming_language)" title="C (programming language)">
+   C programming language
+  </a>
+  and, with long-time colleague,
+  <a href="/wiki/Ken_Thompson" title="Ken Thompson">
+   Ken Thompson
+  </a>
+  , the
+  <a href="/wiki/Unix" title="Unix">
+   UNIX
+  </a>
+  <a href="/wiki/Operating_system" title="Operating system">
+   operating system
+  </a>
+  ,
+  <a class="mw-redirect" href="/wiki/Computer_software" title="Computer software">
+   software
+  </a>
+  that powers everything from
+  <a class="mw-redirect" href="/wiki/Smartphones" title="Smartphones">
+   smartphones
+  </a>
+  to
+  <a href="/wiki/Web_search_engine" title="Web search engine">
+   search engines
+  </a>
+  .
+ Ritchie and Thompson
+ </ins>
+ <del>
+  notable for developing
+  <a href="/wiki/C_(programming_language)" title="C (programming language)">
+   C
+  </a>
+  and for having influence on other
+  <a class="mw-redirect" href="/wiki/Programming_languages" title="Programming languages">
+   programming languages
+  </a>
+  , as well as
+  <a class="mw-redirect" href="/wiki/Operating_systems" title="Operating systems">
+   operating systems
+  </a>
+  such as
+  <a href="/wiki/Multics" title="Multics">
+   Multics
+  </a>
+  and
+  <a href="/wiki/Unix" title="Unix">
+   Unix
+  </a>
+  . He
+ </del>
+ received the
@@ -37 +83,19 @@
- in 1983 and the
+ <ins>
+  from the
+  <a href="/wiki/Association_for_Computing_Machinery" title="Association for Computing Machinery">
+   ACM
+  </a>
+  in 1983, the
+  <a href="/wiki/IEEE_Richard_W._Hamming_Medal" title="IEEE Richard W. Hamming Medal">
+   Hamming Medal
+  </a>
+  from the
+  <a href="/wiki/Institute_of_Electrical_and_Electronics_Engineers" title="Institute of Electrical and Electronics Engineers">
+   IEEE
+  </a>
+  in 1990
+ </ins>
+ <del>
+  in 1983
+ </del>
+ and the
@@ -41 +105,11 @@
- 1998 on April 21, 1999. Ritchie was the head of
+ <ins>
+  from
+  <a href="/wiki/Bill_Clinton" title="Bill Clinton">
+   President Clinton
+  </a>
+  in
+ </ins>
+ <del>
+  1998 on April 21,
+ </del>
+ 1999. Ritchie was the head of
@@ -60,9 +134,31 @@
- Born in
- <a href="/wiki/Bronxville,_New_York" title="Bronxville, New York">
-  Bronxville
- </a>
- ,
- <a href="/wiki/New_York" title="New York">
-  New York
- </a>
- , Ritchie graduated from
+ <ins>
+  Ritchie was born in
+  <a href="/wiki/Bronxville,_New_York" title="Bronxville, New York">
+   Bronxville, New York
+  </a>
+  . His father was Alistair E. Ritchie, a longtime
+  <a href="/wiki/Bell_Labs" title="Bell Labs">
+   Bell Labs
+  </a>
+  scientist and co-author of
+  <i>
+   The Design of Switching Circuits
+  </i>
+  on
+  <a href="/wiki/Switching_circuit_theory" title="Switching circuit theory">
+   switching circuit theory
+  </a>
+  .
+ </ins>
+ <del>
+  Born in
+  <a href="/wiki/Bronxville,_New_York" title="Bronxville, New York">
+   Bronxville
+  </a>
+  ,
+  <a href="/wiki/New_York" title="New York">
+   New York
+  </a>
+  ,
+ </del>
+ Ritchie graduated from
@@ -122 +218,30 @@
- Ritchie is best known as the creator of the C programming language and a key developer of the Unix operating system, and as co-author of the definitive book on C,
+ Ritchie
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ best known as the creator of the C programming
+ <ins>
+  language,
+ </ins>
+ <del>
+  language and
+ </del>
+ a key developer of the
+ <ins>
+  <a href="/wiki/Unix" title="Unix">
+   UNIX
+  </a>
+ </ins>
+ <del>
+  Unix
+ </del>
+ operating system, and
+ <ins>
+  co-author of
+ </ins>
+ <del>
+  as co-author of the definitive book on C,
+ </del>
@@ -124 +249,3 @@
-  The C Programming Language
+  <i>
+   The C Programming Language
+  </i>
@@ -137,9 +264,22 @@
- Ritchie's invention of C and his role in the development of
- <a href="/wiki/Unix" title="Unix">
-  Unix
- </a>
- alongside
- <a class="mw-redirect" href="/wiki/Ken_Thompson_(programmer)" title="Ken Thompson (programmer)">
-  Ken Thompson
- </a>
- have placed him as an important pioneer of modern computing. The C language is still widely used today in application and
+ <del>
+  Ritchie's invention of C and his role in the development of
+  <a href="/wiki/Unix" title="Unix">
+   Unix
+  </a>
+  alongside
+  <a class="mw-redirect" href="/wiki/Ken_Thompson_(programmer)" title="Ken Thompson (programmer)">
+   Ken Thompson
+  </a>
+  have placed him as an important pioneer of modern computing.
+ </del>
+ The C language is
+ <del>
+  still
+ </del>
+ widely used today in
+ <ins>
+  application,
+ </ins>
+ <del>
+  application and
+ </del>
@@ -148,0 +289,6 @@
+ <ins>
+  , and
+  <a href="/wiki/Embedded_system" title="Embedded system">
+   embedded system
+  </a>
+ </ins>
@@ -150,4 +296,13 @@
- <a href="/wiki/Unix" title="Unix">
-  Unix
- </a>
- has also been influential, establishing concepts and principles that are now well-established precepts of computing.
+ <ins>
+  UNIX
+ </ins>
+ <del>
+  <a href="/wiki/Unix" title="Unix">
+   Unix
+  </a>
+ </del>
+ has also been influential, establishing concepts and principles that are now
+ <del>
+  well-established
+ </del>
+ precepts of computing.
@@ -160 +315,8 @@
- in 1988 for "development of the 'C' programming language and for co-development of the UNIX operating system."
+ in 1988 for "development of the
+ <ins>
+  "C"
+ </ins>
+ <del>
+  'C'
+ </del>
+ programming language and for co-development of the UNIX operating system."
@@ -173 +335,4 @@
-   Thompson (left) and Ritchie (center) receiving the National Medal of Technology from President
+   <a href="/wiki/Ken_Thompson" title="Ken Thompson">
+    Thompson
+   </a>
+   (left) and Ritchie (center) receiving the National Medal of Technology from President
@@ -177 +342,6 @@
-   .
+   <ins>
+    in 1999.
+   </ins>
+   <del>
+    .
+   </del>
@@ -188,3 +358,8 @@
- <a class="mw-redirect" href="/wiki/Ken_Thompson_(programmer)" title="Ken Thompson (programmer)">
-  Ken Thompson
- </a>
+ <ins>
+  Thompson
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/Ken_Thompson_(programmer)" title="Ken Thompson (programmer)">
+   Ken Thompson
+  </a>
+ </del>
@@ -206 +381,5 @@
- In 1990, both Ritchie and Ken Thompson received the
+ In 1990, both Ritchie and
+ <del>
+  Ken
+ </del>
+ Thompson received the
@@ -230 +409,14 @@
- for co-inventing the UNIX operating system and the C programming language which together have led to enormous advances in computer hardware, software, and networking systems and stimulated growth of an entire industry, thereby enhancing American leadership in the Information Age.
+ for co-inventing the UNIX operating system and the C programming language
+ <ins>
+  which, according to the citation for the medal, "led
+ </ins>
+ <del>
+  which together have led
+ </del>
+ to enormous advances in computer hardware, software, and networking systems and stimulated growth of an entire industry, thereby enhancing American leadership in the Information
+ <ins>
+  Age".
+ </ins>
+ <del>
+  Age.
+ </del>
@@ -238,5 +430,18 @@
- In 2011 Dennis Ritchie, along with
- <a href="/wiki/Ken_Thompson" title="Ken Thompson">
-  Ken Thompson
- </a>
- , was awarded the
+ In
+ <ins>
+  2011,
+ </ins>
+ <del>
+  2011 Dennis
+ </del>
+ Ritchie, along with
+ <ins>
+  Thompson,
+ </ins>
+ <del>
+  <a href="/wiki/Ken_Thompson" title="Ken Thompson">
+   Ken Thompson
+  </a>
+  ,
+ </del>
+ was awarded the
@@ -249,3 +454,5 @@
- <span class="mw-headline" id="Nicknames">
-  Nicknames
- </span>
+ <ins>
+  <span class="mw-headline" id="Death_and_legacy">
+   Death and legacy
+  </span>
+ </ins>
@@ -254,5 +461,20 @@
- Dennis Ritchie is often referred to as "
- <b>
-  dmr
- </b>
- " (his Bell Labs email address) in technical discussion groups.
+ <ins>
+  Dennis Ritchie was found dead on October 12, 2011 at the age of 70 at his home in
+  <a href="/wiki/Berkeley_Heights,_New_Jersey" title="Berkeley Heights, New Jersey">
+   Berkeley Heights, New Jersey
+  </a>
+  where he lived alone.
+ First news of his death came from his former colleague,
+  <a href="/wiki/Rob_Pike" title="Rob Pike">
+   Rob Pike
+  </a>
+  .
+ The cause and exact time of death have not been disclosed.
+ He had been in frail health for several years following treatment for prostate cancer and heart disease.
+ His death came a week after the death of
+  <a href="/wiki/Steve_Jobs" title="Steve Jobs">
+   Steve Jobs
+  </a>
+  ; although Ritchie's death did not receive as much media coverage,
+ computer historians such as Paul Ceruzzi said his influence was comparable.
+ </ins>
@@ -261,3 +483,47 @@
- <span class="mw-headline" id="Bibliography">
-  Bibliography
- </span>
+ <ins>
+  <span class="mw-headline" id="Notable_books">
+   Notable books
+  </span>
+ </ins>
+</h2>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=K%26R_C.jpg" title="File:K&amp;R C.jpg">
+    File:K&amp;R C.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    <i>
+     <a href="/wiki/The_C_Programming_Language" title="The C Programming Language">
+      The C Programming Language
+     </a>
+    </i>
+    , first edition.
+   </ins>
+  </div>
+ </div>
+</div>
+<h2>
+ <del>
+  <span class="mw-headline" id="Nicknames">
+   Nicknames
+  </span>
+ </del>
+</h2>
+<p>
+ <del>
+  Dennis Ritchie is often referred to as "
+  <b>
+   dmr
+  </b>
+  " (his Bell Labs email address) in technical discussion groups.
+ </del>
+</p>
+<h2>
+ <del>
+  <span class="mw-headline" id="Bibliography">
+   Bibliography
+  </span>
+ </del>
@@ -310 +576,3 @@
-  "We really didn't buy it thinking we'd have this enormous investment."
+  <ins>
+   C has "the power of assembly language and the convenience of... assembly language."
+  </ins>
@@ -312 +580,8 @@
-</ul>
+ <div style="clear:both;">
+ </div>
+ <li>
+  <del>
+   "We really didn't buy it thinking we'd have this enormous investment."
+  </del>
+ </li>
+</ul>
\ No newline at end of file
