--- 699199631
+++ 699259534
@@ -36 +36,8 @@
- , 8 January 1947) is an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
+ , 8 January
+ <ins>
+  1947 - 10 January 2016)
+ </ins>
+ <del>
+  1947)
+ </del>
+ is an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
@@ -191,0 +199,5 @@
+ <ins>
+  Passed away peacefully on 10th January 2016.
+ </ins>
+</p>
+<p>
@@ -4106 +4118 @@
-</ul>
+</ul>
\ No newline at end of file
