--- 699199631
+++ 699260381
@@ -36 +36,9 @@
- , 8 January 1947) is an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
+ , 8 January
+ <ins>
+  1947 – 10 January 2016
+ ) was
+ </ins>
+ <del>
+  1947) is
+ </del>
+ an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
@@ -3387 +3395,8 @@
- The couple resides primarily in New York City and London.
+ The couple
+ <ins>
+  resided
+ </ins>
+ <del>
+  resides
+ </del>
+ primarily in New York City and London.
@@ -3493,0 +3509,12 @@
+<h3>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  David Bowie died January 10, 2015 after an 18 month battle with cancer.
+ </ins>
+</p>
@@ -4106 +4133 @@
-</ul>
+</ul>
\ No newline at end of file
