--- 699199631
+++ 699333471
@@ -1,0 +2,6 @@
+ <ins>
+  <b>
+   David Robert Jones
+  </b>
+  (8 January 1947 – 10 January 2016), known professionally as
+ </ins>
@@ -31 +37,6 @@
- ;
+ <ins>
+  ),
+ was
+ </ins>
+ <del>
+  ;
@@ -33,4 +44,20 @@
- <b>
-  David Robert Jones
- </b>
- , 8 January 1947) is an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
+  <b>
+   David Robert Jones
+  </b>
+  , 8 January 1947) is
+ </del>
+ an English singer, songwriter, multi-instrumentalist, record producer, arranger,
+ <ins>
+  painter and actor. He was
+ </ins>
+ <del>
+  painter, and actor. Bowie has been
+ </del>
+ a figure in popular music for over four decades, and
+ <ins>
+  was considered by critics and other musicians
+ </ins>
+ <del>
+  is known
+ </del>
+ as an innovator, particularly for his work in the 1970s. His
@@ -40,4 +67,17 @@
- appearance was an iconic element of his image, particularly in the 1970s and 1980s.
-</p>
-<p>
- Bowie's first hit song, "
+ appearance was an iconic element of his image,
+ <ins>
+  principally
+ </ins>
+ <del>
+  particularly
+ </del>
+ in the 1970s and 1980s.
+</p>
+<p>
+ <ins>
+  Born and raised in South London, Bowie developed an early interest in music although his attempts to succeed as a pop star during much of the 1960s were frustrated.
+ </ins>
+ <del>
+  Bowie's first hit song,
+ </del>
+ "
@@ -47 +87,8 @@
- " in July 1969, reached the top five of the
+ "
+ <ins>
+  became his first top five entry on
+ </ins>
+ <del>
+  in July 1969, reached the top five of
+ </del>
+ the
@@ -51 +98,7 @@
- . After a three-year period of experimentation, he re-emerged in 1972 during the
+ <ins>
+  after its release in July 1969.
+ </ins>
+ <del>
+  .
+ </del>
+ After a three-year period of experimentation, he re-emerged in 1972 during the
@@ -55,4 +108,10 @@
- era with the flamboyant,
- <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
-  androgynous
- </a>
+ era with
+ <ins>
+  his flamboyant and androgynous
+ </ins>
+ <del>
+  the flamboyant,
+  <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
+   androgynous
+  </a>
+ </del>
@@ -62 +121,8 @@
- Ziggy Stardust, spearheaded by the hit single "
+ Ziggy
+ <ins>
+  Stardust. The character was spearheaded by his
+ </ins>
+ <del>
+  Stardust, spearheaded by the hit
+ </del>
+ single "
@@ -66 +132,5 @@
- " and the album
+ " and
+ <del>
+  the
+ </del>
+ album
@@ -72 +142,7 @@
- . Bowie's impact at that time, as described by biographer David Buckley, "challenged the core belief of the rock music of its day" and "created perhaps the biggest cult in popular culture."
+ . Bowie's impact at that time, as described by biographer David Buckley, "challenged the core belief of the rock music of its day" and "created perhaps the biggest cult in popular
+ <ins>
+  culture".
+ </ins>
+ <del>
+  culture."
+ </del>
@@ -80 +156,5 @@
- " and the hit album
+ " and the
+ <del>
+  hit
+ </del>
+ album
@@ -144 +224,8 @@
- , which yielded several hit singles. Throughout the 1990s and 2000s, Bowie continued to experiment with musical styles, including
+ , which yielded several
+ <ins>
+  successful
+ </ins>
+ <del>
+  hit
+ </del>
+ singles. Throughout the 1990s and 2000s, Bowie continued to experiment with musical styles, including
@@ -160 +247,7 @@
- . He has not toured since the 2003–2004
+ . He
+ <ins>
+  stopped touring after his 2003–04
+ </ins>
+ <del>
+  has not toured since the 2003–2004
+ </del>
@@ -164,7 +257,26 @@
- and has not performed live since 2006. Bowie's latest studio album,
- <i>
-  <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
-   Blackstar
-  </a>
- </i>
- , was released on 8 January 2016, his 69th birthday.
+ <ins>
+  , and last performed live at a charity event in 2006. On
+ </ins>
+ <del>
+  and has not performed live since 2006. Bowie's latest studio album,
+  <i>
+   <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
+    Blackstar
+   </a>
+  </i>
+  , was released on
+ </del>
+ 8 January 2016,
+</p>
+<p>
+ <ins>
+  the date of Bowie's 69th birthday, the studio album
+ </ins>
+ <ins>
+  <i>
+   <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
+    Blackstar
+   </a>
+  </i>
+  was released.
+ </ins>
@@ -173 +285,62 @@
- David Buckley says of Bowie: "His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
+ <ins>
+  Bowie also had a successful, but sporadic
+  <a href="/wiki/David_Bowie_filmography" title="David Bowie filmography">
+   film career
+  </a>
+  . His acting roles include the eponymous character in
+  <i>
+   <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
+    The Man Who Fell to Earth
+   </a>
+  </i>
+  (1976), Major Celliers in
+  <i>
+   <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
+    Merry Christmas, Mr. Lawrence
+   </a>
+  </i>
+  (1983),
+  <a href="/wiki/Jareth" title="Jareth">
+   Jareth, the Goblin King
+  </a>
+  in
+  <i>
+   <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
+    Labyrinth
+   </a>
+  </i>
+  (1986),
+  <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
+   Pontius Pilate
+  </a>
+  in
+  <a href="/wiki/Martin_Scorsese" title="Martin Scorsese">
+   Martin Scorsese
+  </a>
+  's
+  <i>
+   <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
+    The Last Temptation of Christ
+   </a>
+  </i>
+  (1988), and
+  <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
+   Nikola Tesla
+  </a>
+  in
+  <i>
+   <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
+    The Prestige
+   </a>
+  </i>
+  (2006), among other film and television appearances and cameos.
+ </ins>
+</p>
+<ins>
+ David Buckley said
+</ins>
+<del>
+ his 69th birthday.
+David Buckley says
+</del>
+  of Bowie: "His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
@@ -176,2 +349,2 @@
-  100 Greatest Britons
- </a>
+ 100 Greatest Britons
+</a>
@@ -179 +352,2 @@
- In the UK, he has been awarded nine Platinum album certifications, eleven Gold and eight Silver, and in the US, five Platinum and seven Gold certifications. In 2004,
+ In the UK, he has been awarded nine Platinum album certifications, eleven Gold and eight Silver, and in the US, five Platinum and seven Gold certifications. <del>
+ In 2004,
@@ -185 +359,3 @@
- ranked him 39th on their list of the "100 Greatest Artists of All Time" and 23rd on their list of the best singers of all time. He was inducted into the
+ ranked him 39th on their list of the "100 Greatest Artists of All Time" and 23rd on their list of the best singers of all time.
+</del>
+ He was inducted into the
@@ -187,2 +363,2 @@
-  Rock and Roll Hall of Fame
- </a>
+ Rock and Roll Hall of Fame
+</a>
@@ -189,0 +366,12 @@
+ <p>
+</p>
+<p>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Early_life">
+   Early life
+  </span>
+ </ins>
+</h2>
+<p>
@@ -196,3 +384,5 @@
- <span class="mw-headline" id="Biography">
-  Biography
- </span>
+ <del>
+  <span class="mw-headline" id="Biography">
+   Biography
+  </span>
+ </del>
@@ -201,3 +391,5 @@
- <span class="mw-headline" id="Early_life">
-  Early life
- </span>
+ <del>
+  <span class="mw-headline" id="Early_life">
+   Early life
+  </span>
+ </del>
@@ -205,2 +397,12 @@
-<p>
- David Bowie was born David Robert Jones on 8 January 1947, in
+<del>
+ David
+</del>
+ Bowie was born <ins>
+ in
+ <a href="/wiki/Brixton" title="Brixton">
+  Brixton, London
+ </a>
+ .
+</ins>
+<del>
+ David Robert Jones on 8 January 1947, in
@@ -210 +412,3 @@
- , London. His mother, Margaret Mary "Peggy" (née Burns), from Kent,
+ , London.
+</del>
+ His mother, Margaret Mary "Peggy" (née Burns), from Kent,
@@ -213,5 +417,10 @@
- was a promotions officer for
- <a href="/wiki/Barnardo%27s" title="Barnardo's">
-  Barnardo's
- </a>
- . The family lived at 40 Stansfield Road, located near the border of the south London areas of Brixton and
+ was a promotions officer for <ins>
+ the children's charity
+</ins>
+<a href="/wiki/Barnardo%27s" title="Barnardo's">
+ Barnardo's
+</a>
+ . The family lived at 40 Stansfield Road, <del>
+ located
+</del>
+ near the border of the south London areas of Brixton and
@@ -219,2 +428,2 @@
-  Stockwell
- </a>
+ Stockwell
+</a>
@@ -222,2 +431 @@
-</p>
-<p>
+ <p>
@@ -305 +513,7 @@
- It was an unusual technical school, as biographer Christopher Sandford writes:
+ It was an unusual technical school, as biographer Christopher Sandford
+ <ins>
+  wrote:
+ </ins>
+ <del>
+  writes:
+ </del>
@@ -364,0 +579,7 @@
+<h2>
+ <ins>
+  <span class="mw-headline" id="Career">
+   Career
+  </span>
+ </ins>
+</h2>
@@ -380 +601,7 @@
-   Bowie in 1967
+   Bowie in
+   <ins>
+    1967.
+   </ins>
+   <del>
+    1967
+   </del>
@@ -413 +640,8 @@
- blues numbers, Bowie quit the band less than a month later to join the Manish Boys, another blues outfit, who incorporated folk and soul — "I used to dream of being their
+ blues numbers, Bowie quit the band less than a month later to join the Manish Boys, another blues outfit, who incorporated folk and
+ <ins>
+  soul—"I
+ </ins>
+ <del>
+  soul — "I
+ </del>
+ used to dream of being their
@@ -591,0 +826,11 @@
+ <ins>
+  In February and March 1969, he undertook a short tour with
+  <a href="/wiki/Marc_Bolan" title="Marc Bolan">
+   Marc Bolan
+  </a>
+  's duo
+  <a class="mw-redirect" href="/wiki/Tyrannosaurus_Rex_(band)" title="Tyrannosaurus Rex (band)">
+   Tyrannosaurus Rex
+  </a>
+  , as third on the bill, performing a mime act.
+ </ins>
@@ -596 +841,7 @@
- launch, to become a UK top five hit.
+ launch,
+ <ins>
+  and reached the top five in the UK.
+ </ins>
+ <del>
+  to become a UK top five hit.
+ </del>
@@ -606 +857,8 @@
- , this developed into the Beckenham Arts Lab, and became extremely popular. The Arts Lab hosted a free festival in a local park, the subject of his song "
+ , this developed into the
+ <a class="mw-redirect" href="/wiki/Beckenham_Arts_Lab" title="Beckenham Arts Lab">
+  Beckenham Arts Lab
+ </a>
+ <del>
+  Lab,
+ </del>
+ and became extremely popular. The Arts Lab hosted a free festival in a local park, the subject of his song "
@@ -691 +949,8 @@
- appearance, the original cover of the UK version unveiled two months later depicted the singer wearing a dress: taking the garment with him, he wore it during interviews—to the approval of critics, including
+ appearance, the original cover of the UK version unveiled two months later depicted the singer wearing a dress: taking the garment with him, he wore it during
+ <ins>
+  interviews – to
+ </ins>
+ <del>
+  interviews—to
+ </del>
+ the approval of critics, including
@@ -702 +967,7 @@
- "—and in the street, to mixed reaction including laughter and, in the case of one male pedestrian, producing a gun and telling Bowie to "kiss my ass".
+ <ins>
+  " – and
+ </ins>
+ <del>
+  "—and
+ </del>
+ in the street, to mixed reaction including laughter and, in the case of one male pedestrian, producing a gun and telling Bowie to "kiss my ass".
@@ -762 +1033,6 @@
- pastiche. It was not a significant commercial success at the time
+ pastiche. It was not a significant commercial success at the
+ <ins>
+  time.
+ </ins>
+ <del>
+  time
@@ -764,4 +1040,5 @@
- <a href="/wiki/All_Time_Top_1000_Albums" title="All Time Top 1000 Albums">
-  All Time Top 1000 Albums
- </a>
- list.
+  <a href="/wiki/All_Time_Top_1000_Albums" title="All Time Top 1000 Albums">
+   All Time Top 1000 Albums
+  </a>
+  list.
+ </del>
@@ -790,0 +1068,3 @@
+   <ins>
+    .
+   </ins>
@@ -844 +1124,8 @@
- , became UK hits. The
+ ,
+ <ins>
+  were successful in the UK.
+ </ins>
+ <del>
+  became UK hits.
+ </del>
+ The
@@ -890 +1177,2 @@
-  sumo wrestling
+  sumo
+  Link: /wiki/Sumo wrestling wrestling
@@ -1013 +1301,7 @@
-   Bowie filming a video for "Rebel Rebel" in 1974
+   Bowie filming a video for "Rebel Rebel" in
+   <ins>
+    1974.
+   </ins>
+   <del>
+    1974
+   </del>
@@ -1122 +1416,7 @@
-   show, 1975
+   show,
+   <ins>
+    1975.
+   </ins>
+   <del>
+    1975
+   </del>
@@ -1175 +1475,5 @@
- Despite his by now well established superstardom, Bowie, in the words of biographer Christopher Sandford, "for all his record sales (over a million copies of
+ Despite his by now well established superstardom, Bowie, in the words of
+ <del>
+  biographer Christopher
+ </del>
+ Sandford, "for all his record sales (over a million copies of
@@ -1193 +1497,7 @@
-   Bowie as the Thin White Duke at Maple Leaf Gardens, Toronto 1976
+   Bowie as the Thin White Duke at Maple Leaf Gardens, Toronto
+   <ins>
+    1976.
+   </ins>
+   <del>
+    1976
+   </del>
@@ -1292 +1602,8 @@
- ". The core band that coalesced around this album and tour—rhythm guitarist Alomar, bassist
+ ". The core band that coalesced around this album and
+ <ins>
+  tour — rhythm
+ </ins>
+ <del>
+  tour—rhythm
+ </del>
+ guitarist Alomar, bassist
@@ -1300 +1617,7 @@
- —continued as a stable unit for the remainder of the 1970s. The tour was highly successful but mired in political controversy. Bowie was quoted in
+ <ins>
+  — continued
+ </ins>
+ <del>
+  —continued
+ </del>
+ as a stable unit for the remainder of the 1970s. The tour was highly successful but mired in political controversy. Bowie was quoted in
@@ -1308,0 +1632,2 @@
+</p>
+<p>
@@ -1336,13 +1661,17 @@
- <span class="mw-headline" id="1976.E2.80.9379:_The_Berlin_era">
-  1976–79: The Berlin era
- </span>
-</h3>
-<div class="thumb tleft">
- <div class="thumbinner" style="width:152px;">
-  <a class="image" href="/wiki/File:David_bowie_05061978_01_150.jpg">
-   <img alt="" class="thumbimage" data-file-height="221" data-file-width="150" height="221" src="//upload.wikimedia.org/wikipedia/commons/f/ff/David_bowie_05061978_01_150.jpg" width="150"/>
-  </a>
-  <div class="thumbcaption">
-   <div class="magnify">
-    <a class="internal" href="/wiki/File:David_bowie_05061978_01_150.jpg" title="Enlarge">
-    </a>
+ <span class="mw-headline" id="1976.E2.80.9379:_Berlin_era">
+  1976–79:
+  <ins>
+   Berlin era
+  </ins>
+  <del>
+   The Berlin era
+  </del>
+  <div class="thumb tleft">
+   <div class="thumbinner" style="width:152px;">
+    <div class="thumbcaption">
+     <div class="magnify">
+     </div>
+     <del>
+      Bowie performing in Oslo on 5 June 1978
+     </del>
+    </div>
@@ -1350 +1678,0 @@
-   Bowie performing in Oslo on 5 June 1978
@@ -1352,2 +1680,2 @@
- </div>
-</div>
+ </span>
+</h3>
@@ -1367 +1695,8 @@
- in Berlin, Bowie became, in the words of biographer Christopher Sandford, "a prolific producer and collector of contemporary art. […] Not only did he become a well-known patron of expressionist art: locked in Clos des Mésanges he began an intensive self-improvement course in classical music and literature, and started work on an autobiography."
+ in Berlin, Bowie became, in the words of biographer Christopher Sandford, "a prolific producer and collector of contemporary art.
+ <ins>
+  [...]
+ </ins>
+ <del>
+  […]
+ </del>
+ Not only did he become a well-known patron of expressionist art: locked in Clos des Mésanges he began an intensive self-improvement course in classical music and literature, and started work on an autobiography."
@@ -1368,0 +1704,11 @@
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:152px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Bowie performing in Oslo on 5 June 1978
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1371,3 +1716,0 @@
-  <a class="image" href="/wiki/File:Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg">
-   <img alt="" class="thumbimage" data-file-height="2800" data-file-width="3648" height="169" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/220px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/330px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/440px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg 2x" width="220"/>
-  </a>
@@ -1376,2 +1718,0 @@
-    <a class="internal" href="/wiki/File:Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg" title="Enlarge">
-    </a>
@@ -1379,5 +1720,9 @@
-   Apartment building on Hauptstraße 155 in Berlin
-   <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
-    Schöneberg
-   </a>
-   where David Bowie lived 1976–1978
+   <del>
+    Apartment building on Hauptstraße 155 in Berlin
+   </del>
+   <del>
+    <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
+     Schöneberg
+    </a>
+    where David Bowie lived 1976–1978
+   </del>
@@ -1583 +1928,12 @@
- 's televised Christmas special, when he joined Crosby in "
+ 's
+ <ins>
+  final
+  <a href="/wiki/CBS" title="CBS">
+   CBS
+  </a>
+  television
+ </ins>
+ <del>
+  televised
+ </del>
+ Christmas special, when he joined Crosby in "
@@ -1596,0 +1953,17 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Apartment building on Hauptstraße 155 in Berlin
+   </ins>
+   <ins>
+    <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
+     Schöneberg
+    </a>
+    where Bowie lived from 1976 to 1978
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1688 +2061,7 @@
-   Serious Moonlight Tour 1983
+   Serious Moonlight Tour
+   <ins>
+    1983.
+   </ins>
+   <del>
+    1983
+   </del>
@@ -1707 +2086 @@
- <a class="mw-redirect" href="/wiki/New_Romantic" title="New Romantic">
+ <a href="/wiki/New_Romanticism" title="New Romanticism">
@@ -1876 +2255,7 @@
-   , 1987
+   ,
+   <ins>
+    1987.
+   </ins>
+   <del>
+    1987
+   </del>
@@ -2045,0 +2431,6 @@
+   <p>
+    <ins>
+     .
+  Although
+    </ins>
+   </p>
@@ -2049,2 +2440,4 @@
-<p>
- Though he intended Tin Machine to operate as a democracy, Bowie dominated, both in songwriting and in decision-making.
+<del>
+ Though
+</del>
+ he intended Tin Machine to operate as a democracy, Bowie dominated, both in songwriting and in decision-making.
@@ -2053,4 +2446,4 @@
-  <a href="/wiki/Tin_Machine_(album)" title="Tin Machine (album)">
-   Tin Machine
-  </a>
- </i>
+ <a href="/wiki/Tin_Machine_(album)" title="Tin Machine (album)">
+  Tin Machine
+ </a>
+</i>
@@ -2059,0 +2453 @@
+ <p>
@@ -2108,3 +2501,0 @@
-  <a class="image" href="/wiki/File:David_Bowie_1997.jpg">
-   <img alt="" class="thumbimage" data-file-height="1967" data-file-width="1292" height="259" src="//upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/170px-David_Bowie_1997.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/255px-David_Bowie_1997.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/340px-David_Bowie_1997.jpg 2x" width="170"/>
-  </a>
@@ -2113,2 +2503,0 @@
-    <a class="internal" href="/wiki/File:David_Bowie_1997.jpg" title="Enlarge">
-    </a>
@@ -2116 +2505,3 @@
-   Bowie performing in Finland in 1997
+   <del>
+    Bowie performing in Finland in 1997
+   </del>
@@ -2137 +2528,19 @@
- 1993 saw the release of Bowie's first solo offering since his Tin Machine departure, the soul, jazz and
+</p>
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:172px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Bowie performing in Finland in 1997
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ In 1993, Bowie released his
+</ins>
+<del>
+ 1993 saw the release of Bowie's
+</del>
+ first solo offering since his Tin Machine departure, the soul, jazz and
@@ -2139,2 +2548,2 @@
-  hip-hop
- </a>
+ hip-hop
+</a>
@@ -2143,4 +2552,4 @@
-  <a href="/wiki/Black_Tie_White_Noise" title="Black Tie White Noise">
-   Black Tie White Noise
-  </a>
- </i>
+ <a href="/wiki/Black_Tie_White_Noise" title="Black Tie White Noise">
+  Black Tie White Noise
+ </a>
+</i>
@@ -2149,2 +2558,2 @@
-  Let's Dance
- </i>
+ Let's Dance
+</i>
@@ -2153,2 +2562,2 @@
-  Nile Rodgers
- </a>
+ Nile Rodgers
+</a>
@@ -2157,2 +2566,2 @@
-  Jump They Say
- </a>
+ Jump They Say
+</a>
@@ -2162,4 +2571,4 @@
-  <a href="/wiki/The_Buddha_of_Suburbia_(soundtrack)" title="The Buddha of Suburbia (soundtrack)">
-   The Buddha of Suburbia
-  </a>
- </i>
+ <a href="/wiki/The_Buddha_of_Suburbia_(soundtrack)" title="The Buddha of Suburbia (soundtrack)">
+  The Buddha of Suburbia
+ </a>
+</i>
@@ -2168,2 +2577,2 @@
-  Hanif Kureishi's novel
- </a>
+ Hanif Kureishi's novel
+</a>
@@ -2172,2 +2581,2 @@
-  Black Tie White Noise
- </i>
+ Black Tie White Noise
+</i>
@@ -2176,2 +2585,2 @@
-  alternative rock
- </a>
+ alternative rock
+</a>
@@ -2179,2 +2588 @@
-</p>
-<p>
+ <p>
@@ -2253 +2661,9 @@
- ".
+ <ins>
+  <span style="padding-left:.2em;">
+   "
+  </span>
+  .
+ </ins>
+ <del>
+  ".
+ </del>
@@ -2279 +2695,8 @@
-   Bowie on-stage with
+   Bowie
+   <ins>
+    on stage
+   </ins>
+   <del>
+    on-stage
+   </del>
+   with
@@ -2418,0 +2842,3 @@
+   <ins>
+    .
+   </ins>
@@ -2550,10 +2976,19 @@
- <span class="mw-headline" id="2013.E2.80.93present:_The_Next_Day_and_Blackstar">
-  2013–present:
-  <i>
-   The Next Day
-  </i>
-  and
-  <i>
-   Blackstar
-  </i>
- </span>
+ <ins>
+  <span class="mw-headline" id="2013.E2.80.9316:_Final_years">
+   2013–16: Final years
+  </span>
+ </ins>
+</h3>
+<h3>
+ <del>
+  <span class="mw-headline" id="2013.E2.80.93present:_The_Next_Day_and_Blackstar">
+   2013–present:
+   <i>
+    The Next Day
+   </i>
+   and
+   <i>
+    Blackstar
+   </i>
+  </span>
+ </del>
@@ -2588 +3023,12 @@
- A music video for the single was released onto
+</p>
+<p>
+ A music video for
+ <ins>
+  <i>
+   Where Are We Now?
+  </i>
+ </ins>
+ <del>
+  the single
+ </del>
+ was released onto
@@ -2599 +3045,2 @@
-  iTunes Chart
+  iTunes
+  Link: /wiki/ITunes Chart Chart
@@ -2637,2 +3083,0 @@
-</p>
-<p>
@@ -2643 +3088,5 @@
- " has created some controversy, initially being removed from YouTube for
+ "
+ <del>
+  has
+ </del>
+ created some controversy, initially being removed from YouTube for
@@ -2656,6 +3105,30 @@
- , Bowie has ruled out ever giving an interview again, with producer Visconti agreeing with the interviewer's suggestion that he is now Bowie's "voice on earth". Visconti also revealed to
- <i>
-  The Times
- </i>
- that Bowie has no intention of going on tour, noting that all "he wants to do is make records."
- An exhibition of Bowie artifacts, called "David Bowie Is," was shown at the
+ , Bowie
+ <del>
+  has
+ </del>
+ ruled out ever giving an interview
+ <ins>
+  again.
+ </ins>
+ <del>
+  again, with producer Visconti agreeing with the interviewer's suggestion that he is now Bowie's "voice on earth". Visconti also revealed to
+  <i>
+   The Times
+  </i>
+  that Bowie has no intention of going on tour, noting that all "he wants to do is make records."
+ </del>
+ An exhibition of Bowie
+ <ins>
+  artefacts,
+ </ins>
+ <del>
+  artifacts,
+ </del>
+ called "David Bowie
+ <ins>
+  Is",
+ </ins>
+ <del>
+  Is,"
+ </del>
+ was shown at the
@@ -2678 +3151,7 @@
- (The Netherlands).
+ <ins>
+  (the
+ </ins>
+ <del>
+  (The
+ </del>
+ Netherlands).
@@ -2729,5 +3208,18 @@
- In May 2015 it was announced that "Let’s Dance" would be reissued as a yellow vinyl single on 16 July 2015 in conjunction with the
- <i>
-  David Bowie is
- </i>
- ’ exhibition at the
+ In May
+ <ins>
+  2015, "Let’s Dance" was announced to
+ </ins>
+ <del>
+  2015 it was announced that "Let’s Dance" would
+ </del>
+ be reissued as a yellow vinyl single on 16 July 2015 in conjunction with the
+ <ins>
+  "David Bowie is"
+ </ins>
+ <del>
+  <i>
+   David Bowie is
+  </i>
+  ’
+ </del>
+ exhibition at the
@@ -2747 +3239,15 @@
- The show's director, Johan Renck, said of Bowie, "His first response was precise, engaged and curious. The piece of music he laid before us embodied every aspect of our characters and the series itself - dark, brooding, beautiful and sentimental (in the best possible incarnation of this word). All along, the man inspired and intrigued me and as the process passed, I was overwhelmed with his generosity. I still can't fathom what actually happened." The theme that will be used for
+ The show's director, Johan Renck, said of Bowie, "His first response was precise, engaged and curious. The piece of music he laid before us embodied every aspect of our characters and the series itself
+ <ins>
+  –
+ </ins>
+ <del>
+  -
+ </del>
+ dark, brooding, beautiful and sentimental (in the best possible incarnation of this word). All along, the man inspired and intrigued me and as the process passed, I was overwhelmed with his generosity. I still can't fathom what actually happened." The theme that
+ <ins>
+  was
+ </ins>
+ <del>
+  will be
+ </del>
+ used for
@@ -2751 +3257,7 @@
- will also be the title track for his January 2016 release
+ <ins>
+  was also
+ </ins>
+ <del>
+  will also be
+ </del>
+ the title track for his January 2016 release
@@ -2754 +3266,4 @@
-   Blackstar (★)
+   Blackstar
+   <del>
+    (★)
+   </del>
@@ -2766 +3281,7 @@
- may be the oddest work yet from [Bowie],”
+ may be the oddest work yet from
+ <ins>
+  Bowie”.
+ </ins>
+ <del>
+  [Bowie],”
+ </del>
@@ -2776 +3297,3 @@
- Bowie's career has also been punctuated by various roles in film and theatre productions, earning him some acclaim as an actor in his own right.
+ <del>
+  Bowie's career has also been punctuated by various roles in film and theatre productions, earning him some acclaim as an actor in his own right.
+ </del>
@@ -2972 +3495,8 @@
- Two years later he played
+ Two years
+ <ins>
+  later,
+ </ins>
+ <del>
+  later
+ </del>
+ he played
@@ -3152,0 +3683,3 @@
+   <ins>
+    .
+   </ins>
@@ -3157 +3690,9 @@
- From the time of his earliest recordings in the 1960s, Bowie has employed a wide variety of musical styles. His early compositions and performances were strongly influenced by not only rock and rollers like
+ From the time of his earliest recordings in the 1960s, Bowie
+ <del>
+  has
+ </del>
+ employed a wide variety of musical styles. His early compositions and performances were strongly influenced by
+ <del>
+  not only
+ </del>
+ rock and rollers like
@@ -3165 +3706,7 @@
- but also the wider world of show business. He particularly strove to emulate the British musical theatre singer-songwriter and actor
+ <ins>
+  , and
+ </ins>
+ <del>
+  but
+ </del>
+ also the wider world of show business. He particularly strove to emulate the British musical theatre singer-songwriter and actor
@@ -3208 +3755,18 @@
- Bowie is known as a multi-instrumentalist. In addition to his playing of guitar, keyboards, harmonica and saxophone, he plays stylophone,
+ Bowie
+ <ins>
+  was
+ </ins>
+ <del>
+  is
+ </del>
+ known as a multi-instrumentalist. In addition to his playing of guitar, keyboards, harmonica and saxophone, he
+ <ins>
+  played
+  <a href="/wiki/Stylophone" title="Stylophone">
+   stylophone
+  </a>
+  ,
+ </ins>
+ <del>
+  plays stylophone,
+ </del>
@@ -3213,4 +3777,9 @@
- <a href="/wiki/Cello" title="Cello">
-  cello
- </a>
- ,
+ <ins>
+  cello,
+ </ins>
+ <del>
+  <a href="/wiki/Cello" title="Cello">
+   cello
+  </a>
+  ,
+ </del>
@@ -3231 +3800 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -3244,0 +3814,3 @@
+   <ins>
+    .
+   </ins>
@@ -3270,0 +3843,9 @@
+ <ins>
+  Bowie was the principal inspiration for the bisexual glam rock icon Brian Slade in the 1998 film
+  <i>
+   <a href="/wiki/Velvet_Goldmine" title="Velvet Goldmine">
+    Velvet Goldmine
+   </a>
+  </i>
+  .
+ </ins>
@@ -3276,2 +3856,0 @@
-</p>
-<p>
@@ -3310 +3889,5 @@
-   Bowie and wife Iman
+   Bowie and wife
+   <a href="/wiki/Iman_(model)" title="Iman (model)">
+    Iman
+    Link: /wiki/Iman_(model) .
+   </a>
@@ -3323,5 +3906,15 @@
- , London. They had a son together, Zowie Bowie (now known as
- <a href="/wiki/Duncan_Jones" title="Duncan Jones">
-  Duncan Jones
- </a>
- , film director), and divorced on 8 February 1980 in Switzerland.
+ , London. They had a son together,
+ <ins>
+  <a href="/wiki/Duncan_Jones" title="Duncan Jones">
+   Duncan
+  </a>
+  , a film director,
+ </ins>
+ <del>
+  Zowie Bowie (now known as
+  <a href="/wiki/Duncan_Jones" title="Duncan Jones">
+   Duncan Jones
+  </a>
+  , film director),
+ </del>
+ and divorced on 8 February 1980 in Switzerland.
@@ -3331 +3924,5 @@
- Bowie declared himself gay in an interview with Michael Watts in 22 January 1972 issue of
+ Bowie declared himself gay in an interview with Michael Watts in
+ <ins>
+  the
+ </ins>
+ 22 January 1972 issue of
@@ -3372 +3969,11 @@
- and that "it is probably true that Bowie was never gay, nor even consistently actively bisexual ... he did, from time to time, experiment, even if only out of a sense of curiosity and a genuine allegiance with the 'transgressional.' "
+ and that "it is probably true that Bowie was never gay, nor even consistently actively bisexual ... he did, from time to time, experiment, even if only out of a sense of curiosity and a genuine allegiance with the
+ <ins>
+  'transgressional.
+  <span style="padding-right:.2em;">
+   '
+  </span>
+ </ins>
+ <del>
+  'transgressional.'
+ </del>
+ "
@@ -3386,2 +3993,21 @@
- They have one daughter, Alexandria "Lexi" Zahra Jones, born in August 2000.
- The couple resides primarily in New York City and London.
+ They
+ <ins>
+  had
+ </ins>
+ <del>
+  have
+ </del>
+ one daughter, Alexandria "Lexi" Zahra Jones, born in August 2000.
+ The couple
+ <ins>
+  resided
+ </ins>
+ <del>
+  resides
+ </del>
+ primarily in New York City and
+</p>
+<p>
+ <ins>
+  London, as well as owning an apartment in Sydney.
+ </ins>
@@ -3390,3 +4016,5 @@
- <span class="mw-headline" id="Religion">
-  Religion
- </span>
+ <ins>
+  <span class="mw-headline" id="Religion">
+   Religion
+  </span>
+ </ins>
@@ -3394,2 +4022,23 @@
-<p>
- Regarding his religion, in 2005 he said, "Questioning my spiritual life has always been germane to what I was writing. Always." He added that he is bothered by being "not quite an atheist".
+<ins>
+ Regarding
+</ins>
+<del>
+ London.
+</del>
+<h3>
+ <del>
+  <span class="mw-headline" id="Religion">
+   Religion
+  </span>
+ </del>
+</h3>
+<del>
+ Regarding his
+</del>
+  religion, in 2005 he said, "Questioning my spiritual life has always been germane to what I was writing. Always." He added that he <ins>
+ was
+</ins>
+<del>
+ is
+</del>
+ bothered by being "not quite an atheist".
@@ -3398,5 +4047,17 @@
-  <a href="/wiki/Esquire_(magazine)" title="Esquire (magazine)">
-   Esquire
-  </a>
- </i>
- interview "What I've Learned", he stated, "I'm in awe of the universe, but I don't necessarily believe there's an intelligence or agent behind it. I do have a passion for the visual in religious rituals, though, even though they may be completely empty and bereft of substance. The incense is powerful and provocative, whether Buddhist or Catholic."
+ <a href="/wiki/Esquire_(magazine)" title="Esquire (magazine)">
+  Esquire
+ </a>
+</i>
+ interview "What I've Learned", he stated, "I'm in awe of the universe, but I don't necessarily believe there's an intelligence or agent behind it. I do have a passion for the visual in religious rituals, though, even though they may be completely empty and bereft of substance. The incense is powerful and provocative, whether Buddhist or
+ <p>
+ <ins>
+  <a class="mw-redirect" href="/wiki/Roman_Catholic" title="Roman Catholic">
+   Catholic
+  </a>
+  ."
+Bowie showed
+ </ins>
+ <del>
+  Catholic."
+Bowie has shown
+ </del>
@@ -3404,2 +4065 @@
-<p>
- Bowie has shown an interest in
+ an interest in
@@ -3407,3 +4067,9 @@
-  Buddhism
- </a>
- since 1967. He frequently studied in London under the
+ Buddhism
+</a>
+<ins>
+ that began in
+</ins>
+<del>
+ since
+</del>
+ 1967. He frequently studied in London under the
@@ -3411,8 +4077,8 @@
-  Tibetan
- </a>
- <a href="/wiki/Lama" title="Lama">
-  Lama
- </a>
- <a href="/wiki/Chime_Rinpoche" title="Chime Rinpoche">
-  Chime Rinpoche
- </a>
+ Tibetan
+</a>
+<a href="/wiki/Lama" title="Lama">
+ Lama
+</a>
+<a href="/wiki/Chime_Rinpoche" title="Chime Rinpoche">
+ Chime Rinpoche
+</a>
@@ -3421,2 +4087,2 @@
-  '
- </span>
+ '
+</span>
@@ -3426,4 +4092,4 @@
-  <a href="/wiki/David_Bowie_(1967_album)" title="David Bowie (1967 album)">
-   David Bowie
-  </a>
- </i>
+ <a href="/wiki/David_Bowie_(1967_album)" title="David Bowie (1967 album)">
+  David Bowie
+ </a>
+</i>
@@ -3432,8 +4098,14 @@
-  Crazy wisdom
- </a>
- <a href="/wiki/Tulku" title="Tulku">
-  Tulku
- </a>
- <a href="/wiki/Ch%C3%B6gyam_Trungpa" title="Chögyam Trungpa">
-  Chögyam Trungpa
- </a>
+ <ins>
+  crazy
+ </ins>
+ <del>
+  Crazy
+ </del>
+ wisdom
+</a>
+<a href="/wiki/Tulku" title="Tulku">
+ Tulku
+</a>
+<a href="/wiki/Ch%C3%B6gyam_Trungpa" title="Chögyam Trungpa">
+ Chögyam Trungpa
+</a>
@@ -3441,2 +4113 @@
-</p>
-<h3>
+ <h3>
@@ -3450 +4121,7 @@
-  The Thin White Duke
+  <ins>
+   the
+  </ins>
+  <del>
+   The
+  </del>
+  Thin White Duke
@@ -3471,5 +4148,29 @@
- Bowie later retracted these comments and blamed them on mental instability caused by his drug problems at the time, saying: "I was out of my mind, totally, completely crazed."
-</p>
-<h3>
- <span class="mw-headline" id="Legal_issues">
-  Legal issues
+ Bowie later retracted these comments
+ <ins>
+  in an interview with
+  <i>
+   <a href="/wiki/Melody_Maker" title="Melody Maker">
+    Melody Maker
+   </a>
+  </i>
+  in October 1977, blaming
+ </ins>
+ <del>
+  and blamed
+ </del>
+ them on mental instability caused by his drug problems at the time, saying: "I was out of my mind, totally, completely crazed."
+ <ins>
+  In the
+  <i>
+   Melody Maker
+  </i>
+  interview, he claimed to be
+  <a class="mw-redirect" href="/wiki/Apolitical" title="Apolitical">
+   apolitical
+  </a>
+  , stating: "The more I travel and the less sure I am about exactly which political philosophies are commendable. The more government systems I see, the less enticed I am to give my allegiance to any set of people, so it would be disastrous for me to adopt a definitive point of view, or to adopt a party of people and say 'these are my people'".
+ </ins>
+</p>
+<h3>
+ <span class="mw-headline" id="Legal_issues">
+  Legal issues
@@ -3479 +4180,11 @@
- In 1990, British rock band Queen and artist David Bowie filed a lawsuit against
+ In 1990,
+ <ins>
+  Queen and
+ </ins>
+ <del>
+  British rock band Queen and artist David
+ </del>
+ Bowie filed a lawsuit against
+ <ins>
+  American rapper
+ </ins>
@@ -3493,0 +4205,47 @@
+<h3>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  On 10 January 2016, two days after releasing the album
+  <i>
+   <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
+    Blackstar
+   </a>
+  </i>
+  on his 69th birthday, Bowie died from
+  <a href="/wiki/Liver_cancer" title="Liver cancer">
+   liver cancer
+  </a>
+  at his New York home. He had been diagnosed with the malignancy eighteen months earlier.
+ Belgian theatre director
+  <a href="/wiki/Ivo_van_Hove" title="Ivo van Hove">
+   Ivo van Hove
+  </a>
+  , who worked with the singer on his
+  <a href="/wiki/Off-Broadway" title="Off-Broadway">
+   Off-Broadway
+  </a>
+  musical
+  <i>
+   Lazarus
+  </i>
+  , explained that Bowie was unable to attend rehearsals due to progression of the disease. He noted that "Bowie was still writing on his deathbed, I saw a man fighting. He fought like a lion and kept working like a lion through it all."
+ Bowie's producer Tony Visconti wrote:
+ </ins>
+</p>
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   He always did what he wanted to do. And he wanted to do it his way and he wanted to do it the best way. His death was no different from his life – a work of Art. He made
+   <i>
+    Blackstar
+   </i>
+   for us, his parting gift. I knew for a year this was the way it would be. I wasn't, however, prepared for it. He was an extraordinary man, full of love and life. He will always be with us. For now, it is appropriate to cry.
+  </ins>
+ </p>
+</blockquote>
@@ -3504,0 +4263,19 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    The spider
+   </ins>
+   <ins>
+    <i>
+     <a href="/wiki/Heteropoda_davidbowie" title="Heteropoda davidbowie">
+      Heteropoda davidbowie
+     </a>
+    </i>
+    .
+   </ins>
+  </div>
+ </div>
+</div>
@@ -3521,2 +4298 @@
- .
- In the ensuing decades he has been honoured with numerous awards for his music and its accompanying videos, receiving, among others, two
+ . In the ensuing decades he has been honoured with numerous awards for his music and its accompanying videos, receiving, among others, two
@@ -3527,4 +4303,12 @@
- <a class="mw-redirect" href="/wiki/BRIT_Awards" title="BRIT Awards">
-  BRIT Awards
- </a>
- .
+ <ins>
+  <a href="/wiki/Brit_Awards" title="Brit Awards">
+   Brit Awards
+  </a>
+  —winning Best British Male Artist twice and in 1996 the award for Outstanding Contribution to Music.
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/BRIT_Awards" title="BRIT Awards">
+   BRIT Awards
+  </a>
+  .
+ </del>
@@ -3545,2 +4329,9 @@
-  Commander of the British Empire
- </a>
+  Commander of the
+  <ins>
+   Order of the
+  </ins>
+  British Empire
+ </a>
+ <ins>
+  (CBE)
+ </ins>
@@ -3552,3 +4343,17 @@
-</p>
-<p>
- Throughout his career he has sold an estimated 140 million albums. In the United Kingdom, he has been awarded 9 Platinum, 11 Gold and 8 Silver albums, and in the United States, 5 Platinum and 7 Gold.
+ <ins>
+  Bowie explained his reluctance, saying: “I would never have any intention of accepting anything like that. I seriously don’t know what it’s for. It’s not what I spent my life working for.”
+ </ins>
+</p>
+<p>
+ Throughout his career he
+ <del>
+  has
+ </del>
+ sold an estimated 140 million albums. In the United Kingdom, he
+ <ins>
+  was
+ </ins>
+ <del>
+  has been
+ </del>
+ awarded 9 Platinum, 11 Gold and 8 Silver albums, and in the United States, 5 Platinum and 7 Gold.
@@ -3559 +4364,2 @@
- , he was ranked 29. In 2004,
+ , he was ranked 29.
+ In 2004,
@@ -3561 +4367,3 @@
-  Rolling Stone
+  <a class="mw-redirect" href="/wiki/Rolling_Stone_(magazine)" title="Rolling Stone (magazine)">
+   Rolling Stone
+  </a>
@@ -3563 +4371,6 @@
- magazine ranked him 39th on their list of the 100 Greatest Rock Artists of All Time
+ magazine ranked him 39th on their list of the 100 Greatest Rock Artists of All
+ <ins>
+  Time.
+ </ins>
+ <del>
+  Time
@@ -3564,0 +4378 @@
+ </del>
@@ -3574,0 +4389,9 @@
+ <ins>
+  The spider
+  <i>
+   <a href="/wiki/Heteropoda_davidbowie" title="Heteropoda davidbowie">
+    Heteropoda davidbowie
+   </a>
+  </i>
+  is named in his honour.
+ </ins>
@@ -3756 +4579,6 @@
-   (1993)
+   <ins>
+    (1992)
+   </ins>
+   <del>
+    (1993)
+   </del>
@@ -3840,108 +4668,11 @@
-<ul>
- <li>
-  <i>
-   <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
-    The Man Who Fell to Earth
-   </a>
-  </i>
-  (1976) as Thomas Jerome Newton; received
-  <a href="/wiki/Saturn_Award_for_Best_Actor" title="Saturn Award for Best Actor">
-   Saturn Award for Best Actor
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Just_a_Gigolo_(1978_film)" title="Just a Gigolo (1978 film)">
-    Just a Gigolo
-   </a>
-  </i>
-  (1978) as Paul Ambrosius von Przygodski
- </li>
- <li>
-  <i>
-   <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
-    Christiane F.
-   </a>
-  </i>
-  (1981) cameo as himself
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Snowman" title="The Snowman">
-    The Snowman
-   </a>
-  </i>
-  (1982) narrator in re-released version
- </li>
- <li>
-  <i>
-   Baal
-  </i>
-  (1982) as Baal
- </li>
- <li>
-  <i>
-   <a href="/wiki/Yellowbeard" title="Yellowbeard">
-    Yellowbeard
-   </a>
-  </i>
-  (1983) the sailor wearing shark fin - The Shark
- </li>
- <li>
-  <i>
-   <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
-    Merry Christmas, Mr. Lawrence
-   </a>
-  </i>
-  (1983) as Maj. Jack 'Strafer' Celliers
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Hunger_(1983_film)" title="The Hunger (1983 film)">
-    The Hunger
-   </a>
-  </i>
-  (1983) as John Blaylock
- </li>
- <li>
-  <i>
-   <a href="/wiki/Jazzin%27_for_Blue_Jean" title="Jazzin' for Blue Jean">
-    Jazzin' for Blue Jean
-   </a>
-  </i>
-  (1984) as Vic and Screaming Lord Byron
- </li>
- <li>
-  <i>
-   <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
-    Labyrinth
-   </a>
-  </i>
-  (1986) as
-  <a href="/wiki/Jareth" title="Jareth">
-   Jareth the Goblin King
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Absolute_Beginners_(film)" title="Absolute Beginners (film)">
-    Absolute Beginners
-   </a>
-  </i>
-  (1986) as Vendice Partners
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
-    The Last Temptation of Christ
-   </a>
-  </i>
-  (1988) as
-  <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
-   Pontius Pilate
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Linguini_Incident" title="The Linguini Incident">
-    The Linguini Incident
+<div class="div-col columns column-count column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2;">
+ <ul>
+  <li>
+   <i>
+    <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
+     The Man Who Fell to Earth
+    </a>
+   </i>
+   (1976) as Thomas Jerome Newton; received
+   <a href="/wiki/Saturn_Award_for_Best_Actor" title="Saturn Award for Best Actor">
+    Saturn Award for Best Actor
@@ -3949,13 +4680,89 @@
-  </i>
-  (1991) as Monte
- </li>
- <li>
-  <a href="/wiki/Dream_On_(TV_series)" title="Dream On (TV series)">
-   Dream On (TV series)
-  </a>
-  (1991) as Sir Roland Moorecock
- </li>
- <li>
-  <i>
-   <a href="/wiki/Basquiat_(film)" title="Basquiat (film)">
-    Basquiat
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Just_a_Gigolo_(1978_film)" title="Just a Gigolo (1978 film)">
+     Just a Gigolo
+    </a>
+   </i>
+   (1978) as Paul Ambrosius von Przygodski
+  </li>
+  <li>
+   <i>
+    <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
+     Christiane F.
+    </a>
+   </i>
+   (1981) cameo as himself
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Snowman" title="The Snowman">
+     The Snowman
+    </a>
+   </i>
+   (1982) narrator in re-released version
+  </li>
+  <li>
+   <i>
+    Baal
+   </i>
+   (1982) as Baal
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Yellowbeard" title="Yellowbeard">
+     Yellowbeard
+    </a>
+   </i>
+   (1983) the sailor wearing shark fin
+   <ins>
+    –
+   </ins>
+   <del>
+    -
+   </del>
+   The Shark
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
+     Merry Christmas, Mr. Lawrence
+    </a>
+   </i>
+   (1983) as Maj. Jack 'Strafer' Celliers
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Hunger_(1983_film)" title="The Hunger (1983 film)">
+     The Hunger
+    </a>
+   </i>
+   (1983) as John Blaylock
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Jazzin%27_for_Blue_Jean" title="Jazzin' for Blue Jean">
+     Jazzin' for Blue Jean
+    </a>
+   </i>
+   (1984) as Vic and Screaming Lord Byron
+  </li>
+  <li>
+   <ins>
+    <i>
+     <a href="/wiki/Into_the_Night_(film)" title="Into the Night (film)">
+      Into the Night
+     </a>
+    </i>
+    (1985) as Colin Morris
+   </ins>
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
+     Labyrinth
+    </a>
+   </i>
+   (1986) as
+   <a href="/wiki/Jareth" title="Jareth">
+    Jareth
@@ -3963,10 +4770,22 @@
-  </i>
-  (1996) as
-  <a href="/wiki/Andy_Warhol" title="Andy Warhol">
-   Andy Warhol
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Gunslinger%27s_Revenge" title="Gunslinger's Revenge">
-    Gunslinger's Revenge
+   the Goblin King
+   <del>
+    Link: /wiki/Jareth
+   </del>
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Absolute_Beginners_(film)" title="Absolute Beginners (film)">
+     Absolute Beginners
+    </a>
+   </i>
+   (1986) as Vendice Partners
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
+     The Last Temptation of Christ
+    </a>
+   </i>
+   (1988) as
+   <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
+    Pontius Pilate
@@ -3974,7 +4793,30 @@
-  </i>
-  (1998) as Jack Sikora
- </li>
- <li>
-  <i>
-   <a href="/wiki/Everybody_Loves_Sunshine" title="Everybody Loves Sunshine">
-    Everybody Loves Sunshine
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Linguini_Incident" title="The Linguini Incident">
+     The Linguini Incident
+    </a>
+   </i>
+   (1991) as Monte
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Dream_On_(TV_series)" title="Dream On (TV series)">
+     Twin Peaks: Fire Walk with Me
+     Link: /wiki/Twin_Peaks:_Fire_Walk_with_Me   (1992) as
+   Phillip Jeffries
+    Link: /wiki/Phillip_Jeffries   Dream On
+    Dream On (TV series)
+    </a>
+   </i>
+   (1991) as Sir Roland Moorecock
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Basquiat_(film)" title="Basquiat (film)">
+     Basquiat
+    </a>
+   </i>
+   (1996) as
+   <a href="/wiki/Andy_Warhol" title="Andy Warhol">
+    Andy Warhol
@@ -3982,13 +4824,32 @@
-  </i>
-  (1999) as Bernie
- </li>
- <li>
-  <i>
-   Mr. Rice's Secret
-  </i>
-  (2000) as William Rice
- </li>
- <li>
-  <i>
-   <a href="/wiki/Zoolander" title="Zoolander">
-    Zoolander
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Gunslinger%27s_Revenge" title="Gunslinger's Revenge">
+     Gunslinger's Revenge
+    </a>
+   </i>
+   (1998) as Jack Sikora
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Everybody_Loves_Sunshine" title="Everybody Loves Sunshine">
+     Everybody Loves Sunshine
+    </a>
+   </i>
+   (1999) as Bernie
+  </li>
+  <li>
+   <i>
+    Mr. Rice's Secret
+   </i>
+   (2000) as William Rice
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Zoolander" title="Zoolander">
+     Zoolander
+    </a>
+   </i>
+   (2001) cameo as himself; nominated for
+   <a class="mw-redirect" href="/wiki/MTV_Movie_Award" title="MTV Movie Award">
+    MTV Movie Award
@@ -3996,10 +4857,10 @@
-  </i>
-  (2001) cameo as himself; nominated for
-  <a class="mw-redirect" href="/wiki/MTV_Movie_Award" title="MTV Movie Award">
-   MTV Movie Award
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
-    The Prestige
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
+     The Prestige
+    </a>
+   </i>
+   (2006) as
+   <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
+    Nikola Tesla
@@ -4007,7 +4868,3 @@
-  </i>
-  (2006) as
-  <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
-   Nikola Tesla
-  </a>
- </li>
-</ul>
+  </li>
+ </ul>
+</div>
@@ -4106 +4963 @@
-</ul>
+</ul>
\ No newline at end of file
