--- 699199631
+++ 699667984
@@ -1,0 +2,6 @@
+ <ins>
+  <b>
+   David Robert Jones
+  </b>
+  (8 January 1947 – 10 January 2016), known as
+ </ins>
@@ -31 +37,6 @@
- ;
+ <ins>
+  ),
+ was
+ </ins>
+ <del>
+  ;
@@ -33,11 +44,38 @@
- <b>
-  David Robert Jones
- </b>
- , 8 January 1947) is an English singer, songwriter, multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been a figure in popular music for over four decades, and is known as an innovator, particularly for his work in the 1970s. His
- <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
-  androgynous
- </a>
- appearance was an iconic element of his image, particularly in the 1970s and 1980s.
-</p>
-<p>
- Bowie's first hit song, "
+  <b>
+   David Robert Jones
+  </b>
+  , 8 January 1947) is
+ </del>
+ an English singer, songwriter,
+ <ins>
+  record producer, painter and actor. He was
+ </ins>
+ <del>
+  multi-instrumentalist, record producer, arranger, painter, and actor. Bowie has been
+ </del>
+ a figure in
+ <a href="/wiki/Popular_music" title="Popular music">
+  popular music
+ </a>
+ <del>
+  for over four decades, and is known
+ </del>
+ as an innovator, particularly for his work in the 1970s.
+</p>
+<p>
+ <ins>
+  Born and raised in
+  <a href="/wiki/South_London" title="South London">
+   south London
+  </a>
+  , Bowie developed an interest in music while at school, where he excelled at playing the recorder. When he left school he studied art, music and design, and became proficient on the saxophone, forming his first band that year at the age of 15. He embarked on a professional career as a musician in 1963, and received his first management contract shortly afterwards.
+ </ins>
+ <del>
+  His
+  <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
+   androgynous
+  </a>
+  appearance was an iconic element of his image, particularly in the 1970s and 1980s.
+Bowie's first hit song,
+ </del>
+ "
@@ -47 +85,8 @@
- " in July 1969, reached the top five of the
+ "
+ <ins>
+  became his first top five entry on
+ </ins>
+ <del>
+  in July 1969, reached the top five of
+ </del>
+ the
@@ -51 +96,7 @@
- . After a three-year period of experimentation, he re-emerged in 1972 during the
+ <ins>
+  after its release in July 1969. After a
+ </ins>
+ <del>
+  . After a three-year
+ </del>
+ period of experimentation, he re-emerged in 1972 during the
@@ -55,4 +106,10 @@
- era with the flamboyant,
- <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
-  androgynous
- </a>
+ era with
+ <ins>
+  his flamboyant and androgynous
+ </ins>
+ <del>
+  the flamboyant,
+  <a class="mw-redirect" href="/wiki/Androgynous" title="Androgynous">
+   androgynous
+  </a>
+ </del>
@@ -62 +119,8 @@
- Ziggy Stardust, spearheaded by the hit single "
+ Ziggy
+ <ins>
+  Stardust. The character was spearheaded by his
+ </ins>
+ <del>
+  Stardust, spearheaded by the hit
+ </del>
+ single "
@@ -66 +130,5 @@
- " and the album
+ " and
+ <del>
+  the
+ </del>
+ album
@@ -72 +140,4 @@
- . Bowie's impact at that time, as described by biographer David Buckley, "challenged the core belief of the rock music of its day" and "created perhaps the biggest cult in popular culture."
+ .
+ <del>
+  Bowie's impact at that time, as described by biographer David Buckley, "challenged the core belief of the rock music of its day" and "created perhaps the biggest cult in popular culture."
+ </del>
@@ -80 +151,5 @@
- " and the hit album
+ " and the
+ <del>
+  hit
+ </del>
+ album
@@ -100 +175,7 @@
- , the first of three collaborations with
+ <ins>
+  (1977),
+ </ins>
+ <del>
+  ,
+ </del>
+ the first of three collaborations with
@@ -104,11 +185,23 @@
- .
- <i>
-  Low
- </i>
- (1977),
- <i>
-  <a href="/wiki/%22Heroes%22_(David_Bowie_album)" title='"Heroes" (David Bowie album)'>
-   "Heroes"
-  </a>
- </i>
- (1977), and
+ <ins>
+  later known as the "Berlin Trilogy".
+  <i>
+   <a href="/wiki/%22Heroes%22_(David_Bowie_album)" title='"Heroes" (David Bowie album)'>
+    "Heroes"
+   </a>
+  </i>
+  (1977)
+ </ins>
+ <del>
+  .
+  <i>
+   Low
+  </i>
+  (1977),
+  <i>
+   <a href="/wiki/%22Heroes%22_(David_Bowie_album)" title='"Heroes" (David Bowie album)'>
+    "Heroes"
+   </a>
+  </i>
+  (1977),
+ </del>
+ and
@@ -120 +213,7 @@
- (1979)—the so-called "Berlin Trilogy" albums—all reached the UK top five and received lasting critical praise. After uneven commercial success in the late 1970s, Bowie had UK number ones with the 1980 single "
+ <ins>
+  (1979) followed; each album
+ </ins>
+ <del>
+  (1979)—the so-called "Berlin Trilogy" albums—all
+ </del>
+ reached the UK top five and received lasting critical praise. After uneven commercial success in the late 1970s, Bowie had UK number ones with the 1980 single "
@@ -144,13 +243,28 @@
- , which yielded several hit singles. Throughout the 1990s and 2000s, Bowie continued to experiment with musical styles, including
- <a href="/wiki/Blue-eyed_soul" title="Blue-eyed soul">
-  blue-eyed soul
- </a>
- ,
- <a href="/wiki/Industrial_music" title="Industrial music">
-  industrial
- </a>
- ,
- <a href="/wiki/Adult_contemporary_music" title="Adult contemporary music">
-  adult contemporary
- </a>
- , and
+ , which yielded several
+ <ins>
+  successful
+ </ins>
+ <del>
+  hit
+ </del>
+ singles. Throughout the 1990s and 2000s, Bowie continued to experiment with musical styles, including
+ <ins>
+  <a href="/wiki/Industrial_music" title="Industrial music">
+   industrial
+  </a>
+ </ins>
+ <del>
+  <a href="/wiki/Blue-eyed_soul" title="Blue-eyed soul">
+   blue-eyed soul
+  </a>
+  ,
+  <a href="/wiki/Industrial_music" title="Industrial music">
+   industrial
+  </a>
+  ,
+  <a href="/wiki/Adult_contemporary_music" title="Adult contemporary music">
+   adult contemporary
+  </a>
+  ,
+ </del>
+ and
@@ -160,11 +274 @@
- . He has not toured since the 2003–2004
- <a class="mw-redirect" href="/wiki/Reality_Tour" title="Reality Tour">
-  Reality Tour
- </a>
- and has not performed live since 2006. Bowie's latest studio album,
- <i>
-  <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
-   Blackstar
-  </a>
- </i>
- , was released on 8 January 2016, his 69th birthday.
+ .
@@ -173 +277,63 @@
- David Buckley says of Bowie: "His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
+ <ins>
+  Bowie also had a successful but sporadic
+  <a href="/wiki/David_Bowie_filmography" title="David Bowie filmography">
+   film career
+  </a>
+  . His acting roles include the eponymous character in
+  <i>
+   <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
+    The Man Who Fell to Earth
+   </a>
+  </i>
+  (1976), Major Celliers in
+  <i>
+   <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
+    Merry Christmas, Mr. Lawrence
+   </a>
+  </i>
+  (1983), the Goblin King
+  <a href="/wiki/Jareth" title="Jareth">
+   Jareth
+  </a>
+  in
+  <i>
+   <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
+    Labyrinth
+   </a>
+  </i>
+  (1986),
+  <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
+   Pontius Pilate
+  </a>
+  in
+  <i>
+   <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
+    The Last Temptation of Christ
+   </a>
+  </i>
+  (1988), and
+  <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
+   Nikola Tesla
+  </a>
+  in
+  <i>
+   <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
+    The Prestige
+   </a>
+  </i>
+  (2006), among other film and television appearances and cameos.
+Bowie's impact was enormous; he changed the nature of rock music, and changed his own approach repeatedly. During
+ </ins>
+ <del>
+  He has not toured since the 2003–2004
+  <a class="mw-redirect" href="/wiki/Reality_Tour" title="Reality Tour">
+   Reality Tour
+  </a>
+  and has not performed live since 2006. Bowie's latest studio album,
+  <i>
+   <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
+    Blackstar
+   </a>
+  </i>
+  , was released on 8 January 2016, his 69th birthday.
+David Buckley says of Bowie: "His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
@@ -175,5 +341,20 @@
- <a href="/wiki/100_Greatest_Britons" title="100 Greatest Britons">
-  100 Greatest Britons
- </a>
- , Bowie was placed at number 29. Throughout his career, he has sold an estimated 140 million records worldwide.
- In the UK, he has been awarded nine Platinum album certifications, eleven Gold and eight Silver, and in the US, five Platinum and seven Gold certifications. In 2004,
+  <a href="/wiki/100_Greatest_Britons" title="100 Greatest Britons">
+   100 Greatest Britons
+  </a>
+  , Bowie was placed at number 29. Throughout
+ </del>
+</p>
+ his career, he <del>
+ has
+</del>
+ sold an estimated 140 million records worldwide. In the UK, he <ins>
+ was awarded nine platinum
+</ins>
+<del>
+ has been awarded nine Platinum
+</del>
+ album certifications, eleven <ins>
+ gold and eight silver, and in the US received five platinum and seven gold certifications.
+</ins>
+<del>
+ Gold and eight Silver, and in the US, five Platinum and seven Gold certifications. In 2004,
@@ -185 +366,3 @@
- ranked him 39th on their list of the "100 Greatest Artists of All Time" and 23rd on their list of the best singers of all time. He was inducted into the
+ ranked him 39th on their list of the "100 Greatest Artists of All Time" and 23rd on their list of the best singers of all time.
+</del>
+ He was inducted into the
@@ -187 +370,8 @@
-  Rock and Roll Hall of Fame
+ Rock and Roll Hall of Fame
+</a>
+ in 1996. <ins>
+ Bowie stopped concert touring after 2004, and last performed live at a charity event in 2006. In 2013, he returned from a decade-long recording hiatus, remaining musically active until his death from
+</ins>
+<ins>
+ <a href="/wiki/Liver_cancer" title="Liver cancer">
+  liver cancer
@@ -189 +379,14 @@
- in 1996.
+ three years later.
+</ins>
+<p>
+</p>
+<p>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Early_life">
+   Early life
+  </span>
+ </ins>
+</h2>
+<p>
@@ -196,3 +399,5 @@
- <span class="mw-headline" id="Biography">
-  Biography
- </span>
+ <del>
+  <span class="mw-headline" id="Biography">
+   Biography
+  </span>
+ </del>
@@ -201,3 +406,5 @@
- <span class="mw-headline" id="Early_life">
-  Early life
- </span>
+ <del>
+  <span class="mw-headline" id="Early_life">
+   Early life
+  </span>
+ </del>
@@ -205,2 +412,7 @@
-<p>
- David Bowie was born David Robert Jones on 8 January 1947, in
+<del>
+ David
+</del>
+ Bowie was born <del>
+ David Robert Jones on 8 January 1947,
+</del>
+ in
@@ -208 +420,9 @@
-  Brixton
+ Brixton
+</a>
+ , <ins>
+ south
+</ins>
+ London. His mother, Margaret Mary "Peggy" (née Burns), from
+ <ins>
+ <a href="/wiki/Kent" title="Kent">
+  Kent
@@ -210 +430,5 @@
- , London. His mother, Margaret Mary "Peggy" (née Burns), from Kent,
+ ,
+</ins>
+<del>
+ Kent,
+</del>
@@ -212,2 +436,13 @@
- while his father, Haywood Stenton "John" Jones, from Yorkshire,
- was a promotions officer for
+ while his father, Haywood Stenton "John" Jones, from
+ <ins>
+ <a href="/wiki/Yorkshire" title="Yorkshire">
+  Yorkshire
+ </a>
+ ,
+</ins>
+<del>
+ Yorkshire,
+ was
+</del>
+ a promotions officer for <ins>
+ the children's charity
@@ -217,3 +452,5 @@
- . The family lived at 40 Stansfield Road, located near the border of the south London areas of Brixton and
- <a href="/wiki/Stockwell" title="Stockwell">
-  Stockwell
+ , was of Welsh descent.
+</ins>
+<del>
+ <a href="/wiki/Barnardo%27s" title="Barnardo's">
+  Barnardo's
@@ -220,0 +458,9 @@
+ .
+</del>
+ The family lived at 40 Stansfield Road, <del>
+ located
+</del>
+ near the border of the south London areas of Brixton and
+ <a href="/wiki/Stockwell" title="Stockwell">
+ Stockwell
+</a>
@@ -222,3 +468,9 @@
-</p>
-<p>
- In 1953 the family moved to the suburb of
+ <p>
+ In
+ <ins>
+  1953, Bowie moved with his family
+ </ins>
+ <del>
+  1953 the family moved
+ </del>
+ to the suburb of
@@ -228,5 +480,22 @@
- , where, two years later, Bowie progressed to Burnt Ash Junior School. His voice was considered "adequate" by the school choir, and his
- <a href="/wiki/Recorder_(musical_instrument)" title="Recorder (musical instrument)">
-  recorder
- </a>
- playing judged to demonstrate above-average musical ability.
+ , where, two years later,
+ <ins>
+  he
+ </ins>
+ <del>
+  Bowie
+ </del>
+ progressed to Burnt Ash Junior School. His voice was considered "adequate" by the school choir, and
+ <ins>
+  he demonstrated above-average abilities in playing the
+  <a href="/wiki/Recorder_(musical_instrument)" title="Recorder (musical instrument)">
+   recorder
+  </a>
+  .
+ </ins>
+ <del>
+  his
+  <a href="/wiki/Recorder_(musical_instrument)" title="Recorder (musical instrument)">
+   recorder
+  </a>
+  playing judged to demonstrate above-average musical ability.
+ </del>
@@ -267,0 +537,2 @@
+</p>
+<p>
@@ -294 +565,7 @@
- Failing his
+ <ins>
+  After taking
+ </ins>
+ <del>
+  Failing
+ </del>
+ his
@@ -298 +575,7 @@
- at the conclusion of his Burnt Ash Junior education, Bowie joined
+ at the conclusion of his Burnt Ash Junior education, Bowie
+ <ins>
+  went to
+ </ins>
+ <del>
+  joined
+ </del>
@@ -305 +588,7 @@
- It was an unusual technical school, as biographer Christopher Sandford writes:
+ It was an unusual technical school, as biographer Christopher Sandford
+ <ins>
+  wrote:
+ </ins>
+ <del>
+  writes:
+ </del>
@@ -334,4 +623,10 @@
- <a href="/wiki/Jazz" title="Jazz">
-  jazz
- </a>
- , his enthusiasm for players like
+ <ins>
+  jazz,
+ </ins>
+ <del>
+  <a href="/wiki/Jazz" title="Jazz">
+   jazz
+  </a>
+  ,
+ </del>
+ his enthusiasm for players like
@@ -364,0 +660,7 @@
+<h2>
+ <ins>
+  <span class="mw-headline" id="Career">
+   Career
+  </span>
+ </ins>
+</h2>
@@ -366,2 +668,9 @@
- <span class="mw-headline" id="1962.E2.80.9367:_Early_career_to_d.C3.A9but_album">
-  1962–67: Early career to début album
+ <span class="mw-headline" id="1962.E2.80.9367:_Early_career_to_debut_album">
+  1962–67: Early career to
+  <ins>
+   debut
+  </ins>
+  <del>
+   début
+  </del>
+  album
@@ -373 +682 @@
-   <img alt="" class="thumbimage" data-file-height="547" data-file-width="452" height="206" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/170px-David_Bowie_%281967%29.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/255px-David_Bowie_%281967%29.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/340px-David_Bowie_%281967%29.png 2x" width="170"/>
+   <img alt="Bowie looking to the camera" class="thumbimage" data-file-height="547" data-file-width="452" height="206" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/170px-David_Bowie_%281967%29.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/255px-David_Bowie_%281967%29.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/98/David_Bowie_%281967%29.png/340px-David_Bowie_%281967%29.png 2x" width="170"/>
@@ -413 +722,8 @@
- blues numbers, Bowie quit the band less than a month later to join the Manish Boys, another blues outfit, who incorporated folk and soul — "I used to dream of being their
+ blues numbers, Bowie quit the band less than a month later to join the Manish Boys, another blues outfit, who incorporated folk and
+ <ins>
+  soul—"I
+ </ins>
+ <del>
+  soul — "I
+ </del>
+ used to dream of being their
@@ -587 +903,8 @@
- ice cream commercial, but was rejected for another by
+ ice cream commercial,
+ <ins>
+  and
+ </ins>
+ <del>
+  but
+ </del>
+ was rejected for another by
@@ -591,0 +915,11 @@
+ <ins>
+  In February and March 1969, he undertook a short tour with
+  <a href="/wiki/Marc_Bolan" title="Marc Bolan">
+   Marc Bolan
+  </a>
+  's duo
+  <a class="mw-redirect" href="/wiki/Tyrannosaurus_Rex_(band)" title="Tyrannosaurus Rex (band)">
+   Tyrannosaurus Rex
+  </a>
+  , as third on the bill, performing a mime act.
+ </ins>
@@ -596 +930,7 @@
- launch, to become a UK top five hit.
+ launch,
+ <ins>
+  and reached the top five in the UK.
+ </ins>
+ <del>
+  to become a UK top five hit.
+ </del>
@@ -601,0 +942,2 @@
+</p>
+<p>
@@ -606 +948,8 @@
- , this developed into the Beckenham Arts Lab, and became extremely popular. The Arts Lab hosted a free festival in a local park, the subject of his song "
+ , this developed into the
+ <a class="mw-redirect" href="/wiki/Beckenham_Arts_Lab" title="Beckenham Arts Lab">
+  Beckenham Arts Lab
+ </a>
+ <del>
+  Lab,
+ </del>
+ and became extremely popular. The Arts Lab hosted a free festival in a local park, the subject of his song "
@@ -656 +1005,15 @@
- Their initial studio work was marred by a heated disagreement between Bowie and Cambridge over the latter's drumming style; matters came to a head when Bowie, enraged, accused, "You're fucking up my album." Cambridge summarily quit and was replaced by
+ Their initial studio work was marred by a heated disagreement between Bowie and Cambridge over the latter's drumming
+ <ins>
+  style. Matters
+ </ins>
+ <del>
+  style; matters
+ </del>
+ came to a head when
+ <ins>
+  an enraged Bowie accused the drummer of the disturbance, exclaiming
+ </ins>
+ <del>
+  Bowie, enraged, accused,
+ </del>
+ "You're fucking up my album." Cambridge summarily quit and was replaced by
@@ -666,0 +1030,14 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:182px;">
+  <ins>
+   <a class="new" href="//en.wikipedia.org/wiki/Special:Upload?wpDestFile=Jameso77.jpg" title="File:Jameso77.jpg">
+    File:Jameso77.jpg
+   </a>
+  </ins>
+  <div class="thumbcaption">
+   <ins>
+    Bowie crafted the stage persona Ziggy Stardust based in part on future collaborator Iggy Pop
+   </ins>
+  </div>
+ </div>
+</div>
@@ -674 +1051,8 @@
- (1970), which contained references to schizophrenia, paranoia, and delusion.
+ (1970), which contained references to schizophrenia,
+ <ins>
+  paranoia
+ </ins>
+ <del>
+  paranoia,
+ </del>
+ and delusion.
@@ -702 +1086,7 @@
- "—and in the street, to mixed reaction including laughter and, in the case of one male pedestrian, producing a gun and telling Bowie to "kiss my ass".
+ <ins>
+  "  — and
+ </ins>
+ <del>
+  "—and
+ </del>
+ in the street, to mixed reaction including laughter and, in the case of one male pedestrian, producing a gun and telling Bowie to "kiss my ass".
@@ -762 +1152,6 @@
- pastiche. It was not a significant commercial success at the time
+ pastiche. It was not a significant commercial success at the
+ <ins>
+  time.
+ </ins>
+ <del>
+  time
@@ -764,4 +1159,5 @@
- <a href="/wiki/All_Time_Top_1000_Albums" title="All Time Top 1000 Albums">
-  All Time Top 1000 Albums
- </a>
- list.
+  <a href="/wiki/All_Time_Top_1000_Albums" title="All Time Top 1000 Albums">
+   All Time Top 1000 Albums
+  </a>
+  list.
+ </del>
@@ -787 +1183,4 @@
-   David Bowie during the
+   <del>
+    David
+   </del>
+   Bowie during the
@@ -795 +1194,8 @@
- Dressed in a striking costume, his hair dyed red, Bowie launched his Ziggy Stardust stage show with
+ Dressed in a striking costume, his hair dyed
+ <ins>
+  reddish-brown,
+ </ins>
+ <del>
+  red,
+ </del>
+ Bowie launched his Ziggy Stardust stage show with
@@ -804 +1210,5 @@
- The show was hugely popular, catapulting him to stardom as he toured the UK over the course of the next six months and creating, as described by Buckley, a "cult of Bowie" that was "unique—its influence lasted longer and has been more creative than perhaps almost any other force within pop fandom."
+ The show was hugely popular, catapulting him to stardom as he toured the UK over
+ <del>
+  the course of
+ </del>
+ the next six months and creating, as described by Buckley, a "cult of Bowie" that was "unique—its influence lasted longer and has been more creative than perhaps almost any other force within pop fandom."
@@ -844 +1254,8 @@
- , became UK hits. The
+ ,
+ <ins>
+  were successful in the UK.
+ </ins>
+ <del>
+  became UK hits.
+ </del>
+ The
@@ -890 +1307,2 @@
-  sumo wrestling
+  sumo
+  Link: /wiki/Sumo wrestling wrestling
@@ -1013 +1431,12 @@
-   Bowie filming a video for "Rebel Rebel" in 1974
+   Bowie filming a video for
+   <ins>
+    "
+    <a href="/wiki/Rebel_Rebel" title="Rebel Rebel">
+     Rebel Rebel
+    </a>
+    "
+   </ins>
+   <del>
+    "Rebel Rebel"
+   </del>
+   in 1974
@@ -1117,6 +1546,8 @@
-   <i>
-    <a href="/wiki/Cher_(TV_series)" title="Cher (TV series)">
-     Cher
-    </a>
-   </i>
-   show, 1975
+   <a href="/wiki/Cher_(TV_series)" title="Cher (TV series)">
+    Cher
+     show
+   </a>
+   <del>
+    Link: /wiki/Cher_(TV_series) show,
+   </del>
+   1975
@@ -1175 +1606,5 @@
- Despite his by now well established superstardom, Bowie, in the words of biographer Christopher Sandford, "for all his record sales (over a million copies of
+ Despite his by now well established superstardom, Bowie, in the words of
+ <del>
+  biographer Christopher
+ </del>
+ Sandford, "for all his record sales (over a million copies of
@@ -1193 +1628,10 @@
-   Bowie as the Thin White Duke at Maple Leaf Gardens, Toronto 1976
+   Bowie as the Thin White Duke at
+   <a href="/wiki/Maple_Leaf_Gardens" title="Maple Leaf Gardens">
+    Maple Leaf Gardens
+    Link: /wiki/Maple_Leaf_Gardens ,
+   Toronto
+   </a>
+   <del>
+    Gardens, Toronto
+   </del>
+   1976
@@ -1292 +1736,8 @@
- ". The core band that coalesced around this album and tour—rhythm guitarist Alomar, bassist
+ ". The core band that coalesced
+ <ins>
+  to produce
+ </ins>
+ <del>
+  around
+ </del>
+ this album and tour—rhythm guitarist Alomar, bassist
@@ -1308,0 +1760,2 @@
+</p>
+<p>
@@ -1320 +1773,8 @@
- , the singer waved to the crowd in a gesture that some alleged was a Nazi salute, which was captured on camera and published in
+ ,
+ <ins>
+  Bowie
+ </ins>
+ <del>
+  the singer
+ </del>
+ waved to the crowd in a gesture that some alleged was a Nazi salute, which was captured on camera and published in
@@ -1336,13 +1796,17 @@
- <span class="mw-headline" id="1976.E2.80.9379:_The_Berlin_era">
-  1976–79: The Berlin era
- </span>
-</h3>
-<div class="thumb tleft">
- <div class="thumbinner" style="width:152px;">
-  <a class="image" href="/wiki/File:David_bowie_05061978_01_150.jpg">
-   <img alt="" class="thumbimage" data-file-height="221" data-file-width="150" height="221" src="//upload.wikimedia.org/wikipedia/commons/f/ff/David_bowie_05061978_01_150.jpg" width="150"/>
-  </a>
-  <div class="thumbcaption">
-   <div class="magnify">
-    <a class="internal" href="/wiki/File:David_bowie_05061978_01_150.jpg" title="Enlarge">
-    </a>
+ <span class="mw-headline" id="1976.E2.80.9379:_Berlin_era">
+  1976–79:
+  <ins>
+   Berlin era
+  </ins>
+  <del>
+   The Berlin era
+  </del>
+  <div class="thumb tleft">
+   <div class="thumbinner" style="width:152px;">
+    <div class="thumbcaption">
+     <div class="magnify">
+     </div>
+     <del>
+      Bowie performing in Oslo on 5 June 1978
+     </del>
+    </div>
@@ -1350 +1813,0 @@
-   Bowie performing in Oslo on 5 June 1978
@@ -1352,2 +1815,2 @@
- </div>
-</div>
+ </span>
+</h3>
@@ -1367 +1830,8 @@
- in Berlin, Bowie became, in the words of biographer Christopher Sandford, "a prolific producer and collector of contemporary art. […] Not only did he become a well-known patron of expressionist art: locked in Clos des Mésanges he began an intensive self-improvement course in classical music and literature, and started work on an autobiography."
+ in Berlin, Bowie became, in the words of biographer Christopher Sandford, "a prolific producer and collector of contemporary art.
+ <ins>
+  ...
+ </ins>
+ <del>
+  […]
+ </del>
+ Not only did he become a well-known patron of expressionist art: locked in Clos des Mésanges he began an intensive self-improvement course in classical music and literature, and started work on an autobiography."
@@ -1368,0 +1839,17 @@
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:152px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Bowie performing in
+   </ins>
+   <ins>
+    <a href="/wiki/Oslo" title="Oslo">
+     Oslo
+    </a>
+    on 5 June 1978
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1371,3 +1857,0 @@
-  <a class="image" href="/wiki/File:Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg">
-   <img alt="" class="thumbimage" data-file-height="2800" data-file-width="3648" height="169" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/220px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/330px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg/440px-Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg 2x" width="220"/>
-  </a>
@@ -1376,2 +1859,0 @@
-    <a class="internal" href="/wiki/File:Hauptstra%C3%9Fe_155_Berlin-Sch%C3%B6neberg.jpg" title="Enlarge">
-    </a>
@@ -1379,5 +1861,9 @@
-   Apartment building on Hauptstraße 155 in Berlin
-   <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
-    Schöneberg
-   </a>
-   where David Bowie lived 1976–1978
+   <del>
+    Apartment building on Hauptstraße 155 in Berlin
+   </del>
+   <del>
+    <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
+     Schöneberg
+    </a>
+    where David Bowie lived 1976–1978
+   </del>
@@ -1583 +2069,12 @@
- 's televised Christmas special, when he joined Crosby in "
+ 's
+ <ins>
+  final
+  <a href="/wiki/CBS" title="CBS">
+   CBS
+  </a>
+  television
+ </ins>
+ <del>
+  televised
+ </del>
+ Christmas special, when he joined Crosby in "
@@ -1596,0 +2094,17 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Apartment building on Hauptstraße 155 in Berlin
+   </ins>
+   <ins>
+    <a href="/wiki/Sch%C3%B6neberg" title="Schöneberg">
+     Schöneberg
+    </a>
+    where Bowie lived from 1976 to 1978
+   </ins>
+  </div>
+ </div>
+</div>
@@ -1678 +2192 @@
-<div class="thumb tright">
+<div class="thumb tleft">
@@ -1688 +2202,4 @@
-   Serious Moonlight Tour 1983
+   <a href="/wiki/Serious_Moonlight_Tour" title="Serious Moonlight Tour">
+    Serious Moonlight Tour
+   </a>
+   1983
@@ -1707 +2224 @@
- <a class="mw-redirect" href="/wiki/New_Romantic" title="New Romantic">
+ <a href="/wiki/New_Romanticism" title="New Romanticism">
@@ -1731,5 +2248,8 @@
- and
- <a href="/wiki/Chuck_Hammer" title="Chuck Hammer">
-  Chuck Hammer
- </a>
- .
+ and Chuck
+ <ins>
+  Hammer.
+ </ins>
+ <del>
+  Hammer
+  Link: /wiki/Chuck_Hammer .
+ </del>
@@ -1743,13 +2263,15 @@
- The same year, he made a cameo appearance in the German film
- <i>
-  <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
-   Christiane F.
-  </a>
- </i>
- , a real-life story of teenage drug addiction in 1970s Berlin. The
- <i>
-  <a href="/wiki/Christiane_F._(album)" title="Christiane F. (album)">
-   Christiane F.
-  </a>
- </i>
- soundtrack album, which featured Bowie's music prominently, was released a few months later.
+ <del>
+  The same year, he made a cameo appearance in the German film
+  <i>
+   <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
+    Christiane F.
+   </a>
+  </i>
+  , a real-life story of teenage drug addiction in 1970s Berlin. The
+  <i>
+   <a href="/wiki/Christiane_F._(album)" title="Christiane F. (album)">
+    Christiane F.
+   </a>
+  </i>
+  soundtrack album, which featured Bowie's music prominently, was released a few months later.
+ </del>
@@ -1862 +2384 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -2050 +2572,7 @@
- Though he intended Tin Machine to operate as a democracy, Bowie dominated, both in songwriting and in decision-making.
+ <ins>
+  Although
+ </ins>
+ <del>
+  Though
+ </del>
+ he intended Tin Machine to operate as a democracy, Bowie dominated, both in songwriting and in decision-making.
@@ -2059,0 +2588,2 @@
+</p>
+<p>
@@ -2105,0 +2636,5 @@
+<p>
+ <ins>
+  On 20 April 1992,
+ </ins>
+</p>
@@ -2108,3 +2642,0 @@
-  <a class="image" href="/wiki/File:David_Bowie_1997.jpg">
-   <img alt="" class="thumbimage" data-file-height="1967" data-file-width="1292" height="259" src="//upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/170px-David_Bowie_1997.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/255px-David_Bowie_1997.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/57/David_Bowie_1997.jpg/340px-David_Bowie_1997.jpg 2x" width="170"/>
-  </a>
@@ -2113,2 +2644,0 @@
-    <a class="internal" href="/wiki/File:David_Bowie_1997.jpg" title="Enlarge">
-    </a>
@@ -2116 +2646,3 @@
-   Bowie performing in Finland in 1997
+   <del>
+    Bowie performing in Finland in 1997
+   </del>
@@ -2120,2 +2652,4 @@
-<p>
- In April 1992 Bowie appeared at the
+<del>
+ In April 1992
+</del>
+ Bowie appeared at the
@@ -2123,3 +2657,9 @@
-  Freddie Mercury Tribute Concert
- </a>
- , following the Queen frontman's death the previous year. As well as performing "Heroes" and "All the Young Dudes", he was joined on "Under Pressure" by
+ Freddie Mercury Tribute Concert
+</a>
+ , following the Queen <ins>
+ singer's
+</ins>
+<del>
+ frontman's
+</del>
+ death the previous year. As well as performing "Heroes" and "All the Young Dudes", he was joined on "Under Pressure" by
@@ -2127 +2667,10 @@
-  Annie Lennox
+ Annie Lennox
+</a>
+ , who took Mercury's vocal <ins>
+ part; during his appearance, Bowie knelt and recited the
+ <a href="/wiki/Lord%27s_Prayer" title="Lord's Prayer">
+  Lord's Prayer
+ </a>
+ at
+ <a href="/wiki/Wembley_Stadium_(1923)" title="Wembley Stadium (1923)">
+  Wembley Stadium
@@ -2129 +2678,5 @@
- , who took Mercury's vocal part.
+ .
+</ins>
+<del>
+ part.
+</del>
@@ -2132,2 +2685,2 @@
-  1992 Los Angeles riots
- </a>
+ 1992 Los Angeles riots
+</a>
@@ -2134,0 +2688 @@
+ <p>
@@ -2136,2 +2690,24 @@
-<p>
- 1993 saw the release of Bowie's first solo offering since his Tin Machine departure, the soul, jazz and
+<div class="thumb tleft">
+ <div class="thumbinner" style="width:172px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Bowie performing in
+   </ins>
+   <ins>
+    <a href="/wiki/Turku" title="Turku">
+     Turku
+    </a>
+    , Finland, in 1997
+   </ins>
+  </div>
+ </div>
+</div>
+<ins>
+ In 1993, Bowie released his
+</ins>
+<del>
+ 1993 saw the release of Bowie's
+</del>
+ first solo offering since his Tin Machine departure, the soul, jazz and
@@ -2139,2 +2715,2 @@
-  hip-hop
- </a>
+ hip-hop
+</a>
@@ -2143,4 +2719,4 @@
-  <a href="/wiki/Black_Tie_White_Noise" title="Black Tie White Noise">
-   Black Tie White Noise
-  </a>
- </i>
+ <a href="/wiki/Black_Tie_White_Noise" title="Black Tie White Noise">
+  Black Tie White Noise
+ </a>
+</i>
@@ -2149,2 +2725,2 @@
-  Let's Dance
- </i>
+ Let's Dance
+</i>
@@ -2153,2 +2729,2 @@
-  Nile Rodgers
- </a>
+ Nile Rodgers
+</a>
@@ -2157,2 +2733,2 @@
-  Jump They Say
- </a>
+ Jump They Say
+</a>
@@ -2162,4 +2738,4 @@
-  <a href="/wiki/The_Buddha_of_Suburbia_(soundtrack)" title="The Buddha of Suburbia (soundtrack)">
-   The Buddha of Suburbia
-  </a>
- </i>
+ <a href="/wiki/The_Buddha_of_Suburbia_(soundtrack)" title="The Buddha of Suburbia (soundtrack)">
+  The Buddha of Suburbia
+ </a>
+</i>
@@ -2168,2 +2744,2 @@
-  Hanif Kureishi's novel
- </a>
+ Hanif Kureishi's novel
+</a>
@@ -2172,2 +2748,2 @@
-  Black Tie White Noise
- </i>
+ Black Tie White Noise
+</i>
@@ -2176,2 +2752,2 @@
-  alternative rock
- </a>
+ alternative rock
+</a>
@@ -2179,2 +2755 @@
-</p>
-<p>
+ <p>
@@ -2242,0 +2818,11 @@
+ <ins>
+  In November 1997, Bowie performed on the BBC's
+  <a href="/wiki/Children_in_Need" title="Children in Need">
+   Children in Need
+  </a>
+  charity single "
+  <a class="mw-redirect" href="/wiki/Perfect_Day_97" title="Perfect Day 97">
+   Perfect Day
+  </a>
+  ", which reached number one in the UK.
+ </ins>
@@ -2253 +2839,9 @@
- ".
+ <ins>
+  <span style="padding-left:.2em;">
+   "
+  </span>
+  .
+ </ins>
+ <del>
+  ".
+ </del>
@@ -2263,0 +2858,90 @@
+<h4>
+ <ins>
+  <span class="mw-headline" id="Bowie_Bonds">
+   Bowie Bonds
+  </span>
+ </ins>
+</h4>
+<p>
+ <ins>
+  Bowie Bonds, an early example of
+  <a href="/wiki/Celebrity_bond" title="Celebrity bond">
+   celebrity bonds
+  </a>
+  , were
+  <a href="/wiki/Asset-backed_security" title="Asset-backed security">
+   asset-backed securities
+  </a>
+  of current and future revenues of the 25 albums (287 songs) that David Bowie recorded before 1990. Bowie Bonds were pioneered by
+  <a href="/wiki/Rock_and_roll" title="Rock and roll">
+   rock and roll
+  </a>
+  <a class="mw-redirect" href="/wiki/Investment_banker" title="Investment banker">
+   investment banker
+  </a>
+  David Pullman.
+ Issued in 1997, the bonds were bought for US$55 million by the
+  <a href="/wiki/Prudential_Financial" title="Prudential Financial">
+   Prudential Insurance Company of America
+  </a>
+  .
+ The bonds paid an interest rate of 7.9% and had an average life of ten years,
+ a higher rate of return than a 10-year
+  <a class="mw-redirect" href="/wiki/Treasury_note" title="Treasury note">
+   Treasury note
+  </a>
+  (at the time, 6.37%).
+  <a class="mw-redirect" href="/wiki/Royalties" title="Royalties">
+   Royalties
+  </a>
+  from the 25 albums generated the cash flow that secured the bonds' interest payments.
+ Prudential also received guarantees from Bowie's label, EMI Records, which had recently signed a $30m deal with Bowie.
+ By forfeiting ten years worth of royalties, Bowie received a payment of US$55 million up front. Bowie used this income to buy songs owned by his former manager.
+ Bowie's combined catalogue of albums covered by this agreement sold more than 1 million copies annually at the time of the agreement.
+ By March 2004,
+  <a href="/wiki/Moody%27s_Investors_Service" title="Moody's Investors Service">
+   Moody's Investors Service
+  </a>
+  lowered the bonds from an A3 rating (the seventh highest rating) to Baa3, one notch above
+  <a href="/wiki/High-yield_debt" title="High-yield debt">
+   junk status
+  </a>
+  .
+ The downgrade was prompted by lower-than-expected revenue "due to weakness in sales for recorded music" and that an unnamed company guaranteed the issue.
+ Nonetheless, the bonds liquidated in 2007 as originally planned, without default, and the rights to the income from the songs reverted to Bowie.
+ </ins>
+</p>
+<h4>
+ <ins>
+  <span class="mw-headline" id="BowieNet">
+   BowieNet
+  </span>
+ </ins>
+</h4>
+<p>
+ <ins>
+  In September 1998, Bowie launched his own
+  <a class="mw-redirect" href="/wiki/Internet_Service_Provider" title="Internet Service Provider">
+   Internet Service Provider
+  </a>
+  ,
+  <a class="new" href="/w/index.php?title=BowieNet&amp;action=edit&amp;redlink=1" title="BowieNet (page does not exist)">
+   BowieNet
+  </a>
+  , developed in conjunction with
+  <a class="new" href="/w/index.php?title=Robert_Goodale&amp;action=edit&amp;redlink=1" title="Robert Goodale (page does not exist)">
+   Robert Goodale
+  </a>
+  and
+  <a href="/wiki/Ron_Roy_(producer)" title="Ron Roy (producer)">
+   Ron Roy
+  </a>
+  .
+ Subscribers to the
+  <a href="/wiki/Dial-up_Internet_access" title="Dial-up Internet access">
+   dial-up
+  </a>
+  service were offered exclusive content, as well as a BowieNet email address and regular Internet access.
+ The service was closed by 2006.
+ </ins>
+</p>
@@ -2279 +2963,8 @@
-   Bowie on-stage with
+   Bowie
+   <ins>
+    on stage
+   </ins>
+   <del>
+    on-stage
+   </del>
+   with
@@ -2287 +2978,7 @@
-   in 2002.
+   in
+   <ins>
+    2002
+   </ins>
+   <del>
+    2002.
+   </del>
@@ -2318,0 +3016,20 @@
+</p>
+<p>
+ <ins>
+  On 25 June 2000, Bowie made his second appearance at the
+  <a href="/wiki/Glastonbury_Festival" title="Glastonbury Festival">
+   Glastonbury Festival
+  </a>
+  in England, playing 30 years after his first.
+ On 27 June, Bowie performed a concert at
+  <a class="mw-redirect" href="/wiki/BBC_Radio_Theatre" title="BBC Radio Theatre">
+   BBC Radio Theatre
+  </a>
+  in London, which was released in the compilation album
+  <i>
+   <a href="/wiki/Bowie_at_the_Beeb" title="Bowie at the Beeb">
+    Bowie at the Beeb
+   </a>
+  </i>
+  , which also featured BBC recording sessions from 1968 to 1972.
+ </ins>
@@ -2438 +3155,10 @@
- During a relatively quiet 2005, he recorded the vocals for the song "(She Can) Do That", co-written with Brian Transeau, for the film
+</p>
+<p>
+ During a relatively quiet 2005, he recorded the vocals for the song "(She Can) Do That", co-written with
+ <a href="/wiki/BT_(musician)" title="BT (musician)">
+  Brian Transeau
+ </a>
+ <del>
+  Transeau,
+ </del>
+ for the film
@@ -2505 +3231,8 @@
- a performance that marks the last time Bowie performed his music on stage.
+ a performance
+ <ins>
+  marking
+ </ins>
+ <del>
+  that marks
+ </del>
+ the last time Bowie performed his music on stage.
@@ -2528 +3261,8 @@
- On the 40th anniversary of the July 1969 moon landing—and Bowie's accompanying commercial breakthrough with "Space Oddity"—EMI released the individual tracks from the original eight-track studio recording of the song, in a 2009 contest inviting members of the public to create a remix.
+ On the 40th anniversary of the
+ <a class="mw-redirect" href="/wiki/Apollo_Moon_landing" title="Apollo Moon landing">
+  July 1969 moon landing
+ </a>
+ <del>
+  landing—and
+ </del>
+ Bowie's accompanying commercial breakthrough with "Space Oddity"—EMI released the individual tracks from the original eight-track studio recording of the song, in a 2009 contest inviting members of the public to create a remix.
@@ -2550,10 +3290,19 @@
- <span class="mw-headline" id="2013.E2.80.93present:_The_Next_Day_and_Blackstar">
-  2013–present:
-  <i>
-   The Next Day
-  </i>
-  and
-  <i>
-   Blackstar
-  </i>
- </span>
+ <ins>
+  <span class="mw-headline" id="2013.E2.80.9316:_Final_years">
+   2013–16: Final years
+  </span>
+ </ins>
+</h3>
+<h3>
+ <del>
+  <span class="mw-headline" id="2013.E2.80.93present:_The_Next_Day_and_Blackstar">
+   2013–present:
+   <i>
+    The Next Day
+   </i>
+   and
+   <i>
+    Blackstar
+   </i>
+  </span>
+ </del>
@@ -2583,0 +3333,5 @@
+ <ins>
+  Visconti.
+ </ins>
+</p>
+<del>
@@ -2588 +3342,10 @@
- A music video for the single was released onto
+</del>
+<p>
+ A music video for
+ <ins>
+  "Where Are We Now?"
+ </ins>
+ <del>
+  the single
+ </del>
+ was released onto
@@ -2599 +3362,2 @@
-  iTunes Chart
+  iTunes
+  Link: /wiki/ITunes Chart Chart
@@ -2607 +3371,8 @@
- his first single to enter the top 10 for two decades, (since "
+ his first single to enter the top 10 for two
+ <ins>
+  decades
+ </ins>
+ <del>
+  decades,
+ </del>
+ (since "
@@ -2637,2 +3407,0 @@
-</p>
-<p>
@@ -2643 +3412,5 @@
- " has created some controversy, initially being removed from YouTube for
+ "
+ <del>
+  has
+ </del>
+ created some controversy, initially being removed from YouTube for
@@ -2656,6 +3429,30 @@
- , Bowie has ruled out ever giving an interview again, with producer Visconti agreeing with the interviewer's suggestion that he is now Bowie's "voice on earth". Visconti also revealed to
- <i>
-  The Times
- </i>
- that Bowie has no intention of going on tour, noting that all "he wants to do is make records."
- An exhibition of Bowie artifacts, called "David Bowie Is," was shown at the
+ , Bowie
+ <del>
+  has
+ </del>
+ ruled out ever giving an interview
+ <ins>
+  again.
+ </ins>
+ <del>
+  again, with producer Visconti agreeing with the interviewer's suggestion that he is now Bowie's "voice on earth". Visconti also revealed to
+  <i>
+   The Times
+  </i>
+  that Bowie has no intention of going on tour, noting that all "he wants to do is make records."
+ </del>
+ An exhibition of Bowie
+ <ins>
+  artefacts,
+ </ins>
+ <del>
+  artifacts,
+ </del>
+ called "David Bowie
+ <ins>
+  Is",
+ </ins>
+ <del>
+  Is,"
+ </del>
+ was shown at the
@@ -2664,0 +3462,3 @@
+ <ins>
+  in London
+ </ins>
@@ -2678,3 +3478,7 @@
- (The Netherlands).
-</p>
-<p>
+ <ins>
+  (the
+ </ins>
+ <del>
+  (The
+ </del>
+ Netherlands).
@@ -2690,0 +3495,2 @@
+</p>
+<p>
@@ -2708 +3514,5 @@
- garnered a significant reaction on
+ garnered a significant reaction
+ <ins>
+  throughout the UK
+ </ins>
+ on
@@ -2727,7 +3537,18 @@
-</p>
-<p>
- In May 2015 it was announced that "Let’s Dance" would be reissued as a yellow vinyl single on 16 July 2015 in conjunction with the
- <i>
-  David Bowie is
- </i>
- ’ exhibition at the
+ In May
+ <ins>
+  2015, "Let's Dance" was announced to
+ </ins>
+ <del>
+  2015 it was announced that "Let’s Dance" would
+ </del>
+ be reissued as a yellow vinyl single on 16 July 2015 in conjunction with the
+ <ins>
+  "David Bowie is"
+ </ins>
+ <del>
+  <i>
+   David Bowie is
+  </i>
+  ’
+ </del>
+ exhibition at the
@@ -2739,0 +3561,9 @@
+ <ins>
+  In August 2015, it was announced that Bowie was writing songs for a Broadway musical based on the
+  <i>
+   <a href="/wiki/SpongeBob_SquarePants" title="SpongeBob SquarePants">
+    SpongeBob SquarePants
+   </a>
+  </i>
+  cartoon series.
+ </ins>
@@ -2747 +3577,8 @@
- The show's director, Johan Renck, said of Bowie, "His first response was precise, engaged and curious. The piece of music he laid before us embodied every aspect of our characters and the series itself - dark, brooding, beautiful and sentimental (in the best possible incarnation of this word). All along, the man inspired and intrigued me and as the process passed, I was overwhelmed with his generosity. I still can't fathom what actually happened." The theme that will be used for
+ The
+ <ins>
+  theme that was
+ </ins>
+ <del>
+  show's director, Johan Renck, said of Bowie, "His first response was precise, engaged and curious. The piece of music he laid before us embodied every aspect of our characters and the series itself - dark, brooding, beautiful and sentimental (in the best possible incarnation of this word). All along, the man inspired and intrigued me and as the process passed, I was overwhelmed with his generosity. I still can't fathom what actually happened." The theme that will be
+ </del>
+ used for
@@ -2751 +3588,7 @@
- will also be the title track for his January 2016 release
+ <ins>
+  was also
+ </ins>
+ <del>
+  will also be
+ </del>
+ the title track for his January 2016 release
@@ -2754 +3597,4 @@
-   Blackstar (★)
+   Blackstar
+   <del>
+    (★)
+   </del>
@@ -2762 +3608,7 @@
- : “
+ :
+ <ins>
+  "
+ </ins>
+ <del>
+  “
+ </del>
@@ -2766 +3618,4 @@
- may be the oddest work yet from [Bowie],”
+ may be the oddest work yet from
+ <ins>
+  Bowie".
+ </ins>
@@ -2767,0 +3623,18 @@
+<p>
+ <ins>
+  <i>
+   Blackstar
+  </i>
+  was released on 8 January 2016, Bowie's 69th birthday, and was met with critical acclaim.
+ . Following his death on 10 January, producer Tony Visconti revealed that Bowie had planned the album to be his
+  <a href="/wiki/Swan_song" title="Swan song">
+   swan song
+  </a>
+  , and a "parting gift" for his fans before his death.
+ Several reporters and critics subsequently noted that most of the lyrics on the album seem to revolve around his impending death,
+ with CNN noting that the album "reveals a man who appears to be grappling with his own mortality".
+ </ins>
+</p>
+<del>
+ [Bowie],”
+</del>
@@ -2776 +3649,3 @@
- Bowie's career has also been punctuated by various roles in film and theatre productions, earning him some acclaim as an actor in his own right.
+ <del>
+  Bowie's career has also been punctuated by various roles in film and theatre productions, earning him some acclaim as an actor in his own right.
+ </del>
@@ -2848 +3723,12 @@
- Bowie took the title role in the Broadway theatre production
+ Bowie
+ <ins>
+  played
+ </ins>
+ <del>
+  took
+ </del>
+ the
+ <a href="/wiki/Joseph_Merrick" title="Joseph Merrick">
+  title role
+ </a>
+ in the Broadway theatre production
@@ -2870,4 +3756,4 @@
- (1981), featured much material from his
- <a href="/wiki/Berlin_Trilogy" title="Berlin Trilogy">
-  Berlin Trilogy
- </a>
+ (1981), featured much material from his Berlin Trilogy
+ <del>
+  Link: /wiki/Berlin_Trilogy
+ </del>
@@ -2881 +3767,4 @@
- (1983), a revisionist
+ (1983), a
+ <del>
+  revisionist
+ </del>
@@ -2972 +3861,8 @@
- Two years later he played
+ Two years
+ <ins>
+  later,
+ </ins>
+ <del>
+  later
+ </del>
+ he played
@@ -3085 +3981,8 @@
- (2006), which was about the bitter rivalry between two magicians in the late 19th century. He voice-acted in the animated film
+ (2006), which was about the bitter rivalry between two magicians in the late 19th century.
+ <ins>
+  In the same year, he
+ </ins>
+ <del>
+  He
+ </del>
+ voice-acted in the animated film
@@ -3091 +3994,18 @@
- as the powerful villain Maltazard, and lent his voice to the character Lord Royal Highness in the
+ as the powerful villain
+ <ins>
+  Maltazard and appeared as himself in an episode of the
+  <a href="/wiki/Ricky_Gervais" title="Ricky Gervais">
+   Ricky Gervais
+  </a>
+  television series
+  <i>
+   <a href="/wiki/Extras_(TV_series)" title="Extras (TV series)">
+    Extras
+   </a>
+  </i>
+  . In 2007, he
+ </ins>
+ <del>
+  Maltazard, and
+ </del>
+ lent his voice to the character Lord Royal Highness in the
@@ -3145 +4065,7 @@
-   located at the
+   <ins>
+    in
+   </ins>
+   <del>
+    located at
+   </del>
+   the
@@ -3157 +4083,9 @@
- From the time of his earliest recordings in the 1960s, Bowie has employed a wide variety of musical styles. His early compositions and performances were strongly influenced by not only rock and rollers like
+ From the time of his earliest recordings in the 1960s, Bowie
+ <del>
+  has
+ </del>
+ employed a wide variety of musical styles. His early compositions and performances were strongly influenced by
+ <del>
+  not only
+ </del>
+ rock and rollers like
@@ -3165 +4099,7 @@
- but also the wider world of show business. He particularly strove to emulate the British musical theatre singer-songwriter and actor
+ <ins>
+  , and
+ </ins>
+ <del>
+  but
+ </del>
+ also the wider world of show business. He particularly strove to emulate the British musical theatre singer-songwriter and actor
@@ -3196 +4136,8 @@
- Here, too, as in his stagecraft and songwriting, the singer's chamaeleon-like nature is evident: historiographer Michael Campbell says that Bowie's lyrics "arrest our ear, without question. But Bowie continually shifts from person to person as he delivers them ... His voice changes dramatically from section to section."
+ Here, too, as in his stagecraft and songwriting, the singer's
+ <ins>
+  role playing
+ </ins>
+ <del>
+  chamaeleon-like nature
+ </del>
+ is evident: historiographer Michael Campbell says that Bowie's lyrics "arrest our ear, without question. But Bowie continually shifts from person to person as he delivers them ... His voice changes dramatically from section to section."
@@ -3206,3 +4153,14 @@
-</p>
-<p>
- Bowie is known as a multi-instrumentalist. In addition to his playing of guitar, keyboards, harmonica and saxophone, he plays stylophone,
+ <del>
+  Bowie is known as a multi-instrumentalist.
+ </del>
+ In addition to
+ <ins>
+  the guitar, Bowie was also a fluent player of the keyboard, harmonica, saxophone,
+  <a href="/wiki/Stylophone" title="Stylophone">
+   stylophone
+  </a>
+  ,
+ </ins>
+ <del>
+  his playing of guitar, keyboards, harmonica and saxophone, he plays stylophone,
+ </del>
@@ -3213,4 +4171,9 @@
- <a href="/wiki/Cello" title="Cello">
-  cello
- </a>
- ,
+ <ins>
+  cello,
+ </ins>
+ <del>
+  <a href="/wiki/Cello" title="Cello">
+   cello
+  </a>
+  ,
+ </del>
@@ -3226,0 +4190,351 @@
+ <ins>
+  <span class="mw-headline" id="Personal_life">
+   Personal life
+  </span>
+ </ins>
+</h2>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Marriages">
+   Marriages
+  </span>
+ </ins>
+</h3>
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Bowie and wife
+   </ins>
+   <ins>
+    <a href="/wiki/Iman_(model)" title="Iman (model)">
+     Iman
+    </a>
+    in 2009
+   </ins>
+  </div>
+ </div>
+</div>
+<p>
+ <ins>
+  Bowie married
+  <a class="mw-redirect" href="/wiki/Angela_Bowie" title="Angela Bowie">
+   Mary Angela Barnett
+  </a>
+  on 19 March 1970 at Bromley Register Office in
+  <a href="/wiki/Bromley" title="Bromley">
+   Bromley
+  </a>
+  , London. They had a son together,
+  <a href="/wiki/Duncan_Jones" title="Duncan Jones">
+   Duncan
+  </a>
+  , who was born on 30 May 1971.
+ Bowie and Angela divorced on 8 February 1980 in Switzerland.
+ </ins>
+</p>
+<p>
+ <ins>
+  On 24 April 1992, Bowie married the Somali-American model
+  <a href="/wiki/Iman_(model)" title="Iman (model)">
+   Iman
+  </a>
+  in a private ceremony in
+  <a href="/wiki/Lausanne" title="Lausanne">
+   Lausanne
+  </a>
+  . The wedding was later solemnised on 6 June in Florence.
+ They had one daughter, Alexandria "Lexi" Zahra Jones, born in August 2000.
+ The couple resided primarily in New York City and London, as well as owning an apartment in Sydney.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Sexuality">
+   Sexuality
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  Buckley writes, "If Ziggy confused both his creator and his audience, a big part of that confusion centred on the topic of sexuality."
+ Bowie declared himself gay in an interview with Michael Watts in the 22 January 1972 issue of
+  <i>
+   <a href="/wiki/Melody_Maker" title="Melody Maker">
+    Melody Maker
+   </a>
+  </i>
+  ,
+ a move which coincided with the first shots in his campaign for stardom as Ziggy Stardust.
+ In a September 1976 interview with
+  <i>
+   <a href="/wiki/Playboy" title="Playboy">
+    Playboy
+   </a>
+  </i>
+  , Bowie said: "It's true—I am a bisexual. But I can't deny that I've used that fact very well. I suppose it's the best thing that ever happened to me."
+ According to Angie, Bowie had a relationship with Mick Jagger.
+ </ins>
+</p>
+<p>
+ <ins>
+  In a 1983 interview with
+  <i>
+   Rolling Stone
+  </i>
+  , Bowie said his public declaration of bisexuality was "the biggest mistake I ever made" and "I was always a closet heterosexual."
+ On other occasions, he said his interest in homosexual and bisexual culture had been more a product of the times and the situation in which he found himself than his own feelings; as described by Buckley, he said he had been driven more by "a compulsion to flout moral codes than a real biological and psychological state of being."
+ </ins>
+</p>
+<p>
+ <ins>
+  Asked in 2002 by
+  <i>
+   <a href="/wiki/Blender_(magazine)" title="Blender (magazine)">
+    Blender
+   </a>
+  </i>
+  whether he still believed his public declaration was the biggest mistake he ever made, he replied:
+ </ins>
+</p>
+<blockquote class="templatequote">
+ <p>
+  <ins>
+   Interesting. [Long pause] I don't think it was a mistake in Europe, but it was a lot tougher in America. I had no problem with people knowing I was bisexual. But I had no inclination to hold any banners nor be a representative of any group of people. I knew what I wanted to be, which was a songwriter and a performer, and I felt that bisexuality became my headline over here for so long. America is a very puritanical place, and I think it stood in the way of so much I wanted to do.
+  </ins>
+ </p>
+</blockquote>
+<p>
+ <ins>
+  Buckley's view of the period is that Bowie, "a taboo-breaker and a dabbler ... mined sexual intrigue for its ability to shock",
+ and that "it is probably true that Bowie was never gay, nor even consistently actively bisexual ... he did, from time to time, experiment, even if only out of a sense of curiosity and a genuine allegiance with the 'transgressional.
+  <span style="padding-right:.2em;">
+   '
+  </span>
+  "
+ Biographer Christopher Sandford says that according to Mary Finnigan, with whom Bowie had an affair in 1969, the singer and his first wife Angie "lived in a fantasy world ... and they created their bisexual fantasy."
+ Sandford tells how, during the marriage, Bowie "made a positive fetish of repeating the quip that he and his wife had met while 'fucking the same bloke' ... Gay sex was always an anecdotal and laughing matter. That Bowie's actual tastes swung the other way is clear from even a partial tally of his affairs with women."
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Religion_and_spirituality">
+   Religion and spirituality
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  In 2005 he said, "Questioning my spiritual life has always been germane to what I was writing. Always." He added that he was bothered by being "not quite an atheist".
+ In the
+  <i>
+   <a href="/wiki/Esquire_(magazine)" title="Esquire (magazine)">
+    Esquire
+   </a>
+  </i>
+  interview
+ </ins>
+</p>
+<blockquote>
+ <p>
+  <ins>
+   "What I've Learned", he stated, "I'm in awe of the universe, but I don't necessarily believe there's an intelligence or agent behind it. I do have a passion for the visual in religious rituals, though, even though they may be completely empty and bereft of substance. The incense is powerful and provocative, whether Buddhist or
+   <a class="mw-redirect" href="/wiki/Roman_Catholic" title="Roman Catholic">
+    Catholic
+   </a>
+   ."
+  </ins>
+ </p>
+</blockquote>
+<p>
+ <ins>
+  Bowie showed an interest in
+  <a href="/wiki/Buddhism" title="Buddhism">
+   Buddhism
+  </a>
+  that began in 1967. He frequently studied in London under the
+  <a href="/wiki/Tibetan_people" title="Tibetan people">
+   Tibetan
+  </a>
+  <a href="/wiki/Lama" title="Lama">
+   Lama
+  </a>
+  <a href="/wiki/Chime_Rinpoche" title="Chime Rinpoche">
+   Chime Rinpoche
+  </a>
+  before becoming a solo artist. During a 2001 interview, Bowie claimed that "after a few months of study, he told me, 'You don't want to be Buddhist ... You should follow music.
+  <span style="padding-right:.2em;">
+   '
+  </span>
+  "
+ Bowie later wrote the song "Silly Boy Blue" in tribute to Rinpoche on his 1967 album
+  <i>
+   <a href="/wiki/David_Bowie_(1967_album)" title="David Bowie (1967 album)">
+    David Bowie
+   </a>
+  </i>
+  . In the 1960s, he had also studied under the
+  <a class="mw-redirect" href="/wiki/Crazy_wisdom" title="Crazy wisdom">
+   crazy wisdom
+  </a>
+  <a href="/wiki/Tulku" title="Tulku">
+   tulku
+  </a>
+  <a href="/wiki/Ch%C3%B6gyam_Trungpa" title="Chögyam Trungpa">
+   Chögyam Trungpa
+  </a>
+  .
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Politics">
+   Politics
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  In 1976, speaking as
+  <a href="/wiki/The_Thin_White_Duke" title="The Thin White Duke">
+   the Thin White Duke
+  </a>
+  , Bowie's persona at the time, and "at least partially tongue-in-cheek", he made statements that expressed support for fascism and perceived admiration for
+  <a href="/wiki/Adolf_Hitler" title="Adolf Hitler">
+   Adolf Hitler
+  </a>
+  in interviews with
+  <i>
+   <a href="/wiki/Playboy" title="Playboy">
+    Playboy
+   </a>
+  </i>
+  ,
+  <i>
+   NME
+  </i>
+  and a Swedish publication. Bowie was quoted as saying: "Britain is ready for a fascist leader... I think Britain could benefit from a fascist leader. After all, fascism is really nationalism... I believe very strongly in fascism, people have always responded with greater efficiency under a regimental leadership." He was also quoted as saying: "Adolf Hitler was one of the first rock stars" and "You've got to have an
+  <a href="/wiki/Far-right_politics" title="Far-right politics">
+   extreme right
+  </a>
+  front come up and sweep everything off its feet and tidy everything up."
+ Bowie later retracted these comments in an interview with
+  <i>
+   <a href="/wiki/Melody_Maker" title="Melody Maker">
+    Melody Maker
+   </a>
+  </i>
+  in October 1977, blaming them on mental instability caused by his drug problems at the time, saying: "I was out of my mind, totally, completely crazed."
+ </ins>
+</p>
+<p>
+ <ins>
+  In the eighties and nineties, Bowie's public statements shifted sharply towards
+  <a href="/wiki/Anti-racism" title="Anti-racism">
+   anti-racism
+  </a>
+  and
+  <a href="/wiki/Anti-fascism" title="Anti-fascism">
+   anti-fascism
+  </a>
+  . In an interview with
+  <a href="/wiki/MTV" title="MTV">
+   MTV
+  </a>
+  in 1983, Bowie criticised the channel for not providing enough coverage of black musicians,
+ and the music videos for "China Girl" and "Let's Dance" were described by Bowie as a "very simple, very direct" statement against racism.
+ The album
+  <i>
+   Tin Machine
+  </i>
+  took a more direct stance against fascism and Neo-Nazism, and was criticized for being too preachy.
+ </ins>
+</p>
+<h3>
+ <ins>
+  <span class="mw-headline" id="Legal_issues">
+   Legal issues
+  </span>
+ </ins>
+</h3>
+<p>
+ <ins>
+  In 1990, Queen and Bowie filed a lawsuit against American rapper
+  <a href="/wiki/Vanilla_Ice" title="Vanilla Ice">
+   Vanilla Ice
+  </a>
+  for copying the bass line of "
+  <a href="/wiki/Under_Pressure" title="Under Pressure">
+   Under Pressure
+  </a>
+  " with only minor modifications in his song "
+  <a href="/wiki/Ice_Ice_Baby" title="Ice Ice Baby">
+   Ice Ice Baby
+  </a>
+  ".
+ The dispute was later resolved with an undisclosed out-of-court settlement.
+ </ins>
+</p>
+<h2>
+ <ins>
+  <span class="mw-headline" id="Death">
+   Death
+  </span>
+ </ins>
+</h2>
+<p>
+ <ins>
+  On 10 January 2016, two days after his 69th birthday and the release of the album,
+  <i>
+   <a class="mw-redirect" href="/wiki/Blackstar_(David_Bowie_album)" title="Blackstar (David Bowie album)">
+    Blackstar
+   </a>
+  </i>
+  , Bowie died from
+  <a href="/wiki/Liver_cancer" title="Liver cancer">
+   liver cancer
+  </a>
+  at his New York home.
+ He had been diagnosed 18 months earlier, but had not made the news of his illness public.
+ Belgian theatre director
+  <a href="/wiki/Ivo_van_Hove" title="Ivo van Hove">
+   Ivo van Hove
+  </a>
+  , who worked with the singer on his
+  <a href="/wiki/Off-Broadway" title="Off-Broadway">
+   Off-Broadway
+  </a>
+  musical
+  <i>
+   Lazarus
+  </i>
+  , explained that Bowie was unable to attend rehearsals due to progression of the disease. He noted that Bowie kept working during the illness.
+ Bowie's producer Tony Visconti wrote:
+ </ins>
+</p>
+<table class="cquote" role="presentation" style="margin:auto; border-collapse: collapse; border: none; background-color: transparent; width: auto;">
+ <tr>
+  <td style="width: 20px; vertical-align: top; border: none; color: #B2B7F2; font-size: 40px; font-family: 'Times New Roman', Times, serif; font-weight: bold; line-height: .6em; text-align: left; padding: 10px 10px;">
+   <ins>
+    “
+   </ins>
+  </td>
+  <td style="vertical-align: top; border: none; padding: 4px 10px;">
+   <ins>
+    He always did what he wanted to do. And he wanted to do it his way and he wanted to do it the best way. His death was no different from his life — a work of art. He made
+    <i>
+     Blackstar
+    </i>
+    for us, his parting gift. I knew for a year this was the way it would be. I wasn't, however, prepared for it. He was an extraordinary man, full of love and life. He will always be with us. For now, it is appropriate to cry.
+   </ins>
+  </td>
+  <td style="width: 20px; vertical-align: bottom; border: none; color: #B2B7F2; font-size: 40px; font-family: 'Times New Roman', Times, serif; font-weight: bold; line-height: .6em; text-align: right; padding: 10px 10px;">
+   <ins>
+    ”
+   </ins>
+  </td>
+ </tr>
+</table>
+<h2>
@@ -3231 +4545 @@
-<div class="thumb tleft">
+<div class="thumb tright">
@@ -3249,3 +4563,14 @@
- Bowie's innovative songs and stagecraft brought a new dimension to popular music in the early 1970s, strongly influencing both its immediate forms and its subsequent development. A pioneer of glam rock, Bowie, according to music historians Schinder and Schwartz, has joint responsibility with Marc Bolan for creating the genre.
- At the same time, he inspired the innovators of the punk rock music movement—historian Michael Campbell calls him "one of punk's seminal influences". While punk musicians trashed the conventions of pop stardom, Bowie moved on again—into a more abstract style of music making that in turn became a transforming influence. Biographer David Buckley writes, "At a time when punk rock was noisily reclaiming the three-minute pop song in a show of public defiance, Bowie almost completely abandoned traditional rock instrumentation."
- Bowie's record company sought to convey his unique status in popular music with the slogan, "There is old wave, there is new wave, and there is Bowie ..."
+ Bowie's
+ <del>
+  innovative
+ </del>
+ songs and stagecraft brought a new dimension to popular music in the early 1970s, strongly influencing both its immediate forms and its subsequent development. A pioneer of glam rock, Bowie, according to music historians Schinder and Schwartz, has joint responsibility with Marc Bolan for creating the genre.
+ At the same time, he inspired the innovators of the punk rock music movement—historian Michael Campbell calls him "one of punk's seminal influences".
+ While punk musicians trashed the conventions of pop stardom, Bowie moved on again—into a more abstract style of music making that in turn became a transforming influence. Biographer David Buckley writes, "At a time when punk rock was noisily reclaiming the three-minute pop song in a show of public defiance, Bowie almost completely abandoned traditional rock instrumentation."
+ Bowie's record company sought to convey his unique status in popular music with the slogan, "There is old wave, there is new wave, and there is
+ <ins>
+  Bowie."
+ </ins>
+ <del>
+  Bowie ..."
+ </del>
@@ -3255 +4580,5 @@
- Buckley writes that, in an early 1970s pop world that was "Bloated, self-important, leather-clad, self-satisfied, ... Bowie challenged the core belief of the rock music of its day." As described by
+ Buckley writes that, in an early 1970s pop world that was "Bloated, self-important, leather-clad, self-satisfied, ... Bowie challenged the
+ <ins>
+  very
+ </ins>
+ core belief of the rock music of its day." As described by
@@ -3259 +4588 @@
- , "The one distinguishing feature about early-70s progressive rock was that it didn't progress. Before Bowie came along, people didn't want too much change." Buckley says that Bowie "subverted the whole notion of what it was to be a rock star", with the result that "After Bowie there has been no other pop icon of his stature, because the pop world that produces these rock gods doesn't exist any more. ... The fierce partisanship of the cult of Bowie was also unique—its influence lasted longer and has been more creative than perhaps almost any other force within pop fandom." Buckley concludes that "Bowie is both star and icon. The vast body of work he has produced ... has created perhaps the biggest cult in popular culture. ... His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
+ , "The one distinguishing feature about early-70s progressive rock was that it didn't progress. Before Bowie came along, people didn't want too much change." Buckley says that Bowie "subverted the whole notion of what it was to be a rock star", with the result that
@@ -3260,0 +4590,5 @@
+<blockquote>
+ <p>
+  "After Bowie there has been no other pop icon of his stature, because the pop world that produces these rock gods doesn't exist any more. ... The fierce partisanship of the cult of Bowie was also unique—its influence lasted longer and has been more creative than perhaps almost any other force within pop fandom."
+ </p>
+</blockquote>
@@ -3262,6 +4596,24 @@
- Bowie was inducted into the
- <a href="/wiki/Rock_and_Roll_Hall_of_Fame" title="Rock and Roll Hall of Fame">
-  Rock and Roll Hall of Fame
- </a>
- in 1996.
- Through perpetual reinvention, he has seen his influence continue to broaden and extend: music reviewer Brad Filicky writes that over the decades, "Bowie has become known as a musical chameleon, changing and dictating trends as much as he has altered his style to fit, influencing fashion and pop culture."
+ Buckley concludes that "Bowie is both star and icon. The vast body of work he has produced ... has created perhaps the biggest cult in popular culture. ... His influence has been unique in popular culture—he has permeated and altered more lives than any comparable figure."
+</p>
+<p>
+ <del>
+  Bowie was inducted into the
+  <a href="/wiki/Rock_and_Roll_Hall_of_Fame" title="Rock and Roll Hall of Fame">
+   Rock and Roll Hall of Fame
+  </a>
+  in 1996.
+ </del>
+ Through perpetual reinvention, he has seen his influence continue to broaden and extend: music reviewer Brad Filicky writes that over the decades, "Bowie has become known as a musical chameleon, changing and dictating trends as much as he has altered his style to
+ <ins>
+  fit them. His influence on
+ </ins>
+ <del>
+  fit, influencing
+ </del>
+ fashion and pop
+ <ins>
+  culture is second only to Madonna."
+ </ins>
+ <del>
+  culture."
+ </del>
@@ -3268,0 +4621 @@
+ In
@@ -3271 +4624,16 @@
- In 2015, he was named one of
+ <ins>
+  2000, Bowie was named by
+  <a href="/wiki/NME" title="NME">
+   NME
+  </a>
+  as the "most influential artist of all time".
+Bowie was the principal inspiration for the bisexual glam rock icon Brian Slade in the 1998 film
+  <i>
+   <a href="/wiki/Velvet_Goldmine" title="Velvet Goldmine">
+    Velvet Goldmine
+   </a>
+  </i>
+  . In
+ </ins>
+</p>
+ 2015, he was named one of
@@ -3273,2 +4641,2 @@
-  GQ
- </a>
+ GQ
+</a>
@@ -3276,3 +4644,7 @@
-</p>
-<p>
- His alter ego Ziggy Stardust, was the main inspiration for
+ His alter ego Ziggy <ins>
+ Stardust
+</ins>
+<del>
+ Stardust,
+</del>
+ was the main inspiration for
@@ -3280,2 +4652,2 @@
-  Tilda Swinton
- </a>
+ Tilda Swinton
+</a>
@@ -3284,10 +4656,17 @@
-  <a href="/wiki/A_Bigger_Splash_(film)" title="A Bigger Splash (film)">
-   A Bigger Splash
-  </a>
- </i>
- , Rock singer Marianne Lane.
-</p>
-<h2>
- <span class="mw-headline" id="Personal_life">
-  Personal life
- </span>
+ <a href="/wiki/A_Bigger_Splash_(film)" title="A Bigger Splash (film)">
+  A Bigger Splash
+ </a>
+</i>
+ , <ins>
+ rock
+</ins>
+<del>
+ Rock
+</del>
+ singer Marianne Lane.
+ <h2>
+ <del>
+  <span class="mw-headline" id="Personal_life">
+   Personal life
+  </span>
+ </del>
@@ -3296,3 +4675,5 @@
- <span class="mw-headline" id="Relationships_and_sexuality">
-  Relationships and sexuality
- </span>
+ <del>
+  <span class="mw-headline" id="Relationships_and_sexuality">
+   Relationships and sexuality
+  </span>
+ </del>
@@ -3302,3 +4682,0 @@
-  <a class="image" href="/wiki/File:Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg">
-   <img alt="" class="thumbimage" data-file-height="2036" data-file-width="2557" height="175" src="//upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg/220px-Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg/330px-Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg/440px-Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg 2x" width="220"/>
-  </a>
@@ -3307,2 +4684,0 @@
-    <a class="internal" href="/wiki/File:Iman_and_David_Bowie_at_the_premiere_of_Moon.jpg" title="Enlarge">
-    </a>
@@ -3310 +4686,3 @@
-   Bowie and wife Iman
+   <del>
+    Bowie and wife Iman
+   </del>
@@ -3315,13 +4693,15 @@
- Bowie married Mary Angela Barnett (also known as
- <a class="mw-redirect" href="/wiki/Angela_Bowie" title="Angela Bowie">
-  Angie Bowie
- </a>
- ) on 19 March 1970 at Bromley Register Office on Beckenham Lane,
- <a href="/wiki/Bromley" title="Bromley">
-  Bromley
- </a>
- , London. They had a son together, Zowie Bowie (now known as
- <a href="/wiki/Duncan_Jones" title="Duncan Jones">
-  Duncan Jones
- </a>
- , film director), and divorced on 8 February 1980 in Switzerland.
+ <del>
+  Bowie married Mary Angela Barnett (also known as
+  <a class="mw-redirect" href="/wiki/Angela_Bowie" title="Angela Bowie">
+   Angie Bowie
+  </a>
+  ) on 19 March 1970 at Bromley Register Office on Beckenham Lane,
+  <a href="/wiki/Bromley" title="Bromley">
+   Bromley
+  </a>
+  , London. They had a son together, Zowie Bowie (now known as
+  <a href="/wiki/Duncan_Jones" title="Duncan Jones">
+   Duncan Jones
+  </a>
+  , film director), and divorced on 8 February 1980 in Switzerland.
+ </del>
@@ -3330 +4710,2 @@
- Buckley writes, "If Ziggy confused both his creator and his audience, a big part of that confusion centred on the topic of sexuality."
+ <del>
+  Buckley writes, "If Ziggy confused both his creator and his audience, a big part of that confusion centred on the topic of sexuality."
@@ -3332,7 +4713,7 @@
- <i>
-  <a href="/wiki/Melody_Maker" title="Melody Maker">
-   Melody Maker
-  </a>
- </i>
- ,
- a move which coincided with the first shots in his campaign for stardom as Ziggy Stardust.
+  <i>
+   <a href="/wiki/Melody_Maker" title="Melody Maker">
+    Melody Maker
+   </a>
+  </i>
+  ,
+ a move which coincided with the first shots in his campaign for stardom as Ziggy Stardust.
@@ -3340,6 +4721,6 @@
- <i>
-  <a href="/wiki/Playboy" title="Playboy">
-   Playboy
-  </a>
- </i>
- , Bowie said: "It's true—I am a bisexual. But I can't deny that I've used that fact very well. I suppose it's the best thing that ever happened to me."
+  <i>
+   <a href="/wiki/Playboy" title="Playboy">
+    Playboy
+   </a>
+  </i>
+  , Bowie said: "It's true—I am a bisexual. But I can't deny that I've used that fact very well. I suppose it's the best thing that ever happened to me."
@@ -3346,0 +4728 @@
+ </del>
@@ -3349,5 +4731,6 @@
- In a 1983 interview with
- <i>
-  Rolling Stone
- </i>
- , Bowie said his public declaration of bisexuality was "the biggest mistake I ever made" and "I was always a closet heterosexual."
+ <del>
+  In a 1983 interview with
+  <i>
+   Rolling Stone
+  </i>
+  , Bowie said his public declaration of bisexuality was "the biggest mistake I ever made" and "I was always a closet heterosexual."
@@ -3354,0 +4738 @@
+ </del>
@@ -3357,7 +4741,9 @@
- Asked in 2002 by
- <i>
-  <a href="/wiki/Blender_(magazine)" title="Blender (magazine)">
-   Blender
-  </a>
- </i>
- whether he still believed his public declaration was the biggest mistake he ever made, he replied:
+ <del>
+  Asked in 2002 by
+  <i>
+   <a href="/wiki/Blender_(magazine)" title="Blender (magazine)">
+    Blender
+   </a>
+  </i>
+  whether he still believed his public declaration was the biggest mistake he ever made, he replied:
+ </del>
@@ -3367 +4753,3 @@
-  Interesting. [Long pause] I don't think it was a mistake in Europe, but it was a lot tougher in America. I had no problem with people knowing I was bisexual. But I had no inclination to hold any banners nor be a representative of any group of people. I knew what I wanted to be, which was a songwriter and a performer, and I felt that bisexuality became my headline over here for so long. America is a very puritanical place, and I think it stood in the way of so much I wanted to do.
+  <del>
+   Interesting. [Long pause] I don't think it was a mistake in Europe, but it was a lot tougher in America. I had no problem with people knowing I was bisexual. But I had no inclination to hold any banners nor be a representative of any group of people. I knew what I wanted to be, which was a songwriter and a performer, and I felt that bisexuality became my headline over here for so long. America is a very puritanical place, and I think it stood in the way of so much I wanted to do.
+  </del>
@@ -3371 +4759,2 @@
- Buckley's view of the period is that Bowie, "a taboo-breaker and a dabbler ... mined sexual intrigue for its ability to shock",
+ <del>
+  Buckley's view of the period is that Bowie, "a taboo-breaker and a dabbler ... mined sexual intrigue for its ability to shock",
@@ -3374,0 +4764 @@
+ </del>
@@ -3377,9 +4767,10 @@
- On 24 April 1992, David Bowie married Somali-American model
- <a href="/wiki/Iman_(model)" title="Iman (model)">
-  Iman
- </a>
- in a private ceremony in
- <a href="/wiki/Lausanne" title="Lausanne">
-  Lausanne
- </a>
- . The wedding was later solemnized on 6 June in Florence.
+ <del>
+  On 24 April 1992, David Bowie married Somali-American model
+  <a href="/wiki/Iman_(model)" title="Iman (model)">
+   Iman
+  </a>
+  in a private ceremony in
+  <a href="/wiki/Lausanne" title="Lausanne">
+   Lausanne
+  </a>
+  . The wedding was later solemnized on 6 June in Florence.
@@ -3387,0 +4779 @@
+ </del>
@@ -3390,3 +4782,5 @@
- <span class="mw-headline" id="Religion">
-  Religion
- </span>
+ <del>
+  <span class="mw-headline" id="Religion">
+   Religion
+  </span>
+ </del>
@@ -3395 +4789,2 @@
- Regarding his religion, in 2005 he said, "Questioning my spiritual life has always been germane to what I was writing. Always." He added that he is bothered by being "not quite an atheist".
+ <del>
+  Regarding his religion, in 2005 he said, "Questioning my spiritual life has always been germane to what I was writing. Always." He added that he is bothered by being "not quite an atheist".
@@ -3397,6 +4792,7 @@
- <i>
-  <a href="/wiki/Esquire_(magazine)" title="Esquire (magazine)">
-   Esquire
-  </a>
- </i>
- interview "What I've Learned", he stated, "I'm in awe of the universe, but I don't necessarily believe there's an intelligence or agent behind it. I do have a passion for the visual in religious rituals, though, even though they may be completely empty and bereft of substance. The incense is powerful and provocative, whether Buddhist or Catholic."
+  <i>
+   <a href="/wiki/Esquire_(magazine)" title="Esquire (magazine)">
+    Esquire
+   </a>
+  </i>
+  interview "What I've Learned", he stated, "I'm in awe of the universe, but I don't necessarily believe there's an intelligence or agent behind it. I do have a passion for the visual in religious rituals, though, even though they may be completely empty and bereft of substance. The incense is powerful and provocative, whether Buddhist or Catholic."
+ </del>
@@ -3405,19 +4801,20 @@
- Bowie has shown an interest in
- <a href="/wiki/Buddhism" title="Buddhism">
-  Buddhism
- </a>
- since 1967. He frequently studied in London under the
- <a href="/wiki/Tibetan_people" title="Tibetan people">
-  Tibetan
- </a>
- <a href="/wiki/Lama" title="Lama">
-  Lama
- </a>
- <a href="/wiki/Chime_Rinpoche" title="Chime Rinpoche">
-  Chime Rinpoche
- </a>
- before becoming a solo artist. During a 2001 interview, Bowie claimed that "after a few months of study, he told me, 'You don't want to be Buddhist ... You should follow music.
- <span style="padding-right:.2em;">
-  '
- </span>
- "
+ <del>
+  Bowie has shown an interest in
+  <a href="/wiki/Buddhism" title="Buddhism">
+   Buddhism
+  </a>
+  since 1967. He frequently studied in London under the
+  <a href="/wiki/Tibetan_people" title="Tibetan people">
+   Tibetan
+  </a>
+  <a href="/wiki/Lama" title="Lama">
+   Lama
+  </a>
+  <a href="/wiki/Chime_Rinpoche" title="Chime Rinpoche">
+   Chime Rinpoche
+  </a>
+  before becoming a solo artist. During a 2001 interview, Bowie claimed that "after a few months of study, he told me, 'You don't want to be Buddhist ... You should follow music.
+  <span style="padding-right:.2em;">
+   '
+  </span>
+  "
@@ -3425,3 +4822,8 @@
- <i>
-  <a href="/wiki/David_Bowie_(1967_album)" title="David Bowie (1967 album)">
-   David Bowie
+  <i>
+   <a href="/wiki/David_Bowie_(1967_album)" title="David Bowie (1967 album)">
+    David Bowie
+   </a>
+  </i>
+  . Bowie also became a student of the
+  <a class="mw-redirect" href="/wiki/Crazy_wisdom" title="Crazy wisdom">
+   Crazy wisdom
@@ -3429,12 +4831,8 @@
- </i>
- . Bowie also became a student of the
- <a class="mw-redirect" href="/wiki/Crazy_wisdom" title="Crazy wisdom">
-  Crazy wisdom
- </a>
- <a href="/wiki/Tulku" title="Tulku">
-  Tulku
- </a>
- <a href="/wiki/Ch%C3%B6gyam_Trungpa" title="Chögyam Trungpa">
-  Chögyam Trungpa
- </a>
- .
+  <a href="/wiki/Tulku" title="Tulku">
+   Tulku
+  </a>
+  <a href="/wiki/Ch%C3%B6gyam_Trungpa" title="Chögyam Trungpa">
+   Chögyam Trungpa
+  </a>
+  .
+ </del>
@@ -3443,3 +4841,5 @@
- <span class="mw-headline" id="Politics">
-  Politics
- </span>
+ <del>
+  <span class="mw-headline" id="Politics">
+   Politics
+  </span>
+ </del>
@@ -3448,12 +4848,8 @@
- Speaking as
- <a href="/wiki/The_Thin_White_Duke" title="The Thin White Duke">
-  The Thin White Duke
- </a>
- , Bowie's persona at the time, and "at least partially tongue-in-cheek", he made statements that expressed support for fascism and perceived admiration for
- <a href="/wiki/Adolf_Hitler" title="Adolf Hitler">
-  Adolf Hitler
- </a>
- in interviews with
- <i>
-  <a href="/wiki/Playboy" title="Playboy">
-   Playboy
+ <del>
+  Speaking as
+  <a href="/wiki/The_Thin_White_Duke" title="The Thin White Duke">
+   The Thin White Duke
+  </a>
+  , Bowie's persona at the time, and "at least partially tongue-in-cheek", he made statements that expressed support for fascism and perceived admiration for
+  <a href="/wiki/Adolf_Hitler" title="Adolf Hitler">
+   Adolf Hitler
@@ -3461,10 +4857,15 @@
- </i>
- ,
- <i>
-  NME
- </i>
- and a Swedish publication. Bowie was quoted as saying: "Britain is ready for a fascist leader... I think Britain could benefit from a fascist leader. After all, fascism is really nationalism... I believe very strongly in fascism, people have always responded with greater efficiency under a regimental leadership." He was also quoted as saying: "Adolf Hitler was one of the first rock stars" and "You've got to have an
- <a href="/wiki/Far-right_politics" title="Far-right politics">
-  extreme right
- </a>
- front come up and sweep everything off its feet and tidy everything up."
+  in interviews with
+  <i>
+   <a href="/wiki/Playboy" title="Playboy">
+    Playboy
+   </a>
+  </i>
+  ,
+  <i>
+   NME
+  </i>
+  and a Swedish publication. Bowie was quoted as saying: "Britain is ready for a fascist leader... I think Britain could benefit from a fascist leader. After all, fascism is really nationalism... I believe very strongly in fascism, people have always responded with greater efficiency under a regimental leadership." He was also quoted as saying: "Adolf Hitler was one of the first rock stars" and "You've got to have an
+  <a href="/wiki/Far-right_politics" title="Far-right politics">
+   extreme right
+  </a>
+  front come up and sweep everything off its feet and tidy everything up."
@@ -3471,0 +4873 @@
+ </del>
@@ -3474,3 +4876,5 @@
- <span class="mw-headline" id="Legal_issues">
-  Legal issues
- </span>
+ <del>
+  <span class="mw-headline" id="Legal_issues">
+   Legal issues
+  </span>
+ </del>
@@ -3479,13 +4883,14 @@
- In 1990, British rock band Queen and artist David Bowie filed a lawsuit against
- <a href="/wiki/Vanilla_Ice" title="Vanilla Ice">
-  Vanilla Ice
- </a>
- for copying the bass line of "
- <a href="/wiki/Under_Pressure" title="Under Pressure">
-  Under Pressure
- </a>
- " with only minor modifications in his song "
- <a href="/wiki/Ice_Ice_Baby" title="Ice Ice Baby">
-  Ice Ice Baby
- </a>
- ".
+ <del>
+  In 1990, British rock band Queen and artist David Bowie filed a lawsuit against
+  <a href="/wiki/Vanilla_Ice" title="Vanilla Ice">
+   Vanilla Ice
+  </a>
+  for copying the bass line of "
+  <a href="/wiki/Under_Pressure" title="Under Pressure">
+   Under Pressure
+  </a>
+  " with only minor modifications in his song "
+  <a href="/wiki/Ice_Ice_Baby" title="Ice Ice Baby">
+   Ice Ice Baby
+  </a>
+  ".
@@ -3492,0 +4898 @@
+ </del>
@@ -3504,0 +4911,16 @@
+<div class="thumb tright">
+ <div class="thumbinner" style="width:222px;">
+  <div class="thumbcaption">
+   <div class="magnify">
+   </div>
+   <ins>
+    Variety of Bowie's outfits on display at the
+   </ins>
+   <ins>
+    <a href="/wiki/Rock_and_Roll_Hall_of_Fame" title="Rock and Roll Hall of Fame">
+     Rock and Roll Hall of Fame
+    </a>
+   </ins>
+  </div>
+ </div>
+</div>
@@ -3521,2 +4943 @@
- .
- In the ensuing decades he has been honoured with numerous awards for his music and its accompanying videos, receiving, among others, two
+ . In the ensuing decades he has been honoured with numerous awards for his music and its accompanying videos, receiving, among others, two
@@ -3527,4 +4948,12 @@
- <a class="mw-redirect" href="/wiki/BRIT_Awards" title="BRIT Awards">
-  BRIT Awards
- </a>
- .
+ <ins>
+  <a href="/wiki/Brit_Awards" title="Brit Awards">
+   Brit Awards
+  </a>
+  —winning Best British Male Artist twice and in 1996 the award for Outstanding Contribution to Music.
+ </ins>
+ <del>
+  <a class="mw-redirect" href="/wiki/BRIT_Awards" title="BRIT Awards">
+   BRIT Awards
+  </a>
+  .
+ </del>
@@ -3545,2 +4974,9 @@
-  Commander of the British Empire
- </a>
+  Commander of the
+  <ins>
+   Order of the
+  </ins>
+  British Empire
+ </a>
+ <ins>
+  (CBE)
+ </ins>
@@ -3552,3 +4988,17 @@
-</p>
-<p>
- Throughout his career he has sold an estimated 140 million albums. In the United Kingdom, he has been awarded 9 Platinum, 11 Gold and 8 Silver albums, and in the United States, 5 Platinum and 7 Gold.
+ <ins>
+  Bowie later stated "I would never have any intention of accepting anything like that. I seriously don't know what it's for. It's not what I spent my life working for."
+ </ins>
+</p>
+<p>
+ Throughout his career he
+ <del>
+  has
+ </del>
+ sold an estimated 140 million albums. In the United Kingdom, he
+ <ins>
+  was
+ </ins>
+ <del>
+  has been
+ </del>
+ awarded 9 Platinum, 11 Gold and 8 Silver albums, and in the United States, 5 Platinum and 7 Gold.
@@ -3559 +5009,2 @@
- , he was ranked 29. In 2004,
+ , he was ranked 29.
+ In 2004,
@@ -3561 +5012,3 @@
-  Rolling Stone
+  <a class="mw-redirect" href="/wiki/Rolling_Stone_(magazine)" title="Rolling Stone (magazine)">
+   Rolling Stone
+  </a>
@@ -3563 +5016,6 @@
- magazine ranked him 39th on their list of the 100 Greatest Rock Artists of All Time
+ magazine ranked him 39th on their list of the 100 Greatest Rock Artists of All
+ <ins>
+  Time.
+ </ins>
+ <del>
+  Time
@@ -3564,0 +5023 @@
+ </del>
@@ -3574,0 +5034,9 @@
+ <ins>
+  The spider
+  <i>
+   <a href="/wiki/Heteropoda_davidbowie" title="Heteropoda davidbowie">
+    Heteropoda davidbowie
+   </a>
+  </i>
+  is named in his honour.
+ </ins>
@@ -3735,6 +5203,8 @@
-   <i>
-    <a href="/wiki/Tin_Machine_(album)" title="Tin Machine (album)">
-     Tin Machine
-    </a>
-   </i>
-   (1989)
+   <del>
+    <i>
+     <a href="/wiki/Tin_Machine_(album)" title="Tin Machine (album)">
+      Tin Machine
+     </a>
+    </i>
+    (1989)
+   </del>
@@ -3743,6 +5213,8 @@
-   <i>
-    <a href="/wiki/Tin_Machine_II" title="Tin Machine II">
-     Tin Machine II
-    </a>
-   </i>
-   (1991)
+   <del>
+    <i>
+     <a href="/wiki/Tin_Machine_II" title="Tin Machine II">
+      Tin Machine II
+     </a>
+    </i>
+    (1991)
+   </del>
@@ -3756 +5228,6 @@
-   (1993)
+   <ins>
+    (1992)
+   </ins>
+   <del>
+    (1993)
+   </del>
@@ -3840,108 +5317,11 @@
-<ul>
- <li>
-  <i>
-   <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
-    The Man Who Fell to Earth
-   </a>
-  </i>
-  (1976) as Thomas Jerome Newton; received
-  <a href="/wiki/Saturn_Award_for_Best_Actor" title="Saturn Award for Best Actor">
-   Saturn Award for Best Actor
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Just_a_Gigolo_(1978_film)" title="Just a Gigolo (1978 film)">
-    Just a Gigolo
-   </a>
-  </i>
-  (1978) as Paul Ambrosius von Przygodski
- </li>
- <li>
-  <i>
-   <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
-    Christiane F.
-   </a>
-  </i>
-  (1981) cameo as himself
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Snowman" title="The Snowman">
-    The Snowman
-   </a>
-  </i>
-  (1982) narrator in re-released version
- </li>
- <li>
-  <i>
-   Baal
-  </i>
-  (1982) as Baal
- </li>
- <li>
-  <i>
-   <a href="/wiki/Yellowbeard" title="Yellowbeard">
-    Yellowbeard
-   </a>
-  </i>
-  (1983) the sailor wearing shark fin - The Shark
- </li>
- <li>
-  <i>
-   <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
-    Merry Christmas, Mr. Lawrence
-   </a>
-  </i>
-  (1983) as Maj. Jack 'Strafer' Celliers
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Hunger_(1983_film)" title="The Hunger (1983 film)">
-    The Hunger
-   </a>
-  </i>
-  (1983) as John Blaylock
- </li>
- <li>
-  <i>
-   <a href="/wiki/Jazzin%27_for_Blue_Jean" title="Jazzin' for Blue Jean">
-    Jazzin' for Blue Jean
-   </a>
-  </i>
-  (1984) as Vic and Screaming Lord Byron
- </li>
- <li>
-  <i>
-   <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
-    Labyrinth
-   </a>
-  </i>
-  (1986) as
-  <a href="/wiki/Jareth" title="Jareth">
-   Jareth the Goblin King
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Absolute_Beginners_(film)" title="Absolute Beginners (film)">
-    Absolute Beginners
-   </a>
-  </i>
-  (1986) as Vendice Partners
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
-    The Last Temptation of Christ
-   </a>
-  </i>
-  (1988) as
-  <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
-   Pontius Pilate
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Linguini_Incident" title="The Linguini Incident">
-    The Linguini Incident
+<div class="div-col columns column-count column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2;">
+ <ul>
+  <li>
+   <i>
+    <a href="/wiki/The_Man_Who_Fell_to_Earth" title="The Man Who Fell to Earth">
+     The Man Who Fell to Earth
+    </a>
+   </i>
+   (1976) as Thomas Jerome Newton; received
+   <a href="/wiki/Saturn_Award_for_Best_Actor" title="Saturn Award for Best Actor">
+    Saturn Award for Best Actor
@@ -3949,13 +5329,89 @@
-  </i>
-  (1991) as Monte
- </li>
- <li>
-  <a href="/wiki/Dream_On_(TV_series)" title="Dream On (TV series)">
-   Dream On (TV series)
-  </a>
-  (1991) as Sir Roland Moorecock
- </li>
- <li>
-  <i>
-   <a href="/wiki/Basquiat_(film)" title="Basquiat (film)">
-    Basquiat
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Just_a_Gigolo_(1978_film)" title="Just a Gigolo (1978 film)">
+     Just a Gigolo
+    </a>
+   </i>
+   (1978) as Paul Ambrosius von Przygodski
+  </li>
+  <li>
+   <i>
+    <a class="mw-redirect" href="/wiki/Christiane_F._-_Wir_Kinder_vom_Bahnhof_Zoo_(film)" title="Christiane F. - Wir Kinder vom Bahnhof Zoo (film)">
+     Christiane F.
+    </a>
+   </i>
+   (1981) cameo as himself
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Snowman" title="The Snowman">
+     The Snowman
+    </a>
+   </i>
+   (1982) narrator in re-released version
+  </li>
+  <li>
+   <i>
+    Baal
+   </i>
+   (1982) as Baal
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Yellowbeard" title="Yellowbeard">
+     Yellowbeard
+    </a>
+   </i>
+   (1983) the sailor wearing shark fin
+   <ins>
+    –
+   </ins>
+   <del>
+    -
+   </del>
+   The Shark
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Merry_Christmas,_Mr._Lawrence" title="Merry Christmas, Mr. Lawrence">
+     Merry Christmas, Mr. Lawrence
+    </a>
+   </i>
+   (1983) as Maj. Jack 'Strafer' Celliers
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Hunger_(1983_film)" title="The Hunger (1983 film)">
+     The Hunger
+    </a>
+   </i>
+   (1983) as John Blaylock
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Jazzin%27_for_Blue_Jean" title="Jazzin' for Blue Jean">
+     Jazzin' for Blue Jean
+    </a>
+   </i>
+   (1984) as Vic and Screaming Lord Byron
+  </li>
+  <li>
+   <ins>
+    <i>
+     <a href="/wiki/Into_the_Night_(film)" title="Into the Night (film)">
+      Into the Night
+     </a>
+    </i>
+    (1985) as Colin Morris
+   </ins>
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Labyrinth_(film)" title="Labyrinth (film)">
+     Labyrinth
+    </a>
+   </i>
+   (1986) as
+   <a href="/wiki/Jareth" title="Jareth">
+    Jareth
@@ -3963,10 +5419,22 @@
-  </i>
-  (1996) as
-  <a href="/wiki/Andy_Warhol" title="Andy Warhol">
-   Andy Warhol
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/Gunslinger%27s_Revenge" title="Gunslinger's Revenge">
-    Gunslinger's Revenge
+   the Goblin King
+   <del>
+    Link: /wiki/Jareth
+   </del>
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Absolute_Beginners_(film)" title="Absolute Beginners (film)">
+     Absolute Beginners
+    </a>
+   </i>
+   (1986) as Vendice Partners
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Last_Temptation_of_Christ_(film)" title="The Last Temptation of Christ (film)">
+     The Last Temptation of Christ
+    </a>
+   </i>
+   (1988) as
+   <a href="/wiki/Pontius_Pilate" title="Pontius Pilate">
+    Pontius Pilate
@@ -3974,7 +5442,30 @@
-  </i>
-  (1998) as Jack Sikora
- </li>
- <li>
-  <i>
-   <a href="/wiki/Everybody_Loves_Sunshine" title="Everybody Loves Sunshine">
-    Everybody Loves Sunshine
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Linguini_Incident" title="The Linguini Incident">
+     The Linguini Incident
+    </a>
+   </i>
+   (1991) as Monte
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Dream_On_(TV_series)" title="Dream On (TV series)">
+     Twin Peaks: Fire Walk with Me
+     Link: /wiki/Twin_Peaks:_Fire_Walk_with_Me   (1992) as
+   Phillip Jeffries
+    Link: /wiki/Phillip_Jeffries   Dream On
+    Dream On (TV series)
+    </a>
+   </i>
+   (1991) as Sir Roland Moorecock
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Basquiat_(film)" title="Basquiat (film)">
+     Basquiat
+    </a>
+   </i>
+   (1996) as
+   <a href="/wiki/Andy_Warhol" title="Andy Warhol">
+    Andy Warhol
@@ -3982,13 +5473,32 @@
-  </i>
-  (1999) as Bernie
- </li>
- <li>
-  <i>
-   Mr. Rice's Secret
-  </i>
-  (2000) as William Rice
- </li>
- <li>
-  <i>
-   <a href="/wiki/Zoolander" title="Zoolander">
-    Zoolander
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Gunslinger%27s_Revenge" title="Gunslinger's Revenge">
+     Gunslinger's Revenge
+    </a>
+   </i>
+   (1998) as Jack Sikora
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Everybody_Loves_Sunshine" title="Everybody Loves Sunshine">
+     Everybody Loves Sunshine
+    </a>
+   </i>
+   (1999) as Bernie
+  </li>
+  <li>
+   <i>
+    Mr. Rice's Secret
+   </i>
+   (2000) as William Rice
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/Zoolander" title="Zoolander">
+     Zoolander
+    </a>
+   </i>
+   (2001) cameo as himself; nominated for
+   <a class="mw-redirect" href="/wiki/MTV_Movie_Award" title="MTV Movie Award">
+    MTV Movie Award
@@ -3996,10 +5506,10 @@
-  </i>
-  (2001) cameo as himself; nominated for
-  <a class="mw-redirect" href="/wiki/MTV_Movie_Award" title="MTV Movie Award">
-   MTV Movie Award
-  </a>
- </li>
- <li>
-  <i>
-   <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
-    The Prestige
+  </li>
+  <li>
+   <i>
+    <a href="/wiki/The_Prestige_(film)" title="The Prestige (film)">
+     The Prestige
+    </a>
+   </i>
+   (2006) as
+   <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
+    Nikola Tesla
@@ -4007,7 +5517,3 @@
-  </i>
-  (2006) as
-  <a href="/wiki/Nikola_Tesla" title="Nikola Tesla">
-   Nikola Tesla
-  </a>
- </li>
-</ul>
+  </li>
+ </ul>
+</div>
@@ -4066,0 +5573,7 @@
+  <ins>
+   <a class="mw-redirect" href="/wiki/Nat_Tate:_An_American_Artist_1928-1960" title="Nat Tate: An American Artist 1928-1960">
+    Nat Tate: An American Artist 1928-1960
+   </a>
+  </ins>
+ </li>
+ <li>
@@ -4106 +5619 @@
-</ul>
+</ul>
\ No newline at end of file
